diff --git "a/_8.0.0/tantan/game_SpecialAirHiStart2.txt" "b/_9.0.0/tantan/game_SpecialAirHiStart2.txt"
index 473ad9d..fc06280 100644
--- "a/_8.0.0/tantan/game_SpecialAirHiStart2.txt"
+++ "b/_9.0.0/tantan/game_SpecialAirHiStart2.txt"
@@ -1,17 +1,17 @@
 frame(Frame=4)
 if(is_excute){
-	WorkModule::on_flag(Flag=53288)
+	WorkModule::on_flag(Flag=53788)
 }
 frame(Frame=5)
 if(is_excute){
 	WorkModule::on_flag(Flag=FIGHTER_STATUS_AIR_LASSO_FLAG_CHECK)
 	WorkModule::on_flag(Flag=FIGHTER_STATUS_AIR_LASSO_FLAG_HANG_IMMIDIATE)
 }
 frame(Frame=6)
 if(is_excute){
-	GroundModule::select_cliff_hangdata(56600)
+	GroundModule::select_cliff_hangdata(58476)
 	WorkModule::off_flag(Flag=FIGHTER_STATUS_AIR_LASSO_FLAG_HANG_IMMIDIATE)
 }
 frame(Frame=8)
 if(is_excute){
 	WorkModule::off_flag(Flag=FIGHTER_STATUS_AIR_LASSO_FLAG_CHECK)
