diff --git "a/0.0.0/purin/game_FinalAirStartL.txt" "b/0.1.0/purin/game_FinalAirStartL.txt"
index 06fe208..12bc15e 100644
--- "a/0.0.0/purin/game_FinalAirStartL.txt"
+++ "b/0.1.0/purin/game_FinalAirStartL.txt"
@@ -9,11 +9,11 @@ if(0xf40a0(WorkModule::is_flag(FIGHTER_INSTANCE_WORK_ID_FLAG_DISABLE_FINAL_START
 		REQ_FINAL_START_CAMERA(hash40("d04finalstartr.nuanmb"), false)
 	}
 	else{
 		if(is_excute){
 			PostureModule::scale(5, 0)
-			0xf40a0(1709844057, 2.1)
+			0xf40a0(896462908, 2.1)
 			CAM_ZOOM_IN_arg5(0, 0)
 			FT_START_CUTIN()
 		}
 	}
 }
