diff --git "a/g_9.0.0/inkling/game_EscapeAir.txt" "b/_10.0.0/inkling/game_EscapeAir.txt"
index dbaa91b..936ed0a 100644
--- "a/g_9.0.0/inkling/game_EscapeAir.txt"
+++ "b/_10.0.0/inkling/game_EscapeAir.txt"
@@ -6,12 +6,12 @@ if(method.lib::L2CValue.operator___lib::L2CValueconst__const(WorkModule::is_flag
 		ArticleModule::generate_article(FIGHTER_INKLING_GENERATE_ARTICLE_SQUID)
 	}
 }
 MotionModule::motion_kind()
 MotionModule::frame(0x00000002b0)
-MotionModule::rate(989672188)
-as_hash__const(989672188, FIGHTER_INKLING_GENERATE_ARTICLE_SQUID, false)
+MotionModule::rate(1652099283)
+as_hash__const(1652099283, FIGHTER_INKLING_GENERATE_ARTICLE_SQUID, false)
 ArticleModule::change_motion()
 if(is_excute){
 	ArticleModule::set_frame(FIGHTER_INKLING_GENERATE_ARTICLE_SQUID)
 	ArticleModule::set_rate(FIGHTER_INKLING_GENERATE_ARTICLE_SQUID)
 	ArticleModule::set_visibility_whole(FIGHTER_INKLING_GENERATE_ARTICLE_SQUID, false)
