diff --git "a/_9.0.0/tantan_punch3/game_AttackLongHold.txt" "b/10.0.0/tantan_punch3/game_AttackLongHold.txt"
index ebceea6..ba6d48b 100644
--- "a/_9.0.0/tantan_punch3/game_AttackLongHold.txt"
+++ "b/10.0.0/tantan_punch3/game_AttackLongHold.txt"
@@ -1,25 +1,25 @@
 if(is_excute){
-	ATTACK(ID=0, Part=0, Bone=0x0703d6cf75, Damage=11.5, Angle=30, KBG=67, FKB=0, BKB=25, Size=0.7, X=0.0, Y=2.0, Z=0.0, X2=LUA_VOID, Y2=LUA_VOID, Z2=LUA_VOID, Hitlag=0.9, SDI=1.0, Clang_Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=false, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=false, Absorbable=false, Flinchless=false, DisableHitlag=false, Direct_Hitbox=true, Ground_or_Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=false, Effect=hash40("collision_attr_fire"), SFXLevel=ATTACK_SOUND_LEVEL_L, SFXType=58472, Type=ATTACK_REGION_PUNCH)
+	ATTACK(ID=0, Part=0, Bone=0x0703d6cf75, Damage=11.5, Angle=30, KBG=67, FKB=0, BKB=25, Size=0.7, X=0.0, Y=2.0, Z=0.0, X2=LUA_VOID, Y2=LUA_VOID, Z2=LUA_VOID, Hitlag=0.9, SDI=1.0, Clang_Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=false, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=false, Absorbable=false, Flinchless=false, DisableHitlag=false, Direct_Hitbox=true, Ground_or_Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=false, Effect=hash40("collision_attr_fire"), SFXLevel=ATTACK_SOUND_LEVEL_L, SFXType=59144, Type=ATTACK_REGION_PUNCH)
 	AttackModule::enable_safe_pos()
 	AttackModule::disable_tip()
 	AttackModule::set_damage_shake_scale(0.5)
 }
 frame(Frame=1)
 if(is_excute){
-	ATTACK(ID=0, Part=0, Bone=0x0703d6cf75, Damage=11.5, Angle=30, KBG=67, FKB=0, BKB=25, Size=4.0, X=0.0, Y=2.0, Z=0.0, X2=LUA_VOID, Y2=LUA_VOID, Z2=LUA_VOID, Hitlag=0.9, SDI=1.0, Clang_Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=false, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=false, Absorbable=false, Flinchless=false, DisableHitlag=false, Direct_Hitbox=true, Ground_or_Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=false, Effect=hash40("collision_attr_fire"), SFXLevel=ATTACK_SOUND_LEVEL_L, SFXType=58472, Type=ATTACK_REGION_PUNCH)
+	ATTACK(ID=0, Part=0, Bone=0x0703d6cf75, Damage=11.5, Angle=30, KBG=67, FKB=0, BKB=25, Size=4.0, X=0.0, Y=2.0, Z=0.0, X2=LUA_VOID, Y2=LUA_VOID, Z2=LUA_VOID, Hitlag=0.9, SDI=1.0, Clang_Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=false, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=false, Absorbable=false, Flinchless=false, DisableHitlag=false, Direct_Hitbox=true, Ground_or_Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=false, Effect=hash40("collision_attr_fire"), SFXLevel=ATTACK_SOUND_LEVEL_L, SFXType=59144, Type=ATTACK_REGION_PUNCH)
 	as_hash__const(0x1145350e55, 24, false)
 	ControlModule::set_rumble()
 }
 frame(Frame=3)
-if(0x3a06b0(WorkModule::is_flag(53912), false, true)){
+if(0x3a06b0(WorkModule::is_flag(54028), false, true)){
 	if(is_excute){
 		AttackModule::clear_all()
 	}
 	else{
 		if(is_excute){
-			ATTACK(ID=0, Part=0, Bone=0x0703d6cf75, Damage=13.8, Angle=30, KBG=57, FKB=0, BKB=25, Size=4.0, X=0.0, Y=2.0, Z=0.0, X2=LUA_VOID, Y2=LUA_VOID, Z2=LUA_VOID, Hitlag=0.9, SDI=1.0, Clang_Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=false, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=false, Absorbable=false, Flinchless=false, DisableHitlag=false, Direct_Hitbox=true, Ground_or_Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=false, Effect=hash40("collision_attr_fire"), SFXLevel=ATTACK_SOUND_LEVEL_L, SFXType=58472, Type=ATTACK_REGION_PUNCH)
+			ATTACK(ID=0, Part=0, Bone=0x0703d6cf75, Damage=13.8, Angle=30, KBG=57, FKB=0, BKB=25, Size=4.0, X=0.0, Y=2.0, Z=0.0, X2=LUA_VOID, Y2=LUA_VOID, Z2=LUA_VOID, Hitlag=0.9, SDI=1.0, Clang_Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=false, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=false, Absorbable=false, Flinchless=false, DisableHitlag=false, Direct_Hitbox=true, Ground_or_Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=false, Effect=hash40("collision_attr_fire"), SFXLevel=ATTACK_SOUND_LEVEL_L, SFXType=59144, Type=ATTACK_REGION_PUNCH)
 		}
 	}
 }
 frame(Frame=10)
 if(is_excute){
