diff --git "a/_3.0.0/packun/game_Catch.txt" "b/_3.1.0/packun/game_Catch.txt"
index c769e94..1fac272 100644
--- "a/_3.0.0/packun/game_Catch.txt"
+++ "b/_3.1.0/packun/game_Catch.txt"
@@ -2,12 +2,12 @@ frame(Frame=5)
 if(is_excute){
 	GrabModule__set_rebound(CanCatchRebound=True)
 }
 frame(Frame=6)
 if(is_excute){
-	CATCH(ID=0, Bone=top, Size=3.3, X=0.0, Y=1449984, Z=4.0, X2=0.0, Y2=1449984, Z2=11.0, Status=FIGHTER_STATUS_KIND_CAPTURE_PULLED, Ground/Air=COLLISION_SITUATION_MASK_G)
-	CATCH(ID=1, Bone=top, Size=1.65, X=0.0, Y=1449984, Z=2.35, X2=0.0, Y2=1449984, Z2=12.65, Status=FIGHTER_STATUS_KIND_CAPTURE_PULLED, Ground/Air=COLLISION_SITUATION_MASK_A)
+	CATCH(ID=0, Bone=top, Size=3.3, X=0.0, Y=6.6, Z=4.0, X2=0.0, Y2=6.6, Z2=11.0, Status=FIGHTER_STATUS_KIND_CAPTURE_PULLED, Ground/Air=COLLISION_SITUATION_MASK_G)
+	CATCH(ID=1, Bone=top, Size=1.65, X=0.0, Y=6.6, Z=2.35, X2=0.0, Y2=6.6, Z2=12.65, Status=FIGHTER_STATUS_KIND_CAPTURE_PULLED, Ground/Air=COLLISION_SITUATION_MASK_A)
 }
 game_CaptureCutCommon()
 wait(Frames=2)
 if(is_excute){
 	grab(MA_MSC_CMD_GRAB_CLEAR_ALL)
