diff --git "a/10.0.0/pickel_trolley/game_SpecialSWallM.txt" "b/10.1.0/pickel_trolley/game_SpecialSWallM.txt"
index 5b5f865..df3a20d 100644
--- "a/10.0.0/pickel_trolley/game_SpecialSWallM.txt"
+++ "b/10.1.0/pickel_trolley/game_SpecialSWallM.txt"
@@ -1,5 +1,5 @@
 if(is_excute){
 	as_hash__const(hash40("rbkind_nohitl"), 0, false)
 	ControlModule::set_rumble()
-	WorkModule::off_flag(Flag=59244)
+	WorkModule::off_flag(Flag=59332)
 }
