diff --git "a/12.0.0/elight/game_AttackAirN.txt" "b/13.0.0/elight/game_AttackAirN.txt"
index b656d7d..a4e439a 100644
--- "a/12.0.0/elight/game_AttackAirN.txt"
+++ "b/13.0.0/elight/game_AttackAirN.txt"
@@ -2,18 +2,18 @@ frame(Frame=1)
 FT_MOTION_RATE(FSM=4)
 frame(Frame=2)
 FT_MOTION_RATE(FSM=0.5)
 frame(Frame=3)
 IS_EXIST_ARTICLE(FIGHTER_ELIGHT_GENERATE_ARTICLE_ESWORD)
-if(0x354230(false, true)){
+if(0x357740(false, true)){
 	if(is_excute){
 		methodlib::L2CValue::as_hash()const(FIGHTER_ELIGHT_GENERATE_ARTICLE_ESWORD, WEAPON_ELIGHT_ESWORD_MOTION_PART_SET_KIND_OPEM_CLOSE, 0x07439e926b, 5, 5, false, false, 0, false, true, false)
 		ArticleModule::add_motion_partial()
 	}
 }
 MotionModule::is_changing()
-if(0x354230(false, true)){
+if(0x357740(false, true)){
 	if(is_excute){
 		WorkModule::on_flag(Flag=FIGHTER_ELIGHT_INSTANCE_WORK_ID_FLAG_ADD_PARTIAL_MTION_SWORD_WHEN_CHANGEING)
 	}
 }
 FT_MOTION_RATE(0, 1)
@@ -38,18 +38,18 @@ frame(Frame=18)
 if(is_excute){
 	AttackModule::clear_all()
 }
 frame(Frame=24)
 IS_EXIST_ARTICLE(FIGHTER_ELIGHT_GENERATE_ARTICLE_ESWORD)
-if(0x354230(false, true)){
+if(0x357740(false, true)){
 	if(is_excute){
 		methodlib::L2CValue::as_hash()const(FIGHTER_ELIGHT_GENERATE_ARTICLE_ESWORD, WEAPON_ELIGHT_ESWORD_MOTION_PART_SET_KIND_OPEM_CLOSE, 0x08183db0f4, 5, 5, false, false, 0, false, true, false)
 		ArticleModule::add_motion_partial()
 	}
 }
 MotionModule::is_changing()
-if(0x354230(false, true)){
+if(0x357740(false, true)){
 	if(is_excute){
 		WorkModule::on_flag(Flag=FIGHTER_ELIGHT_INSTANCE_WORK_ID_FLAG_ADD_PARTIAL_MTION_SWORD_WHEN_CHANGEING)
 	}
 }
 frame(0, 36)
