diff --git "a/12.0.0/elight/game_FinalAttackCommon.txt" "b/13.0.0/elight/game_FinalAttackCommon.txt"
index 4b7028e..4bad274 100644
--- "a/12.0.0/elight/game_FinalAttackCommon.txt"
+++ "b/13.0.0/elight/game_FinalAttackCommon.txt"
@@ -4,40 +4,40 @@ if(is_excute){
 	WorkModule::on_flag(Flag=FIGHTER_STATUS_VISUAL_SCENE_FLAG_ABS_SET)
 	CAM_ZOOM_OUT()
 }
 frame(Frame=2)
 WorkModule::is_flag(FIGHTER_ELIGHT_STATUS_FINAL_FLAG_CAMERA_CLIP_IN)
-if(0x354230(false, true)){
+if(0x357740(false, true)){
 	WorkModule::is_flag(FIGHTER_ELIGHT_STATUS_FINAL_FLAG_DEMON_DOJO_ZOOM)
-	if(0x354230(false, true)){
+	if(0x357740(false, true)){
 		WorkModule::is_flag(FIGHTER_ELIGHT_STATUS_FINAL_FLAG_TRAINING_ZOOM)
-		if(0x354230(false, false)){
+		if(0x357740(false, false)){
 			get_value_float(SO_VAR_FLOAT_LR)
-			if(0x354230(1984, 0)){
+			if(0x357740(1984, 0)){
 				if(is_excute){
 					PostureModule::scale(1, 0)
-					0x354230(-707422823, 5)
+					0x357740(619365007, 5)
 					CAM_ZOOM_IN_arg5(0, 0.1)
 					camera(MA_MSC_CMD_CAMERA_CAM_OFFSET, -20, 25)
 				}
 				else{
 					if(is_excute){
 						PostureModule::scale(1, 0)
-						0x354230(-707422823, 5)
+						0x357740(619365007, 5)
 						CAM_ZOOM_IN_arg5(0, 0)
 					}
 					WorkModule::is_flag(FIGHTER_ELIGHT_STATUS_FINAL_FLAG_TRAINING_ZOOM)
-					if(0x354230(false, false)){
+					if(0x357740(false, false)){
 						get_value_float(SO_VAR_FLOAT_LR)
-						if(0x354230(1984, 0)){
+						if(0x357740(1984, 0)){
 							if(is_excute){
 								camera(MA_MSC_CMD_CAMERA_CAM_OFFSET, -40, 25)
 							}
 							else{
 								if(is_excute){
 									PostureModule::scale(1, 0)
-									0x354230(-707422823, 5)
+									0x357740(619365007, 5)
 									CAM_ZOOM_IN_arg5(0, -0.1)
 									camera(MA_MSC_CMD_CAMERA_CAM_OFFSET, 20, 25)
 								}
 								else{
 									if(is_excute){
