diff --git "a/to_3.0.0/murabito/game_FinalAirMoneyR.txt" "b/to_3.1.0/murabito/game_FinalAirMoneyR.txt"
index 2e0ed7a..2ca54ad 100644
--- "a/to_3.0.0/murabito/game_FinalAirMoneyR.txt"
+++ "b/to_3.1.0/murabito/game_FinalAirMoneyR.txt"
@@ -1,4 +1,4 @@
 frame(Frame=30)
 if(is_excute){
-	FILL_SCREEN_MODEL_COLOR(1, 10, 0.3, 0.3, 0.3, 0, 0, 0, 1, 1.3, 50052, EFFECT_SCREEN_PRIO_FINAL)
+	FILL_SCREEN_MODEL_COLOR(1, 10, 0.3, 0.3, 0.3, 0, 0, 0, 1, 1.3, 50068, EFFECT_SCREEN_PRIO_FINAL)
 }
