diff --git "a/to_3.0.0/murabito/game_AttackAirLw.txt" "b/to_3.1.0/murabito/game_AttackAirLw.txt"
index 8c74fb4..87b6409 100644
--- "a/to_3.0.0/murabito/game_AttackAirLw.txt"
+++ "b/to_3.1.0/murabito/game_AttackAirLw.txt"
@@ -1,16 +1,16 @@
 frame(Frame=14)
 if(is_excute){
 	WorkModule__on_flag(Flag=FIGHTER_STATUS_ATTACK_AIR_FLAG_ENABLE_LANDING)
 }
-if(0x126100(FIGHTER_MURABITO_INSTANCE_WORK_ID_INT_TURNIP_NUM, 1)){
+if(0x1262c0(FIGHTER_MURABITO_INSTANCE_WORK_ID_INT_TURNIP_NUM, 1)){
 	if(is_excute){
 		ATTACK(ID=0, Part=0, Bone=top, Damage=8.0, Angle=270, KBG=80, FKB=0, BKB=0, Size=2.7, X=0.0, Y=-6.0, Z=0.0, Hitlag=1.0, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_normal, SFXLevel=ATTACK_SOUND_LEVEL_M, SFXType=COLLISION_SOUND_ATTR_PUNCH, Type=ATTACK_REGION_OBJECT)
 		ATTACK(ID=1, Part=0, Bone=top, Damage=8.0, Angle=45, KBG=100, FKB=0, BKB=30, Size=5.2, X=0.0, Y=-6.0, Z=0.0, Hitlag=1.0, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_normal, SFXLevel=ATTACK_SOUND_LEVEL_M, SFXType=COLLISION_SOUND_ATTR_PUNCH, Type=ATTACK_REGION_OBJECT)
 	}
 	else{
-		if(0x126100(FIGHTER_MURABITO_INSTANCE_WORK_ID_INT_TURNIP_NUM, 2)){
+		if(0x1262c0(FIGHTER_MURABITO_INSTANCE_WORK_ID_INT_TURNIP_NUM, 2)){
 			if(is_excute){
 				ATTACK(ID=0, Part=0, Bone=top, Damage=10.0, Angle=270, KBG=80, FKB=0, BKB=10, Size=3.0, X=0.0, Y=-5.5, Z=0.5, Hitlag=1.0, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_normal, SFXLevel=ATTACK_SOUND_LEVEL_L, SFXType=COLLISION_SOUND_ATTR_PUNCH, Type=ATTACK_REGION_OBJECT)
 				ATTACK(ID=1, Part=0, Bone=top, Damage=10.0, Angle=45, KBG=100, FKB=0, BKB=30, Size=6.5, X=0.0, Y=-5.5, Z=0.5, Hitlag=1.0, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_normal, SFXLevel=ATTACK_SOUND_LEVEL_L, SFXType=COLLISION_SOUND_ATTR_PUNCH, Type=ATTACK_REGION_OBJECT)
 			}
 			else{
@@ -21,17 +21,17 @@ if(0x126100(FIGHTER_MURABITO_INSTANCE_WORK_ID_INT_TURNIP_NUM, 1)){
 			}
 		}
 	}
 }
 wait(Frames=2)
-if(0x126100(FIGHTER_MURABITO_INSTANCE_WORK_ID_INT_TURNIP_NUM, 1)){
+if(0x1262c0(FIGHTER_MURABITO_INSTANCE_WORK_ID_INT_TURNIP_NUM, 1)){
 	if(is_excute){
 		ATTACK(ID=0, Part=0, Bone=top, Damage=4.0, Angle=45, KBG=100, FKB=0, BKB=30, Size=3.8, X=0.0, Y=-4.0, Z=0.5, Hitlag=1.0, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_normal, SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=COLLISION_SOUND_ATTR_PUNCH, Type=ATTACK_REGION_OBJECT)
 		AttackModule__clear(ID=1)
 	}
 	else{
-		if(0x126100(FIGHTER_MURABITO_INSTANCE_WORK_ID_INT_TURNIP_NUM, 2)){
+		if(0x1262c0(FIGHTER_MURABITO_INSTANCE_WORK_ID_INT_TURNIP_NUM, 2)){
 			if(is_excute){
 				ATTACK(ID=0, Part=0, Bone=top, Damage=5.0, Angle=45, KBG=100, FKB=0, BKB=30, Size=4.5, X=0.0, Y=-4.0, Z=0.5, Hitlag=1.0, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_normal, SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=COLLISION_SOUND_ATTR_PUNCH, Type=ATTACK_REGION_OBJECT)
 				AttackModule__clear(ID=1)
 			}
 			else{
