diff --git "a/3.0.0/chrom/game_FinalAirStart.txt" "b/3.0.1/chrom/game_FinalAirStart.txt"
index 374bf5f..4d7786b 100644
--- "a/3.0.0/chrom/game_FinalAirStart.txt"
+++ "b/3.0.1/chrom/game_FinalAirStart.txt"
@@ -11,11 +11,11 @@ if(methodlib::L2CValue::operator==(lib::L2CValueconst&)const(false, false)){
 		REQ_FINAL_START_CAMERA(hash40("d04finalstart.nuanmb"), false)
 	}
 	else{
 		if(is_excute){
 			PostureModule::scale(3, 0)
-			0x12bc40(1807820073, 2)
+			0x12bc40(1608592093, 2)
 			CAM_ZOOM_IN_arg5(0, 0)
 			FT_START_CUTIN()
 		}
 	}
 }
