diff --git "a/3.1.0/lucas/game_Final.txt" "b/4.0.0/lucas/game_Final.txt"
index 54b51a0..d0225ed 100644
--- "a/3.1.0/lucas/game_Final.txt"
+++ "b/4.0.0/lucas/game_Final.txt"
@@ -1,10 +1,10 @@
 if(is_excute){
 	ArticleModule__generate_article(FIGHTER_LUCAS_GENERATE_ARTICLE_KUMATORA)
 	ArticleModule__generate_article(FIGHTER_LUCAS_GENERATE_ARTICLE_BONNIE)
 	BattleObjectWorld__scale_z()
-	if(0xfffe0(0, 1.0)){
+	if(0x100190(0, 1.0)){
 		article(MA_MSC_CMD_ARTICLE_SET_OFFSET_ARTICLE, FIGHTER_LUCAS_GENERATE_ARTICLE_KUMATORA, -10, 0, -10)
 		article(MA_MSC_CMD_ARTICLE_SET_OFFSET_ARTICLE, FIGHTER_LUCAS_GENERATE_ARTICLE_BONNIE, 10, 0, -10)
 	}
 	else{
 		article(FIGHTER_LUCAS_GENERATE_ARTICLE_BONNIE, FIGHTER_LUCAS_GENERATE_ARTICLE_KUMATORA, -10, 0, -0.01)
@@ -20,11 +20,11 @@ if(method.lib::L2CValue.operator___lib::L2CValueconst__const(WorkModule__is_flag
 		REQ_FINAL_START_CAMERA(d04final.nuanmb, False)
 	}
 	else{
 		if(is_excute){
 			PostureModule__scale(5, 0)
-			0xfffe0(955865829, 2.2)
+			0x100190(-918582943, 2.2)
 			CAM_ZOOM_IN_arg5(0, 0)
 			FT_START_CUTIN()
 		}
 	}
 }
