diff --git "a/5.0.0/brave/game_SpecialAirN3.txt" "b/6.0.0/brave/game_SpecialAirN3.txt"
index ed8ed94..bf660ea 100644
--- "a/5.0.0/brave/game_SpecialAirN3.txt"
+++ "b/6.0.0/brave/game_SpecialAirN3.txt"
@@ -4,11 +4,11 @@ if(is_excute){
 frame(Frame=14)
 if(is_excute){
 	WorkModule__on_flag(Flag=FIGHTER_BRAVE_STATUS_SPECIAL_N_FLAG_HOP)
 }
 frame(Frame=16)
-if(0x21bab0(WorkModule__is_flag(FIGHTER_BRAVE_STATUS_SPECIAL_N_FLAG_SUCCESS_SP), False, True)){
+if(0x21b9e0(WorkModule__is_flag(FIGHTER_BRAVE_STATUS_SPECIAL_N_FLAG_SUCCESS_SP), False, True)){
 	if(is_excute){
 		ArticleModule__generate_article(FIGHTER_BRAVE_GENERATE_ARTICLE_FIREBALL)
 		KineticModule__add_speed(-0.3, 0, 0.0)
 	}
 }
