diff --git "a/4.0.0/peach/game_Final_Com.txt" "b/5.0.0/peach/game_Final_Com.txt"
index 5330ea9..03fa7b0 100644
--- "a/4.0.0/peach/game_Final_Com.txt"
+++ "b/5.0.0/peach/game_Final_Com.txt"
@@ -1,8 +1,8 @@
 if(is_excute){
 	PostureModule__scale(3, 0)
-	0xe4290(-545051865, 1.8)
+	0xe42a0(346621976, 1.8)
 	CAM_ZOOM_IN_arg5(0, 0)
 	camera(MA_MSC_CMD_CAMERA_CAM_OFFSET, 0, 2)
 	CHECK_VALID_FINAL_START_CAMERA(0, 7, 20, 0, 0, 0)
 }
 if(method.lib::L2CValue.operator___lib::L2CValueconst__const(WorkModule__is_flag(FIGHTER_INSTANCE_WORK_ID_FLAG_DISABLE_FINAL_START_CAMERA), False, False)){
@@ -25,11 +25,11 @@ if(is_excute){
 }
 frame(Frame=40)
 if(is_excute){
 	CAM_ZOOM_OUT()
 	PostureModule__scale(120, 0)
-	0xe4290(-545051865, 5)
+	0xe42a0(346621976, 5)
 	CAM_ZOOM_IN_arg5(0, 0)
 	WorkModule__on_flag(Flag=FIGHTER_PEACH_STATUS_FINAL_FLAG_AFFECT_ENABLE)
 	ATTACK(ID=0, Part=0, Bone=throw, Damage=30.0, Angle=361, KBG=0, FKB=0, BKB=0, Size=25.0, X=0.0, Y=5.0, Z=0.0, Hitlag=0.0, SDI=0.0, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=False, ShieldDamage=no, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_FIGHTER, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_normal, SFXLevel=ATTACK_SOUND_LEVEL_M, SFXType=COLLISION_SOUND_ATTR_NONE, Type=ATTACK_REGION_NONE)
 	ATTACK(ID=1, Part=0, Bone=throw, Damage=20.0, Angle=361, KBG=0, FKB=0, BKB=0, Size=50.0, X=0.0, Y=5.0, Z=0.0, Hitlag=0.0, SDI=0.0, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=False, ShieldDamage=no, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_FIGHTER, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_normal, SFXLevel=ATTACK_SOUND_LEVEL_M, SFXType=COLLISION_SOUND_ATTR_NONE, Type=ATTACK_REGION_NONE)
 	ATTACK(ID=2, Part=0, Bone=throw, Damage=10.0, Angle=361, KBG=0, FKB=0, BKB=0, Size=100.0, X=0.0, Y=5.0, Z=0.0, Hitlag=0.0, SDI=0.0, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=False, ShieldDamage=no, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_FIGHTER, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_normal, SFXLevel=ATTACK_SOUND_LEVEL_M, SFXType=COLLISION_SOUND_ATTR_NONE, Type=ATTACK_REGION_NONE)
