diff --git "a/_3.0.0/pacman/game_FinalStart.txt" "b/_3.1.0/pacman/game_FinalStart.txt"
index 4d3d6b2..ab016a9 100644
--- "a/_3.0.0/pacman/game_FinalStart.txt"
+++ "b/_3.1.0/pacman/game_FinalStart.txt"
@@ -12,11 +12,11 @@ if(0x119860(WorkModule__is_flag(FIGHTER_INSTANCE_WORK_ID_FLAG_DISABLE_FINAL_STAR
 		FT_START_CUTIN()
 	}
 	else{
 		if(is_excute){
 			PostureModule__scale(3, 0)
-			0x119860(-1531385283, 2.7)
+			0x119860(-1931570904, 2.7)
 			CAM_ZOOM_IN_arg5(0, 0)
 			FT_START_CUTIN()
 			camera(MA_MSC_CMD_CAMERA_CAM_OFFSET, 0, 15)
 		}
 	}
