diff --git "a/0.0/ryu/game_AttackNearw.txt" "b/1.0/ryu/game_AttackNearw.txt"
index 3d13df0..92f1279 100644
--- "a/0.0/ryu/game_AttackNearw.txt"
+++ "b/1.0/ryu/game_AttackNearw.txt"
@@ -13,9 +13,9 @@ if(is_excute){
 wait(Frames=2)
 if(is_excute){
 	AttackModule__clear_all()
 	WorkModule__off_flag(Flag=FIGHTER_RYU_INSTANCE_WORK_ID_FLAG_FINAL_HIT_CANCEL)
 }
-frame(Frame=15)
+frame(Frame=25)
 if(is_excute){
 	WorkModule__off_flag(Flag=FIGHTER_RYU_STATUS_ATTACK_FLAG_HIT_CANCEL)
 }
