diff --git "a/.0.0/jack/game_SpecialAirNEscapeB_Ex.txt" "b/.0.0/jack/game_SpecialAirNEscapeB_Ex.txt"
index 31711fe..bba3063 100644
--- "a/.0.0/jack/game_SpecialAirNEscapeB_Ex.txt"
+++ "b/.0.0/jack/game_SpecialAirNEscapeB_Ex.txt"
@@ -6,11 +6,11 @@ frame(Frame=17)
 if(is_excute){
 	JostleModule::set_status(true)
 }
 frame(Frame=22)
 WorkModule::is_flag(FIGHTER_JACK_INSTANCE_WORK_ID_FLAG_DOYLE)
-if(0x304760(false, true)){
+if(0x304a00(false, true)){
 	if(is_excute){
 		ATTACK(ID=0, Part=0, Bone=hash40("top"), Damage=6.0, Angle=361, KBG=50, FKB=0, BKB=60, Size=2.8, X=0.0, Y=7.0, Z=9.0, X2=0.0, Y2=7.0, Z2=24.0, Hitlag=0.0, SDI=0.0, Clang_Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=false, ShieldDamage=-3, Trip=0.0, Rehit=0, Reflectable=false, Absorbable=false, Flinchless=false, DisableHitlag=false, Direct_Hitbox=false, Ground_or_Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=false, Effect=hash40("collision_attr_jack_bullet"), SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=COLLISION_SOUND_ATTR_JACK_SHOT, Type=ATTACK_REGION_OBJECT)
 		ATTACK(ID=1, Part=0, Bone=hash40("top"), Damage=3.0, Angle=361, KBG=50, FKB=0, BKB=3, Size=2.8, X=0.0, Y=7.0, Z=9.0, X2=0.0, Y2=7.0, Z2=34.0, Hitlag=0.0, SDI=0.0, Clang_Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=false, ShieldDamage=-1, Trip=0.0, Rehit=0, Reflectable=false, Absorbable=false, Flinchless=false, DisableHitlag=false, Direct_Hitbox=false, Ground_or_Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=false, Effect=hash40("collision_attr_jack_bullet"), SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=COLLISION_SOUND_ATTR_JACK_SHOT, Type=ATTACK_REGION_OBJECT)
 		ATTACK(ID=2, Part=0, Bone=hash40("top"), Damage=1.0, Angle=361, KBG=0, FKB=0, BKB=0, Size=2.8, X=0.0, Y=7.0, Z=9.0, X2=0.0, Y2=7.0, Z2=69.0, Hitlag=0.0, SDI=0.0, Clang_Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=false, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=false, Absorbable=false, Flinchless=false, DisableHitlag=false, Direct_Hitbox=false, Ground_or_Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=false, Effect=hash40("collision_attr_jack_bullet"), SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=COLLISION_SOUND_ATTR_JACK_SHOT, Type=ATTACK_REGION_OBJECT)
 		FT_SHOOTING_ATTACK_GROUND_CHECK_NEW(0, 4, 4)
@@ -22,11 +22,11 @@ wait(Frames=1)
 if(is_excute){
 	AttackModule::clear_all()
 }
 frame(Frame=28)
 WorkModule::is_flag(FIGHTER_JACK_INSTANCE_WORK_ID_FLAG_DOYLE)
-if(0x304760(false, true)){
+if(0x304a00(false, true)){
 	if(is_excute){
 		ATTACK(ID=0, Part=0, Bone=hash40("top"), Damage=2.0, Angle=361, KBG=0, FKB=0, BKB=0, Size=2.8, X=0.0, Y=7.0, Z=9.0, X2=0.0, Y2=7.0, Z2=24.0, Hitlag=0.0, SDI=0.0, Clang_Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=false, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=false, Absorbable=false, Flinchless=false, DisableHitlag=false, Direct_Hitbox=false, Ground_or_Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=false, Effect=hash40("collision_attr_jack_bullet"), SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=COLLISION_SOUND_ATTR_JACK_SHOT, Type=ATTACK_REGION_OBJECT)
 		ATTACK(ID=1, Part=0, Bone=hash40("top"), Damage=1.5, Angle=361, KBG=0, FKB=0, BKB=0, Size=2.8, X=0.0, Y=7.0, Z=9.0, X2=0.0, Y2=7.0, Z2=34.0, Hitlag=0.0, SDI=0.0, Clang_Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=false, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=false, Absorbable=false, Flinchless=false, DisableHitlag=false, Direct_Hitbox=false, Ground_or_Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=false, Effect=hash40("collision_attr_jack_bullet"), SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=COLLISION_SOUND_ATTR_JACK_SHOT, Type=ATTACK_REGION_OBJECT)
 		ATTACK(ID=2, Part=0, Bone=hash40("top"), Damage=1.0, Angle=361, KBG=0, FKB=0, BKB=0, Size=2.8, X=0.0, Y=7.0, Z=9.0, X2=0.0, Y2=7.0, Z2=69.0, Hitlag=0.0, SDI=0.0, Clang_Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=false, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=false, Absorbable=false, Flinchless=false, DisableHitlag=false, Direct_Hitbox=false, Ground_or_Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=false, Effect=hash40("collision_attr_jack_bullet"), SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=COLLISION_SOUND_ATTR_JACK_SHOT, Type=ATTACK_REGION_OBJECT)
 		FT_SHOOTING_ATTACK_GROUND_CHECK_NEW(0, 4, 4)
@@ -38,11 +38,11 @@ wait(Frames=1)
 if(is_excute){
 	AttackModule::clear_all()
 }
 frame(Frame=34)
 WorkModule::is_flag(FIGHTER_JACK_INSTANCE_WORK_ID_FLAG_DOYLE)
-if(0x304760(false, true)){
+if(0x304a00(false, true)){
 	if(is_excute){
 		ATTACK(ID=0, Part=0, Bone=hash40("top"), Damage=2.0, Angle=361, KBG=0, FKB=0, BKB=0, Size=2.8, X=0.0, Y=7.0, Z=9.0, X2=0.0, Y2=7.0, Z2=24.0, Hitlag=0.0, SDI=0.0, Clang_Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=false, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=false, Absorbable=false, Flinchless=false, DisableHitlag=false, Direct_Hitbox=false, Ground_or_Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=false, Effect=hash40("collision_attr_jack_bullet"), SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=COLLISION_SOUND_ATTR_JACK_SHOT, Type=ATTACK_REGION_OBJECT)
 		ATTACK(ID=1, Part=0, Bone=hash40("top"), Damage=1.5, Angle=361, KBG=0, FKB=0, BKB=0, Size=2.8, X=0.0, Y=7.0, Z=9.0, X2=0.0, Y2=7.0, Z2=34.0, Hitlag=0.0, SDI=0.0, Clang_Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=false, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=false, Absorbable=false, Flinchless=false, DisableHitlag=false, Direct_Hitbox=false, Ground_or_Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=false, Effect=hash40("collision_attr_jack_bullet"), SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=COLLISION_SOUND_ATTR_JACK_SHOT, Type=ATTACK_REGION_OBJECT)
 		ATTACK(ID=2, Part=0, Bone=hash40("top"), Damage=1.0, Angle=361, KBG=0, FKB=0, BKB=0, Size=2.8, X=0.0, Y=7.0, Z=9.0, X2=0.0, Y2=7.0, Z2=69.0, Hitlag=0.0, SDI=0.0, Clang_Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=false, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=false, Absorbable=false, Flinchless=false, DisableHitlag=false, Direct_Hitbox=false, Ground_or_Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=false, Effect=hash40("collision_attr_jack_bullet"), SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=COLLISION_SOUND_ATTR_JACK_SHOT, Type=ATTACK_REGION_OBJECT)
 		FT_SHOOTING_ATTACK_GROUND_CHECK_NEW(0, 4, 4)
