diff --git "a/.0.0/wolf/game_SpecialAirN.txt" "b/.1.0/wolf/game_SpecialAirN.txt"
index a746cf5..43d0599 100644
--- "a/.0.0/wolf/game_SpecialAirN.txt"
+++ "b/.1.0/wolf/game_SpecialAirN.txt"
@@ -1,11 +1,11 @@
 frame(Frame=2)
 if(is_excute){
 	ArticleModule__generate_article(FIGHTER_WOLF_GENERATE_ARTICLE_BLASTER)
 }
 IS_EXIST_ARTICLE(FIGHTER_WOLF_GENERATE_ARTICLE_BLASTER)
-if(0x107e30(False, True)){
+if(0x108080(False, True)){
 	if(is_excute){
 		as_hash__const(FIGHTER_WOLF_GENERATE_ARTICLE_BLASTER, 0x04a47083a4)
 		ArticleModule__change_motion()
 	}
 }
@@ -22,13 +22,13 @@ if(is_excute){
 frame(Frame=20)
 if(is_excute){
 	AttackModule__clear_all()
 }
 frame(Frame=38)
-FT_MOTION_RATE(FSM=1.7)
+FT_MOTION_RATE(FSM=1.8)
 IS_EXIST_ARTICLE(FIGHTER_WOLF_GENERATE_ARTICLE_BLASTER)
-if(0x107e30(False, True)){
+if(0x108080(False, True)){
 	if(is_excute){
 		as_hash__const(FIGHTER_WOLF_GENERATE_ARTICLE_BLASTER, 0x05130181c4)
 		ArticleModule__change_motion()
 	}
 }
