diff --git "a/3.0.0/cloud/game_FinalAirHit.txt" "b/3.0.1/cloud/game_FinalAirHit.txt"
index 7b49fdc..6fa00cb 100644
--- "a/3.0.0/cloud/game_FinalAirHit.txt"
+++ "b/3.0.1/cloud/game_FinalAirHit.txt"
@@ -9,22 +9,22 @@ if(is_excute){
 frame(Frame=5)
 WorkModule::is_flag(FIGHTER_CLOUD_STATUS_FINAL_FLAG_HIT_ALL_OPPONENT)
 if(methodlib::L2CValue::operator==(lib::L2CValueconst&)const(false, true)){
 	FT_MOTION_RATE(FSM=0.5)
 	WorkModule::get_float(FIGHTER_CLOUD_STATUS_FINAL_WORK_FLOAT_ATTACK_TARGET_LR)
-	if(0x1801d0(647911431, 0)){
+	if(0x1801b0(1699148503, 0)){
 		if(is_excute){
 			CAM_ZOOM_OUT()
 			PostureModule::scale(32, 0)
-			0x1801d0(647911431, 3.2)
+			0x1801b0(1699148503, 3.2)
 			CAM_ZOOM_IN_FINAL(0, 0, 30, -10, 195, true, FIGHTER_CLOUD_STATUS_FINAL_WORK_INT_ATTACK_HIT_OBJECT_ID, 10)
 		}
 		else{
 			if(is_excute){
 				CAM_ZOOM_OUT()
 				PostureModule::scale(32, 0)
-				0x1801d0(647911431, 2.8)
+				0x1801b0(1699148503, 2.8)
 				CAM_ZOOM_IN_FINAL(0, 0, -30, -10, 195, true, FIGHTER_CLOUD_STATUS_FINAL_WORK_INT_ATTACK_HIT_OBJECT_ID, 10)
 			}
 		}
 	}
 }
