diff --git "a/ght_10.0.0/metaknight/game_FinalAirStartR.txt" "b/ght_10.1.0/metaknight/game_FinalAirStartR.txt"
index be0ae73..8395334 100644
--- "a/ght_10.0.0/metaknight/game_FinalAirStartR.txt"
+++ "b/ght_10.1.0/metaknight/game_FinalAirStartR.txt"
@@ -21,11 +21,11 @@ if(0x12eaf0(32, 0)){
 		}
 		frame(Frame=10)
 		if(is_excute){
 			ArticleModule::set_visibility_whole(FIGHTER_METAKNIGHT_GENERATE_ARTICLE_FOURWINGS, true)
 			PostureModule::scale(5, 0)
-			0x12eaf0(120702201, 2)
+			0x12eaf0(966639530, 2)
 			CAM_ZOOM_IN_arg5(0, 0)
 			FT_START_CUTIN()
 		}
 		frame(Frame=40)
 		if(is_excute){
