diff --git "a/5.0.0/simon/game_FinalStart_Com2.txt" "b/6.0.0/simon/game_FinalStart_Com2.txt"
index 71977ca..792337d 100644
--- "a/5.0.0/simon/game_FinalStart_Com2.txt"
+++ "b/6.0.0/simon/game_FinalStart_Com2.txt"
@@ -1,8 +1,9 @@
 wait(Frames=15)
 if(is_excute){
 	AttackModule__clear_all()
 }
+wait(Frames=1)
 operator___lib::L2CValueconst__const(WorkModule__is_flag(FIGHTER_SIMON_INSTANCE_WORK_ID_FLAG_FINAL_HIT), False, True)
 if(is_excute){
 	WorkModule__on_flag(Flag=FIGHTER_SIMON_STATUS_FINAL_WORK_ID_FLAG_START_COFFIN_ROT_Y)
 }
