diff --git "a/_6.0.0/samusd/game_GuardOn.txt" "b/_7.0.0/samusd/game_GuardOn.txt"
index 77bbe7f..0e5adbb 100644
--- "a/_6.0.0/samusd/game_GuardOn.txt"
+++ "b/_7.0.0/samusd/game_GuardOn.txt"
@@ -1,10 +1,15 @@
 frame(Frame=1)
 if(is_excute){
 	HIT_NODE(head, HIT_STATUS_XLU)
 	HIT_NODE(0x0ce6e8740d, HIT_STATUS_XLU)
 	HIT_NODE(0x0c1ce7496e, HIT_STATUS_XLU)
+	HIT_NODE(armr, HIT_STATUS_XLU)
+}
+frame(Frame=3)
+if(is_excute){
+	HIT_NODE(armr, HIT_STATUS_NORMAL)
 }
 frame(Frame=6)
 if(is_excute){
 	HIT_NODE(head, HIT_STATUS_NORMAL)
 	HIT_NODE(0x0ce6e8740d, HIT_STATUS_NORMAL)
