diff --git "a/_7.0.0/samusd/game_Final.txt" "b/_8.0.0/samusd/game_Final.txt"
index d360c47..96a5623 100644
--- "a/_7.0.0/samusd/game_Final.txt"
+++ "b/_8.0.0/samusd/game_Final.txt"
@@ -1,7 +1,7 @@
 IS_EXIST_ARTICLE(FIGHTER_SAMUSD_GENERATE_ARTICLE_LASER2)
-if(0x111dc0(false, false)){
+if(0x111f00(false, false)){
 	if(is_excute){
 		ArticleModule::shoot(FIGHTER_SAMUSD_GENERATE_ARTICLE_LASER, ARTICLE_OPE_TARGET_ALL, false)
 		WorkModule::set_float(14, FIGHTER_SAMUS_STATUS_FINAL_WORK_FLOAT_OY)
 		ArticleModule::generate_article(FIGHTER_SAMUSD_GENERATE_ARTICLE_LASER2)
 		WorkModule::set_float(-14, FIGHTER_SAMUS_STATUS_FINAL_WORK_FLOAT_OY)
