diff --git "a/10.1.0/wiifit/game_FinalStart.txt" "b/11.0.0/wiifit/game_FinalStart.txt"
index 575e71d..10e5d33 100644
--- "a/10.1.0/wiifit/game_FinalStart.txt"
+++ "b/11.0.0/wiifit/game_FinalStart.txt"
@@ -9,11 +9,11 @@ if(method.lib::L2CValue.operator___lib::L2CValueconst__const(WorkModule::is_flag
 		REQ_FINAL_START_CAMERA_arg3(hash40("d04finalstart.nuanmb"), false, false)
 	}
 	else{
 		if(is_excute){
 			PostureModule::scale(3, 0)
-			0x12ab70(873051465, 1.8)
+			0x12ab70(-275707937, 1.8)
 			CAM_ZOOM_IN_arg5(0, 0)
 			FT_START_CUTIN()
 		}
 	}
 }
