diff --git "a/9.0.0/shulk/game_Final_Com.txt" "b/0.0.0/shulk/game_Final_Com.txt"
index 4abfec1..9253604 100644
--- "a/9.0.0/shulk/game_Final_Com.txt"
+++ "b/0.0.0/shulk/game_Final_Com.txt"
@@ -15,11 +15,11 @@ if(is_excute){
 	SlowModule::set_whole(4, 0)
 }
 else{
 	if(is_excute){
 		PostureModule::scale(3, 0)
-		0x127210(-529798371, 2.1)
+		0x127210(-469006104, 2.1)
 		CAM_ZOOM_IN_arg5(0, 0)
 		FT_START_CUTIN()
 	}
 }
 frame(Frame=19)
