diff --git "a/4.0.0/brave/game_SpecialS2.txt" "b/5.0.0/brave/game_SpecialS2.txt"
index 799f5f2..68930ce 100644
--- "a/4.0.0/brave/game_SpecialS2.txt"
+++ "b/5.0.0/brave/game_SpecialS2.txt"
@@ -1,10 +1,10 @@
 frame(Frame=10)
 if(is_excute){
 	WorkModule__on_flag(Flag=FIGHTER_BRAVE_STATUS_SPECIAL_S_FLAG_ENABLE_SPARK)
 }
-if(0x218bc0(WorkModule__is_flag(FIGHTER_BRAVE_STATUS_SPECIAL_S_FLAG_SUCCESS_SP), False, True)){
+if(0x21bab0(WorkModule__is_flag(FIGHTER_BRAVE_STATUS_SPECIAL_S_FLAG_SUCCESS_SP), False, True)){
 	if(is_excute){
 		ATTACK(ID=0, Part=0, Bone=top, Damage=6.0, Angle=361, KBG=90, FKB=0, BKB=25, Size=4.5, X=0.0, Y=11.5, Z=12.0, X2=0.0, Y2=11.5, Z2=56.0, Hitlag=1.0, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=False, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_elec, SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=COLLISION_SOUND_ATTR_ELEC, Type=ATTACK_REGION_MAGIC)
 		AttackModule__set_no_finish_camera(0, True, False)
 		FT_SHOOTING_ATTACK_GROUND_CHECK_NEW(0, 10, 4)
 	}
