diff --git "a/4.0.0/brave/game_SpecialAirN1.txt" "b/5.0.0/brave/game_SpecialAirN1.txt"
index e574a2e..4fdd108 100644
--- "a/4.0.0/brave/game_SpecialAirN1.txt"
+++ "b/5.0.0/brave/game_SpecialAirN1.txt"
@@ -1,10 +1,10 @@
 if(is_excute){
 	WorkModule__on_flag(Flag=FIGHTER_BRAVE_STATUS_SPECIAL_N_FLAG_ENABLE_CONTROL_ENERGY)
 }
 frame(Frame=10)
-if(0x218bc0(WorkModule__is_flag(FIGHTER_BRAVE_STATUS_SPECIAL_N_FLAG_SUCCESS_SP), False, True)){
+if(0x21bab0(WorkModule__is_flag(FIGHTER_BRAVE_STATUS_SPECIAL_N_FLAG_SUCCESS_SP), False, True)){
 	if(is_excute){
 		ArticleModule__generate_article(FIGHTER_BRAVE_GENERATE_ARTICLE_FIREBALL)
 	}
 }
 frame(Frame=24)
