diff --git "a/_4.0.0/dedede/game_SpecialAirNShotObjectSWallow.txt" "b/_5.0.0/dedede/game_SpecialAirNShotObjectSWallow.txt"
index ee7882c..5eac0f1 100644
--- "a/_4.0.0/dedede/game_SpecialAirNShotObjectSWallow.txt"
+++ "b/_5.0.0/dedede/game_SpecialAirNShotObjectSWallow.txt"
@@ -1,10 +1,10 @@
 FT_MOTION_RATE(FSM=0.6)
 frame(Frame=4)
 if(is_excute){
 	FighterAreaModuleImpl__enable_fix_jostle_area(14.7, 4)
 }
-if(0x148480(FIGHTER_INSTANCE_WORK_ID_INT_KIND, FIGHTER_KIND_KIRBY)){
+if(0x1486b0(FIGHTER_INSTANCE_WORK_ID_INT_KIND, FIGHTER_KIND_KIRBY)){
 	if(is_excute){
 		FighterAreaModuleImpl__enable_fix_jostle_area(4, 4)
 	}
 }
