diff --git "a/.0.0/jack/game_SpecialAirNEscapeB.txt" "b/.1.0/jack/game_SpecialAirNEscapeB.txt"
index 536dd91..236b178 100644
--- "a/.0.0/jack/game_SpecialAirNEscapeB.txt"
+++ "b/.1.0/jack/game_SpecialAirNEscapeB.txt"
@@ -6,19 +6,19 @@ frame(Frame=17)
 if(is_excute){
 	JostleModule__set_status(True)
 }
 frame(Frame=22)
 if(is_excute){
-	ATTACK(ID=0, Part=0, Bone=top, Damage=5.0, Angle=361, KBG=75, FKB=0, BKB=5, Size=2.5, X=0.0, Y=7.5, Z=9.0, X2=0.0, Y2=7.5, Z2=14.0, Hitlag=0.0, SDI=0.0, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=False, ShieldDamage=-2, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=False, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=0x1a6f665bee, SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=48796, Type=ATTACK_REGION_OBJECT)
-	ATTACK(ID=1, Part=0, Bone=top, Damage=3.0, Angle=361, KBG=50, FKB=0, BKB=3, Size=2.5, X=0.0, Y=7.5, Z=9.0, X2=0.0, Y2=7.5, Z2=34.0, Hitlag=0.0, SDI=0.0, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=False, ShieldDamage=-1, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=False, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=0x1a6f665bee, SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=48796, Type=ATTACK_REGION_OBJECT)
-	ATTACK(ID=2, Part=0, Bone=top, Damage=1.0, Angle=361, KBG=0, FKB=0, BKB=0, Size=2.5, X=0.0, Y=7.5, Z=9.0, X2=0.0, Y2=7.5, Z2=69.0, Hitlag=0.0, SDI=0.0, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=False, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=0x1a6f665bee, SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=48796, Type=ATTACK_REGION_OBJECT)
+	ATTACK(ID=0, Part=0, Bone=top, Damage=5.0, Angle=361, KBG=75, FKB=0, BKB=5, Size=2.5, X=0.0, Y=7.5, Z=9.0, X2=0.0, Y2=7.5, Z2=14.0, Hitlag=0.0, SDI=0.0, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=False, ShieldDamage=-2, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=False, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=0x1a6f665bee, SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=48812, Type=ATTACK_REGION_OBJECT)
+	ATTACK(ID=1, Part=0, Bone=top, Damage=3.0, Angle=361, KBG=50, FKB=0, BKB=3, Size=2.5, X=0.0, Y=7.5, Z=9.0, X2=0.0, Y2=7.5, Z2=34.0, Hitlag=0.0, SDI=0.0, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=False, ShieldDamage=-1, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=False, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=0x1a6f665bee, SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=48812, Type=ATTACK_REGION_OBJECT)
+	ATTACK(ID=2, Part=0, Bone=top, Damage=1.0, Angle=361, KBG=0, FKB=0, BKB=0, Size=2.5, X=0.0, Y=7.5, Z=9.0, X2=0.0, Y2=7.5, Z2=69.0, Hitlag=0.0, SDI=0.0, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=False, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=0x1a6f665bee, SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=48812, Type=ATTACK_REGION_OBJECT)
 }
-if(0x28fe90(FIGHTER_INSTANCE_WORK_ID_INT_KIND, FIGHTER_KIND_KIRBY)){
+if(0x2901d0(FIGHTER_INSTANCE_WORK_ID_INT_KIND, FIGHTER_KIND_KIRBY)){
 	if(is_excute){
-		ATTACK(ID=0, Part=0, Bone=top, Damage=5.0, Angle=361, KBG=75, FKB=0, BKB=5, Size=2.5, X=0.0, Y=4.0, Z=9.0, X2=0.0, Y2=4.0, Z2=14.0, Hitlag=0.0, SDI=0.0, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=False, ShieldDamage=-2, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=False, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=0x1a6f665bee, SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=48796, Type=ATTACK_REGION_OBJECT)
-		ATTACK(ID=1, Part=0, Bone=top, Damage=3.0, Angle=361, KBG=50, FKB=0, BKB=3, Size=2.5, X=0.0, Y=4.0, Z=9.0, X2=0.0, Y2=4.0, Z2=34.0, Hitlag=0.0, SDI=0.0, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=False, ShieldDamage=-1, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=False, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=0x1a6f665bee, SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=48796, Type=ATTACK_REGION_OBJECT)
-		ATTACK(ID=2, Part=0, Bone=top, Damage=1.0, Angle=361, KBG=0, FKB=0, BKB=0, Size=2.5, X=0.0, Y=4.0, Z=9.0, X2=0.0, Y2=4.0, Z2=69.0, Hitlag=0.0, SDI=0.0, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=False, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=0x1a6f665bee, SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=48796, Type=ATTACK_REGION_OBJECT)
+		ATTACK(ID=0, Part=0, Bone=top, Damage=5.0, Angle=361, KBG=75, FKB=0, BKB=5, Size=2.5, X=0.0, Y=4.0, Z=9.0, X2=0.0, Y2=4.0, Z2=14.0, Hitlag=0.0, SDI=0.0, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=False, ShieldDamage=-2, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=False, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=0x1a6f665bee, SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=48812, Type=ATTACK_REGION_OBJECT)
+		ATTACK(ID=1, Part=0, Bone=top, Damage=3.0, Angle=361, KBG=50, FKB=0, BKB=3, Size=2.5, X=0.0, Y=4.0, Z=9.0, X2=0.0, Y2=4.0, Z2=34.0, Hitlag=0.0, SDI=0.0, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=False, ShieldDamage=-1, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=False, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=0x1a6f665bee, SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=48812, Type=ATTACK_REGION_OBJECT)
+		ATTACK(ID=2, Part=0, Bone=top, Damage=1.0, Angle=361, KBG=0, FKB=0, BKB=0, Size=2.5, X=0.0, Y=4.0, Z=9.0, X2=0.0, Y2=4.0, Z2=69.0, Hitlag=0.0, SDI=0.0, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=False, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=0x1a6f665bee, SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=48812, Type=ATTACK_REGION_OBJECT)
 	}
 }
 if(is_excute){
 	FT_SHOOTING_ATTACK_GROUND_CHECK_NEW(0, 4, 4)
 	FT_SHOOTING_ATTACK_GROUND_CHECK_NEW(1, 4, 4)
@@ -28,21 +28,21 @@ wait(Frames=1)
 if(is_excute){
 	AttackModule__clear_all()
 }
 frame(Frame=32)
 if(is_excute){
-	WorkModule__on_flag(Flag=16432)
+	WorkModule__on_flag(Flag=16436)
 }
 frame(Frame=36)
 if(is_excute){
-	WorkModule__on_flag(Flag=16392)
+	WorkModule__on_flag(Flag=16396)
 }
 frame(Frame=37)
 frame(Frame=39)
 if(is_excute){
+	WorkModule__on_flag(Flag=16424)
 	WorkModule__on_flag(Flag=16420)
-	WorkModule__on_flag(Flag=16416)
 }
 frame(Frame=42)
 if(is_excute){
-	WorkModule__on_flag(Flag=16388)
+	WorkModule__on_flag(Flag=16392)
 }
