diff --git "a/.1.0/link_ancientbowarrow/game_Stick.txt" "b/.0.0/link_ancientbowarrow/game_Stick.txt"
index f1fba87..35e477b 100644
--- "a/.1.0/link_ancientbowarrow/game_Stick.txt"
+++ "b/.0.0/link_ancientbowarrow/game_Stick.txt"
@@ -1,13 +1,13 @@
 if(is_excute){
 	CAM_ZOOM_OUT_FINAL()
 }
 get_value_float(SO_VAR_FLOAT_LR)
-if(0x118f80(0, 0)){
+if(0x119170(0, 0)){
 	if(is_excute){
 		PostureModule::scale(3, 0)
-		0x118f80(1779259368, 2)
+		0x119170(-828170864, 2)
 		CAM_ZOOM_IN_FINAL_arg13(1, 0, -1, -10, 10, true, WEAPON_LINK_ANCIENTBOWARROW_INSTANCE_WORK_ID_INT_HIT_OBJECT_ID, 0, 10, 0)
 	}
 	else{
 		if(is_excute){
 			CAM_ZOOM_IN_FINAL_arg13(3, 0, 2, -1, 0, 1, -10, 10, true, WEAPON_LINK_ANCIENTBOWARROW_INSTANCE_WORK_ID_INT_HIT_OBJECT_ID, 0, -10, 0)
