diff --git "a/_13.0.0/lucario/game_FinalAirStart.txt" "b/_13.0.1/lucario/game_FinalAirStart.txt"
index 4525eec..f6aea2e 100644
--- "a/_13.0.0/lucario/game_FinalAirStart.txt"
+++ "b/_13.0.1/lucario/game_FinalAirStart.txt"
@@ -11,11 +11,11 @@ if(methodlib::L2CValue::operator==(lib::L2CValueconst&)const(false, false)){
 	}
 	else{
 		if(is_excute){
 			camera(MA_MSC_CMD_CAMERA_CAM_RECT, 0, 4, 6, 0)
 			PostureModule::scale(3, 0)
-			0x1035e0(-28233133, 2)
+			0x1035e0(1821741189, 2)
 			CAM_ZOOM_IN_arg5(0, 0)
 			camera(MA_MSC_CMD_CAMERA_CAM_OFFSET, 0, 6)
 			FT_START_CUTIN()
 		}
 	}
