diff --git "a/5.0.0/snake_c4/game_Explosion.txt" "b/6.0.0/snake_c4/game_Explosion.txt"
index ba65ca7..6cc44b0 100644
--- "a/5.0.0/snake_c4/game_Explosion.txt"
+++ "b/6.0.0/snake_c4/game_Explosion.txt"
@@ -9,9 +9,17 @@ frame(Frame=1)
 if(is_excute){
 	AttackModule__set_size(ID=0, Size=17)
 	as_hash__const(rbkind_explosion, 0, False)
 	ControlModule__set_rumble()
 }
+frame(Frame=2)
+if(method.lib::L2CValue.operator___lib::L2CValueconst__const(WorkModule__is_flag(WEAPON_SNAKE_C4_INSTANCE_WORK_ID_FLAG_GROUND), False, False)){
+	if(is_excute){
+		AttackModule__clear_all()
+	}
+}
 frame(Frame=3)
-if(is_excute){
-	AttackModule__clear_all()
+if(method.lib::L2CValue.operator___lib::L2CValueconst__const(WorkModule__is_flag(WEAPON_SNAKE_C4_INSTANCE_WORK_ID_FLAG_GROUND), False, True)){
+	if(is_excute){
+		AttackModule__clear_all()
+	}
 }
