diff --git "a/12.0.0/pikmin/game_SpecialAirS.txt" "b/13.0.0/pikmin/game_SpecialAirS.txt"
index d40ce1a..b2b4d70 100644
--- "a/12.0.0/pikmin/game_SpecialAirS.txt"
+++ "b/13.0.0/pikmin/game_SpecialAirS.txt"
@@ -3,11 +3,11 @@ if(methodlib::L2CValue::operator==(lib::L2CValueconst&)const(false, true)){
 	frame(Frame=9)
 	if(is_excute){
 		WorkModule::on_flag(Flag=FIGHTER_PIKMIN_STATUS_SPECIAL_S_FLAG_THROW)
 	}
 	else{
-		frame(Frame=604417583)
+		frame(Frame=-353523446)
 		if(is_excute){
 			MotionModule::set_rate(0.6)
 		}
 	}
 }
