diff --git "a/12.0.0/pikmin/game_FinalStartR.txt" "b/13.0.0/pikmin/game_FinalStartR.txt"
index e563d5b..c010bdb 100644
--- "a/12.0.0/pikmin/game_FinalStartR.txt"
+++ "b/13.0.0/pikmin/game_FinalStartR.txt"
@@ -15,11 +15,11 @@ if(methodlib::L2CValue::operator==(lib::L2CValueconst&)const(false, false)){
 		REQ_FINAL_START_CAMERA_arg3(hash40("d04finalstartr.nuanmb"), false, false)
 	}
 	else{
 		if(is_excute){
 			PostureModule::scale(5, 0)
-			0x14bd40(604417583, 2.2)
+			0x14bd40(-353523446, 2.2)
 			CAM_ZOOM_IN_arg5(0, 0)
 		}
 	}
 }
 frame(Frame=41)
