diff --git "a/1.0/fox/game_FinalLockOn.txt" "b/0.0/fox/game_FinalLockOn.txt"
index 6168147..1dff68d 100644
--- "a/1.0/fox/game_FinalLockOn.txt"
+++ "b/0.0/fox/game_FinalLockOn.txt"
@@ -12,25 +12,26 @@ if(is_excute){
 	ArticleModule::generate_article(FIGHTER_FOX_GENERATE_ARTICLE_ARWING)
 	ArticleModule::generate_article(FIGHTER_FOX_GENERATE_ARTICLE_ARWING)
 	ArticleModule::generate_article(FIGHTER_FOX_GENERATE_ARTICLE_ARWING)
 	ArticleModule::generate_article(FIGHTER_FOX_GENERATE_ARTICLE_ARWING)
 	BattleObjectWorld__scale_z()
-	operator___lib::L2CValueconst__const(0, 1.0)
-	ArticleModule::set_visibility_whole(FIGHTER_FOX_GENERATE_ARTICLE_ARWING, false)
-	SLOW_OPPONENT(4, 30)
+	if(method.lib::L2CValue.operator___lib::L2CValueconst__const(0, 1.0)){
+		ArticleModule::set_visibility_whole(FIGHTER_FOX_GENERATE_ARTICLE_ARWING, false)
+	}
 }
+SLOW_OPPONENT(4, 30)
 if(0x10cfb0(WorkModule::is_flag(FIGHTER_INSTANCE_WORK_ID_FLAG_DISABLE_FINAL_START_CAMERA), false, false)){
 	frame(Frame=5)
 	if(is_excute){
 		SlowModule::set_whole(2, 0)
 		FT_SET_FINAL_FEAR_FACE(25)
 		REQ_FINAL_START_CAMERA_arg3(hash40("d04finallockon.nuanmb"), false, false)
 	}
 	else{
 		if(is_excute){
 			PostureModule::scale(4, 0)
-			0x10cfb0(-813640488, 1.2)
+			0x10cfb0(798607211, 1.2)
 			CAM_ZOOM_IN_arg5(0, 0)
 			FT_START_CUTIN()
 		}
 	}
 }
