diff --git "a/1.0.0/falco/game_SpecialNEnd.txt" "b/2.0.0/falco/game_SpecialNEnd.txt"
index a96f2c1..b9f411b 100644
--- "a/1.0.0/falco/game_SpecialNEnd.txt"
+++ "b/2.0.0/falco/game_SpecialNEnd.txt"
@@ -1,9 +1,9 @@
 frame(Frame=1)
 FT_MOTION_RATE(FSM=0.75)
 IS_EXIST_ARTICLE(FIGHTER_FALCO_GENERATE_ARTICLE_BLASTER)
-if(0x11c960(false, true)){
+if(0x11c9e0(false, true)){
 	if(is_excute){
 		methodlib::L2CValue::as_hash()const(FIGHTER_FALCO_GENERATE_ARTICLE_BLASTER, hash40("close"))
 		ArticleModule::change_motion()
 	}
 }
