diff --git "a/.1.0/popo/game_SpecialAirHiStart.txt" "b/.0.0/popo/game_SpecialAirHiStart.txt"
index 937fc0c..60f42dc 100644
--- "a/.1.0/popo/game_SpecialAirHiStart.txt"
+++ "b/.0.0/popo/game_SpecialAirHiStart.txt"
@@ -6,11 +6,11 @@ if(is_excute){
 	WorkModule::on_flag(Flag=FIGHTER_POPO_STATUS_SPECIAL_HI_FLAG_CHECK_COUPLE)
 }
 frame(Frame=15)
 get_value_float(FT_VAR_FLOAT_STICK_X_BACK)
 WorkModule::get_param_float(0, hash40("param_special_hi"), hash40("start_turn_cont_x"))
-if(0x1614b0(32)){
+if(0x161600(32)){
 	if(is_excute){
 		STICK_LR()
 		UPDATE_ROT()
 	}
 }
