diff --git "a/nk_3.1.0/toonlink/game_AttackAirLwOffGravity.txt" "b/nk_4.0.0/toonlink/game_AttackAirLwOffGravity.txt"
index 65cb8d8..c5281eb 100644
--- "a/nk_3.1.0/toonlink/game_AttackAirLwOffGravity.txt"
+++ "b/nk_4.0.0/toonlink/game_AttackAirLwOffGravity.txt"
@@ -1,5 +1,5 @@
 get_value_int(SO_VAR_INT_PREV_STATUS)
-0x1164b0(0, FIGHTER_STATUS_KIND_DAMAGE_FLY)
+0x116a60(0, FIGHTER_STATUS_KIND_DAMAGE_FLY)
 if(is_excute){
 	WorkModule__on_flag(Flag=FIGHTER_STATUS_WORK_ID_FLAG_RESERVE_GRAVITY_STABLE_UNABLE)
 }
