diff --git "a/2.0.0/robot/game_FinalAirStart.txt" "b/3.0.0/robot/game_FinalAirStart.txt"
index 5094631..a801f89 100644
--- "a/2.0.0/robot/game_FinalAirStart.txt"
+++ "b/3.0.0/robot/game_FinalAirStart.txt"
@@ -11,11 +11,11 @@ if(0x1050b0(false, false)){
 		REQ_FINAL_START_CAMERA(hash40("d04finalstart.nuanmb"), false)
 	}
 	else{
 		if(is_excute){
 			PostureModule::scale(5, 0)
-			0x1050b0(531394267, 2)
+			0x1050b0(-1692598717, 2)
 			CAM_ZOOM_IN_arg5(0, 0)
 			FT_START_CUTIN()
 		}
 	}
 }
