diff --git "a/0.0/pit/game_FinalRide.txt" "b/0.0/pit/game_FinalRide.txt"
index e3d1a99..886188c 100644
--- "a/0.0/pit/game_FinalRide.txt"
+++ "b/0.0/pit/game_FinalRide.txt"
@@ -15,37 +15,37 @@ if(method.lib::L2CValue.operator___lib::L2CValueconst__const(0, 1.0)){
 	if(is_excute){
 		SlowModule__set_whole(90, 0)
 	}
 }
 get_value_float(SO_VAR_FLOAT_LR)
-if(0x10f8f0(0, 0)){
+if(0x10f930(0, 0)){
 	if(is_excute){
 		PostureModule__scale(1, 0)
-		0x10f8f0(1237842842, 1.5)
+		0x10f930(-119851890, 1.5)
 		CAM_ZOOM_IN_arg5(-0.3, 0.5)
 	}
 	else{
 		if(is_excute){
 			SlowModule__set_whole(90, 0)
 		}
 		get_value_float(SO_VAR_FLOAT_LR)
-		if(0x10f8f0(0, 0)){
+		if(0x10f930(0, 0)){
 			if(is_excute){
 				PostureModule__scale(1, 0)
-				0x10f8f0(1237842842, 1.5)
+				0x10f930(-119851890, 1.5)
 				CAM_ZOOM_IN_arg5(0, 0)
 			}
 			else{
 				if(is_excute){
 					PostureModule__scale(1, 0)
-					0x10f8f0(1237842842, 1.5)
+					0x10f930(-119851890, 1.5)
 					CAM_ZOOM_IN_arg5(-0.3, -0.5)
 				}
 				else{
 					if(is_excute){
 						PostureModule__scale(1, 0)
-						0x10f8f0(1237842842, 1.5)
+						0x10f930(-119851890, 1.5)
 						CAM_ZOOM_IN_arg5(0, 0)
 					}
 				}
 			}
 		}
