diff --git "a/7.0.0/falco/game_AttackAirLw.txt" "b/8.0.0/falco/game_AttackAirLw.txt"
index 34f63ac..3d39271 100644
--- "a/7.0.0/falco/game_AttackAirLw.txt"
+++ "b/8.0.0/falco/game_AttackAirLw.txt"
@@ -18,5 +18,9 @@ if(is_excute){
 }
 frame(Frame=30)
 if(is_excute){
 	WorkModule::off_flag(Flag=FIGHTER_STATUS_ATTACK_AIR_FLAG_ENABLE_LANDING)
 }
+frame(Frame=32)
+FT_MOTION_RATE(FSM=0.6)
+frame(Frame=52)
+FT_MOTION_RATE(FSM=1)
