diff --git "a/_9.0.0/pacman/game_Win3.txt" "b/10.0.0/pacman/game_Win3.txt"
index 5caa580..a0be62d 100644
--- "a/_9.0.0/pacman/game_Win3.txt"
+++ "b/10.0.0/pacman/game_Win3.txt"
@@ -1,6 +1,6 @@
-if(0x119ed0(WorkModule::is_flag(FIGHTER_INSTANCE_WORK_ID_FLAG_RESULT_FIGHTER_SINGLE), false, false)){
+if(0x119f70(WorkModule::is_flag(FIGHTER_INSTANCE_WORK_ID_FLAG_RESULT_FIGHTER_SINGLE), false, false)){
 	if(is_excute){
 		ArticleModule::generate_article(FIGHTER_PACMAN_GENERATE_ARTICLE_ARTISTICPOINT)
 		as_hash__const(FIGHTER_PACMAN_GENERATE_ARTICLE_ARTISTICPOINT, hash40("win3"))
 		ArticleModule::change_motion()
 		ArticleModule::generate_article(FIGHTER_PACMAN_GENERATE_ARTICLE_FAIRY)
