diff --git "a/_8.0.0/pikmin_pikmin/game_SpNTakenOutStart_b.txt" "b/_9.0.0/pikmin_pikmin/game_SpNTakenOutStart_b.txt"
index 72a4ade..8b880f7 100644
--- "a/_8.0.0/pikmin_pikmin/game_SpNTakenOutStart_b.txt"
+++ "b/_9.0.0/pikmin_pikmin/game_SpNTakenOutStart_b.txt"
@@ -1,9 +1,9 @@
 frame(Frame=1)
 if(is_excute){
 	WorkModule::get_float(WEAPON_PIKMIN_PIKMIN_STATUS_PULL_OUT_START_WORK_FLOAT_MOT_RATE)
-	MotionModule::set_rate(-502677940)
+	MotionModule::set_rate(617544972)
 }
 frame(Frame=5)
 if(is_excute){
 	MotionModule::set_rate(1)
 }
