diff --git "a/3.0.0/demon/game_FinalAirStart.txt" "b/3.0.1/demon/game_FinalAirStart.txt"
index 671fed0..98f55bc 100644
--- "a/3.0.0/demon/game_FinalAirStart.txt"
+++ "b/3.0.1/demon/game_FinalAirStart.txt"
@@ -40,11 +40,11 @@ if(0x3cfb70(false, false)){
 			REQ_FINAL_START_CAMERA_arg3(hash40("d04finalstart.nuanmb"), true, false)
 		}
 		else{
 			if(is_excute){
 				PostureModule::scale(10, 0)
-				0x3cfb70(-155490015, 2.5)
+				0x3cfb70(-1037516665, 2.5)
 				CAM_ZOOM_IN_arg5(0, 0)
 				camera(MA_MSC_CMD_CAMERA_CAM_OFFSET, 0, 0)
 			}
 			else{
 				if(is_excute){
