diff --git "a/_9.0.0/tantan/game_AttackAirN.txt" "b/10.0.0/tantan/game_AttackAirN.txt"
index e3891b9..e943823 100644
--- "a/_9.0.0/tantan/game_AttackAirN.txt"
+++ "b/10.0.0/tantan/game_AttackAirN.txt"
@@ -5,11 +5,11 @@ if(is_excute){
 frame(Frame=10)
 if(is_excute){
 	HIT_NODE(hash40("handl"), HIT_STATUS_XLU)
 	HIT_NODE(hash40("arml5"), HIT_STATUS_XLU)
 }
-if(0x3a06b0(WorkModule::is_flag(53292), false, true)){
+if(0x3a06b0(WorkModule::is_flag(53408), false, true)){
 	if(is_excute){
 		ATTACK(ID=0, Part=0, Bone=hash40("arml5"), Damage=9.775, Angle=50, KBG=100, FKB=0, BKB=50, Size=2.5, X=1.0, Y=0.0, Z=0.0, X2=LUA_VOID, Y2=LUA_VOID, Z2=LUA_VOID, Hitlag=1.0, SDI=1.0, Clang_Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=false, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=false, Absorbable=false, Flinchless=false, DisableHitlag=false, Direct_Hitbox=true, Ground_or_Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=false, Effect=hash40("collision_attr_normal"), SFXLevel=ATTACK_SOUND_LEVEL_L, SFXType=COLLISION_SOUND_ATTR_PUNCH, Type=ATTACK_REGION_PUNCH)
 		ATTACK(ID=1, Part=0, Bone=hash40("handl"), Damage=9.775, Angle=50, KBG=100, FKB=0, BKB=50, Size=5.5, X=3.5, Y=0.0, Z=0.0, X2=LUA_VOID, Y2=LUA_VOID, Z2=LUA_VOID, Hitlag=1.0, SDI=1.0, Clang_Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=false, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=false, Absorbable=false, Flinchless=false, DisableHitlag=false, Direct_Hitbox=true, Ground_or_Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=false, Effect=hash40("collision_attr_normal"), SFXLevel=ATTACK_SOUND_LEVEL_L, SFXType=COLLISION_SOUND_ATTR_PUNCH, Type=ATTACK_REGION_PUNCH)
 	}
 	else{
