diff --git "a/_9.0.0/tantan/game_SpecialAirHiStart.txt" "b/10.0.0/tantan/game_SpecialAirHiStart.txt"
index fc06280..67f82be 100644
--- "a/_9.0.0/tantan/game_SpecialAirHiStart.txt"
+++ "b/10.0.0/tantan/game_SpecialAirHiStart.txt"
@@ -1,17 +1,17 @@
 frame(Frame=4)
 if(is_excute){
-	WorkModule::on_flag(Flag=53788)
+	WorkModule::on_flag(Flag=53904)
 }
 frame(Frame=5)
 if(is_excute){
 	WorkModule::on_flag(Flag=FIGHTER_STATUS_AIR_LASSO_FLAG_CHECK)
 	WorkModule::on_flag(Flag=FIGHTER_STATUS_AIR_LASSO_FLAG_HANG_IMMIDIATE)
 }
 frame(Frame=6)
 if(is_excute){
-	GroundModule::select_cliff_hangdata(58476)
+	GroundModule::select_cliff_hangdata(59148)
 	WorkModule::off_flag(Flag=FIGHTER_STATUS_AIR_LASSO_FLAG_HANG_IMMIDIATE)
 }
 frame(Frame=8)
 if(is_excute){
 	WorkModule::off_flag(Flag=FIGHTER_STATUS_AIR_LASSO_FLAG_CHECK)
