diff --git "a/_9.0.0/tantan/game_AttackLongStartR1.txt" "b/10.0.0/tantan/game_AttackLongStartR1.txt"
index cf6f48c..4e99c24 100644
--- "a/_9.0.0/tantan/game_AttackLongStartR1.txt"
+++ "b/10.0.0/tantan/game_AttackLongStartR1.txt"
@@ -1,11 +1,11 @@
 if(is_excute){
 	FighterAreaModuleImpl::enable_fix_jostle_area(3.5, 3.5)
 }
 frame(Frame=9)
 if(is_excute){
-	WorkModule::on_flag(Flag=53200)
+	WorkModule::on_flag(Flag=53316)
 }
 frame(Frame=15)
 if(is_excute){
 	HIT_NO(2, HIT_STATUS_OFF)
 	HIT_NO(4, HIT_STATUS_OFF)
