diff --git "a/_9.0.0/tantan/game_SpecialAirHi.txt" "b/10.0.0/tantan/game_SpecialAirHi.txt"
index 2d89d81..a4c5f79 100644
--- "a/_9.0.0/tantan/game_SpecialAirHi.txt"
+++ "b/10.0.0/tantan/game_SpecialAirHi.txt"
@@ -8,7 +8,7 @@ if(is_excute){
 	HIT_NO(13, HIT_STATUS_OFF)
 	HIT_NO(19, HIT_STATUS_OFF)
 }
 frame(0, 2)
 if(is_excute){
-	WorkModule::on_flag(Flag=53768)
+	WorkModule::on_flag(Flag=53884)
 }
