diff --git "a/to_4.0.0/murabito/game_AttackAirHi.txt" "b/to_5.0.0/murabito/game_AttackAirHi.txt"
index b8f2daf..436d4e9 100644
--- "a/to_4.0.0/murabito/game_AttackAirHi.txt"
+++ "b/to_5.0.0/murabito/game_AttackAirHi.txt"
@@ -1,15 +1,15 @@
 frame(Frame=6)
 if(is_excute){
 	WorkModule__on_flag(Flag=FIGHTER_STATUS_ATTACK_AIR_FLAG_ENABLE_LANDING)
 }
-if(0x126b60(FIGHTER_MURABITO_INSTANCE_WORK_ID_INT_TURNIP_NUM, 1)){
+if(0x126b70(FIGHTER_MURABITO_INSTANCE_WORK_ID_INT_TURNIP_NUM, 1)){
 	if(is_excute){
 		ATTACK(ID=0, Part=0, Bone=top, Damage=8.0, Angle=85, KBG=100, FKB=0, BKB=30, Size=5.2, X=0.0, Y=20.0, Z=0.0, Hitlag=1.0, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_normal, SFXLevel=ATTACK_SOUND_LEVEL_M, SFXType=COLLISION_SOUND_ATTR_PUNCH, Type=ATTACK_REGION_OBJECT)
 	}
 	else{
-		if(0x126b60(FIGHTER_MURABITO_INSTANCE_WORK_ID_INT_TURNIP_NUM, 2)){
+		if(0x126b70(FIGHTER_MURABITO_INSTANCE_WORK_ID_INT_TURNIP_NUM, 2)){
 			if(is_excute){
 				ATTACK(ID=0, Part=0, Bone=top, Damage=10.0, Angle=85, KBG=100, FKB=0, BKB=30, Size=6.5, X=0.0, Y=20.0, Z=0.0, Hitlag=1.0, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_normal, SFXLevel=ATTACK_SOUND_LEVEL_L, SFXType=COLLISION_SOUND_ATTR_PUNCH, Type=ATTACK_REGION_OBJECT)
 			}
 			else{
 				if(is_excute){
@@ -18,16 +18,16 @@ if(0x126b60(FIGHTER_MURABITO_INSTANCE_WORK_ID_INT_TURNIP_NUM, 1)){
 			}
 		}
 	}
 }
 wait(Frames=2)
-if(0x126b60(FIGHTER_MURABITO_INSTANCE_WORK_ID_INT_TURNIP_NUM, 1)){
+if(0x126b70(FIGHTER_MURABITO_INSTANCE_WORK_ID_INT_TURNIP_NUM, 1)){
 	if(is_excute){
 		ATTACK(ID=0, Part=0, Bone=top, Damage=4.0, Angle=85, KBG=100, FKB=0, BKB=30, Size=3.8, X=0.0, Y=18.0, Z=0.0, Hitlag=1.0, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_normal, SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=COLLISION_SOUND_ATTR_PUNCH, Type=ATTACK_REGION_OBJECT)
 	}
 	else{
-		if(0x126b60(FIGHTER_MURABITO_INSTANCE_WORK_ID_INT_TURNIP_NUM, 2)){
+		if(0x126b70(FIGHTER_MURABITO_INSTANCE_WORK_ID_INT_TURNIP_NUM, 2)){
 			if(is_excute){
 				ATTACK(ID=0, Part=0, Bone=top, Damage=5.0, Angle=85, KBG=100, FKB=0, BKB=30, Size=4.5, X=0.0, Y=18.0, Z=0.0, Hitlag=1.0, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_normal, SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=COLLISION_SOUND_ATTR_PUNCH, Type=ATTACK_REGION_OBJECT)
 			}
 			else{
 				if(is_excute){
