diff --git "a/r_3.1.0/richter/game_FinalStart_Com.txt" "b/r_4.0.0/richter/game_FinalStart_Com.txt"
index 7ca3537..e03dacf 100644
--- "a/r_3.1.0/richter/game_FinalStart_Com.txt"
+++ "b/r_4.0.0/richter/game_FinalStart_Com.txt"
@@ -15,11 +15,11 @@ if(0x16e000(40, 0)){
 		REQ_FINAL_START_CAMERA_arg3(0x1654ea70d3, False, False)
 	}
 	else{
 		if(is_excute){
 			PostureModule__scale(5, 0)
-			0x114aa0(-73020781, 2.1)
+			0x114ba0(527401287, 2.1)
 			CAM_ZOOM_IN_arg5(0, 0)
 		}
 		else{
 			if(is_excute){
 				REQ_FINAL_START_CAMERA_arg3(d04finalstart.nuanmb, False, False)
