diff --git "a/.0.0/ness/game_FinalAir.txt" "b/.0.0/ness/game_FinalAir.txt"
index ef87f82..0fa87d3 100644
--- "a/.0.0/ness/game_FinalAir.txt"
+++ "b/.0.0/ness/game_FinalAir.txt"
@@ -20,11 +20,11 @@ if(0xfc1b0(WorkModule__is_flag(FIGHTER_INSTANCE_WORK_ID_FLAG_DISABLE_FINAL_START
 		REQ_FINAL_START_CAMERA(d04final.nuanmb, False)
 	}
 	else{
 		if(is_excute){
 			PostureModule__scale(5, 0)
-			0xfc1b0(-926713872, 2.2)
+			0xfc1b0(-1612462446, 2.2)
 			CAM_ZOOM_IN_arg5(0, 0)
 			FT_START_CUTIN()
 		}
 	}
 }
