diff --git "a/.0.0/jack/game_SpecialAirNEscapeB_Ex.txt" "b/.0.0/jack/game_SpecialAirNEscapeB_Ex.txt"
index a9650a8..c2152c7 100644
--- "a/.0.0/jack/game_SpecialAirNEscapeB_Ex.txt"
+++ "b/.0.0/jack/game_SpecialAirNEscapeB_Ex.txt"
@@ -5,11 +5,11 @@ if(is_excute){
 frame(Frame=17)
 if(is_excute){
 	JostleModule__set_status(True)
 }
 frame(Frame=22)
-if(0x290440(WorkModule__is_flag(FIGHTER_JACK_INSTANCE_WORK_ID_FLAG_DOYLE), False, True)){
+if(0x2905a0(WorkModule__is_flag(FIGHTER_JACK_INSTANCE_WORK_ID_FLAG_DOYLE), False, True)){
 	if(is_excute){
 		ATTACK(ID=0, Part=0, Bone=top, Damage=6.0, Angle=361, KBG=50, FKB=0, BKB=60, Size=2.8, X=0.0, Y=7.0, Z=9.0, X2=0.0, Y2=7.0, Z2=24.0, Hitlag=0.0, SDI=0.0, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=False, ShieldDamage=-3, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=False, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=0x1a6f665bee, SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=COLLISION_SOUND_ATTR_JACK_SHOT, Type=ATTACK_REGION_OBJECT)
 		ATTACK(ID=1, Part=0, Bone=top, Damage=3.0, Angle=361, KBG=50, FKB=0, BKB=3, Size=2.8, X=0.0, Y=7.0, Z=9.0, X2=0.0, Y2=7.0, Z2=34.0, Hitlag=0.0, SDI=0.0, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=False, ShieldDamage=-1, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=False, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=0x1a6f665bee, SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=COLLISION_SOUND_ATTR_JACK_SHOT, Type=ATTACK_REGION_OBJECT)
 		ATTACK(ID=2, Part=0, Bone=top, Damage=1.0, Angle=361, KBG=0, FKB=0, BKB=0, Size=2.8, X=0.0, Y=7.0, Z=9.0, X2=0.0, Y2=7.0, Z2=69.0, Hitlag=0.0, SDI=0.0, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=False, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=0x1a6f665bee, SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=COLLISION_SOUND_ATTR_JACK_SHOT, Type=ATTACK_REGION_OBJECT)
 		FT_SHOOTING_ATTACK_GROUND_CHECK_NEW(0, 4, 4)
@@ -20,11 +20,11 @@ if(0x290440(WorkModule__is_flag(FIGHTER_JACK_INSTANCE_WORK_ID_FLAG_DOYLE), False
 wait(Frames=1)
 if(is_excute){
 	AttackModule__clear_all()
 }
 frame(Frame=28)
-if(0x290440(WorkModule__is_flag(FIGHTER_JACK_INSTANCE_WORK_ID_FLAG_DOYLE), False, True)){
+if(0x2905a0(WorkModule__is_flag(FIGHTER_JACK_INSTANCE_WORK_ID_FLAG_DOYLE), False, True)){
 	if(is_excute){
 		ATTACK(ID=0, Part=0, Bone=top, Damage=2.0, Angle=361, KBG=0, FKB=0, BKB=0, Size=2.8, X=0.0, Y=7.0, Z=9.0, X2=0.0, Y2=7.0, Z2=24.0, Hitlag=0.0, SDI=0.0, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=False, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=0x1a6f665bee, SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=COLLISION_SOUND_ATTR_JACK_SHOT, Type=ATTACK_REGION_OBJECT)
 		ATTACK(ID=1, Part=0, Bone=top, Damage=1.5, Angle=361, KBG=0, FKB=0, BKB=0, Size=2.8, X=0.0, Y=7.0, Z=9.0, X2=0.0, Y2=7.0, Z2=34.0, Hitlag=0.0, SDI=0.0, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=False, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=0x1a6f665bee, SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=COLLISION_SOUND_ATTR_JACK_SHOT, Type=ATTACK_REGION_OBJECT)
 		ATTACK(ID=2, Part=0, Bone=top, Damage=1.0, Angle=361, KBG=0, FKB=0, BKB=0, Size=2.8, X=0.0, Y=7.0, Z=9.0, X2=0.0, Y2=7.0, Z2=69.0, Hitlag=0.0, SDI=0.0, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=False, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=0x1a6f665bee, SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=COLLISION_SOUND_ATTR_JACK_SHOT, Type=ATTACK_REGION_OBJECT)
 		FT_SHOOTING_ATTACK_GROUND_CHECK_NEW(0, 4, 4)
@@ -35,11 +35,11 @@ if(0x290440(WorkModule__is_flag(FIGHTER_JACK_INSTANCE_WORK_ID_FLAG_DOYLE), False
 wait(Frames=1)
 if(is_excute){
 	AttackModule__clear_all()
 }
 frame(Frame=34)
-if(0x290440(WorkModule__is_flag(FIGHTER_JACK_INSTANCE_WORK_ID_FLAG_DOYLE), False, True)){
+if(0x2905a0(WorkModule__is_flag(FIGHTER_JACK_INSTANCE_WORK_ID_FLAG_DOYLE), False, True)){
 	if(is_excute){
 		ATTACK(ID=0, Part=0, Bone=top, Damage=2.0, Angle=361, KBG=0, FKB=0, BKB=0, Size=2.8, X=0.0, Y=7.0, Z=9.0, X2=0.0, Y2=7.0, Z2=24.0, Hitlag=0.0, SDI=0.0, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=False, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=0x1a6f665bee, SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=COLLISION_SOUND_ATTR_JACK_SHOT, Type=ATTACK_REGION_OBJECT)
 		ATTACK(ID=1, Part=0, Bone=top, Damage=1.5, Angle=361, KBG=0, FKB=0, BKB=0, Size=2.8, X=0.0, Y=7.0, Z=9.0, X2=0.0, Y2=7.0, Z2=34.0, Hitlag=0.0, SDI=0.0, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=False, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=0x1a6f665bee, SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=COLLISION_SOUND_ATTR_JACK_SHOT, Type=ATTACK_REGION_OBJECT)
 		ATTACK(ID=2, Part=0, Bone=top, Damage=1.0, Angle=361, KBG=0, FKB=0, BKB=0, Size=2.8, X=0.0, Y=7.0, Z=9.0, X2=0.0, Y2=7.0, Z2=69.0, Hitlag=0.0, SDI=0.0, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=False, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=0x1a6f665bee, SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=COLLISION_SOUND_ATTR_JACK_SHOT, Type=ATTACK_REGION_OBJECT)
 		FT_SHOOTING_ATTACK_GROUND_CHECK_NEW(0, 4, 4)
