diff --git "a/n_3.0.0/captain/game_SpecialSEnd.txt" "b/n_3.1.0/captain/game_SpecialSEnd.txt"
index 04672c9..c1298ab 100644
--- "a/n_3.0.0/captain/game_SpecialSEnd.txt"
+++ "b/n_3.1.0/captain/game_SpecialSEnd.txt"
@@ -1,20 +1,25 @@
 frame(Frame=1)
+FT_MOTION_RATE(FSM=0.8)
 if(is_excute){
 	JostleModule__set_status(False)
-	damage(MA_MSC_DAMAGE_DAMAGE_NO_REACTION, DAMAGE_NO_REACTION_MODE_DAMAGE_POWER, 8)
+	damage(MA_MSC_DAMAGE_DAMAGE_NO_REACTION, DAMAGE_NO_REACTION_MODE_DAMAGE_POWER, 10)
 }
 frame(Frame=6)
 if(is_excute){
 	ATTACK(ID=0, Part=0, Bone=handl, Damage=10.0, Angle=76, KBG=55, FKB=0, BKB=82, Size=6.0, X=0.0, Y=-2.0, Z=-1.0, Hitlag=1.5, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=False, ShieldDamage=4, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_fire, SFXLevel=ATTACK_SOUND_LEVEL_L, SFXType=COLLISION_SOUND_ATTR_FIRE, Type=ATTACK_REGION_PUNCH)
 	ATTACK(ID=1, Part=0, Bone=top, Damage=10.0, Angle=76, KBG=55, FKB=0, BKB=82, Size=5.5, X=0.0, Y=9.0, Z=5.5, Hitlag=1.5, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=False, ShieldDamage=4, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_fire, SFXLevel=ATTACK_SOUND_LEVEL_L, SFXType=COLLISION_SOUND_ATTR_FIRE, Type=ATTACK_REGION_PUNCH)
+	AttackModule__set_add_reaction_frame(ID=0, Frames=3, Unk=False)
+	AttackModule__set_add_reaction_frame(ID=1, Frames=3, Unk=False)
 }
 wait(Frames=1)
+FT_MOTION_RATE(FSM=1)
 if(is_excute){
 	AttackModule__clear(ID=1)
 	damage(MA_MSC_DAMAGE_DAMAGE_NO_REACTION, DAMAGE_NO_REACTION_MODE_NORMAL, 0)
 	ATTACK(ID=0, Part=0, Bone=handl, Damage=10.0, Angle=76, KBG=55, FKB=0, BKB=82, Size=4.5, X=0.0, Y=0.0, Z=0.0, Hitlag=1.5, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_THRU, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=False, ShieldDamage=4, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_fire, SFXLevel=ATTACK_SOUND_LEVEL_M, SFXType=COLLISION_SOUND_ATTR_FIRE, Type=ATTACK_REGION_PUNCH)
+	AttackModule__set_add_reaction_frame(ID=0, Frames=3, Unk=False)
 }
 wait(Frames=2)
 FT_MOTION_RATE(FSM=0.75)
 if(is_excute){
 	AttackModule__clear_all()
