diff --git "a/ac_10.1.0/littlemac/game_Win2Wait.txt" "b/ac_11.0.0/littlemac/game_Win2Wait.txt"
index 9650407..ac40320 100644
--- "a/ac_10.1.0/littlemac/game_Win2Wait.txt"
+++ "b/ac_11.0.0/littlemac/game_Win2Wait.txt"
@@ -1,8 +1,9 @@
-operator___lib::L2CValueconst__const(WorkModule::is_flag(FIGHTER_INSTANCE_WORK_ID_FLAG_RESULT_FIGHTER_SINGLE), false, true)
-IS_EXIST_ARTICLE(FIGHTER_LITTLEMAC_GENERATE_ARTICLE_DOCLOUIS)
-if(method.lib::L2CValue.operator___lib::L2CValueconst__const(false, true)){
-	if(is_excute){
-		as_hash__const(FIGHTER_LITTLEMAC_GENERATE_ARTICLE_DOCLOUIS, hash40("win2_wait"))
-		ArticleModule::change_motion()
+if(method.lib::L2CValue.operator___lib::L2CValueconst__const(WorkModule::is_flag(FIGHTER_INSTANCE_WORK_ID_FLAG_RESULT_FIGHTER_SINGLE), false, true)){
+	IS_EXIST_ARTICLE(FIGHTER_LITTLEMAC_GENERATE_ARTICLE_DOCLOUIS)
+	if(method.lib::L2CValue.operator___lib::L2CValueconst__const(false, true)){
+		if(is_excute){
+			as_hash__const(FIGHTER_LITTLEMAC_GENERATE_ARTICLE_DOCLOUIS, hash40("win2_wait"))
+			ArticleModule::change_motion()
+		}
 	}
 }
