diff --git "a/n_7.0.0/rockman/game_SpecialNTurn.txt" "b/n_8.0.0/rockman/game_SpecialNTurn.txt"
index 32c54f1..b9260b2 100644
--- "a/n_7.0.0/rockman/game_SpecialNTurn.txt"
+++ "b/n_8.0.0/rockman/game_SpecialNTurn.txt"
@@ -1,12 +1,12 @@
 frame(Frame=7)
-if(0xf4400(WorkModule::is_flag(FIGHTER_ROCKMAN_STATUS_SPECIAL_N_WORK_ID_FLAG_ALREADY_EXIST_METALBLADE), false, false)){
+if(0xf49c0(WorkModule::is_flag(FIGHTER_ROCKMAN_STATUS_SPECIAL_N_WORK_ID_FLAG_ALREADY_EXIST_METALBLADE), false, false)){
 	if(is_excute){
 		ArticleModule::generate_article_enable(FIGHTER_ROCKMAN_GENERATE_ARTICLE_METALBLADE)
 	}
 }
 frame(Frame=18)
-if(0xf4400(WorkModule::is_flag(FIGHTER_ROCKMAN_STATUS_SPECIAL_N_WORK_ID_FLAG_ALREADY_EXIST_METALBLADE), false, false)){
+if(0xf49c0(WorkModule::is_flag(FIGHTER_ROCKMAN_STATUS_SPECIAL_N_WORK_ID_FLAG_ALREADY_EXIST_METALBLADE), false, false)){
 	if(is_excute){
 		ArticleModule::shoot_exist(FIGHTER_ROCKMAN_GENERATE_ARTICLE_METALBLADE, ARTICLE_OPE_TARGET_ALL, false)
 	}
 }
