diff --git "a/0.0/fox/game_SpecialAirNEnd.txt" "b/0.0/fox/game_SpecialAirNEnd.txt"
index 9bf70bd..00bfc6d 100644
--- "a/0.0/fox/game_SpecialAirNEnd.txt"
+++ "b/0.0/fox/game_SpecialAirNEnd.txt"
@@ -1,7 +1,7 @@
 IS_EXIST_ARTICLE(FIGHTER_FOX_GENERATE_ARTICLE_BLASTER)
-if(0x10cfa0(false, true)){
+if(0x10cfb0(false, true)){
 	if(is_excute){
 		as_hash__const(FIGHTER_FOX_GENERATE_ARTICLE_BLASTER, hash40("close"))
 		ArticleModule::change_motion()
 	}
 }
