diff --git "a/rdsman_3.1.0/miiswordsman/game_SpecialAirN3EndTurn.txt" "b/rdsman_4.0.0/miiswordsman/game_SpecialAirN3EndTurn.txt"
index 2d9d0da..2f20b8e 100644
--- "a/rdsman_3.1.0/miiswordsman/game_SpecialAirN3EndTurn.txt"
+++ "b/rdsman_4.0.0/miiswordsman/game_SpecialAirN3EndTurn.txt"
@@ -1,21 +1,30 @@
 if(is_excute){
 	SET_SPEED_EX(-0.5, 1.2, KINETIC_ENERGY_RESERVE_ATTRIBUTE_MAIN)
 }
 frame(Frame=6)
-for(4 Iterations){
+for(2 Iterations){
 	if(is_excute){
-		ATTACK(ID=0, Part=0, Bone=haver, Damage=0.8, Angle=45, KBG=100, FKB=10, BKB=0, Size=3.2, X=0.0, Y=-2.0, Z=0.0, X2=0.0, Y2=9.0, Z2=0.0, Hitlag=0.5, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_fire, SFXLevel=ATTACK_SOUND_LEVEL_M, SFXType=COLLISION_SOUND_ATTR_CUTUP, Type=ATTACK_REGION_SWORD)
+		ATTACK(ID=0, Part=0, Bone=haver, Damage=0.8, Angle=90, KBG=100, FKB=1, BKB=0, Size=3.2, X=0.0, Y=-2.0, Z=0.0, X2=0.0, Y2=9.0, Z2=0.0, Hitlag=0.5, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_fire, SFXLevel=ATTACK_SOUND_LEVEL_M, SFXType=COLLISION_SOUND_ATTR_CUTUP, Type=ATTACK_REGION_SWORD)
+	}
+	wait(Frames=2)
+	if(is_excute){
+		AttackModule__clear_all()
+	}
+	wait(Frames=2)
+	if(is_excute){
+		ATTACK(ID=0, Part=0, Bone=haver, Damage=0.8, Angle=285, KBG=100, FKB=10, BKB=0, Size=3.2, X=0.0, Y=-2.0, Z=0.0, X2=0.0, Y2=9.0, Z2=0.0, Hitlag=0.5, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_fire, SFXLevel=ATTACK_SOUND_LEVEL_M, SFXType=COLLISION_SOUND_ATTR_CUTUP, Type=ATTACK_REGION_SWORD)
 	}
 	wait(Frames=2)
 	if(is_excute){
 		AttackModule__clear_all()
 	}
 	wait(Frames=2)
 }
 if(is_excute){
-	ATTACK(ID=0, Part=0, Bone=haver, Damage=0.8, Angle=45, KBG=100, FKB=25, BKB=0, Size=3.2, X=0.0, Y=-2.0, Z=0.0, X2=0.0, Y2=9.0, Z2=0.0, Hitlag=0.5, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=False, ShieldDamage=1, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_fire, SFXLevel=ATTACK_SOUND_LEVEL_M, SFXType=COLLISION_SOUND_ATTR_CUTUP, Type=ATTACK_REGION_SWORD)
+	ATTACK(ID=0, Part=0, Bone=haver, Damage=0.8, Angle=120, KBG=100, FKB=20, BKB=0, Size=4.0, X=0.0, Y=8.2, Z=0.0, Hitlag=0.5, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=False, ShieldDamage=1, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_fire, SFXLevel=ATTACK_SOUND_LEVEL_M, SFXType=COLLISION_SOUND_ATTR_CUTUP, Type=ATTACK_REGION_SWORD)
+	ATTACK(ID=1, Part=0, Bone=haver, Damage=0.8, Angle=45, KBG=100, FKB=25, BKB=0, Size=4.0, X=0.0, Y=-1.2, Z=0.0, X2=0.0, Y2=8.2, Z2=0.0, Hitlag=0.5, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=False, ShieldDamage=1, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_fire, SFXLevel=ATTACK_SOUND_LEVEL_M, SFXType=COLLISION_SOUND_ATTR_CUTUP, Type=ATTACK_REGION_SWORD)
 }
 wait(Frames=2)
 if(is_excute){
 	AttackModule__clear_all()
 }
