diff --git "a/rdsman_3.1.0/miiswordsman/game_FinalAirStart.txt" "b/rdsman_4.0.0/miiswordsman/game_FinalAirStart.txt"
index b52349f..e2033b2 100644
--- "a/rdsman_3.1.0/miiswordsman/game_FinalAirStart.txt"
+++ "b/rdsman_4.0.0/miiswordsman/game_FinalAirStart.txt"
@@ -7,18 +7,18 @@ if(method.lib::L2CValue.operator___lib::L2CValueconst__const(WorkModule__is_flag
 	if(is_excute){
 		FT_SET_FINAL_FEAR_FACE(60)
 	}
 }
 get_value_float(SO_VAR_FLOAT_LR)
-if(0x18ea80(0, 0)){
+if(0x190b60(0, 0)){
 	if(is_excute){
 		REQ_FINAL_START_CAMERA(0x1654ea70d3, False)
 	}
 	else{
 		if(is_excute){
 			PostureModule__scale(5, 0)
-			0x18ea80(-176937914, 2.2)
+			0x190b60(1603479518, 2.2)
 			CAM_ZOOM_IN_arg5(0, 0)
 		}
 		else{
 			if(is_excute){
 				REQ_FINAL_START_CAMERA(d04finalstart.nuanmb, False)
