diff --git "a/n_5.0.0/gaogaen/game_SpecialLwTurn.txt" "b/n_6.0.0/gaogaen/game_SpecialLwTurn.txt"
index c699152..65cb806 100644
--- "a/n_5.0.0/gaogaen/game_SpecialLwTurn.txt"
+++ "b/n_6.0.0/gaogaen/game_SpecialLwTurn.txt"
@@ -6,12 +6,12 @@ frame(Frame=4)
 if(is_excute){
 	REVERSE_LR()
 }
 frame(Frame=8)
 if(is_excute){
-	ATTACK(ID=0, Part=0, Bone=top, Damage=2.4, Angle=52, KBG=100, FKB=38, BKB=36, Size=10.0, X=0.0, Y=11.0, Z=-1.0, Hitlag=1.0, SDI=0.0, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_fire, SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=COLLISION_SOUND_ATTR_FIRE, Type=ATTACK_REGION_NONE)
-	ATTACK(ID=1, Part=0, Bone=top, Damage=2.4, Angle=56, KBG=100, FKB=36, BKB=34, Size=20.0, X=0.0, Y=11.0, Z=-1.0, Hitlag=1.0, SDI=0.0, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_fire, SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=COLLISION_SOUND_ATTR_FIRE, Type=ATTACK_REGION_NONE)
+	ATTACK(ID=0, Part=0, Bone=top, Damage=2.4, Angle=51, KBG=100, FKB=40, BKB=40, Size=10.0, X=0.0, Y=11.0, Z=-1.0, Hitlag=1.0, SDI=0.0, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=True, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_fire, SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=COLLISION_SOUND_ATTR_FIRE, Type=ATTACK_REGION_NONE)
+	ATTACK(ID=1, Part=0, Bone=top, Damage=2.4, Angle=54, KBG=100, FKB=38, BKB=38, Size=20.0, X=0.0, Y=11.0, Z=-1.0, Hitlag=1.0, SDI=0.0, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=True, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_fire, SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=COLLISION_SOUND_ATTR_FIRE, Type=ATTACK_REGION_NONE)
 }
 wait(Frames=2)
 if(is_excute){
 	AttackModule__clear_all()
 }
