diff --git "a/n_6.0.0/gaogaen/game_SpecialSShoulder.txt" "b/n_7.0.0/gaogaen/game_SpecialSShoulder.txt"
index 923380c..1409efa 100644
--- "a/n_6.0.0/gaogaen/game_SpecialSShoulder.txt"
+++ "b/n_7.0.0/gaogaen/game_SpecialSShoulder.txt"
@@ -6,11 +6,11 @@ if(is_excute){
 frame(Frame=1)
 FT_MOTION_RATE(FSM=0.75)
 frame(Frame=10)
 FT_MOTION_RATE(FSM=1)
 if(is_excute){
-	ATTACK(ID=0, Part=0, Bone=top, Damage=12.0, Angle=105, KBG=50, FKB=0, BKB=80, Size=6.0, X=0.0, Y=6.0, Z=0.0, Hitlag=0.4, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_B, SetWeight=False, ShieldDamage=1, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=True, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_normal, SFXLevel=ATTACK_SOUND_LEVEL_M, SFXType=COLLISION_SOUND_ATTR_PUNCH, Type=ATTACK_REGION_BODY)
+	ATTACK(ID=0, Part=0, Bone=top, Damage=12.0, Angle=105, KBG=34, FKB=0, BKB=128, Size=6.0, X=0.0, Y=6.0, Z=0.0, Hitlag=0.4, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_B, SetWeight=False, ShieldDamage=1, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=True, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_normal, SFXLevel=ATTACK_SOUND_LEVEL_M, SFXType=COLLISION_SOUND_ATTR_PUNCH, Type=ATTACK_REGION_BODY)
 }
 frame(Frame=17)
 if(is_excute){
 	CHECK_FINISH_CAMERA_IF_NOT_HP_MODE(0, 0)
 	FighterCutInManager__set_throw_finish_zoom_rate(ZoomRate=1.5)
