diff --git "a/_3.1.0/packun/game_FinalAir.txt" "b/_4.0.0/packun/game_FinalAir.txt"
index 48fa93b..2011658 100644
--- "a/_3.1.0/packun/game_FinalAir.txt"
+++ "b/_4.0.0/packun/game_FinalAir.txt"
@@ -10,16 +10,16 @@ if(method.lib::L2CValue.operator___lib::L2CValueconst__const(WorkModule__is_flag
 		FT_START_CUTIN()
 	}
 	else{
 		if(is_excute){
 			PostureModule__scale(3, 0)
-			0x14b1e0(-1100921532, 2.05)
+			0x14b420(1529819476, 2.05)
 			CAM_ZOOM_IN_arg5(0, 0)
 			FT_START_CUTIN()
 		}
 		get_value_float(SO_VAR_FLOAT_LR)
-		if(0x14b1e0(0, 0)){
+		if(0x14b420(0, 0)){
 			if(is_excute){
 				camera(MA_MSC_CMD_CAMERA_CAM_OFFSET, 0, 0)
 			}
 			else{
 				if(is_excute){
