diff --git "a/0.0.0/diddy_bunshin/game_FinalHi.txt" "b/0.1.0/diddy_bunshin/game_FinalHi.txt"
index 91989be..a49abae 100644
--- "a/0.0.0/diddy_bunshin/game_FinalHi.txt"
+++ "b/0.1.0/diddy_bunshin/game_FinalHi.txt"
@@ -1,14 +1,14 @@
 PostureModule::scale()
-if(0x1483f0(-767783683, 1.4)){
+if(0x1483f0(-1236325714, 1.4)){
 	if(is_excute){
 		ATTACK(ID=1, Part=0, Bone=hash40("rot"), Damage=2.0, Angle=130, KBG=60, FKB=0, BKB=45, Size=10.0, X=0.0, Y=7.0, Z=0.0, X2=0.0, Y2=-10.0, Z2=0.0, Hitlag=0.5, SDI=1.0, Clang_Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_SPEED, SetWeight=false, ShieldDamage=hash40("no"), Trip=0.0, Rehit=0, Reflectable=false, Absorbable=false, Flinchless=false, DisableHitlag=false, Direct_Hitbox=false, Ground_or_Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=false, Effect=hash40("collision_attr_fire"), SFXLevel=ATTACK_SOUND_LEVEL_M, SFXType=COLLISION_SOUND_ATTR_FIRE, Type=ATTACK_REGION_NONE)
 		AttackModule::set_force_reaction(1, true, false)
 	}
 	else{
 		PostureModule::scale()
-		if(0x1483f0(-767783683, 0.5)){
+		if(0x1483f0(-1236325714, 0.5)){
 			if(is_excute){
 				ATTACK(ID=1, Part=0, Bone=hash40("rot"), Damage=2.0, Angle=130, KBG=60, FKB=0, BKB=45, Size=10.0, X=0.0, Y=7.0, Z=0.0, X2=0.0, Y2=-30.0, Z2=0.0, Hitlag=0.5, SDI=1.0, Clang_Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_SPEED, SetWeight=false, ShieldDamage=hash40("no"), Trip=0.0, Rehit=0, Reflectable=false, Absorbable=false, Flinchless=false, DisableHitlag=false, Direct_Hitbox=false, Ground_or_Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=false, Effect=hash40("collision_attr_fire"), SFXLevel=ATTACK_SOUND_LEVEL_M, SFXType=COLLISION_SOUND_ATTR_FIRE, Type=ATTACK_REGION_NONE)
 				AttackModule::set_force_reaction(1, true, false)
 			}
 			else{
