diff --git "a/7.0.0/simon/game_FinalStart.txt" "b/8.0.0/simon/game_FinalStart.txt"
index 9f41d98..512168e 100644
--- "a/7.0.0/simon/game_FinalStart.txt"
+++ "b/8.0.0/simon/game_FinalStart.txt"
@@ -8,18 +8,18 @@ if(method.lib::L2CValue.operator___lib::L2CValueconst__const(WorkModule::is_flag
 		SlowModule::set_whole(2, 0)
 		FT_SET_FINAL_FEAR_FACE(40)
 	}
 }
 get_value_float(SO_VAR_FLOAT_LR)
-if(0x119b10(32, 0)){
+if(0x119c40(32, 0)){
 	if(is_excute){
 		REQ_FINAL_START_CAMERA_arg3(hash40("d04finalstart02.nuanmb"), false, false)
 	}
 	else{
 		if(is_excute){
 			PostureModule::scale(5, 0)
-			0x119b10(-1443810554, 2.1)
+			0x119c40(1413573805, 2.1)
 			CAM_ZOOM_IN_arg5(0, 0)
 		}
 		else{
 			if(is_excute){
 				REQ_FINAL_START_CAMERA_arg3(hash40("d04finalstart.nuanmb"), false, false)
