diff --git "a/5.0.0/krool/game_SpecialAirLwTurn.txt" "b/6.0.0/krool/game_SpecialAirLwTurn.txt"
index 7c69609..dda0f1f 100644
--- "a/5.0.0/krool/game_SpecialAirLwTurn.txt"
+++ "b/6.0.0/krool/game_SpecialAirLwTurn.txt"
@@ -5,11 +5,11 @@ frame(Frame=4)
 if(is_excute){
 	REVERSE_LR()
 }
 frame(Frame=9)
 if(is_excute){
-	ATTACK(ID=0, Part=0, Bone=top, Damage=0.0, Angle=45, KBG=56, FKB=0, BKB=90, Size=10.0, X=0.0, Y=11.0, Z=-9.0, Hitlag=1.2, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=False, ShieldDamage=1, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_normal, SFXLevel=ATTACK_SOUND_LEVEL_M, SFXType=COLLISION_SOUND_ATTR_KICK, Type=ATTACK_REGION_BODY)
+	ATTACK(ID=0, Part=0, Bone=top, Damage=0.0, Angle=45, KBG=56, FKB=0, BKB=90, Size=12.0, X=0.0, Y=12.5, Z=-10.5, Hitlag=1.2, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=False, ShieldDamage=1, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_normal, SFXLevel=ATTACK_SOUND_LEVEL_M, SFXType=COLLISION_SOUND_ATTR_KICK, Type=ATTACK_REGION_BODY)
 	AttackModule__set_force_reaction(0, True, False)
 	ATTACK(ID=1, Part=0, Bone=top, Damage=0.0, Angle=45, KBG=50, FKB=0, BKB=80, Size=14.0, X=0.0, Y=13.0, Z=-18.0, Hitlag=1.1, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=False, ShieldDamage=1, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_normal, SFXLevel=ATTACK_SOUND_LEVEL_M, SFXType=COLLISION_SOUND_ATTR_PUNCH, Type=ATTACK_REGION_BODY)
 	AttackModule__set_force_reaction(1, True, False)
 }
 frame(Frame=13)
@@ -19,9 +19,9 @@ if(is_excute){
 }
 frame(Frame=14)
 if(is_excute){
 	WHOLE_HIT(HIT_STATUS_NORMAL)
 }
-frame(Frame=28)
-FT_MOTION_RATE(FSM=0.8)
+frame(Frame=23)
+FT_MOTION_RATE(FSM=0.7)
 frame(Frame=53)
 FT_MOTION_RATE(FSM=1)
