diff --git "a/_9.0.0/wiifit/game_AttackAirF.txt" "b/10.0.0/wiifit/game_AttackAirF.txt"
index c3bdf6a..198fe51 100644
--- "a/_9.0.0/wiifit/game_AttackAirF.txt"
+++ "b/10.0.0/wiifit/game_AttackAirF.txt"
@@ -19,11 +19,11 @@ if(is_excute){
 wait(Frames=5)
 if(is_excute){
 	AttackModule::clear_all()
 }
 frame(Frame=16)
-FT_MOTION_RATE(FSM=0.64)
+FT_MOTION_RATE(FSM=0.76)
 frame(Frame=27)
 if(is_excute){
 	WorkModule::off_flag(Flag=FIGHTER_STATUS_ATTACK_AIR_FLAG_ENABLE_LANDING)
 }
 frame(Frame=46)
