diff --git "a/2.0.0/brave/game_SpecialAirS3.txt" "b/3.0.0/brave/game_SpecialAirS3.txt"
index c8b9a8b..d14448d 100644
--- "a/2.0.0/brave/game_SpecialAirS3.txt"
+++ "b/3.0.0/brave/game_SpecialAirS3.txt"
@@ -1,7 +1,7 @@
 frame(Frame=10)
 WorkModule::is_flag(FIGHTER_BRAVE_STATUS_SPECIAL_S_FLAG_SUCCESS_SP)
-if(0x239f10(false, true)){
+if(0x23a020(false, true)){
 	if(is_excute){
 		WorkModule::on_flag(Flag=FIGHTER_BRAVE_STATUS_SPECIAL_S_FLAG_ACTIVATE_LIGHTNING)
 	}
 }
