diff --git "a/8.0.0/koopa/game_FinalStart.txt" "b/9.0.0/koopa/game_FinalStart.txt"
index cb093a4..5898f4c 100644
--- "a/8.0.0/koopa/game_FinalStart.txt"
+++ "b/9.0.0/koopa/game_FinalStart.txt"
@@ -9,11 +9,11 @@ if(method.lib::L2CValue.operator___lib::L2CValueconst__const(WorkModule::is_flag
 		FT_START_CUTIN()
 	}
 	else{
 		if(is_excute){
 			PostureModule::scale(10, 0)
-			0xe7440(1955617966, 1.5)
+			0xe7450(-229470730, 1.5)
 			CAM_ZOOM_IN_arg5(0, 0)
 			FT_START_CUTIN()
 		}
 		frame(Frame=35)
 		if(is_excute){
