diff --git "a/3.0.0/cloud/game_SpecialLw.txt" "b/3.0.1/cloud/game_SpecialLw.txt"
index 43db6fd..a51f016 100644
--- "a/3.0.0/cloud/game_SpecialLw.txt"
+++ "b/3.0.1/cloud/game_SpecialLw.txt"
@@ -1,7 +1,7 @@
 frame(Frame=1)
-FT_MOTION_RATE(FSM=0.812)
+FT_MOTION_RATE(FSM=0.688)
 frame(Frame=16)
 FT_MOTION_RATE(FSM=1)
 if(is_excute){
 	WorkModule::on_flag(Flag=FIGHTER_CLOUD_STATUS_SPECIAL_LW_FLAG_LB_SCENE)
 	ATTACK(ID=0, Part=0, Bone=hash40("top"), Damage=1.0, Angle=80, KBG=431, FKB=0, BKB=60, Size=10.0, X=0.0, Y=10.0, Z=13.0, X2=0.0, Y2=10.0, Z2=9.0, Hitlag=2.0, SDI=1.0, Clang_Rebound=ATTACK_SETOFF_KIND_THRU, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=false, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=false, Absorbable=false, Flinchless=false, DisableHitlag=false, Direct_Hitbox=true, Ground_or_Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=false, Effect=hash40("collision_attr_normal"), SFXLevel=ATTACK_SOUND_LEVEL_L, SFXType=COLLISION_SOUND_ATTR_PUNCH, Type=ATTACK_REGION_SWORD)
