diff --git "a/_4.0.0/mewtwo_bindball/game_Shoot.txt" "b/_5.0.0/mewtwo_bindball/game_Shoot.txt"
index fd642bf..72505ca 100644
--- "a/_4.0.0/mewtwo_bindball/game_Shoot.txt"
+++ "b/_5.0.0/mewtwo_bindball/game_Shoot.txt"
@@ -1,6 +1,6 @@
 if(is_excute){
-	ATTACK(ID=0, Part=0, Bone=top, Damage=1.0, Angle=361, KBG=140, FKB=0, BKB=0, Size=2.3, X=0.0, Y=-1.7, Z=2.5, Hitlag=1.0, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=True, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=False, Ground/Air=49912, Hitbits=COLLISION_CATEGORY_MASK_FEB, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=0x199d57f593, SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=COLLISION_SOUND_ATTR_ELEC, Type=ATTACK_REGION_NONE)
+	ATTACK(ID=0, Part=0, Bone=top, Damage=1.0, Angle=361, KBG=140, FKB=0, BKB=0, Size=2.3, X=0.0, Y=-1.7, Z=2.5, Hitlag=1.0, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=True, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=False, Ground/Air=50752, Hitbits=COLLISION_CATEGORY_MASK_FEB, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=0x199d57f593, SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=COLLISION_SOUND_ATTR_ELEC, Type=ATTACK_REGION_NONE)
 	ATTACK(ID=1, Part=0, Bone=top, Damage=1.0, Angle=361, KBG=180, FKB=0, BKB=20, Size=3.0, X=0.0, Y=-1.7, Z=2.5, Hitlag=1.0, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=True, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=False, Ground/Air=COLLISION_SITUATION_MASK_A, Hitbits=COLLISION_CATEGORY_MASK_FEB, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=0x199d57f593, SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=COLLISION_SOUND_ATTR_ELEC, Type=ATTACK_REGION_NONE)
 	AttackModule__set_lr_check_front_lr(0)
 	AttackModule__set_no_finish_camera(0, True, False)
 }
