diff --git "a/_11.0.0/richter/game_FinalStart.txt" "b/_12.0.0/richter/game_FinalStart.txt"
index 93e4969..6b57269 100644
--- "a/_11.0.0/richter/game_FinalStart.txt"
+++ "b/_12.0.0/richter/game_FinalStart.txt"
@@ -9,18 +9,18 @@ if(methodlib::L2CValue::operator==(lib::L2CValueconst&)const(false, false)){
 		SlowModule::set_whole(2, 0)
 		FT_SET_FINAL_FEAR_FACE(40)
 	}
 }
 get_value_float(SO_VAR_FLOAT_LR)
-if(0x1155f0(32, 0)){
+if(0x115660(32, 0)){
 	if(is_excute){
 		REQ_FINAL_START_CAMERA_arg3(hash40("d04finalstart02.nuanmb"), false, false)
 	}
 	else{
 		if(is_excute){
 			PostureModule::scale(5, 0)
-			0x1155f0(2010766555, 2.1)
+			0x115660(-488527171, 2.1)
 			CAM_ZOOM_IN_arg5(0, 0)
 		}
 		else{
 			if(is_excute){
 				REQ_FINAL_START_CAMERA_arg3(hash40("d04finalstart.nuanmb"), false, false)
