diff --git "a/0.0/fox/game_SpecialNLoop.txt" "b/0.0/fox/game_SpecialNLoop.txt"
index bb947d4..e1d73b5 100644
--- "a/0.0/fox/game_SpecialNLoop.txt"
+++ "b/0.0/fox/game_SpecialNLoop.txt"
@@ -5,11 +5,11 @@ if(is_excute){
 frame(Frame=4)
 if(is_excute){
 	ArticleModule::generate_article(FIGHTER_FOX_GENERATE_ARTICLE_BLASTER_BULLET)
 }
 IS_EXIST_ARTICLE(FIGHTER_FOX_GENERATE_ARTICLE_BLASTER)
-if(0x10ce70(false, true)){
+if(0x10cfa0(false, true)){
 	if(is_excute){
 		as_hash__const(FIGHTER_FOX_GENERATE_ARTICLE_BLASTER, hash40("loop"))
 		ArticleModule::change_motion()
 	}
 }
