diff --git "a/0.0/fox/game_SpecialAirNStart.txt" "b/0.0/fox/game_SpecialAirNStart.txt"
index ac75020..eb1360f 100644
--- "a/0.0/fox/game_SpecialAirNStart.txt"
+++ "b/0.0/fox/game_SpecialAirNStart.txt"
@@ -1,8 +1,8 @@
 frame(Frame=4)
 IS_EXIST_ARTICLE(FIGHTER_FOX_GENERATE_ARTICLE_BLASTER)
-if(0x10ce70(false, true)){
+if(0x10cfa0(false, true)){
 	if(is_excute){
 		as_hash__const(FIGHTER_FOX_GENERATE_ARTICLE_BLASTER, hash40("open"))
 		ArticleModule::change_motion()
 	}
 }
