diff --git "a/o_3.0.0/lucario/game_AttackAirHi.txt" "b/o_3.1.0/lucario/game_AttackAirHi.txt"
index 6ce0c26..82282d5 100644
--- "a/o_3.0.0/lucario/game_AttackAirHi.txt"
+++ "b/o_3.1.0/lucario/game_AttackAirHi.txt"
@@ -1,19 +1,23 @@
+if(is_excute){
+	FighterAreaModuleImpl__enable_fix_jostle_area_xy(1, 4, 4, 4)
+}
 frame(Frame=2)
 if(is_excute){
 	WorkModule__on_flag(Flag=FIGHTER_STATUS_ATTACK_AIR_FLAG_ENABLE_LANDING)
 }
 frame(Frame=10)
 if(is_excute){
-	ATTACK(ID=0, Part=0, Bone=top, Damage=11.0, Angle=90, KBG=110, FKB=0, BKB=30, Size=4.0, X=0.0, Y=21.0, Z=0.0, X2=0.0, Y2=12.0, Z2=0.0, Hitlag=1.0, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=0x13313725f6, SFXLevel=ATTACK_SOUND_LEVEL_M, SFXType=COLLISION_SOUND_ATTR_LUCARIO, Type=ATTACK_REGION_KICK)
+	ATTACK(ID=0, Part=0, Bone=top, Damage=11.0, Angle=90, KBG=110, FKB=0, BKB=30, Size=5.0, X=0.0, Y=20.0, Z=0.0, X2=0.0, Y2=13.0, Z2=0.0, Hitlag=1.0, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=0x13313725f6, SFXLevel=ATTACK_SOUND_LEVEL_M, SFXType=COLLISION_SOUND_ATTR_LUCARIO, Type=ATTACK_REGION_KICK)
 }
 wait(Frames=2)
 if(is_excute){
-	ATTACK(ID=0, Part=0, Bone=top, Damage=6.0, Angle=90, KBG=110, FKB=0, BKB=30, Size=5.0, X=0.0, Y=17.0, Z=0.0, X2=0.0, Y2=12.0, Z2=0.0, Hitlag=1.0, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=0x13313725f6, SFXLevel=ATTACK_SOUND_LEVEL_M, SFXType=COLLISION_SOUND_ATTR_LUCARIO, Type=ATTACK_REGION_KICK)
+	ATTACK(ID=0, Part=0, Bone=top, Damage=6.0, Angle=90, KBG=110, FKB=0, BKB=30, Size=5.0, X=0.0, Y=17.0, Z=0.0, X2=0.0, Y2=13.0, Z2=0.0, Hitlag=1.0, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=0x13313725f6, SFXLevel=ATTACK_SOUND_LEVEL_M, SFXType=COLLISION_SOUND_ATTR_LUCARIO, Type=ATTACK_REGION_KICK)
 }
 wait(Frames=2)
 if(is_excute){
+	FighterAreaModuleImpl__enable_fix_jostle_area_xy(4, 4, 8, 4)
 	AttackModule__clear_all()
 }
 frame(Frame=29)
 if(is_excute){
 	WorkModule__off_flag(Flag=FIGHTER_STATUS_ATTACK_AIR_FLAG_ENABLE_LANDING)
