diff --git "a/o_3.0.0/lucario/game_FinalAirStart.txt" "b/o_3.1.0/lucario/game_FinalAirStart.txt"
index a4045c3..5fefbb9 100644
--- "a/o_3.0.0/lucario/game_FinalAirStart.txt"
+++ "b/o_3.1.0/lucario/game_FinalAirStart.txt"
@@ -11,11 +11,11 @@ if(method.lib::L2CValue.operator___lib::L2CValueconst__const(WorkModule__is_flag
 	}
 	else{
 		if(is_excute){
 			camera(MA_MSC_CMD_CAMERA_CAM_RECT, 0, 4, 6, 0)
 			PostureModule__scale(3, 0)
-			0xfaf60(1860873677, 2)
+			0xfd210(-23050645, 2)
 			CAM_ZOOM_IN_arg5(0, 0)
 			camera(MA_MSC_CMD_CAMERA_CAM_OFFSET, 0, 6)
 		}
 	}
 }
