diff --git "a/7.0.0/lucas/game_FinalAir.txt" "b/8.0.0/lucas/game_FinalAir.txt"
index 77b31dc..a6cebf6 100644
--- "a/7.0.0/lucas/game_FinalAir.txt"
+++ "b/8.0.0/lucas/game_FinalAir.txt"
@@ -1,10 +1,10 @@
 if(is_excute){
 	ArticleModule::generate_article(FIGHTER_LUCAS_GENERATE_ARTICLE_KUMATORA)
 	ArticleModule::generate_article(FIGHTER_LUCAS_GENERATE_ARTICLE_BONNIE)
 	BattleObjectWorld__scale_z()
-	if(0x1001c0(0, 1.0)){
+	if(0x100e30(0, 1.0)){
 		article(MA_MSC_CMD_ARTICLE_SET_OFFSET_ARTICLE, FIGHTER_LUCAS_GENERATE_ARTICLE_KUMATORA, -10, 0, -10)
 		article(MA_MSC_CMD_ARTICLE_SET_OFFSET_ARTICLE, FIGHTER_LUCAS_GENERATE_ARTICLE_BONNIE, 10, 0, -10)
 	}
 	else{
 		article(FIGHTER_LUCAS_GENERATE_ARTICLE_BONNIE, FIGHTER_LUCAS_GENERATE_ARTICLE_KUMATORA, -10, 0, -0.01)
@@ -20,11 +20,11 @@ if(method.lib::L2CValue.operator___lib::L2CValueconst__const(WorkModule::is_flag
 		REQ_FINAL_START_CAMERA(hash40("d04final.nuanmb"), false)
 	}
 	else{
 		if(is_excute){
 			PostureModule::scale(5, 0)
-			0x1001c0(-1482617503, 2.2)
+			0x100e30(-363716770, 2.2)
 			CAM_ZOOM_IN_arg5(0, 0)
 			FT_START_CUTIN()
 		}
 	}
 }
