diff --git "a/.0.0/edge/game_SpecialNStart.txt" "b/.1.0/edge/game_SpecialNStart.txt"
index 6a3f4ac..879c2b9 100644
--- "a/.0.0/edge/game_SpecialNStart.txt"
+++ "b/.1.0/edge/game_SpecialNStart.txt"
@@ -1,28 +1,28 @@
 frame(Frame=20)
 if(is_excute){
-	WorkModule::set_int(17296, 17276)
+	WorkModule::set_int(17316, 17296)
 }
 frame(Frame=32)
 if(is_excute){
-	WorkModule::set_int(17308, 17276)
+	WorkModule::set_int(17328, 17296)
 }
 frame(Frame=79)
 FT_MOTION_RATE(FSM=1.2)
 frame(Frame=99)
 FT_MOTION_RATE(FSM=1)
 frame(Frame=100)
 if(is_excute){
-	WorkModule::set_int(17288, 17276)
+	WorkModule::set_int(17308, 17296)
 }
 frame(Frame=105)
 FT_MOTION_RATE(FSM=1.6)
 frame(Frame=115)
 FT_MOTION_RATE(FSM=1)
 frame(Frame=120)
 if(is_excute){
-	ArticleModule::generate_article(17324)
+	ArticleModule::generate_article(17344)
 }
 frame(Frame=140)
 if(is_excute){
-	WorkModule::on_flag(Flag=17316)
+	WorkModule::on_flag(Flag=17336)
 }
