diff --git "a/10.0.0/pickel/game_SleepEnd.txt" "b/10.1.0/pickel/game_SleepEnd.txt"
index 374730b..9f7e270 100644
--- "a/10.0.0/pickel/game_SleepEnd.txt"
+++ "b/10.1.0/pickel/game_SleepEnd.txt"
@@ -1,7 +1,7 @@
 if(is_excute){
-	ArticleModule::generate_article(59184)
+	ArticleModule::generate_article(59272)
 }
 frame(Frame=41)
 if(is_excute){
-	ArticleModule::remove_exist(59184)
+	ArticleModule::remove_exist(59272)
 }
