diff --git "a/.1.0/jack/0x10ee85b4b7.txt" "b/.0.0/jack/0x10ee85b4b7.txt"
index 91d8928..9302973 100644
--- "a/.1.0/jack/0x10ee85b4b7.txt"
+++ "b/.0.0/jack/0x10ee85b4b7.txt"
@@ -1,37 +1,37 @@
 frame(Frame=76)
 if(is_excute){
-	WorkModule__on_flag(Flag=51004)
+	WorkModule__on_flag(Flag=52132)
 	FT_START_CUTIN_JACK_FINAL()
 }
 frame(Frame=100)
 if(is_excute){
-	WorkModule__on_flag(Flag=51008)
+	WorkModule__on_flag(Flag=52136)
 }
 frame(Frame=117)
 if(is_excute){
-	WorkModule__on_flag(Flag=51008)
+	WorkModule__on_flag(Flag=52136)
 }
 frame(Frame=134)
 if(is_excute){
-	WorkModule__on_flag(Flag=51008)
+	WorkModule__on_flag(Flag=52136)
 }
 frame(Frame=151)
 if(is_excute){
-	WorkModule__on_flag(Flag=51008)
+	WorkModule__on_flag(Flag=52136)
 }
 frame(Frame=168)
 if(is_excute){
-	WorkModule__on_flag(Flag=51008)
+	WorkModule__on_flag(Flag=52136)
 }
 frame(Frame=185)
 if(is_excute){
-	WorkModule__on_flag(Flag=51008)
+	WorkModule__on_flag(Flag=52136)
 }
 frame(Frame=195)
 if(is_excute){
-	WorkModule__on_flag(Flag=51008)
+	WorkModule__on_flag(Flag=52136)
 }
 frame(Frame=200)
 if(is_excute){
-	WorkModule__on_flag(Flag=47928)
+	WorkModule__on_flag(Flag=48144)
 }
