diff --git "a/.1.0/jack/game_SpecialAirHiThrow.txt" "b/.0.0/jack/game_SpecialAirHiThrow.txt"
index 1bd32b4..81ddcfb 100644
--- "a/.1.0/jack/game_SpecialAirHiThrow.txt"
+++ "b/.0.0/jack/game_SpecialAirHiThrow.txt"
@@ -1,8 +1,8 @@
 if(is_excute){
-	ArticleModule__generate_article(47992)
-	as_hash__const(47992, 0x108cd60ace)
+	ArticleModule__generate_article(FIGHTER_JACK_GENERATE_ARTICLE_WIREROPE)
+	as_hash__const(FIGHTER_JACK_GENERATE_ARTICLE_WIREROPE, 0x108cd60ace)
 	ArticleModule__change_motion()
 	ATTACK_ABS(Kind=FIGHTER_ATTACK_ABSOLUTE_KIND_THROW, ID=0, Damage=11.0, Angle=78, KBG=40, FKB=0, BKB=75, Hitlag=0.0, Unk=1.0, FacingRestrict=ATTACK_LR_CHECK_F, Unk=0.0, Unk=True, Effect=collision_attr_normal, SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=COLLISION_SOUND_ATTR_NONE, Type=ATTACK_REGION_THROW)
 }
 frame(Frame=4)
 if(is_excute){
@@ -13,7 +13,7 @@ frame(Frame=6)
 if(is_excute){
 	ATK_HIT_ABS(FIGHTER_ATTACK_ABSOLUTE_KIND_THROW, throw, FIGHTER_STATUS_THROW_WORK_INT_TARGET_OBJECT, FIGHTER_STATUS_THROW_WORK_INT_TARGET_HIT_GROUP, FIGHTER_STATUS_THROW_WORK_INT_TARGET_HIT_NO)
 }
 frame(Frame=8)
 if(is_excute){
-	ArticleModule__remove_exist(47992)
+	ArticleModule__remove_exist(FIGHTER_JACK_GENERATE_ARTICLE_WIREROPE)
 }
