diff --git "a/.1.0/jack/game_SpecialNEscapeF.txt" "b/.0.0/jack/game_SpecialNEscapeF.txt"
index ef0d5b2..da7820b 100644
--- "a/.1.0/jack/game_SpecialNEscapeF.txt"
+++ "b/.0.0/jack/game_SpecialNEscapeF.txt"
@@ -6,19 +6,19 @@ frame(Frame=13)
 if(is_excute){
 	JostleModule__set_status(True)
 }
 frame(Frame=20)
 if(is_excute){
-	ATTACK(ID=0, Part=0, Bone=top, Damage=6.0, Angle=361, KBG=67, FKB=0, BKB=30, Size=2.5, X=0.0, Y=8.5, Z=9.0, X2=0.0, Y2=8.5, Z2=14.0, Hitlag=0.25, SDI=0.0, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=False, ShieldDamage=-3, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=False, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=0x1a6f665bee, SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=48812, Type=ATTACK_REGION_OBJECT)
-	ATTACK(ID=1, Part=0, Bone=top, Damage=4.0, Angle=361, KBG=45, FKB=0, BKB=4, Size=2.5, X=0.0, Y=8.5, Z=9.0, X2=0.0, Y2=8.5, Z2=44.0, Hitlag=0.0, SDI=0.0, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=False, ShieldDamage=-2, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=False, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=0x1a6f665bee, SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=48812, Type=ATTACK_REGION_OBJECT)
-	ATTACK(ID=2, Part=0, Bone=top, Damage=2.0, Angle=361, KBG=0, FKB=0, BKB=0, Size=2.5, X=0.0, Y=8.5, Z=9.0, X2=0.0, Y2=8.5, Z2=84.0, Hitlag=0.0, SDI=0.0, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=False, ShieldDamage=-1, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=False, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=0x1a6f665bee, SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=48812, Type=ATTACK_REGION_OBJECT)
+	ATTACK(ID=0, Part=0, Bone=top, Damage=6.0, Angle=361, KBG=67, FKB=0, BKB=30, Size=2.5, X=0.0, Y=8.5, Z=9.0, X2=0.0, Y2=8.5, Z2=14.0, Hitlag=0.25, SDI=0.0, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=False, ShieldDamage=-3, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=False, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=0x1a6f665bee, SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=49936, Type=ATTACK_REGION_OBJECT)
+	ATTACK(ID=1, Part=0, Bone=top, Damage=4.0, Angle=361, KBG=45, FKB=0, BKB=4, Size=2.5, X=0.0, Y=8.5, Z=9.0, X2=0.0, Y2=8.5, Z2=44.0, Hitlag=0.0, SDI=0.0, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=False, ShieldDamage=-2, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=False, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=0x1a6f665bee, SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=49936, Type=ATTACK_REGION_OBJECT)
+	ATTACK(ID=2, Part=0, Bone=top, Damage=2.0, Angle=361, KBG=0, FKB=0, BKB=0, Size=2.5, X=0.0, Y=8.5, Z=9.0, X2=0.0, Y2=8.5, Z2=84.0, Hitlag=0.0, SDI=0.0, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=False, ShieldDamage=-1, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=False, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=0x1a6f665bee, SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=49936, Type=ATTACK_REGION_OBJECT)
 }
-if(0x2901d0(FIGHTER_INSTANCE_WORK_ID_INT_KIND, FIGHTER_KIND_KIRBY)){
+if(0x290440(FIGHTER_INSTANCE_WORK_ID_INT_KIND, FIGHTER_KIND_KIRBY)){
 	if(is_excute){
-		ATTACK(ID=0, Part=0, Bone=top, Damage=6.0, Angle=361, KBG=67, FKB=0, BKB=30, Size=2.5, X=0.0, Y=4.5, Z=9.0, X2=0.0, Y2=4.5, Z2=14.0, Hitlag=0.25, SDI=0.0, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=False, ShieldDamage=-3, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=False, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=0x1a6f665bee, SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=48812, Type=ATTACK_REGION_OBJECT)
-		ATTACK(ID=1, Part=0, Bone=top, Damage=4.0, Angle=361, KBG=45, FKB=0, BKB=4, Size=2.5, X=0.0, Y=4.5, Z=9.0, X2=0.0, Y2=4.5, Z2=44.0, Hitlag=0.0, SDI=0.0, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=False, ShieldDamage=-2, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=False, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=0x1a6f665bee, SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=48812, Type=ATTACK_REGION_OBJECT)
-		ATTACK(ID=2, Part=0, Bone=top, Damage=2.0, Angle=361, KBG=0, FKB=0, BKB=0, Size=2.5, X=0.0, Y=4.5, Z=9.0, X2=0.0, Y2=4.5, Z2=84.0, Hitlag=0.0, SDI=0.0, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=False, ShieldDamage=-1, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=False, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=0x1a6f665bee, SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=48812, Type=ATTACK_REGION_OBJECT)
+		ATTACK(ID=0, Part=0, Bone=top, Damage=6.0, Angle=361, KBG=67, FKB=0, BKB=30, Size=2.5, X=0.0, Y=4.5, Z=9.0, X2=0.0, Y2=4.5, Z2=14.0, Hitlag=0.25, SDI=0.0, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=False, ShieldDamage=-3, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=False, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=0x1a6f665bee, SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=49936, Type=ATTACK_REGION_OBJECT)
+		ATTACK(ID=1, Part=0, Bone=top, Damage=4.0, Angle=361, KBG=45, FKB=0, BKB=4, Size=2.5, X=0.0, Y=4.5, Z=9.0, X2=0.0, Y2=4.5, Z2=44.0, Hitlag=0.0, SDI=0.0, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=False, ShieldDamage=-2, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=False, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=0x1a6f665bee, SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=49936, Type=ATTACK_REGION_OBJECT)
+		ATTACK(ID=2, Part=0, Bone=top, Damage=2.0, Angle=361, KBG=0, FKB=0, BKB=0, Size=2.5, X=0.0, Y=4.5, Z=9.0, X2=0.0, Y2=4.5, Z2=84.0, Hitlag=0.0, SDI=0.0, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=False, ShieldDamage=-1, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=False, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=0x1a6f665bee, SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=49936, Type=ATTACK_REGION_OBJECT)
 	}
 }
 if(is_excute){
 	FT_SHOOTING_ATTACK_GROUND_CHECK_NEW(0, 4, 4)
 	FT_SHOOTING_ATTACK_GROUND_CHECK_NEW(1, 4, 4)
@@ -28,12 +28,12 @@ wait(Frames=1)
 if(is_excute){
 	AttackModule__clear_all()
 }
 frame(Frame=33)
 if(is_excute){
-	WorkModule__on_flag(Flag=16424)
-	WorkModule__on_flag(Flag=16420)
+	WorkModule__on_flag(Flag=16660)
+	WorkModule__on_flag(Flag=16656)
 }
 frame(Frame=36)
 if(is_excute){
-	WorkModule__on_flag(Flag=16392)
+	WorkModule__on_flag(Flag=16628)
 }
