diff --git "a/7.0.0/sheik/game_FinalAttack.txt" "b/8.0.0/sheik/game_FinalAttack.txt"
index 78d6756..2333ade 100644
--- "a/7.0.0/sheik/game_FinalAttack.txt"
+++ "b/8.0.0/sheik/game_FinalAttack.txt"
@@ -1,11 +1,11 @@
 if(is_excute){
 	CAM_ZOOM_OUT()
 }
-if(0x107280(WorkModule::is_flag(FIGHTER_SHEIK_STATUS_FINAL_FLAG_HIT_ALL_OPPONENT), false, true)){
+if(0x107ac0(WorkModule::is_flag(FIGHTER_SHEIK_STATUS_FINAL_FLAG_HIT_ALL_OPPONENT), false, true)){
 	get_value_float(SO_VAR_FLOAT_LR)
-	if(0x107280(0, 0)){
+	if(0x107ac0(0, 0)){
 		if(is_excute){
 			camera(MA_MSC_CMD_CAMERA_CAM_OFFSET, -10, 0)
 		}
 		else{
 			if(is_excute){
