diff --git "a/uit_4.0.0/szerosuit/game_FinalStart.txt" "b/uit_5.0.0/szerosuit/game_FinalStart.txt"
index e8cefd4..caf5be4 100644
--- "a/uit_4.0.0/szerosuit/game_FinalStart.txt"
+++ "b/uit_5.0.0/szerosuit/game_FinalStart.txt"
@@ -10,11 +10,11 @@ if(0x116880(WorkModule__is_flag(FIGHTER_INSTANCE_WORK_ID_FLAG_DISABLE_FINAL_STAR
 		SlowModule__set_whole(2, 0)
 	}
 	else{
 		if(is_excute){
 			PostureModule__scale(3, 0)
-			0x116880(-1705532002, 2)
+			0x116880(-1805361718, 2)
 			CAM_ZOOM_IN_arg5(0, 0)
 			FT_START_CUTIN()
 		}
 	}
 }
