diff --git "a/10.1.0/tantan/game_SpecialAirHi.txt" "b/11.0.0/tantan/game_SpecialAirHi.txt"
index 4b90fc5..dae280b 100644
--- "a/10.1.0/tantan/game_SpecialAirHi.txt"
+++ "b/11.0.0/tantan/game_SpecialAirHi.txt"
@@ -8,7 +8,7 @@ if(is_excute){
 	HIT_NO(13, HIT_STATUS_OFF)
 	HIT_NO(19, HIT_STATUS_OFF)
 }
 frame(0, 2)
 if(is_excute){
-	WorkModule::on_flag(Flag=53956)
+	WorkModule::on_flag(Flag=54200)
 }
