diff --git "a/0.1.0/dolly/game_SpecialAirLwRiseW.txt" "b/1.0.0/dolly/game_SpecialAirLwRiseW.txt"
index 1e0cc62..ad19597 100644
--- "a/0.1.0/dolly/game_SpecialAirLwRiseW.txt"
+++ "b/1.0.0/dolly/game_SpecialAirLwRiseW.txt"
@@ -2,11 +2,11 @@ if(is_excute){
 	WorkModule::on_flag(Flag=FIGHTER_DOLLY_INSTANCE_WORK_ID_FLAG_FINAL_HIT_CANCEL)
 	WorkModule::on_flag(Flag=FIGHTER_INSTANCE_WORK_ID_FLAG_NO_SPEED_OPERATION_CHK)
 	KineticModule::clear_speed_all()
 	WorkModule::off_flag(Flag=FIGHTER_INSTANCE_WORK_ID_FLAG_NO_SPEED_OPERATION_CHK)
 }
-if(0x24ff00(WorkModule::is_flag(FIGHTER_DOLLY_STATUS_SPECIAL_COMMON_WORK_FLAG_COMMAND), false, true)){
+if(0x24ffc0(WorkModule::is_flag(FIGHTER_DOLLY_STATUS_SPECIAL_COMMON_WORK_FLAG_COMMAND), false, true)){
 	if(is_excute){
 		WHOLE_HIT(HIT_STATUS_XLU)
 	}
 }
 if(is_excute){
