diff --git "a/ink_7.0.0/younglink/game_ThrowLw.txt" "b/ink_8.0.0/younglink/game_ThrowLw.txt"
index 221a0c3..0f2111e 100644
--- "a/ink_7.0.0/younglink/game_ThrowLw.txt"
+++ "b/ink_8.0.0/younglink/game_ThrowLw.txt"
@@ -1,6 +1,8 @@
 if(is_excute){
+	FT_LEAVE_NEAR_OTTOTTO(-2, 2)
+	FT_LEAVE_NEAR_OTTOTTO(2.5, 2.5)
 	ATTACK_ABS(Kind=FIGHTER_ATTACK_ABSOLUTE_KIND_THROW, ID=0, Damage=3.0, Angle=86, KBG=115, FKB=0, BKB=70, Hitlag=0.0, Unk=1.0, FacingRestrict=ATTACK_LR_CHECK_F, Unk=0.0, Unk=true, Effect=hash40("collision_attr_normal"), SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=COLLISION_SOUND_ATTR_NONE, Type=ATTACK_REGION_THROW)
 	ATTACK_ABS(Kind=FIGHTER_ATTACK_ABSOLUTE_KIND_CATCH, ID=0, Damage=3.0, Angle=361, KBG=100, FKB=0, BKB=40, Hitlag=0.0, Unk=1.0, FacingRestrict=ATTACK_LR_CHECK_F, Unk=0.0, Unk=true, Effect=hash40("collision_attr_normal"), SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=COLLISION_SOUND_ATTR_NONE, Type=ATTACK_REGION_THROW)
 }
 frame(Frame=22)
 if(is_excute){
