diff --git "a/10.0.0/tantan_punch1/game_SpecialAirHiAttackDragon.txt" "b/10.1.0/tantan_punch1/game_SpecialAirHiAttackDragon.txt"
index e1a6f67..fd57259 100644
--- "a/10.0.0/tantan_punch1/game_SpecialAirHiAttackDragon.txt"
+++ "b/10.1.0/tantan_punch1/game_SpecialAirHiAttackDragon.txt"
@@ -1,11 +1,11 @@
 if(is_excute){
-	ATTACK(ID=0, Part=0, Bone=hash40("have"), Damage=10.35, Angle=45, KBG=85, FKB=0, BKB=66, Size=5.5, X=4.0, Y=0.0, Z=0.4, X2=-6.0, Y2=0.0, Z2=0.4, Hitlag=1.0, SDI=1.0, Clang_Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=false, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=false, Absorbable=false, Flinchless=false, DisableHitlag=true, Direct_Hitbox=true, Ground_or_Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=false, Effect=hash40("collision_attr_normal"), SFXLevel=ATTACK_SOUND_LEVEL_L, SFXType=59136, Type=ATTACK_REGION_PUNCH)
+	ATTACK(ID=0, Part=0, Bone=hash40("have"), Damage=10.35, Angle=45, KBG=85, FKB=0, BKB=66, Size=5.5, X=4.0, Y=0.0, Z=0.4, X2=-6.0, Y2=0.0, Z2=0.4, Hitlag=1.0, SDI=1.0, Clang_Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=false, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=false, Absorbable=false, Flinchless=false, DisableHitlag=true, Direct_Hitbox=true, Ground_or_Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=false, Effect=hash40("collision_attr_normal"), SFXLevel=ATTACK_SOUND_LEVEL_L, SFXType=59224, Type=ATTACK_REGION_PUNCH)
 }
 frame(Frame=1)
 if(is_excute){
-	ATTACK(ID=0, Part=0, Bone=hash40("have"), Damage=10.35, Angle=45, KBG=85, FKB=0, BKB=66, Size=5.5, X=4.0, Y=0.0, Z=0.4, X2=0.0, Y2=0.0, Z2=0.4, Hitlag=1.0, SDI=1.0, Clang_Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=false, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=false, Absorbable=false, Flinchless=false, DisableHitlag=true, Direct_Hitbox=true, Ground_or_Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=false, Effect=hash40("collision_attr_normal"), SFXLevel=ATTACK_SOUND_LEVEL_L, SFXType=59136, Type=ATTACK_REGION_PUNCH)
+	ATTACK(ID=0, Part=0, Bone=hash40("have"), Damage=10.35, Angle=45, KBG=85, FKB=0, BKB=66, Size=5.5, X=4.0, Y=0.0, Z=0.4, X2=0.0, Y2=0.0, Z2=0.4, Hitlag=1.0, SDI=1.0, Clang_Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=false, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=false, Absorbable=false, Flinchless=false, DisableHitlag=true, Direct_Hitbox=true, Ground_or_Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=false, Effect=hash40("collision_attr_normal"), SFXLevel=ATTACK_SOUND_LEVEL_L, SFXType=59224, Type=ATTACK_REGION_PUNCH)
 }
 frame(Frame=9)
 if(is_excute){
 	AttackModule::clear_all()
 }
