diff --git "a/1.0.0/brave/game_SpecialAirN1.txt" "b/2.0.0/brave/game_SpecialAirN1.txt"
index e669d28..075eead 100644
--- "a/1.0.0/brave/game_SpecialAirN1.txt"
+++ "b/2.0.0/brave/game_SpecialAirN1.txt"
@@ -1,11 +1,11 @@
 if(is_excute){
 	WorkModule::on_flag(Flag=FIGHTER_BRAVE_STATUS_SPECIAL_N_FLAG_ENABLE_CONTROL_ENERGY)
 }
 frame(Frame=10)
 WorkModule::is_flag(FIGHTER_BRAVE_STATUS_SPECIAL_N_FLAG_SUCCESS_SP)
-if(0x239f70(false, true)){
+if(0x239f10(false, true)){
 	if(is_excute){
 		ArticleModule::generate_article(FIGHTER_BRAVE_GENERATE_ARTICLE_FIREBALL)
 	}
 }
 frame(Frame=24)
