diff --git "a/n_6.0.0/gaogaen/game_SpecialSLariat.txt" "b/n_7.0.0/gaogaen/game_SpecialSLariat.txt"
index 3b9890a..e73dfea 100644
--- "a/n_6.0.0/gaogaen/game_SpecialSLariat.txt"
+++ "b/n_7.0.0/gaogaen/game_SpecialSLariat.txt"
@@ -4,11 +4,11 @@ if(is_excute){
 	AttackModule__set_force_reaction(0, True, True)
 	WorkModule__set_float(9, FIGHTER_GAOGAEN_STATUS_SPECIAL_S_FLOAT_LARIAT_HIT_FRAME)
 }
 frame(Frame=9)
 if(is_excute){
-	ATTACK(ID=0, Part=0, Bone=arml, Damage=20.0, Angle=145, KBG=76, FKB=0, BKB=20, Size=7.0, X=0.0, Y=0.0, Z=0.0, Hitlag=1.4, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_B, SetWeight=False, ShieldDamage=2, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=True, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_normal, SFXLevel=ATTACK_SOUND_LEVEL_L, SFXType=COLLISION_SOUND_ATTR_HEAVY, Type=ATTACK_REGION_PUNCH)
+	ATTACK(ID=0, Part=0, Bone=arml, Damage=20.0, Angle=145, KBG=38, FKB=0, BKB=88, Size=7.0, X=0.0, Y=0.0, Z=0.0, Hitlag=1.4, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_B, SetWeight=False, ShieldDamage=2, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=True, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_normal, SFXLevel=ATTACK_SOUND_LEVEL_L, SFXType=COLLISION_SOUND_ATTR_HEAVY, Type=ATTACK_REGION_PUNCH)
 	CHECK_FINISH_CAMERA(0, 0)
 	FighterCutInManager__set_throw_finish_zoom_rate(ZoomRate=1.5)
 	FighterCutInManager__set_throw_finish_offset(X=0, Y=0, Z=0)
 }
 frame(Frame=14)
