diff --git "a/0.1.0/buddy/game_SpecialHiStart.txt" "b/1.0.0/buddy/game_SpecialHiStart.txt"
index dc9dbdb..38e845b 100644
--- "a/0.1.0/buddy/game_SpecialHiStart.txt"
+++ "b/1.0.0/buddy/game_SpecialHiStart.txt"
@@ -1,7 +1,7 @@
 IS_EXIST_ARTICLE(FIGHTER_BUDDY_GENERATE_ARTICLE_PAD)
-if(0x18e0b0(false, true)){
+if(0x18e1e0(false, true)){
 	if(is_excute){
 		ArticleModule::remove_exist(FIGHTER_BUDDY_GENERATE_ARTICLE_PAD)
 	}
 }
 if(is_excute){
