diff --git "a/0.1.0/brave/game_SpecialAirN1.txt" "b/1.0.0/brave/game_SpecialAirN1.txt"
index 94fc9ff..7d0d6f6 100644
--- "a/0.1.0/brave/game_SpecialAirN1.txt"
+++ "b/1.0.0/brave/game_SpecialAirN1.txt"
@@ -1,10 +1,10 @@
 if(is_excute){
 	WorkModule::on_flag(Flag=FIGHTER_BRAVE_STATUS_SPECIAL_N_FLAG_ENABLE_CONTROL_ENERGY)
 }
 frame(Frame=10)
-if(0x239e40(WorkModule::is_flag(FIGHTER_BRAVE_STATUS_SPECIAL_N_FLAG_SUCCESS_SP), false, true)){
+if(0x239f70(WorkModule::is_flag(FIGHTER_BRAVE_STATUS_SPECIAL_N_FLAG_SUCCESS_SP), false, true)){
 	if(is_excute){
 		ArticleModule::generate_article(FIGHTER_BRAVE_GENERATE_ARTICLE_FIREBALL)
 	}
 }
 frame(Frame=24)
