diff --git "a/.0.0/jack/game_Win2Wait.txt" "b/.1.0/jack/game_Win2Wait.txt"
index 0d737e9..1fa94e2 100644
--- "a/.0.0/jack/game_Win2Wait.txt"
+++ "b/.1.0/jack/game_Win2Wait.txt"
@@ -1,6 +1,6 @@
 if(is_excute){
-	as_hash__const(50972, 0x0ab87848fa)
+	as_hash__const(50988, 0x0ab87848fa)
 	ArticleModule__change_motion()
-	as_hash__const(47812, 0x0ab87848fa)
+	as_hash__const(47828, 0x0ab87848fa)
 	ArticleModule__change_motion()
 }
