diff --git "a/.0.0/jack/game_SpecialAirHiStart.txt" "b/.1.0/jack/game_SpecialAirHiStart.txt"
index b952ec6..f7ca8ee 100644
--- "a/.0.0/jack/game_SpecialAirHiStart.txt"
+++ "b/.1.0/jack/game_SpecialAirHiStart.txt"
@@ -1,9 +1,9 @@
 if(is_excute){
-	ArticleModule__generate_article(48104)
-	as_hash__const(48104, special_hi2)
+	ArticleModule__generate_article(48120)
+	as_hash__const(48120, special_hi2)
 	ArticleModule__change_motion()
 }
 frame(Frame=5)
 if(is_excute){
-	WorkModule__on_flag(Flag=48096)
+	WorkModule__on_flag(Flag=48112)
 }
