diff --git "a/.0.0/jack/game_SpecialAirS1.txt" "b/.1.0/jack/game_SpecialAirS1.txt"
index c52650e..50e72a4 100644
--- "a/.0.0/jack/game_SpecialAirS1.txt"
+++ "b/.1.0/jack/game_SpecialAirS1.txt"
@@ -1,9 +1,9 @@
 frame(Frame=16)
 if(is_excute){
-	ArticleModule__generate_article(50976)
-	WorkModule__on_flag(Flag=48192)
+	ArticleModule__generate_article(50992)
+	WorkModule__on_flag(Flag=48208)
 }
 frame(Frame=32)
 if(is_excute){
-	WorkModule__on_flag(Flag=48188)
+	WorkModule__on_flag(Flag=48204)
 }
