diff --git "a/.0.0/jack/game_AttackLw3.txt" "b/.1.0/jack/game_AttackLw3.txt"
index f680fe2..3bd1349 100644
--- "a/.0.0/jack/game_AttackLw3.txt"
+++ "b/.1.0/jack/game_AttackLw3.txt"
@@ -1,11 +1,15 @@
+frame(Frame=7)
+if(is_excute){
+	JostleModule__set_status(False)
+}
 frame(Frame=8)
 if(is_excute){
 	ATTACK(ID=0, Part=0, Bone=kneer, Damage=6.0, Angle=75, KBG=66, FKB=0, BKB=75, Size=2.6, X=-4.0, Y=0.0, Z=1.5, X2=5.0, Y2=0.0, Z2=1.5, Hitlag=0.75, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_normal, SFXLevel=ATTACK_SOUND_LEVEL_M, SFXType=COLLISION_SOUND_ATTR_KICK, Type=ATTACK_REGION_KICK)
 	AttackModule__set_attack_height_all(ATTACK_HEIGHT_LOW, False)
 }
-if(0x28fe90(WorkModule__is_flag(16428), False, True)){
+if(0x2901d0(WorkModule__is_flag(16432), False, True)){
 	if(is_excute){
 		ATTACK(ID=0, Part=0, Bone=top, Damage=6.0, Angle=75, KBG=66, FKB=0, BKB=55, Size=3.5, X=0.0, Y=3.2, Z=0.0, X2=0.0, Y2=3.2, Z2=8.5, Hitlag=1.5, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_normal, SFXLevel=ATTACK_SOUND_LEVEL_M, SFXType=COLLISION_SOUND_ATTR_KICK, Type=ATTACK_REGION_KICK)
 		ATTACK(ID=1, Part=1, Bone=top, Damage=7.0, Angle=82, KBG=95, FKB=0, BKB=70, Size=3.5, X=0.0, Y=3.2, Z=0.0, X2=0.0, Y2=3.2, Z2=8.5, Hitlag=1.5, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_normal, SFXLevel=ATTACK_SOUND_LEVEL_M, SFXType=COLLISION_SOUND_ATTR_HEAVY, Type=ATTACK_REGION_KICK)
 		ATK_SET_SHIELD_SETOFF_MUL(ID=0, ShieldstunMul=1.5)
 		ATK_SET_SHIELD_SETOFF_MUL(ID=1, ShieldstunMul=1.5)
@@ -13,11 +17,11 @@ if(0x28fe90(WorkModule__is_flag(16428), False, True)){
 }
 frame(Frame=10)
 if(is_excute){
 	ATTACK(ID=0, Part=0, Bone=kneer, Damage=6.0, Angle=82, KBG=66, FKB=0, BKB=75, Size=2.6, X=-4.0, Y=0.0, Z=1.5, X2=5.0, Y2=0.0, Z2=1.5, Hitlag=1.0, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_normal, SFXLevel=ATTACK_SOUND_LEVEL_M, SFXType=COLLISION_SOUND_ATTR_KICK, Type=ATTACK_REGION_KICK)
 }
-if(0x28fe90(WorkModule__is_flag(16428), False, True)){
+if(0x2901d0(WorkModule__is_flag(16432), False, True)){
 	if(is_excute){
 		ATTACK(ID=0, Part=0, Bone=top, Damage=6.0, Angle=82, KBG=66, FKB=0, BKB=55, Size=3.5, X=0.0, Y=3.2, Z=0.0, X2=0.0, Y2=3.2, Z2=8.5, Hitlag=1.5, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_normal, SFXLevel=ATTACK_SOUND_LEVEL_M, SFXType=COLLISION_SOUND_ATTR_KICK, Type=ATTACK_REGION_KICK)
 		ATTACK(ID=1, Part=1, Bone=top, Damage=7.0, Angle=82, KBG=98, FKB=0, BKB=70, Size=3.5, X=0.0, Y=3.2, Z=0.0, X2=0.0, Y2=3.2, Z2=8.5, Hitlag=1.5, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_normal, SFXLevel=ATTACK_SOUND_LEVEL_M, SFXType=COLLISION_SOUND_ATTR_HEAVY, Type=ATTACK_REGION_KICK)
 		ATK_SET_SHIELD_SETOFF_MUL(ID=0, ShieldstunMul=1.5)
 		ATK_SET_SHIELD_SETOFF_MUL(ID=1, ShieldstunMul=1.5)
@@ -25,17 +29,18 @@ if(0x28fe90(WorkModule__is_flag(16428), False, True)){
 }
 frame(Frame=12)
 if(is_excute){
 	ATTACK(ID=0, Part=0, Bone=kneer, Damage=6.0, Angle=85, KBG=66, FKB=0, BKB=75, Size=2.6, X=-4.0, Y=0.0, Z=1.5, X2=5.0, Y2=0.0, Z2=1.5, Hitlag=1.0, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_normal, SFXLevel=ATTACK_SOUND_LEVEL_M, SFXType=COLLISION_SOUND_ATTR_KICK, Type=ATTACK_REGION_KICK)
 }
-if(0x28fe90(WorkModule__is_flag(16428), False, True)){
+if(0x2901d0(WorkModule__is_flag(16432), False, True)){
 	if(is_excute){
 		ATTACK(ID=0, Part=0, Bone=top, Damage=6.0, Angle=85, KBG=66, FKB=0, BKB=55, Size=3.5, X=0.0, Y=3.2, Z=0.0, X2=0.0, Y2=3.2, Z2=8.5, Hitlag=1.5, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_normal, SFXLevel=ATTACK_SOUND_LEVEL_M, SFXType=COLLISION_SOUND_ATTR_KICK, Type=ATTACK_REGION_KICK)
 		ATTACK(ID=1, Part=1, Bone=top, Damage=7.0, Angle=82, KBG=99, FKB=0, BKB=70, Size=3.5, X=0.0, Y=3.2, Z=0.0, X2=0.0, Y2=3.2, Z2=8.5, Hitlag=1.5, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_normal, SFXLevel=ATTACK_SOUND_LEVEL_M, SFXType=COLLISION_SOUND_ATTR_HEAVY, Type=ATTACK_REGION_KICK)
 		ATK_SET_SHIELD_SETOFF_MUL(ID=0, ShieldstunMul=1.5)
 		ATK_SET_SHIELD_SETOFF_MUL(ID=1, ShieldstunMul=1.5)
 	}
 }
 frame(Frame=15)
 if(is_excute){
 	AttackModule__clear_all()
+	JostleModule__set_status(True)
 }
