diff --git "a/don_5.0.0/plizardon/game_Final.txt" "b/don_6.0.0/plizardon/game_Final.txt"
index d6cb365..edfcb96 100644
--- "a/don_5.0.0/plizardon/game_Final.txt"
+++ "b/don_6.0.0/plizardon/game_Final.txt"
@@ -10,11 +10,11 @@ if(method.lib::L2CValue.operator___lib::L2CValueconst__const(WorkModule__is_flag
 		FT_START_CUTIN_arg1(True)
 	}
 	else{
 		if(is_excute){
 			PostureModule__scale(4, 0)
-			0xdf4c0(-1674226339, 2.1)
+			0xdf4e0(-858830818, 2.1)
 			CAM_ZOOM_IN_arg5(0, 0)
 			FT_START_CUTIN_arg1(True)
 		}
 		frame(Frame=25)
 		if(is_excute){
@@ -30,11 +30,11 @@ frame(Frame=60)
 if(is_excute){
 	AttackModule__clear_all()
 	camera(MA_MSC_CMD_CAMERA_CAM_RESET)
 }
 get_value_float(SO_VAR_FLOAT_LR)
-if(0xdf4c0(448, 0)){
+if(0xdf4e0(448, 0)){
 	if(is_excute){
 		camera(MA_MSC_CMD_CAMERA_CAM_OFFSET, -60, 0, 0)
 		camera(MA_MSC_CMD_CAMERA_CAM_RECT, 60, -56, 18, -8)
 	}
 	else{
