diff --git "a/don_5.0.0/plizardon/game_AttackHi4.txt" "b/don_6.0.0/plizardon/game_AttackHi4.txt"
index 95b14f8..b03185f 100644
--- "a/don_5.0.0/plizardon/game_AttackHi4.txt"
+++ "b/don_6.0.0/plizardon/game_AttackHi4.txt"
@@ -23,11 +23,11 @@ if(is_excute){
 	ATTACK(ID=3, Part=0, Bone=wingl4, Damage=5.0, Angle=320, KBG=100, FKB=42, BKB=0, Size=4.2, X=7.0, Y=0.0, Z=2.5, Hitlag=1.0, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=True, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_cutup, SFXLevel=ATTACK_SOUND_LEVEL_L, SFXType=COLLISION_SOUND_ATTR_CUTUP, Type=ATTACK_REGION_BODY)
 }
 frame(Frame=10)
 if(is_excute){
 	ATTACK(ID=3, Part=0, Bone=wingl4, Damage=5.0, Angle=340, KBG=100, FKB=60, BKB=0, Size=4.2, X=7.0, Y=0.0, Z=2.5, Hitlag=1.0, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=True, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_A, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_cutup, SFXLevel=ATTACK_SOUND_LEVEL_L, SFXType=COLLISION_SOUND_ATTR_CUTUP, Type=ATTACK_REGION_BODY)
-	ATTACK(ID=0, Part=0, Bone=wingl4, Damage=0.0, Angle=368, KBG=100, FKB=0, BKB=0, Size=4.2, X=7.0, Y=0.0, Z=2.5, Hitlag=1.0, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_cutup, SFXLevel=ATTACK_SOUND_LEVEL_L, SFXType=COLLISION_SOUND_ATTR_CUTUP, Type=ATTACK_REGION_BODY)
+	ATTACK(ID=0, Part=0, Bone=wingl4, Damage=5.0, Angle=368, KBG=100, FKB=0, BKB=0, Size=4.2, X=7.0, Y=0.0, Z=2.5, Hitlag=1.0, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_cutup, SFXLevel=ATTACK_SOUND_LEVEL_L, SFXType=COLLISION_SOUND_ATTR_CUTUP, Type=ATTACK_REGION_BODY)
 	as_hash__const(0, top, 0, 18, 8)
 	AttackModule__set_vec_target_pos()
 }
 frame(Frame=11)
 if(is_excute){
