diff --git "a/3.0.0/sonic/game_FinalStart.txt" "b/3.0.1/sonic/game_FinalStart.txt"
index baae003..60a28c5 100644
--- "a/3.0.0/sonic/game_FinalStart.txt"
+++ "b/3.0.1/sonic/game_FinalStart.txt"
@@ -12,25 +12,25 @@ if(0x12c570(false, false)){
 }
 get_value_float(SO_VAR_FLOAT_LR)
 if(0x12c570(0, 0)){
 	if(is_excute){
 		PostureModule::scale(6, 0)
-		0x12c570(129983098, 2)
+		0x12c570(1964135695, 2)
 		CAM_ZOOM_IN_arg5(-0.3, 0)
 		camera(MA_MSC_CMD_CAMERA_CAM_OFFSET, -2, 8)
 	}
 	else{
 		if(is_excute){
 			PostureModule::scale(6, 0)
-			0x12c570(129983098, 2)
+			0x12c570(1964135695, 2)
 			CAM_ZOOM_IN_arg5(0, 0)
 			FT_START_CUTIN()
 		}
 		else{
 			if(is_excute){
 				PostureModule::scale(6, 0)
-				0x12c570(129983098, 2)
+				0x12c570(1964135695, 2)
 				CAM_ZOOM_IN_arg5(-0.3, 0)
 				camera(MA_MSC_CMD_CAMERA_CAM_OFFSET, 2, 8)
 			}
 			if(is_excute){
 				FT_START_CUTIN()
@@ -53,22 +53,22 @@ if(is_excute){
 }
 frame(Frame=17)
 if(is_excute){
 	CAM_ZOOM_OUT()
 	PostureModule::scale(40, 0)
-	0x12c570(129983098, 2)
+	0x12c570(1964135695, 2)
 	CAM_ZOOM_IN_arg5(0.3, 0)
 }
 frame(Frame=20)
 if(is_excute){
 	SlowModule::clear_whole()
 }
 frame(Frame=33)
 if(is_excute){
 	CAM_ZOOM_OUT()
 	PostureModule::scale(1, 0)
-	0x12c570(129983098, 2)
+	0x12c570(1964135695, 2)
 	CAM_ZOOM_IN_arg5(0, 0)
 }
 frame(Frame=35)
 if(is_excute){
 	WorkModule::on_flag(Flag=FIGHTER_SONIC_STATUS_FINAL_FLAG_FINAL_CAMERA)
