diff --git "a/.0.0/pitb/game_FinalAir.txt" "b/.0.1/pitb/game_FinalAir.txt"
index 10c6e6b..d2c11cc 100644
--- "a/.0.0/pitb/game_FinalAir.txt"
+++ "b/.0.1/pitb/game_FinalAir.txt"
@@ -10,11 +10,11 @@ if(0x10d570(false, false)){
 		REQ_FINAL_START_CAMERA(hash40("d04final.nuanmb"), false)
 	}
 	else{
 		if(is_excute){
 			PostureModule::scale(4, 0)
-			0x10d570(-1313760193, 2)
+			0x10d570(-1396590502, 2)
 			CAM_ZOOM_IN_arg5(0, 0)
 			FT_START_CUTIN()
 		}
 	}
 }
