diff --git "a/3.0.0/cloud/game_FinalHit.txt" "b/3.1.0/cloud/game_FinalHit.txt"
index 92d4e56..0d569f1 100644
--- "a/3.0.0/cloud/game_FinalHit.txt"
+++ "b/3.1.0/cloud/game_FinalHit.txt"
@@ -8,22 +8,22 @@ if(is_excute){
 }
 frame(Frame=5)
 if(method.lib::L2CValue.operator___lib::L2CValueconst__const(WorkModule__is_flag(FIGHTER_CLOUD_STATUS_FINAL_FLAG_HIT_ALL_OPPONENT), False, True)){
 	FT_MOTION_RATE(FSM=0.5)
 	WorkModule__get_float(FIGHTER_CLOUD_STATUS_FINAL_WORK_FLOAT_ATTACK_TARGET_LR)
-	if(0x14cb70(1986414738, 0)){
+	if(0x14cb70(814204757, 0)){
 		if(is_excute){
 			CAM_ZOOM_OUT()
 			PostureModule__scale(32, 0)
-			0x14cb70(1986414738, 3.2)
+			0x14cb70(814204757, 3.2)
 			CAM_ZOOM_IN_FINAL(0, 0, 30, -10, 195, True, FIGHTER_CLOUD_STATUS_FINAL_WORK_INT_ATTACK_HIT_OBJECT_ID, 10)
 		}
 		else{
 			if(is_excute){
 				CAM_ZOOM_OUT()
 				PostureModule__scale(32, 0)
-				0x14cb70(1986414738, 2.8)
+				0x14cb70(814204757, 2.8)
 				CAM_ZOOM_IN_FINAL(0, 0, -30, -10, 195, True, FIGHTER_CLOUD_STATUS_FINAL_WORK_INT_ATTACK_HIT_OBJECT_ID, 10)
 			}
 		}
 	}
 }
