diff --git "a/r_8.0.0/richter/game_AttackHi4.txt" "b/r_9.0.0/richter/game_AttackHi4.txt"
index 590d733..d37ce7a 100644
--- "a/r_8.0.0/richter/game_AttackHi4.txt"
+++ "b/r_9.0.0/richter/game_AttackHi4.txt"
@@ -6,15 +6,15 @@ frame(Frame=7)
 if(is_excute){
 	WorkModule::on_flag(Flag=FIGHTER_STATUS_ATTACK_FLAG_START_SMASH_HOLD)
 }
 frame(Frame=13)
 FT_MOTION_RATE(FSM=0.5)
-frame(Frame=16)
+frame(Frame=21)
 FT_MOTION_RATE(FSM=1)
 frame(Frame=22)
 if(is_excute){
-	SEARCH(0, 0, hash40("top"), 3.0, 0.0, 18.0, 2.0, 0.0, 52.0, 2.0, COLLISION_KIND_MASK_HIT, HIT_STATUS_MASK_NORMAL, 1, COLLISION_SITUATION_MASK_GA, COLLISION_CATEGORY_MASK_STAGE_GIMMICK, COLLISION_PART_MASK_ALL, false)
+	SEARCH(0, 0, hash40("top"), 3.0, 0.0, 18.0, 2.0, 0.0, 52.0, 2.0, COLLISION_KIND_MASK_HIT, HIT_STATUS_MASK_NORMAL, 1, COLLISION_SITUATION_MASK_GA, 57980, COLLISION_PART_MASK_ALL, false)
 }
 frame(Frame=24)
 if(is_excute){
 	battle_object()
 	L2CValue()
@@ -22,12 +22,12 @@ if(is_excute){
 else{
 	L2CValue_lib::L2CValueconst()
 }
 as_pointer__const(0, 1)
 set_whip_reflect_attack_off_id_app::Fighter__int_int_int_int_int_int_int_int_int_int()
-ATTACK(ID=0, Part=0, Bone=hash40("top"), Damage=16.0, Angle=90, KBG=88, FKB=0, BKB=61, Size=3.0, X=0.0, Y=52.0, Z=2.0, X2=LUA_VOID, Y2=LUA_VOID, Z2=LUA_VOID, Hitlag=0.8, SDI=1.0, Clang_Rebound=ATTACK_SETOFF_KIND_THRU, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=false, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=false, Absorbable=false, Flinchless=false, DisableHitlag=false, Direct_Hitbox=true, Ground_or_Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=false, Effect=hash40("collision_attr_whip"), SFXLevel=ATTACK_SOUND_LEVEL_L, SFXType=COLLISION_SOUND_ATTR_RICHTER_WHIP, Type=ATTACK_REGION_WHIP)
-ATTACK(ID=1, Part=0, Bone=hash40("top"), Damage=14.0, Angle=90, KBG=82, FKB=0, BKB=60, Size=3.0, X=0.0, Y=18.0, Z=2.0, X2=0.0, Y2=52.0, Z2=2.0, Hitlag=0.8, SDI=1.0, Clang_Rebound=ATTACK_SETOFF_KIND_THRU, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=false, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=false, Absorbable=false, Flinchless=false, DisableHitlag=false, Direct_Hitbox=true, Ground_or_Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=false, Effect=hash40("collision_attr_whip"), SFXLevel=ATTACK_SOUND_LEVEL_L, SFXType=COLLISION_SOUND_ATTR_RICHTER_WHIP, Type=ATTACK_REGION_WHIP)
+ATTACK(ID=0, Part=0, Bone=hash40("top"), Damage=16.0, Angle=90, KBG=91, FKB=0, BKB=64, Size=3.0, X=0.0, Y=52.0, Z=2.0, X2=LUA_VOID, Y2=LUA_VOID, Z2=LUA_VOID, Hitlag=0.8, SDI=1.0, Clang_Rebound=ATTACK_SETOFF_KIND_THRU, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=false, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=false, Absorbable=false, Flinchless=false, DisableHitlag=false, Direct_Hitbox=true, Ground_or_Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=false, Effect=hash40("collision_attr_whip"), SFXLevel=ATTACK_SOUND_LEVEL_L, SFXType=COLLISION_SOUND_ATTR_RICHTER_WHIP, Type=ATTACK_REGION_WHIP)
+ATTACK(ID=1, Part=0, Bone=hash40("top"), Damage=14.0, Angle=90, KBG=85, FKB=0, BKB=63, Size=3.0, X=0.0, Y=18.0, Z=2.0, X2=0.0, Y2=52.0, Z2=2.0, Hitlag=0.8, SDI=1.0, Clang_Rebound=ATTACK_SETOFF_KIND_THRU, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=false, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=false, Absorbable=false, Flinchless=false, DisableHitlag=false, Direct_Hitbox=true, Ground_or_Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=false, Effect=hash40("collision_attr_whip"), SFXLevel=ATTACK_SOUND_LEVEL_L, SFXType=COLLISION_SOUND_ATTR_RICHTER_WHIP, Type=ATTACK_REGION_WHIP)
 search(MA_MSC_CMD_SEARCH_SEARCH_SCH_CLR_ALL)
 frame(Frame=27)
 if(is_excute){
 	AttackModule::clear_all()
 }
