diff --git "a/2.0.0/lucas/game_FinalAir.txt" "b/3.0.0/lucas/game_FinalAir.txt"
index 7e2a620..17d7f94 100644
--- "a/2.0.0/lucas/game_FinalAir.txt"
+++ "b/3.0.0/lucas/game_FinalAir.txt"
@@ -1,10 +1,10 @@
 if(is_excute){
 	ArticleModule::generate_article(FIGHTER_LUCAS_GENERATE_ARTICLE_KUMATORA)
 	ArticleModule::generate_article(FIGHTER_LUCAS_GENERATE_ARTICLE_BONNIE)
 	BattleObjectWorld__scale_z()
-	if(0x101670(0, 1.0)){
+	if(0x101650(0, 1.0)){
 		article(MA_MSC_CMD_ARTICLE_SET_OFFSET_ARTICLE, FIGHTER_LUCAS_GENERATE_ARTICLE_KUMATORA, -10, 0, -10)
 		article(MA_MSC_CMD_ARTICLE_SET_OFFSET_ARTICLE, FIGHTER_LUCAS_GENERATE_ARTICLE_BONNIE, 10, 0, -10)
 	}
 	else{
 		article(FIGHTER_LUCAS_GENERATE_ARTICLE_BONNIE, FIGHTER_LUCAS_GENERATE_ARTICLE_KUMATORA, -10, 0, -0.01)
@@ -21,11 +21,11 @@ if(methodlib::L2CValue::operator==(lib::L2CValueconst&)const(false, false)){
 		REQ_FINAL_START_CAMERA(hash40("d04final.nuanmb"), false)
 	}
 	else{
 		if(is_excute){
 			PostureModule::scale(5, 0)
-			0x101670(948132434, 2.2)
+			0x101650(-1047495411, 2.2)
 			CAM_ZOOM_IN_arg5(0, 0)
 			FT_START_CUTIN()
 		}
 	}
 }
