diff --git "a/0.0/ryu/game_Final.txt" "b/0.0/ryu/game_Final.txt"
index 067bce4..4e79f2e 100644
--- "a/0.0/ryu/game_Final.txt"
+++ "b/0.0/ryu/game_Final.txt"
@@ -11,11 +11,11 @@ if(0x160530(WorkModule::is_flag(FIGHTER_INSTANCE_WORK_ID_FLAG_DISABLE_FINAL_STAR
 		FT_START_CUTIN()
 	}
 	else{
 		if(is_excute){
 			PostureModule::scale(3, 0)
-			0x160530(48432809, 1.8)
+			0x160530(-557596978, 1.8)
 			CAM_ZOOM_IN_arg5(0, 0)
 			FT_START_CUTIN()
 		}
 		frame(Frame=28)
 		if(is_excute){
