diff --git "a/_8.0.0/dedede/game_SpecialNFood.txt" "b/_9.0.0/dedede/game_SpecialNFood.txt"
index 187d521..e2687cf 100644
--- "a/_8.0.0/dedede/game_SpecialNFood.txt"
+++ "b/_9.0.0/dedede/game_SpecialNFood.txt"
@@ -3,29 +3,29 @@ if(is_excute){
 }
 frame(Frame=2)
 if(is_excute){
 	FighterAreaModuleImpl::enable_fix_jostle_area(12, 4)
 }
-if(0x156bb0(FIGHTER_INSTANCE_WORK_ID_INT_KIND, FIGHTER_KIND_KIRBY)){
+if(0x156650(FIGHTER_INSTANCE_WORK_ID_INT_KIND, FIGHTER_KIND_KIRBY)){
 	if(is_excute){
 		FighterAreaModuleImpl::enable_fix_jostle_area(4, 4)
 	}
 }
 frame(Frame=3)
 if(is_excute){
 	FighterAreaModuleImpl::enable_fix_jostle_area(14.7, 4)
 }
-if(0x156bb0(FIGHTER_INSTANCE_WORK_ID_INT_KIND, FIGHTER_KIND_KIRBY)){
+if(0x156650(FIGHTER_INSTANCE_WORK_ID_INT_KIND, FIGHTER_KIND_KIRBY)){
 	if(is_excute){
 		FighterAreaModuleImpl::enable_fix_jostle_area(4, 4)
 	}
 }
 frame(Frame=7)
 if(is_excute){
 	FighterAreaModuleImpl::enable_fix_jostle_area(12, 4)
 }
-if(0x156bb0(FIGHTER_INSTANCE_WORK_ID_INT_KIND, FIGHTER_KIND_KIRBY)){
+if(0x156650(FIGHTER_INSTANCE_WORK_ID_INT_KIND, FIGHTER_KIND_KIRBY)){
 	if(is_excute){
 		FighterAreaModuleImpl::enable_fix_jostle_area(4, 4)
 	}
 }
 frame(Frame=11)
