diff --git "a/13.0.0/eflame/game_Attack13.txt" "b/13.0.1/eflame/game_Attack13.txt"
index bcf5a48..3f5172d 100644
--- "a/13.0.0/eflame/game_Attack13.txt"
+++ "b/13.0.1/eflame/game_Attack13.txt"
@@ -1,15 +1,15 @@
 frame(Frame=3)
 IS_EXIST_ARTICLE(FIGHTER_EFLAME_GENERATE_ARTICLE_ESWORD)
-if(0x3cc0b0(false, true)){
+if(0x3de840(false, true)){
 	if(is_excute){
 		methodlib::L2CValue::as_hash()const(FIGHTER_EFLAME_GENERATE_ARTICLE_ESWORD, WEAPON_EFLAME_ESWORD_MOTION_PART_SET_KIND_OPEM_CLOSE, hash40("to_open"), 5, 5, false, false, 0, false, true, false)
 		ArticleModule::add_motion_partial()
 	}
 }
 MotionModule::is_changing()
-if(0x3cc0b0(false, true)){
+if(0x3de840(false, true)){
 	if(is_excute){
 		WorkModule::on_flag(Flag=FIGHTER_EFLAME_INSTANCE_WORK_ID_FLAG_ADD_PARTIAL_MTION_SWORD_WHEN_CHANGEING)
 	}
 }
 frame(0, 5)
@@ -40,17 +40,17 @@ frame(Frame=8)
 if(is_excute){
 	AttackModule::clear_all()
 }
 frame(Frame=21)
 IS_EXIST_ARTICLE(FIGHTER_EFLAME_GENERATE_ARTICLE_ESWORD)
-if(0x3cc0b0(false, true)){
+if(0x3de840(false, true)){
 	if(is_excute){
 		methodlib::L2CValue::as_hash()const(FIGHTER_EFLAME_GENERATE_ARTICLE_ESWORD, WEAPON_EFLAME_ESWORD_MOTION_PART_SET_KIND_OPEM_CLOSE, hash40("to_close"), 5, 5, false, false, 0, false, true, false)
 		ArticleModule::add_motion_partial()
 	}
 }
 MotionModule::is_changing()
-if(0x3cc0b0(false, true)){
+if(0x3de840(false, true)){
 	if(is_excute){
 		WorkModule::on_flag(Flag=FIGHTER_EFLAME_INSTANCE_WORK_ID_FLAG_ADD_PARTIAL_MTION_SWORD_WHEN_CHANGEING)
 	}
 }
