diff --git "a/11.0.0/elight_bunshin/game_SpecialS0.txt" "b/12.0.0/elight_bunshin/game_SpecialS0.txt"
index da5b2f8..b3528e4 100644
--- "a/11.0.0/elight_bunshin/game_SpecialS0.txt"
+++ "b/12.0.0/elight_bunshin/game_SpecialS0.txt"
@@ -1,9 +1,9 @@
-IS_EXIST_ARTICLE(60356)
-if(0x353770(false, true)){
+IS_EXIST_ARTICLE(61384)
+if(0x354230(false, true)){
 	if(is_excute){
-		methodlib::L2CValue::as_hash()const(60356, 60344, 0x07439e926b, 10, 10, false, false, 0, false, true, false)
+		methodlib::L2CValue::as_hash()const(61384, 61372, 0x07439e926b, 10, 10, false, false, 0, false, true, false)
 		ArticleModule::add_motion_partial()
 	}
 }
 frame(0, 2)
 if(is_excute){
