diff --git "a/11.0.0/elight_bunshin/0x18c4b9779e.txt" "b/12.0.0/elight_bunshin/0x18c4b9779e.txt"
index 40f7dba..e38dfa6 100644
--- "a/11.0.0/elight_bunshin/0x18c4b9779e.txt"
+++ "b/12.0.0/elight_bunshin/0x18c4b9779e.txt"
@@ -1,7 +1,7 @@
-IS_EXIST_ARTICLE(60356)
-if(0x353770(false, true)){
+IS_EXIST_ARTICLE(61384)
+if(0x354230(false, true)){
 	if(is_excute){
-		methodlib::L2CValue::as_hash()const(60356, 60344, 0x08183db0f4, 10, 10, false, false, 0, false, true, false)
+		methodlib::L2CValue::as_hash()const(61384, 61372, 0x08183db0f4, 10, 10, false, false, 0, false, true, false)
 		ArticleModule::add_motion_partial()
 	}
 }
