diff --git "a/8.0.0/falco/game_SpecialAirNLoop.txt" "b/9.0.0/falco/game_SpecialAirNLoop.txt"
index 72c3740..ed47fdf 100644
--- "a/8.0.0/falco/game_SpecialAirNLoop.txt"
+++ "b/9.0.0/falco/game_SpecialAirNLoop.txt"
@@ -4,11 +4,11 @@ if(is_excute){
 frame(Frame=4)
 if(is_excute){
 	ArticleModule::generate_article(FIGHTER_FALCO_GENERATE_ARTICLE_BLASTER_BULLET)
 }
 IS_EXIST_ARTICLE(FIGHTER_FALCO_GENERATE_ARTICLE_BLASTER)
-if(0x11c800(false, true)){
+if(0x11c960(false, true)){
 	if(is_excute){
 		as_hash__const(FIGHTER_FALCO_GENERATE_ARTICLE_BLASTER, hash40("loop"))
 		ArticleModule::change_motion()
 	}
 }
