diff --git "a/5.0.0/wario_wariobike/game_SpecialSItem.txt" "b/6.0.0/wario_wariobike/game_SpecialSItem.txt"
index e84449c..276a4e3 100644
--- "a/5.0.0/wario_wariobike/game_SpecialSItem.txt"
+++ "b/6.0.0/wario_wariobike/game_SpecialSItem.txt"
@@ -1,6 +1,6 @@
-if(0x102ee0(WorkModule__is_flag(WEAPON_WARIO_WARIOBIKE_INSTANCE_WORK_ID_FLAG_BROKEN_ATTACK), False, True)){
+if(0x102ef0(WorkModule__is_flag(WEAPON_WARIO_WARIOBIKE_INSTANCE_WORK_ID_FLAG_BROKEN_ATTACK), False, True)){
 	if(is_excute){
 		ATTACK_ABS(Kind=FIGHTER_ATTACK_ABSOLUTE_KIND_THROW, ID=0, Damage=6.0, Angle=70, KBG=68, FKB=0, BKB=74, Hitlag=0.0, Unk=1.0, FacingRestrict=ATTACK_LR_CHECK_F, Unk=0.0, Unk=True, Effect=collision_attr_fire, SFXLevel=ATTACK_SOUND_LEVEL_L, SFXType=COLLISION_SOUND_ATTR_NONE, Type=ATTACK_REGION_THROW)
 		ATK_HIT_ABS(FIGHTER_ATTACK_ABSOLUTE_KIND_THROW, top, WEAPON_WARIO_WARIOBIKE_STATUS_WORK_INT_TARGET_OBJECT, WEAPON_WARIO_WARIOBIKE_STATUS_WORK_INT_TARGET_HIT_GROUP, WEAPON_WARIO_WARIOBIKE_STATUS_WORK_INT_TARGET_HIT_NO)
 	}
 }
