diff --git "a/10.1.0/shizue/game_AttackLw4.txt" "b/11.0.0/shizue/game_AttackLw4.txt"
index c778df7..65e8df6 100644
--- "a/10.1.0/shizue/game_AttackLw4.txt"
+++ "b/11.0.0/shizue/game_AttackLw4.txt"
@@ -6,11 +6,11 @@ if(is_excute){
 frame(Frame=4)
 execute(4)
 if(is_excute){
 	WorkModule::on_flag(Flag=FIGHTER_STATUS_ATTACK_FLAG_START_SMASH_HOLD)
 	WorkModule::get_float(FIGHTER_SHIZUE_GENERATE_ARTICLE_BUCKET, hash40("attack"), true, FIGHTER_STATUS_ATTACK_WORK_FLOAT_SMASH_RESTART_FRAME)
-	as_hash__const(1820595963)
+	as_hash__const(1870685302)
 	ArticleModule::change_motion()
 }
 frame(Frame=8)
 if(is_excute){
 	ATTACK(ID=0, Part=0, Bone=hash40("top"), Damage=10.0, Angle=28, KBG=100, FKB=0, BKB=35, Size=4.0, X=0.0, Y=3.6, Z=12.5, X2=LUA_VOID, Y2=LUA_VOID, Z2=LUA_VOID, Hitlag=1.0, SDI=1.0, Clang_Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=false, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=false, Absorbable=false, Flinchless=false, DisableHitlag=false, Direct_Hitbox=true, Ground_or_Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=false, Effect=hash40("collision_attr_normal"), SFXLevel=ATTACK_SOUND_LEVEL_M, SFXType=COLLISION_SOUND_ATTR_WATER, Type=ATTACK_REGION_WATER)
