diff --git "a/6.0.0/sonic/game_SpecialHiSquat.txt" "b/7.0.0/sonic/game_SpecialHiSquat.txt"
index 4851895..578e463 100644
--- "a/6.0.0/sonic/game_SpecialHiSquat.txt"
+++ "b/7.0.0/sonic/game_SpecialHiSquat.txt"
@@ -1,7 +1,7 @@
 frame(Frame=2)
 IS_EXIST_ARTICLE(FIGHTER_SONIC_GENERATE_ARTICLE_GIMMICKJUMP)
-if(0x129090(False, True)){
+if(0x1290a0(False, True)){
 	if(is_excute){
 		ArticleModule__set_visibility_whole(FIGHTER_SONIC_GENERATE_ARTICLE_GIMMICKJUMP, True)
 	}
 }
