diff --git "a/12.0.0/eflame/game_AttackAirLw.txt" "b/13.0.0/eflame/game_AttackAirLw.txt"
index 48b5b66..1ce523f 100644
--- "a/12.0.0/eflame/game_AttackAirLw.txt"
+++ "b/13.0.0/eflame/game_AttackAirLw.txt"
@@ -6,18 +6,18 @@ frame(Frame=6)
 FT_MOTION_RATE(FSM=2)
 frame(Frame=8)
 FT_MOTION_RATE(FSM=1)
 frame(Frame=9)
 IS_EXIST_ARTICLE(FIGHTER_EFLAME_GENERATE_ARTICLE_ESWORD)
-if(0x3cc3a0(false, true)){
+if(0x3cc0b0(false, true)){
 	if(is_excute){
 		methodlib::L2CValue::as_hash()const(FIGHTER_EFLAME_GENERATE_ARTICLE_ESWORD, WEAPON_EFLAME_ESWORD_MOTION_PART_SET_KIND_OPEM_CLOSE, 0x07439e926b, 5, 5, false, false, 0, false, true, false)
 		ArticleModule::add_motion_partial()
 	}
 }
 MotionModule::is_changing()
-if(0x3cc3a0(false, true)){
+if(0x3cc0b0(false, true)){
 	if(is_excute){
 		WorkModule::on_flag(Flag=FIGHTER_EFLAME_INSTANCE_WORK_ID_FLAG_ADD_PARTIAL_MTION_SWORD_WHEN_CHANGEING)
 	}
 }
 frame(0, 15)
@@ -56,18 +56,18 @@ frame(Frame=19)
 if(is_excute){
 	AttackModule::clear_all()
 }
 frame(Frame=26)
 IS_EXIST_ARTICLE(FIGHTER_EFLAME_GENERATE_ARTICLE_ESWORD)
-if(0x3cc3a0(false, true)){
+if(0x3cc0b0(false, true)){
 	if(is_excute){
 		methodlib::L2CValue::as_hash()const(FIGHTER_EFLAME_GENERATE_ARTICLE_ESWORD, WEAPON_EFLAME_ESWORD_MOTION_PART_SET_KIND_OPEM_CLOSE, 0x08183db0f4, 5, 5, false, false, 0, false, true, false)
 		ArticleModule::add_motion_partial()
 	}
 }
 MotionModule::is_changing()
-if(0x3cc3a0(false, true)){
+if(0x3cc0b0(false, true)){
 	if(is_excute){
 		WorkModule::on_flag(Flag=FIGHTER_EFLAME_INSTANCE_WORK_ID_FLAG_ADD_PARTIAL_MTION_SWORD_WHEN_CHANGEING)
 	}
 }
 frame(0, 33)
