diff --git "a/3.0.0/purin/game_Attack12.txt" "b/3.1.0/purin/game_Attack12.txt"
index 6dbdeb8..aba85d3 100644
--- "a/3.0.0/purin/game_Attack12.txt"
+++ "b/3.1.0/purin/game_Attack12.txt"
@@ -1,9 +1,9 @@
 frame(Frame=5)
 if(is_excute){
 	ATTACK(ID=0, Part=0, Bone=top, Damage=3.0, Angle=361, KBG=50, FKB=0, BKB=50, Size=3.5, X=0.0, Y=5.0, Z=4.0, Hitlag=1.7, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_normal, SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=COLLISION_SOUND_ATTR_PUNCH, Type=ATTACK_REGION_PUNCH)
-	ATTACK(ID=1, Part=0, Bone=top, Damage=3.0, Angle=361, KBG=50, FKB=0, BKB=50, Size=3.9, X=0.0, Y=5.0, Z=8.5, Hitlag=1.7, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_normal, SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=COLLISION_SOUND_ATTR_PUNCH, Type=ATTACK_REGION_PUNCH)
+	ATTACK(ID=1, Part=0, Bone=top, Damage=3.0, Angle=361, KBG=50, FKB=0, BKB=50, Size=3.9, X=0.0, Y=5.0, Z=8.5, X2=0.0, Y2=5.0, Z2=9.0, Hitlag=1.7, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_normal, SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=COLLISION_SOUND_ATTR_PUNCH, Type=ATTACK_REGION_PUNCH)
 }
 wait(Frames=2)
 if(is_excute){
 	AttackModule__clear_all()
 }
