diff --git "a/_9.0.0/mariod/game_Win2Wait.txt" "b/10.0.0/mariod/game_Win2Wait.txt"
index 6966940..79dec69 100644
--- "a/_9.0.0/mariod/game_Win2Wait.txt"
+++ "b/10.0.0/mariod/game_Win2Wait.txt"
@@ -1,7 +1,7 @@
 IS_EXIST_ARTICLE(FIGHTER_MARIOD_GENERATE_ARTICLE_STETHOSCOPE)
-if(0xda890(false, false)){
+if(0xd9460(false, false)){
 	if(is_excute){
 		ArticleModule::generate_article_enable(FIGHTER_MARIOD_GENERATE_ARTICLE_STETHOSCOPE)
 	}
 }
 if(is_excute){
