diff --git "a/12.0.0/elight/game_Win2Wait.txt" "b/13.0.0/elight/game_Win2Wait.txt"
index 31d0ee2..a354eef 100644
--- "a/12.0.0/elight/game_Win2Wait.txt"
+++ "b/13.0.0/elight/game_Win2Wait.txt"
@@ -1,7 +1,7 @@
 WorkModule::is_flag(FIGHTER_INSTANCE_WORK_ID_FLAG_RESULT_FIGHTER_SINGLE)
-if(0x354230(false, false)){
+if(0x357740(false, false)){
 	if(is_excute){
 		methodlib::L2CValue::as_hash()const(FIGHTER_ELIGHT_GENERATE_ARTICLE_DIVER, 0x0ce639ee88)
 		ArticleModule::change_motion()
 	}
 }
