diff --git "a/0.1.0/marth/game_FinalStart.txt" "b/1.0.0/marth/game_FinalStart.txt"
index aabd34e..b3de414 100644
--- "a/0.1.0/marth/game_FinalStart.txt"
+++ "b/1.0.0/marth/game_FinalStart.txt"
@@ -16,11 +16,11 @@ if(0x139640(0, 0)){
 	}
 	else{
 		if(is_excute){
 			camera(MA_MSC_CMD_CAMERA_CAM_RECT, 0, 10, 0, 0)
 			PostureModule::scale(5, 0)
-			0x139640(1193257343, 2)
+			0x139640(-1933449883, 2)
 			CAM_ZOOM_IN_arg5(0, 0)
 			FT_START_CUTIN()
 		}
 		frame(Frame=25)
 		if(is_excute){
