diff --git "a/5.0.0/purin/game_AttackAirLw.txt" "b/6.0.0/purin/game_AttackAirLw.txt"
index b557106..3e68bbe 100644
--- "a/5.0.0/purin/game_AttackAirLw.txt"
+++ "b/6.0.0/purin/game_AttackAirLw.txt"
@@ -19,9 +19,9 @@ if(is_excute){
 }
 wait(Frames=2)
 if(is_excute){
 	AttackModule__clear_all()
 }
-frame(Frame=45)
+frame(Frame=42)
 if(is_excute){
 	WorkModule__off_flag(Flag=FIGHTER_STATUS_ATTACK_AIR_FLAG_ENABLE_LANDING)
 }
