diff --git "a/13.0.0/dedede/game_Win2Wait.txt" "b/13.0.1/dedede/game_Win2Wait.txt"
index eeaf4fc..fc0b985 100644
--- "a/13.0.0/dedede/game_Win2Wait.txt"
+++ "b/13.0.1/dedede/game_Win2Wait.txt"
@@ -1,7 +1,7 @@
 WorkModule::is_flag(FIGHTER_INSTANCE_WORK_ID_FLAG_RESULT_FIGHTER_SINGLE)
-if(0x157710(false, false)){
+if(0x1579b0(false, false)){
 	if(is_excute){
 		methodlib::L2CValue::as_hash()const(FIGHTER_DEDEDE_GENERATE_ARTICLE_WADDLEDEE, hash40("win2_wait"))
 		ArticleModule::change_motion()
 	}
 }
