diff --git "a/_8.0.0/pikmin/game_SpecialAirS.txt" "b/_9.0.0/pikmin/game_SpecialAirS.txt"
index 0049bb3..6b4f781 100644
--- "a/_8.0.0/pikmin/game_SpecialAirS.txt"
+++ "b/_9.0.0/pikmin/game_SpecialAirS.txt"
@@ -2,11 +2,11 @@ if(method.lib::L2CValue.operator___lib::L2CValueconst__const(WorkModule::is_flag
 	frame(Frame=9)
 	if(is_excute){
 		WorkModule::on_flag(Flag=FIGHTER_PIKMIN_STATUS_SPECIAL_S_FLAG_THROW)
 	}
 	else{
-		frame(Frame=-502677940)
+		frame(Frame=617544972)
 		if(is_excute){
 			MotionModule::set_rate(0.6)
 		}
 	}
 }
