diff --git "a/10.1.0/dedede/game_SpecialAirSGet.txt" "b/11.0.0/dedede/game_SpecialAirSGet.txt"
index 7be864e..f85e52d 100644
--- "a/10.1.0/dedede/game_SpecialAirSGet.txt"
+++ "b/11.0.0/dedede/game_SpecialAirSGet.txt"
@@ -1,4 +1,8 @@
+frame(Frame=1)
+FT_MOTION_RATE(FSM=0.8)
+frame(Frame=20)
+FT_MOTION_RATE(FSM=1)
 frame(Frame=29)
 if(is_excute){
 	WorkModule::on_flag(Flag=FIGHTER_DEDEDE_STATUS_GORDO_THROW_FLAG_THROW)
 }
