diff --git "a/.0.0/link/game_FinalAir.txt" "b/.0.0/link/game_FinalAir.txt"
index 72f4078..b29bb17 100644
--- "a/.0.0/link/game_FinalAir.txt"
+++ "b/.0.0/link/game_FinalAir.txt"
@@ -11,11 +11,11 @@ if(0x110910(WorkModule__is_flag(FIGHTER_INSTANCE_WORK_ID_FLAG_DISABLE_FINAL_STAR
 		FT_START_CUTIN()
 	}
 	else{
 		if(is_excute){
 			PostureModule__scale(3, 0)
-			0x110910(1967980257, 1.8)
+			0x110910(-155209434, 1.8)
 			CAM_ZOOM_IN_arg5(0, 0)
 			FT_START_CUTIN()
 			camera(MA_MSC_CMD_CAMERA_CAM_RECT, 0, -5, -2, 0)
 		}
 	}
