diff --git "a/13.0.0/packun/game_SpecialNShootB.txt" "b/13.0.1/packun/game_SpecialNShootB.txt"
index 9dcb32d..1c425a7 100644
--- "a/13.0.0/packun/game_SpecialNShootB.txt"
+++ "b/13.0.1/packun/game_SpecialNShootB.txt"
@@ -9,12 +9,12 @@ if(methodlib::L2CValue::operator==(lib::L2CValueconst&)const(FIGHTER_INSTANCE_WO
 frame(Frame=1)
 WorkModule::is_flag(FIGHTER_PACKUN_STATUS_SPECIAL_N_FLAG_SHOOT_PREVIOUS_WAIT)
 if(methodlib::L2CValue::operator==(lib::L2CValueconst&)const(false, true)){
 	WorkModule::get_float(FIGHTER_PACKUN_STATUS_SPECIAL_N_WORK_FLOAT_SHOOT_START_SPIKEBALL_DISTANCE)
 	PostureModule::scale(0)
-	0x157dc0(0, 30)
-	if(0x157dc0()){
+	0x1646b0(0, 30)
+	if(0x1646b0()){
 		FT_MOTION_RATE(FSM=0.6)
 		frame(Frame=8)
 		if(is_excute){
 			ArticleModule::shoot_exist(FIGHTER_PACKUN_GENERATE_ARTICLE_SPIKEBALL, ARTICLE_OPE_TARGET_ALL, false)
 		}
@@ -60,12 +60,12 @@ if(methodlib::L2CValue::operator==(lib::L2CValueconst&)const(false, true)){
 		FT_MOTION_RATE(FSM=1)
 	}
 }
 WorkModule::get_float(0, FIGHTER_PACKUN_STATUS_SPECIAL_N_WORK_FLOAT_SHOOT_START_SPIKEBALL_DISTANCE)
 PostureModule::scale(0)
-0x157dc0(0, 40)
-if(0x157dc0()){
+0x1646b0(0, 40)
+if(0x1646b0()){
 	FT_MOTION_RATE(FSM=0.7)
 	frame(Frame=8)
 	if(is_excute){
 		ArticleModule::shoot_exist(FIGHTER_PACKUN_GENERATE_ARTICLE_SPIKEBALL, ARTICLE_OPE_TARGET_ALL, false)
 	}
@@ -87,12 +87,12 @@ if(is_excute){
 	WorkModule::off_flag(Flag=FIGHTER_PACKUN_STATUS_SPECIAL_N_FLAG_SHOOT_BLOW_SPIKEBALL)
 }
 else{
 	WorkModule::get_float(FIGHTER_PACKUN_STATUS_SPECIAL_N_WORK_FLOAT_SHOOT_START_SPIKEBALL_DISTANCE)
 	PostureModule::scale(0)
-	0x157dc0(0, 50)
-	if(0x157dc0()){
+	0x1646b0(0, 50)
+	if(0x1646b0()){
 		FT_MOTION_RATE(FSM=0.9)
 		frame(Frame=10)
 		if(is_excute){
 			search(MA_MSC_CMD_SEARCH_SEARCH_SCH_CLR, 0)
 		}
@@ -112,12 +112,12 @@ else{
 			WorkModule::off_flag(Flag=FIGHTER_PACKUN_STATUS_SPECIAL_N_FLAG_SHOOT_BLOW_SPIKEBALL)
 		}
 		else{
 			WorkModule::get_float(FIGHTER_PACKUN_STATUS_SPECIAL_N_WORK_FLOAT_SHOOT_START_SPIKEBALL_DISTANCE)
 			PostureModule::scale(0)
-			0x157dc0(0, 100)
-			if(0x157dc0()){
+			0x1646b0(0, 100)
+			if(0x1646b0()){
 				FT_MOTION_RATE(FSM=1.2)
 				frame(Frame=10)
 				if(is_excute){
 					search(MA_MSC_CMD_SEARCH_SEARCH_SCH_CLR, 0)
 				}
