diff --git "a/_3.1.0/mariod/game_Win1Wait.txt" "b/_4.0.0/mariod/game_Win1Wait.txt"
index 0e6306f..b857bce 100644
--- "a/_3.1.0/mariod/game_Win1Wait.txt"
+++ "b/_4.0.0/mariod/game_Win1Wait.txt"
@@ -1,7 +1,7 @@
 IS_EXIST_ARTICLE(FIGHTER_MARIOD_GENERATE_ARTICLE_DRCAPSULE)
-if(0xd45d0(False, False)){
+if(0xd45f0(False, False)){
 	if(is_excute){
 		ArticleModule__generate_article_enable(FIGHTER_MARIOD_GENERATE_ARTICLE_DRCAPSULE)
 		ArticleModule__generate_article_enable(FIGHTER_MARIOD_GENERATE_ARTICLE_DRCAPSULE)
 	}
 }
