diff --git "a/7.0.0/diddy/game_FinalCharge.txt" "b/8.0.0/diddy/game_FinalCharge.txt"
index b71b0b8..fc13bb3 100644
--- "a/7.0.0/diddy/game_FinalCharge.txt"
+++ "b/8.0.0/diddy/game_FinalCharge.txt"
@@ -1,8 +1,8 @@
 if(is_excute){
 	PostureModule::scale(10, 0)
-	0x13c860(-1925395313, 2.1)
+	0x148970(1485675813, 2.1)
 	CAM_ZOOM_IN_arg5(0, 0)
 	SLOW_OPPONENT(30, 60)
 }
 frame(Frame=40)
 if(is_excute){
