diff --git "a/ga_6.0.0/gekkouga_gekkougas/game_FinalAttack.txt" "b/ga_7.0.0/gekkouga_gekkougas/game_FinalAttack.txt"
index ab6e7b7..4bb571b 100644
--- "a/ga_6.0.0/gekkouga_gekkougas/game_FinalAttack.txt"
+++ "b/ga_7.0.0/gekkouga_gekkougas/game_FinalAttack.txt"
@@ -1,23 +1,23 @@
 if(is_excute){
 	CAM_ZOOM_OUT()
 }
 frame(Frame=2)
 get_fov()
-if(0x1410f0(1, 25.0)){
+if(0x141100(1, 25.0)){
 	if(is_excute){
 		get_fov(5, 0)
-		0x1410f0(0, 60)
+		0x141100(0, 60)
 		CAM_ZOOM_IN_arg5(0.1, 0)
 	}
 }
 frame(Frame=3)
 get_fov()
-if(0x1410f0(1, 5.0)){
+if(0x141100(1, 5.0)){
 	if(is_excute){
 		get_fov(5, 0)
-		0x1410f0(0, 90)
+		0x141100(0, 90)
 		CAM_ZOOM_IN_arg5(0.1, 0)
 	}
 }
 frame(Frame=4)
 get_fov()
