diff --git "a/a_10.0.0/gekkouga/game_FinalAirChange.txt" "b/a_10.1.0/gekkouga/game_FinalAirChange.txt"
index 03760af..63c6177 100644
--- "a/a_10.0.0/gekkouga/game_FinalAirChange.txt"
+++ "b/a_10.1.0/gekkouga/game_FinalAirChange.txt"
@@ -13,11 +13,11 @@ if(0x141250(WorkModule::is_flag(FIGHTER_INSTANCE_WORK_ID_FLAG_DISABLE_FINAL_STAR
 		REQ_FINAL_START_CAMERA_arg3(hash40("d04finalchange.nuanmb"), false, false)
 	}
 	else{
 		if(is_excute){
 			PostureModule::scale(3, 0)
-			0x141250(601692341, 2.3)
+			0x141250(1427875192, 2.3)
 			CAM_ZOOM_IN_arg5(0, 0)
 		}
 		get_value_float(SO_VAR_FLOAT_LR)
 		if(0x141250(0, 0)){
 			if(is_excute){
