diff --git "a/2.0.0/demon/0x17628cba9c.txt" "b/3.0.0/demon/0x17628cba9c.txt"
index a19a864..ea92aea 100644
--- "a/2.0.0/demon/0x17628cba9c.txt"
+++ "b/3.0.0/demon/0x17628cba9c.txt"
@@ -1,7 +1,7 @@
 WorkModule::is_flag(FIGHTER_DEMON_INSTANCE_WORK_ID_FLAG_ATTACK_RAGE_CAPTURE)
-if(0x3cfb50(false, false)){
+if(0x3cfb70(false, false)){
 	battle_object_module_accessor()
 	methodlib::L2CValue::L2CValue(void*)()
 	methodlib::L2CValue::L2CValue(lib::L2CValueconst&)()
 	methodlib::L2CValue::as_pointer()const(true, 10)
 	set_devil()
