diff --git "a/2.0.0/demon/game_SpecialHi.txt" "b/3.0.0/demon/game_SpecialHi.txt"
index 80b4d15..bbbe611 100644
--- "a/2.0.0/demon/game_SpecialHi.txt"
+++ "b/3.0.0/demon/game_SpecialHi.txt"
@@ -1,7 +1,7 @@
 WorkModule::is_flag(FIGHTER_DEMON_STATUS_SPECIAL_HI_FLAG_AIR)
-if(0x3cfb50(false, true)){
+if(0x3cfb70(false, true)){
 	battle_object_module_accessor()
 	methodlib::L2CValue::L2CValue(void*)()
 	methodlib::L2CValue::L2CValue(lib::L2CValueconst&)()
 	methodlib::L2CValue::as_pointer()const(true, 2)
 	set_devil()
