diff --git "a/7.0.0/brave/game_SpecialAirN3.txt" "b/8.0.0/brave/game_SpecialAirN3.txt"
index 541a667..6b549f6 100644
--- "a/7.0.0/brave/game_SpecialAirN3.txt"
+++ "b/8.0.0/brave/game_SpecialAirN3.txt"
@@ -4,11 +4,11 @@ if(is_excute){
 frame(Frame=14)
 if(is_excute){
 	WorkModule::on_flag(Flag=FIGHTER_BRAVE_STATUS_SPECIAL_N_FLAG_HOP)
 }
 frame(Frame=16)
-if(0x21bdd0(WorkModule::is_flag(FIGHTER_BRAVE_STATUS_SPECIAL_N_FLAG_SUCCESS_SP), false, true)){
+if(0x21c050(WorkModule::is_flag(FIGHTER_BRAVE_STATUS_SPECIAL_N_FLAG_SUCCESS_SP), false, true)){
 	if(is_excute){
 		ArticleModule::generate_article(FIGHTER_BRAVE_GENERATE_ARTICLE_FIREBALL)
 		KineticModule::add_speed(-0.3, 0, 0.0)
 	}
 }
