diff --git "a/.1.0/edge/game_SpecialLwEnd.txt" "b/.0.0/edge/game_SpecialLwEnd.txt"
index 1abf8d9..61c9b1a 100644
--- "a/.1.0/edge/game_SpecialLwEnd.txt"
+++ "b/.0.0/edge/game_SpecialLwEnd.txt"
@@ -1,8 +1,8 @@
 if(is_excute){
 	SET_SPEED_EX(-1.4, 0, KINETIC_ENERGY_RESERVE_ATTRIBUTE_MAIN)
 }
 frame(Frame=18)
 if(is_excute){
-	WorkModule::on_flag(Flag=56068)
-	WorkModule::on_flag(Flag=56076)
+	WorkModule::on_flag(Flag=56996)
+	WorkModule::on_flag(Flag=57004)
 }
