diff --git "a/.1.0/edge/game_SpecialAirHi2End.txt" "b/.0.0/edge/game_SpecialAirHi2End.txt"
index 6c81c8f..34895fe 100644
--- "a/.1.0/edge/game_SpecialAirHi2End.txt"
+++ "b/.0.0/edge/game_SpecialAirHi2End.txt"
@@ -16,7 +16,7 @@ frame(Frame=6)
 if(is_excute){
 	notify_event_msc_cmd(0x2127e37c07, GROUND_CLIFF_CHECK_KIND_ALWAYS_BOTH_SIDES)
 }
 frame(Frame=18)
 if(is_excute){
-	WorkModule::on_flag(Flag=56064)
+	WorkModule::on_flag(Flag=56992)
 }
