diff --git "a/.1.0/edge/game_SpecialAirNStart.txt" "b/.0.0/edge/game_SpecialAirNStart.txt"
index 879c2b9..fb733d7 100644
--- "a/.1.0/edge/game_SpecialAirNStart.txt"
+++ "b/.0.0/edge/game_SpecialAirNStart.txt"
@@ -1,28 +1,28 @@
 frame(Frame=20)
 if(is_excute){
-	WorkModule::set_int(17316, 17296)
+	WorkModule::set_int(17420, 17400)
 }
 frame(Frame=32)
 if(is_excute){
-	WorkModule::set_int(17328, 17296)
+	WorkModule::set_int(17432, 17400)
 }
 frame(Frame=79)
 FT_MOTION_RATE(FSM=1.2)
 frame(Frame=99)
 FT_MOTION_RATE(FSM=1)
 frame(Frame=100)
 if(is_excute){
-	WorkModule::set_int(17308, 17296)
+	WorkModule::set_int(17412, 17400)
 }
 frame(Frame=105)
 FT_MOTION_RATE(FSM=1.6)
 frame(Frame=115)
 FT_MOTION_RATE(FSM=1)
 frame(Frame=120)
 if(is_excute){
-	ArticleModule::generate_article(17344)
+	ArticleModule::generate_article(17448)
 }
 frame(Frame=140)
 if(is_excute){
-	WorkModule::on_flag(Flag=17336)
+	WorkModule::on_flag(Flag=17440)
 }
