diff --git "a/.1.0/edge/game_SpecialN1.txt" "b/.0.0/edge/game_SpecialN1.txt"
index c708b44..b0afc6d 100644
--- "a/.1.0/edge/game_SpecialN1.txt"
+++ "b/.0.0/edge/game_SpecialN1.txt"
@@ -2,11 +2,11 @@ frame(Frame=1)
 FT_MOTION_RATE(FSM=1.9)
 frame(Frame=11)
 FT_MOTION_RATE(FSM=1)
 frame(Frame=13)
 if(is_excute){
-	ArticleModule::generate_article(17344)
+	ArticleModule::generate_article(17448)
 }
 frame(Frame=35)
 if(is_excute){
-	WorkModule::on_flag(Flag=17336)
+	WorkModule::on_flag(Flag=17440)
 }
