diff --git "a/11.0.0/elight/game_SpecialAirHi2.txt" "b/12.0.0/elight/game_SpecialAirHi2.txt"
index 1edf725..b1b8c3d 100644
--- "a/11.0.0/elight/game_SpecialAirHi2.txt"
+++ "b/12.0.0/elight/game_SpecialAirHi2.txt"
@@ -1,51 +1,51 @@
 frame(Frame=1)
 if(is_excute){
 	notify_event_msc_cmd(0x2127e37c07, GROUND_CLIFF_CHECK_KIND_NONE)
 }
-IS_EXIST_ARTICLE(17456)
-if(0x353770(false, true)){
+IS_EXIST_ARTICLE(17632)
+if(0x354230(false, true)){
 	if(is_excute){
-		methodlib::L2CValue::as_hash()const(17456, 60344, 0x07439e926b, 5, 5, false, false, 0, false, true, false)
+		methodlib::L2CValue::as_hash()const(17632, 61372, 0x07439e926b, 5, 5, false, false, 0, false, true, false)
 		ArticleModule::add_motion_partial()
 	}
 }
 MotionModule::is_changing()
-if(0x353770(false, true)){
+if(0x354230(false, true)){
 	if(is_excute){
-		WorkModule::on_flag(Flag=60348)
+		WorkModule::on_flag(Flag=61376)
 	}
 }
 frame(0, 12)
 if(is_excute){
-	ArticleModule::generate_article(60332)
+	ArticleModule::generate_article(61360)
 }
 frame(Frame=13)
 if(is_excute){
-	ArticleModule::generate_article(60332)
+	ArticleModule::generate_article(61360)
 }
 frame(Frame=14)
 if(is_excute){
-	ArticleModule::generate_article(60332)
+	ArticleModule::generate_article(61360)
 }
 frame(Frame=15)
 if(is_excute){
-	ArticleModule::generate_article(60332)
+	ArticleModule::generate_article(61360)
 }
 frame(Frame=16)
 if(is_excute){
-	ArticleModule::generate_article(60332)
+	ArticleModule::generate_article(61360)
 }
 frame(Frame=18)
-IS_EXIST_ARTICLE(17456)
-if(0x353770(false, true)){
+IS_EXIST_ARTICLE(17632)
+if(0x354230(false, true)){
 	if(is_excute){
-		methodlib::L2CValue::as_hash()const(17456, 60344, 0x08183db0f4, 5, 5, false, false, 0, false, true, false)
+		methodlib::L2CValue::as_hash()const(17632, 61372, 0x08183db0f4, 5, 5, false, false, 0, false, true, false)
 		ArticleModule::add_motion_partial()
 	}
 }
 MotionModule::is_changing()
-if(0x353770(false, true)){
+if(0x354230(false, true)){
 	if(is_excute){
-		WorkModule::on_flag(Flag=60348)
+		WorkModule::on_flag(Flag=61376)
 	}
 }
