diff --git "a/11.0.0/elight/game_FinalAir.txt" "b/12.0.0/elight/game_FinalAir.txt"
index c7302a2..5908e60 100644
--- "a/11.0.0/elight/game_FinalAir.txt"
+++ "b/12.0.0/elight/game_FinalAir.txt"
@@ -10,23 +10,23 @@ if(is_excute){
 	ATTACK(ID=7, Part=0, Bone=hash40("top"), Damage=0.0, Angle=366, KBG=0, FKB=0, BKB=30, Size=11.0, X=0.0, Y=4.0, Z=-14.5, X2=0.0, Y2=18.0, Z2=-14.5, Hitlag=0.0, SDI=1.0, Clang_Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=true, ShieldDamage=hash40("no"), Trip=0.0, Rehit=1, Reflectable=false, Absorbable=false, Flinchless=true, DisableHitlag=true, Direct_Hitbox=true, Ground_or_Air=COLLISION_SITUATION_MASK_A, Hitbits=COLLISION_CATEGORY_MASK_FIGHTER, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=false, Effect=hash40("collision_attr_normal"), SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=COLLISION_SOUND_ATTR_NONE, Type=ATTACK_REGION_NONE)
 }
 frame(Frame=5)
 if(is_excute){
 	CHECK_VALID_FINAL_START_CAMERA(0, 6.5, 20, 0, 0, 0)
-	WorkModule::on_flag(Flag=56548)
+	WorkModule::on_flag(Flag=56732)
 	FT_SET_FINAL_FEAR_FACE(50)
 	FT_START_CUTIN()
 }
 WorkModule::is_flag(FIGHTER_INSTANCE_WORK_ID_FLAG_DISABLE_FINAL_START_CAMERA)
-if(0x353770(false, false)){
+if(0x354230(false, false)){
 	if(is_excute){
 		REQ_FINAL_START_CAMERA_arg3(hash40("d04finalstart.nuanmb"), true, false)
 	}
 	else{
 		if(is_excute){
 			PostureModule::scale(4, 0)
-			0x353770(-1154986274, 2.1)
+			0x354230(-707422823, 2.1)
 			CAM_ZOOM_IN_arg5(0, 0)
 		}
 	}
 }
 frame(Frame=6)
@@ -39,21 +39,21 @@ else{
 }
 methodlib::L2CValue::as_pointer()const()
 attach_esword_diver()
 SLOW_OPPONENT(30, 80)
 frame(Frame=22)
-IS_EXIST_ARTICLE(17456)
-if(0x353770(false, true)){
+IS_EXIST_ARTICLE(17632)
+if(0x354230(false, true)){
 	if(is_excute){
-		methodlib::L2CValue::as_hash()const(17456, 60344, 0x07439e926b, 5, 5, false, false, 0, false, true, false)
+		methodlib::L2CValue::as_hash()const(17632, 61372, 0x07439e926b, 5, 5, false, false, 0, false, true, false)
 		ArticleModule::add_motion_partial()
 	}
 }
 MotionModule::is_changing()
-if(0x353770(false, true)){
+if(0x354230(false, true)){
 	if(is_excute){
-		WorkModule::on_flag(Flag=60348)
+		WorkModule::on_flag(Flag=61376)
 	}
 }
 frame(0, 25)
 if(is_excute){
 	CAM_ZOOM_OUT()
@@ -98,21 +98,21 @@ if(is_excute){
 frame(Frame=61)
 if(is_excute){
 	AttackModule::clear_all()
 }
 frame(Frame=70)
-IS_EXIST_ARTICLE(17456)
-if(0x353770(false, true)){
+IS_EXIST_ARTICLE(17632)
+if(0x354230(false, true)){
 	if(is_excute){
-		methodlib::L2CValue::as_hash()const(17456, 60344, 0x08183db0f4, 5, 5, false, false, 0, false, true, false)
+		methodlib::L2CValue::as_hash()const(17632, 61372, 0x08183db0f4, 5, 5, false, false, 0, false, true, false)
 		ArticleModule::add_motion_partial()
 	}
 }
 MotionModule::is_changing()
-if(0x353770(false, true)){
+if(0x354230(false, true)){
 	if(is_excute){
-		WorkModule::on_flag(Flag=60348)
+		WorkModule::on_flag(Flag=61376)
 	}
 }
 frame(0, 90)
 if(is_excute){
 	battle_object_module_accessor()
@@ -121,6 +121,6 @@ if(is_excute){
 else{
 	methodlib::L2CValue::L2CValue(lib::L2CValueconst&)()
 }
 methodlib::L2CValue::as_pointer()const()
 detach_esword_diver()
-WorkModule::on_flag(Flag=60288)
+WorkModule::on_flag(Flag=61316)
