diff --git "a/ter_10.1.0/miifighter/game_FinalStart.txt" "b/ter_11.0.0/miifighter/game_FinalStart.txt"
index f815ee2..cebdf89 100644
--- "a/ter_10.1.0/miifighter/game_FinalStart.txt"
+++ "b/ter_11.0.0/miifighter/game_FinalStart.txt"
@@ -27,11 +27,11 @@ if(is_excute){
 	SlowModule::set_whole(2, 0)
 }
 else{
 	if(is_excute){
 		PostureModule::scale(3, 0)
-		0x18b7d0(-1589736992, 2.4)
+		0x18b900(1665199257, 2.4)
 		CAM_ZOOM_IN_arg5(0, 0)
 		FT_START_CUTIN()
 	}
 }
 frame(Frame=20)
