diff --git "a/2.0.0/kirby/game_Win3.txt" "b/3.0.0/kirby/game_Win3.txt"
index 846c3a2..b6c074c 100644
--- "a/2.0.0/kirby/game_Win3.txt"
+++ "b/3.0.0/kirby/game_Win3.txt"
@@ -1,7 +1,7 @@
 WorkModule::is_flag(FIGHTER_INSTANCE_WORK_ID_FLAG_RESULT_FIGHTER_SINGLE)
-0x52d460(false, true)
+0x543c50(false, true)
 if(is_excute){
 	ArticleModule::generate_article(FIGHTER_KIRBY_GENERATE_ARTICLE_WINDUMMY)
 	ArticleModule::generate_article(FIGHTER_KIRBY_GENERATE_ARTICLE_WINDUMMY)
 	methodlib::L2CValue::as_hash()const(FIGHTER_KIRBY_GENERATE_ARTICLE_WINDUMMY, hash40("win_3"))
 	ArticleModule::change_motion()
