diff --git "a/0.0/ken/game_FinalAir2.txt" "b/0.0/ken/game_FinalAir2.txt"
index f92e37b..9675f27 100644
--- "a/0.0/ken/game_FinalAir2.txt"
+++ "b/0.0/ken/game_FinalAir2.txt"
@@ -14,28 +14,28 @@ if(0x168b20(WorkModule__is_flag(FIGHTER_INSTANCE_WORK_ID_FLAG_DISABLE_FINAL_STAR
 		REQ_FINAL_START_CAMERA(0x100bb16777, True)
 	}
 	else{
 		if(is_excute){
 			PostureModule__scale(3, 0)
-			0x168b20(-1098132098, 1.8)
+			0x168b20(-857381893, 1.8)
 			CAM_ZOOM_IN_arg5(0, 0)
 			FT_START_CUTIN()
 		}
 		frame(Frame=28)
 		if(is_excute){
 			CAM_ZOOM_OUT()
 		}
 	}
 }
 PostureModule__scale()
-if(method.lib::L2CValue.operator___lib::L2CValueconst__const(-1098132098, 1.4)){
+if(method.lib::L2CValue.operator___lib::L2CValueconst__const(-857381893, 1.4)){
 	if(is_excute){
 		ATTACK(ID=0, Part=0, Bone=top, Damage=0.0, Angle=366, KBG=100, FKB=140, BKB=0, Size=35.0, X=0.0, Y=5.0, Z=-1.0, Hitlag=0.0, SDI=0.0, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=True, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=True, DisableHitlag=True, Direct/Indirect=False, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_normal, SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=COLLISION_SOUND_ATTR_NONE, Type=ATTACK_REGION_NONE)
 	}
 	else{
 		PostureModule__scale()
-		if(method.lib::L2CValue.operator___lib::L2CValueconst__const(-1098132098, 0.5)){
+		if(method.lib::L2CValue.operator___lib::L2CValueconst__const(-857381893, 0.5)){
 			if(is_excute){
 				ATTACK(ID=0, Part=0, Bone=top, Damage=0.0, Angle=366, KBG=100, FKB=22, BKB=0, Size=35.0, X=0.0, Y=5.0, Z=-1.0, Hitlag=0.0, SDI=0.0, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=True, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=True, DisableHitlag=True, Direct/Indirect=False, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_normal, SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=COLLISION_SOUND_ATTR_NONE, Type=ATTACK_REGION_NONE)
 			}
 			else{
 				if(is_excute){
