diff --git "a/1.0.0/dolly/game_SpecialAirSFEnd.txt" "b/2.0.0/dolly/game_SpecialAirSFEnd.txt"
index 8b76242..86d25fc 100644
--- "a/1.0.0/dolly/game_SpecialAirSFEnd.txt"
+++ "b/2.0.0/dolly/game_SpecialAirSFEnd.txt"
@@ -1,7 +1,7 @@
 frame(Frame=0)
-if(0x24ffc0(FIGHTER_DOLLY_STATUS_SPECIAL_COMMON_WORK_INT_STRENGTH, FIGHTER_DOLLY_STRENGTH_W)){
+if(0x2508c0(FIGHTER_DOLLY_STATUS_SPECIAL_COMMON_WORK_INT_STRENGTH, FIGHTER_DOLLY_STRENGTH_W)){
 	if(is_excute){
 		MotionModule::set_rate(1)
 	}
 	else{
 		if(is_excute){
