diff --git "a/2.0.0/cloud/game_Final2AirStart.txt" "b/3.0.0/cloud/game_Final2AirStart.txt"
index b028f12..eed9547 100644
--- "a/2.0.0/cloud/game_Final2AirStart.txt"
+++ "b/3.0.0/cloud/game_Final2AirStart.txt"
@@ -18,11 +18,11 @@ if(0x1801d0(2001240, 0)){
 		REQ_FINAL_START_CAMERA(hash40("d04finalstart02.nuanmb"), true)
 	}
 	else{
 		if(is_excute){
 			PostureModule::scale(4, 0)
-			0x1801d0(527381693, 1.8)
+			0x1801d0(647911431, 1.8)
 			CAM_ZOOM_IN_arg5(-0.1, 0)
 			FT_START_CUTIN()
 			camera(MA_MSC_CMD_CAMERA_CAM_OFFSET, 0, 6)
 		}
 		else{
