diff --git "a/_5.0.0/pikmin_pikmin/game_SpNTakenOutStart_y.txt" "b/_6.0.0/pikmin_pikmin/game_SpNTakenOutStart_y.txt"
index a731991..2844bbb 100644
--- "a/_5.0.0/pikmin_pikmin/game_SpNTakenOutStart_y.txt"
+++ "b/_6.0.0/pikmin_pikmin/game_SpNTakenOutStart_y.txt"
@@ -1,9 +1,9 @@
 frame(Frame=1)
 if(is_excute){
 	WorkModule__get_float(WEAPON_PIKMIN_PIKMIN_STATUS_PULL_OUT_START_WORK_FLOAT_MOT_RATE)
-	MotionModule__set_rate(463762201)
+	MotionModule__set_rate(-1298307000)
 }
 frame(Frame=5)
 if(is_excute){
 	MotionModule__set_rate(1)
 }
