diff --git "a/9.0.0/luigi/game_FinalStart.txt" "b/0.0.0/luigi/game_FinalStart.txt"
index feb28c3..184df8e 100644
--- "a/9.0.0/luigi/game_FinalStart.txt"
+++ "b/0.0.0/luigi/game_FinalStart.txt"
@@ -14,11 +14,11 @@ if(method.lib::L2CValue.operator__lib::L2CValueconst__const(0, 0)){
 		REQ_FINAL_START_CAMERA_arg3(hash40("d04finalstart02.nuanmb"), false, false)
 	}
 	else{
 		if(is_excute){
 			PostureModule::scale(5, 0)
-			0xe2730(106349022, 1.8)
+			0xe2730(-1311273485, 1.8)
 			CAM_ZOOM_IN_arg5(0, 0)
 		}
 		else{
 			if(is_excute){
 				REQ_FINAL_START_CAMERA_arg3(hash40("d04finalstart.nuanmb"), false, false)
