diff --git "a/10.0.0/tantan/game_SpecialAirHiStart2.txt" "b/10.1.0/tantan/game_SpecialAirHiStart2.txt"
index 67f82be..609fb22 100644
--- "a/10.0.0/tantan/game_SpecialAirHiStart2.txt"
+++ "b/10.1.0/tantan/game_SpecialAirHiStart2.txt"
@@ -1,17 +1,17 @@
 frame(Frame=4)
 if(is_excute){
-	WorkModule::on_flag(Flag=53904)
+	WorkModule::on_flag(Flag=53976)
 }
 frame(Frame=5)
 if(is_excute){
 	WorkModule::on_flag(Flag=FIGHTER_STATUS_AIR_LASSO_FLAG_CHECK)
 	WorkModule::on_flag(Flag=FIGHTER_STATUS_AIR_LASSO_FLAG_HANG_IMMIDIATE)
 }
 frame(Frame=6)
 if(is_excute){
-	GroundModule::select_cliff_hangdata(59148)
+	GroundModule::select_cliff_hangdata(59236)
 	WorkModule::off_flag(Flag=FIGHTER_STATUS_AIR_LASSO_FLAG_HANG_IMMIDIATE)
 }
 frame(Frame=8)
 if(is_excute){
 	WorkModule::off_flag(Flag=FIGHTER_STATUS_AIR_LASSO_FLAG_CHECK)
