diff --git "a/hter_7.0.0/miifighter/game_FinalStart.txt" "b/hter_8.0.0/miifighter/game_FinalStart.txt"
index ea7e886..94f4cc3 100644
--- "a/hter_7.0.0/miifighter/game_FinalStart.txt"
+++ "b/hter_8.0.0/miifighter/game_FinalStart.txt"
@@ -27,11 +27,11 @@ if(is_excute){
 	SlowModule::set_whole(2, 0)
 }
 else{
 	if(is_excute){
 		PostureModule::scale(3, 0)
-		0x18b670(-688717853, 2.4)
+		0x18b7a0(-1974002816, 2.4)
 		CAM_ZOOM_IN_arg5(0, 0)
 		FT_START_CUTIN()
 	}
 }
 frame(Frame=20)
