diff --git "a/.0.0/wolf/game_SpecialN.txt" "b/.0.0/wolf/game_SpecialN.txt"
index c45cdd0..f65e2f6 100644
--- "a/.0.0/wolf/game_SpecialN.txt"
+++ "b/.0.0/wolf/game_SpecialN.txt"
@@ -1,11 +1,11 @@
 frame(Frame=2)
 if(is_excute){
 	ArticleModule::generate_article(FIGHTER_WOLF_GENERATE_ARTICLE_BLASTER)
 }
 IS_EXIST_ARTICLE(FIGHTER_WOLF_GENERATE_ARTICLE_BLASTER)
-if(0x1081e0(false, true)){
+if(0x108090(false, true)){
 	if(is_excute){
 		as_hash__const(FIGHTER_WOLF_GENERATE_ARTICLE_BLASTER, hash40("open"))
 		ArticleModule::change_motion()
 	}
 }
@@ -23,11 +23,11 @@ frame(Frame=20)
 if(is_excute){
 	AttackModule::clear_all()
 }
 frame(Frame=38)
 IS_EXIST_ARTICLE(FIGHTER_WOLF_GENERATE_ARTICLE_BLASTER)
-if(0x1081e0(false, true)){
+if(0x108090(false, true)){
 	if(is_excute){
 		as_hash__const(FIGHTER_WOLF_GENERATE_ARTICLE_BLASTER, hash40("close"))
 		ArticleModule::change_motion()
 	}
 }
