diff --git "a/10.0.0/tantan_punch2/game_AttackShort.txt" "b/10.1.0/tantan_punch2/game_AttackShort.txt"
index 9b04653..bb1781e 100644
--- "a/10.0.0/tantan_punch2/game_AttackShort.txt"
+++ "b/10.1.0/tantan_punch2/game_AttackShort.txt"
@@ -1,30 +1,30 @@
-if(0x3a06b0(WorkModule::is_flag(59168), false, true)){
+if(0x3a06b0(WorkModule::is_flag(59256), false, true)){
 	if(is_excute){
-		ATTACK(ID=0, Part=0, Bone=hash40("center"), Damage=11.0, Angle=45, KBG=92, FKB=0, BKB=50, Size=0.7, X=0.0, Y=0.0, Z=0.0, X2=LUA_VOID, Y2=LUA_VOID, Z2=LUA_VOID, Hitlag=1.2, SDI=1.0, Clang_Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=false, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=false, Absorbable=false, Flinchless=false, DisableHitlag=false, Direct_Hitbox=true, Ground_or_Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=false, Effect=hash40("collision_attr_normal"), SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=59140, Type=ATTACK_REGION_PUNCH)
+		ATTACK(ID=0, Part=0, Bone=hash40("center"), Damage=11.0, Angle=45, KBG=92, FKB=0, BKB=50, Size=0.7, X=0.0, Y=0.0, Z=0.0, X2=LUA_VOID, Y2=LUA_VOID, Z2=LUA_VOID, Hitlag=1.2, SDI=1.0, Clang_Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=false, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=false, Absorbable=false, Flinchless=false, DisableHitlag=false, Direct_Hitbox=true, Ground_or_Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=false, Effect=hash40("collision_attr_normal"), SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=59228, Type=ATTACK_REGION_PUNCH)
 		AttackModule::enable_safe_pos()
 		AttackModule::disable_tip()
 		AttackModule::set_damage_shake_scale(0.5)
 	}
 }
 frame(Frame=1)
 if(is_excute){
-	ATTACK(ID=0, Part=0, Bone=hash40("center"), Damage=11.0, Angle=45, KBG=92, FKB=0, BKB=50, Size=3.7, X=-1.0, Y=0.0, Z=0.0, X2=LUA_VOID, Y2=LUA_VOID, Z2=LUA_VOID, Hitlag=1.2, SDI=1.0, Clang_Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=false, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=false, Absorbable=false, Flinchless=false, DisableHitlag=false, Direct_Hitbox=true, Ground_or_Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=false, Effect=hash40("collision_attr_normal"), SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=59140, Type=ATTACK_REGION_PUNCH)
+	ATTACK(ID=0, Part=0, Bone=hash40("center"), Damage=11.0, Angle=45, KBG=92, FKB=0, BKB=50, Size=3.7, X=-1.0, Y=0.0, Z=0.0, X2=LUA_VOID, Y2=LUA_VOID, Z2=LUA_VOID, Hitlag=1.2, SDI=1.0, Clang_Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=false, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=false, Absorbable=false, Flinchless=false, DisableHitlag=false, Direct_Hitbox=true, Ground_or_Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=false, Effect=hash40("collision_attr_normal"), SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=59228, Type=ATTACK_REGION_PUNCH)
 	as_hash__const(0x123a5b1b8e, 18, false)
 	ControlModule::set_rumble()
 }
 frame(Frame=14)
 operatorbool__const(is_excute)
 if(is_excute){
-	ATTACK(ID=0, Part=0, Bone=hash40("center"), Damage=11.0, Angle=45, KBG=97, FKB=0, BKB=50, Size=0.7, X=0.0, Y=0.0, Z=0.0, X2=LUA_VOID, Y2=LUA_VOID, Z2=LUA_VOID, Hitlag=1.2, SDI=1.0, Clang_Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=false, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=false, Absorbable=false, Flinchless=false, DisableHitlag=false, Direct_Hitbox=true, Ground_or_Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=false, Effect=hash40("collision_attr_normal"), SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=59140, Type=ATTACK_REGION_PUNCH)
+	ATTACK(ID=0, Part=0, Bone=hash40("center"), Damage=11.0, Angle=45, KBG=97, FKB=0, BKB=50, Size=0.7, X=0.0, Y=0.0, Z=0.0, X2=LUA_VOID, Y2=LUA_VOID, Z2=LUA_VOID, Hitlag=1.2, SDI=1.0, Clang_Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=false, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=false, Absorbable=false, Flinchless=false, DisableHitlag=false, Direct_Hitbox=true, Ground_or_Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=false, Effect=hash40("collision_attr_normal"), SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=59228, Type=ATTACK_REGION_PUNCH)
 	AttackModule::enable_safe_pos()
 	AttackModule::disable_tip()
 	AttackModule::set_damage_shake_scale(0.5)
 }
 frame(Frame=1)
 if(is_excute){
-	ATTACK(ID=0, Part=0, Bone=hash40("center"), Damage=11.0, Angle=45, KBG=97, FKB=0, BKB=50, Size=3.7, X=-1.0, Y=0.0, Z=0.0, X2=LUA_VOID, Y2=LUA_VOID, Z2=LUA_VOID, Hitlag=1.2, SDI=1.0, Clang_Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=false, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=false, Absorbable=false, Flinchless=false, DisableHitlag=false, Direct_Hitbox=true, Ground_or_Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=false, Effect=hash40("collision_attr_normal"), SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=59140, Type=ATTACK_REGION_PUNCH)
+	ATTACK(ID=0, Part=0, Bone=hash40("center"), Damage=11.0, Angle=45, KBG=97, FKB=0, BKB=50, Size=3.7, X=-1.0, Y=0.0, Z=0.0, X2=LUA_VOID, Y2=LUA_VOID, Z2=LUA_VOID, Hitlag=1.2, SDI=1.0, Clang_Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=false, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=false, Absorbable=false, Flinchless=false, DisableHitlag=false, Direct_Hitbox=true, Ground_or_Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=false, Effect=hash40("collision_attr_normal"), SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=59228, Type=ATTACK_REGION_PUNCH)
 	as_hash__const(0x123a5b1b8e, 18, false)
 	ControlModule::set_rumble()
 }
 frame(Frame=19)
 if(is_excute){
