diff --git "a/hter_5.0.0/miifighter/game_SpecialLw3Catch.txt" "b/hter_6.0.0/miifighter/game_SpecialLw3Catch.txt"
index 26179fa..c4ba92a 100644
--- "a/hter_5.0.0/miifighter/game_SpecialLw3Catch.txt"
+++ "b/hter_6.0.0/miifighter/game_SpecialLw3Catch.txt"
@@ -1,11 +1,11 @@
 if(is_excute){
 	WHOLE_HIT(HIT_STATUS_XLU)
 }
 frame(Frame=4)
 if(is_excute){
-	SEARCH(0, 0, top, 8.5, 0.0, 5.0, 9.0, 0.0, 5.0, 1.0, COLLISION_KIND_MASK_HIT, HIT_STATUS_MASK_NORMAL, 0, COLLISION_SITUATION_MASK_GA, COLLISION_CATEGORY_MASK_FIEB, COLLISION_PART_MASK_ALL, False)
+	SEARCH(0, 0, top, 8.5, 0.0, 5.0, 9.0, 0.0, 5.0, 7.5, COLLISION_KIND_MASK_HIT, HIT_STATUS_MASK_NORMAL, 0, COLLISION_SITUATION_MASK_GA, COLLISION_CATEGORY_MASK_FIEB, COLLISION_PART_MASK_ALL, False)
 }
 frame(Frame=6)
 if(is_excute){
 	search(MA_MSC_CMD_SEARCH_SEARCH_SCH_CLR_ALL)
 }
