diff --git "a/3.0.0/mario/game_FinalAir.txt" "b/3.0.1/mario/game_FinalAir.txt"
index 70bbb2c..0ed24cb 100644
--- "a/3.0.0/mario/game_FinalAir.txt"
+++ "b/3.0.1/mario/game_FinalAir.txt"
@@ -16,11 +16,11 @@ if(0xe1ba0(false, false)){
 		FT_START_CUTIN()
 	}
 	else{
 		if(is_excute){
 			PostureModule::scale(3, 0)
-			0xe1ba0(-534644470, 2.05)
+			0xe1ba0(-1233724739, 2.05)
 			CAM_ZOOM_IN_arg5(0, 0)
 			FT_START_CUTIN()
 		}
 		get_value_float(SO_VAR_FLOAT_LR)
 		if(0xe1ba0(0, 0)){
