diff --git "a/k_10.0.0/toonlink/game_AttackAirLwOffGravity.txt" "b/k_10.1.0/toonlink/game_AttackAirLwOffGravity.txt"
index 214f449..8ea0a6c 100644
--- "a/k_10.0.0/toonlink/game_AttackAirLwOffGravity.txt"
+++ "b/k_10.1.0/toonlink/game_AttackAirLwOffGravity.txt"
@@ -1,6 +1,6 @@
 get_value_int(SO_VAR_INT_PREV_STATUS)
-if(0x120860(0, FIGHTER_STATUS_KIND_DAMAGE_FLY)){
+if(0x120d40(0, FIGHTER_STATUS_KIND_DAMAGE_FLY)){
 	if(is_excute){
 		WorkModule::on_flag(Flag=FIGHTER_STATUS_WORK_ID_FLAG_RESERVE_GRAVITY_STABLE_UNABLE)
 	}
 }
