diff --git "a/13.0.0/pikmin_pikmin/game_SpNTakenOutStart_Y.txt" "b/13.0.1/pikmin_pikmin/game_SpNTakenOutStart_Y.txt"
index 640bdd1..6b2a3c5 100644
--- "a/13.0.0/pikmin_pikmin/game_SpNTakenOutStart_Y.txt"
+++ "b/13.0.1/pikmin_pikmin/game_SpNTakenOutStart_Y.txt"
@@ -1,9 +1,9 @@
 frame(Frame=1)
 if(is_excute){
 	WorkModule::get_float(WEAPON_PIKMIN_PIKMIN_STATUS_PULL_OUT_START_WORK_FLOAT_MOT_RATE)
-	MotionModule::set_rate(FSR=-353523446)
+	MotionModule::set_rate(FSR=-378992935)
 }
 frame(Frame=5)
 if(is_excute){
 	MotionModule::set_rate(FSR=1)
 }
