diff --git "a/10.0.0/donkey/game_FinalHit.txt" "b/10.1.0/donkey/game_FinalHit.txt"
index d413b77..b2e33bb 100644
--- "a/10.0.0/donkey/game_FinalHit.txt"
+++ "b/10.1.0/donkey/game_FinalHit.txt"
@@ -5,18 +5,18 @@ if(method.lib::L2CValue.operator___lib::L2CValueconst__const(WorkModule::is_flag
 	if(method.lib::L2CValue.operator___lib::L2CValueconst__const(WorkModule::is_flag(FIGHTER_DONKEY_STATUS_FINAL_FLAG_HIT_ALL_OPPONENT), false, true)){
 		get_value_float(SO_VAR_FLOAT_LR)
 		if(0x11e000(0, 0)){
 			if(is_excute){
 				PostureModule::scale(1, 0)
-				0x11e000(-1263937947, 1.5)
+				0x11e000(1202330612, 1.5)
 				CAM_ZOOM_IN_arg5(-0.1, -0.3)
 				camera(MA_MSC_CMD_CAMERA_CAM_OFFSET, -20, 3)
 			}
 			else{
 				if(is_excute){
 					PostureModule::scale(1, 0)
-					0x11e000(-1263937947, 1.5)
+					0x11e000(1202330612, 1.5)
 					CAM_ZOOM_IN_arg5(-0.1, 0.3)
 					camera(MA_MSC_CMD_CAMERA_CAM_OFFSET, 20, 3)
 				}
 			}
 		}
