diff --git "a/t_13.0.0/duckhunt/game_FinalStart_Com.txt" "b/t_13.0.1/duckhunt/game_FinalStart_Com.txt"
index d626785..85ce230 100644
--- "a/t_13.0.0/duckhunt/game_FinalStart_Com.txt"
+++ "b/t_13.0.1/duckhunt/game_FinalStart_Com.txt"
@@ -28,18 +28,18 @@ else{
 get_value_float(SO_VAR_FLOAT_LR)
 if(0x119a00(0, 0)){
 	if(is_excute){
 		camera(MA_MSC_CMD_CAMERA_CAM_OFFSET, -10, 0)
 		PostureModule::scale(4, 0)
-		0x119a00(1466293516, 2)
+		0x119a00(1942401420, 2)
 		CAM_ZOOM_IN_arg5(0, 0)
 	}
 	else{
 		if(is_excute){
 			camera(MA_MSC_CMD_CAMERA_CAM_OFFSET, 10, 0)
 			PostureModule::scale(4, 0)
-			0x119a00(1466293516, 2)
+			0x119a00(1942401420, 2)
 			CAM_ZOOM_IN_arg5(0, 0)
 		}
 	}
 }
 frame(Frame=12)
