diff --git "a/n_5.0.0/gaogaen/game_SpecialAirLwStart.txt" "b/n_6.0.0/gaogaen/game_SpecialAirLwStart.txt"
index 7766956..98dd88d 100644
--- "a/n_5.0.0/gaogaen/game_SpecialAirLwStart.txt"
+++ "b/n_6.0.0/gaogaen/game_SpecialAirLwStart.txt"
@@ -3,16 +3,19 @@ FT_MOTION_RATE(FSM=0.25)
 frame(Frame=8)
 if(is_excute){
 	WorkModule__on_flag(Flag=FIGHTER_GAOGAEN_STATUS_SPECIAL_LW_FLAG_STANCE_START)
 }
 frame(Frame=9)
-FT_MOTION_RATE(FSM=1.2)
-frame(Frame=14)
+FT_MOTION_RATE(FSM=1.5)
+frame(Frame=21)
 FT_MOTION_RATE(FSM=1)
 frame(Frame=28)
 if(is_excute){
 	WorkModule__on_flag(Flag=FIGHTER_GAOGAEN_STATUS_SPECIAL_LW_FLAG_STANCE_END)
 }
-frame(Frame=35)
+FT_MOTION_RATE(FSM=0.5)
+frame(Frame=32)
 if(is_excute){
 	notify_event_msc_cmd(0x2127e37c07, GROUND_CLIFF_CHECK_KIND_ALWAYS)
 }
+frame(Frame=38)
+FT_MOTION_RATE(FSM=1)
