diff --git "a/1.0/ike/game_FinalStart.txt" "b/0.0/ike/game_FinalStart.txt"
index 0ccb579..b255f40 100644
--- "a/1.0/ike/game_FinalStart.txt"
+++ "b/0.0/ike/game_FinalStart.txt"
@@ -20,11 +20,11 @@ if(is_excute){
 	SlowModule::set_whole(20, 0)
 }
 else{
 	if(is_excute){
 		PostureModule::scale(5, 0)
-		0x140320(-880976900, 2)
+		0x140320(1857541619, 2)
 		CAM_ZOOM_IN_arg5(0, 0)
 		FT_START_CUTIN()
 	}
 }
 frame(Frame=9)
