diff --git "a/12.0.0/pickel/game_EntryR.txt" "b/13.0.0/pickel/game_EntryR.txt"
index 07d65ea..5de94b5 100644
--- "a/12.0.0/pickel/game_EntryR.txt"
+++ "b/13.0.0/pickel/game_EntryR.txt"
@@ -1,11 +1,11 @@
 if(is_excute){
 	WorkModule::set_int(FIGHTER_PICKEL_CRAFT_WEAPON_KIND_PICK, FIGHTER_PICKEL_INSTANCE_WORK_ID_INT_REQUEST_HAVE_CRAFT_WEAPON_KIND)
 }
 frame(Frame=90)
 WorkModule::is_flag(FIGHTER_PICKEL_INSTANCE_WORK_ID_FLAG_IS_SUPER_SUDDEN_DEATH)
-if(0x36c760(false, true)){
+if(0x370800(false, true)){
 	if(is_excute){
 		battle_object_module_accessor()
 		methodlib::L2CValue::L2CValue(void*)()
 	}
 	else{
@@ -14,11 +14,11 @@ if(0x36c760(false, true)){
 }
 methodlib::L2CValue::as_pointer()const()
 create_table()
 frame(Frame=118)
 WorkModule::is_flag(FIGHTER_PICKEL_INSTANCE_WORK_ID_FLAG_IS_SUPER_SUDDEN_DEATH)
-if(0x36c760(false, false)){
+if(0x370800(false, false)){
 	if(is_excute){
 		battle_object_module_accessor()
 		methodlib::L2CValue::L2CValue(void*)()
 	}
 	else{
