diff --git "a/11.0.0/pickel_trolley/game_SpecialSWallM.txt" "b/12.0.0/pickel_trolley/game_SpecialSWallM.txt"
index 5ea0262..659a48c 100644
--- "a/11.0.0/pickel_trolley/game_SpecialSWallM.txt"
+++ "b/12.0.0/pickel_trolley/game_SpecialSWallM.txt"
@@ -1,5 +1,5 @@
 if(is_excute){
 	methodlib::L2CValue::as_hash()const(hash40("rbkind_nohitl"), 0, false)
 	ControlModule::set_rumble()
-	WorkModule::off_flag(Flag=60256)
+	WorkModule::off_flag(Flag=61284)
 }
