diff --git "a/o_5.0.0/lucario/game_FinalAirStart.txt" "b/o_6.0.0/lucario/game_FinalAirStart.txt"
index ecbb88e..73aacd3 100644
--- "a/o_5.0.0/lucario/game_FinalAirStart.txt"
+++ "b/o_6.0.0/lucario/game_FinalAirStart.txt"
@@ -5,19 +5,19 @@ if(is_excute){
 if(method.lib::L2CValue.operator___lib::L2CValueconst__const(WorkModule__is_flag(FIGHTER_INSTANCE_WORK_ID_FLAG_DISABLE_FINAL_START_CAMERA), False, False)){
 	frame(Frame=10)
 	if(is_excute){
 		FT_SET_FINAL_FEAR_FACE(60)
 		REQ_FINAL_START_CAMERA(d04finalstart.nuanmb, False)
-		FT_START_CUTIN()
 	}
 	else{
 		if(is_excute){
 			camera(MA_MSC_CMD_CAMERA_CAM_RECT, 0, 4, 6, 0)
 			PostureModule__scale(3, 0)
-			0xfd1c0(2100879093, 2)
+			0xfd1f0(1582018871, 2)
 			CAM_ZOOM_IN_arg5(0, 0)
 			camera(MA_MSC_CMD_CAMERA_CAM_OFFSET, 0, 6)
+			FT_START_CUTIN()
 		}
 	}
 }
 frame(Frame=20)
 if(is_excute){
