diff --git "a/on_10.0.0/plizardon/game_FinalAir.txt" "b/on_10.1.0/plizardon/game_FinalAir.txt"
index 6ca6e65..1d06b2f 100644
--- "a/on_10.0.0/plizardon/game_FinalAir.txt"
+++ "b/on_10.1.0/plizardon/game_FinalAir.txt"
@@ -10,11 +10,11 @@ if(method.lib::L2CValue.operator___lib::L2CValueconst__const(WorkModule::is_flag
 		FT_START_CUTIN_arg1(true)
 	}
 	else{
 		if(is_excute){
 			PostureModule::scale(4, 0)
-			0xdf650(2056057233, 2.1)
+			0xdf650(-696384609, 2.1)
 			CAM_ZOOM_IN_arg5(0, 0)
 			FT_START_CUTIN_arg1(true)
 		}
 		frame(Frame=25)
 		if(is_excute){
