diff --git "a/_5.0.0/pikmin/game_SpecialNStart.txt" "b/_6.0.0/pikmin/game_SpecialNStart.txt"
index cf1086d..3917200 100644
--- "a/_5.0.0/pikmin/game_SpecialNStart.txt"
+++ "b/_6.0.0/pikmin/game_SpecialNStart.txt"
@@ -2,11 +2,11 @@ if(is_excute){
 	ArticleModule__generate_article(FIGHTER_PIKMIN_GENERATE_ARTICLE_PIKMIN)
 }
 frame(Frame=3)
 if(is_excute){
 	WorkModule__get_float(FIGHTER_PIKMIN_STATUS_PULL_OUT_WORK_FLOAT_MOT_RATE)
-	MotionModule__set_rate(463762201)
+	MotionModule__set_rate(-1298307000)
 }
 frame(Frame=6)
 if(is_excute){
 	MotionModule__set_rate(1)
 }
