diff --git "a/11.0.0/pacman/game_Win3.txt" "b/12.0.0/pacman/game_Win3.txt"
index f67e33b..c5b0e45 100644
--- "a/11.0.0/pacman/game_Win3.txt"
+++ "b/12.0.0/pacman/game_Win3.txt"
@@ -1,7 +1,7 @@
 WorkModule::is_flag(FIGHTER_INSTANCE_WORK_ID_FLAG_RESULT_FIGHTER_SINGLE)
-if(0x11a050(false, false)){
+if(0x11a0d0(false, false)){
 	if(is_excute){
 		ArticleModule::generate_article(FIGHTER_PACMAN_GENERATE_ARTICLE_ARTISTICPOINT)
 		methodlib::L2CValue::as_hash()const(FIGHTER_PACMAN_GENERATE_ARTICLE_ARTISTICPOINT, hash40("win3"))
 		ArticleModule::change_motion()
 		ArticleModule::generate_article(FIGHTER_PACMAN_GENERATE_ARTICLE_FAIRY)
