diff --git "a/g_7.0.0/inkling/game_SpecialSTurn.txt" "b/g_8.0.0/inkling/game_SpecialSTurn.txt"
index 123502b..48be3b8 100644
--- "a/g_7.0.0/inkling/game_SpecialSTurn.txt"
+++ "b/g_8.0.0/inkling/game_SpecialSTurn.txt"
@@ -1,7 +1,7 @@
 if(is_excute){
-	FighterAreaModuleImpl::enable_fix_jostle_area(5, 8)
+	FighterAreaModuleImpl::enable_fix_jostle_area(3, 8)
 }
 frame(Frame=10)
 if(is_excute){
 	WorkModule::on_flag(Flag=FIGHTER_INKLING_STATUS_SPECIAL_S_FLAG_TURN)
 }
