diff --git "a/g_7.0.0/inkling/game_Dash.txt" "b/g_8.0.0/inkling/game_Dash.txt"
index a6c6bb3..7a09805 100644
--- "a/g_7.0.0/inkling/game_Dash.txt"
+++ "b/g_8.0.0/inkling/game_Dash.txt"
@@ -1,10 +1,10 @@
 if(is_excute){
 	VisibilityModule::set_whole(false)
 }
 get_value_float(SO_VAR_FLOAT_LR)
-if(0x13cfa0(0, 0)){
+if(0x13d0e0(0, 0)){
 	if(is_excute){
 		WorkModule::on_flag(Flag=FIGHTER_INKLING_INSTANCE_WORK_ID_FLAG_NO_FLIP_SQUID)
 	}
 }
 if(method.lib::L2CValue.operator___lib::L2CValueconst__const(WorkModule::is_flag(FIGHTER_INKLING_INSTANCE_WORK_ID_FLAG_EXIST_SQUID), false, false)){
@@ -12,12 +12,12 @@ if(method.lib::L2CValue.operator___lib::L2CValueconst__const(WorkModule::is_flag
 		ArticleModule::generate_article(FIGHTER_INKLING_GENERATE_ARTICLE_SQUID)
 	}
 }
 MotionModule::motion_kind()
 MotionModule::frame(hash40("0x00000002b0"))
-MotionModule::rate(271506559)
-as_hash__const(271506559, FIGHTER_INKLING_GENERATE_ARTICLE_SQUID, false)
+MotionModule::rate(780876990)
+as_hash__const(780876990, FIGHTER_INKLING_GENERATE_ARTICLE_SQUID, false)
 ArticleModule::change_motion()
 if(is_excute){
 	ArticleModule::set_frame(FIGHTER_INKLING_GENERATE_ARTICLE_SQUID)
 	ArticleModule::set_rate(FIGHTER_INKLING_GENERATE_ARTICLE_SQUID)
 	ArticleModule::set_visibility_whole(FIGHTER_INKLING_GENERATE_ARTICLE_SQUID, false)
@@ -25,11 +25,11 @@ if(is_excute){
 	StatusModule::status_kind()
 	operator___lib::L2CValueconst__const(0, FIGHTER_STATUS_KIND_REBIRTH)
 	WorkModule::on_flag(Flag=FIGHTER_INSTANCE_WORK_ID_FLAG_FORCE_LOUPE)
 }
 get_value_float(0, SO_VAR_FLOAT_LR)
-if(0x13cfa0(0, 0)){
+if(0x13d0e0(0, 0)){
 	if(is_excute){
 		as_hash__const(FIGHTER_INKLING_GENERATE_ARTICLE_SQUID, hash40("dash_l"))
 		ArticleModule::change_motion()
 	}
 	else{
