diff --git "a/ner_5.0.0/miigunner/game_FinalStart.txt" "b/ner_6.0.0/miigunner/game_FinalStart.txt"
index c5044f0..99e9c02 100644
--- "a/ner_5.0.0/miigunner/game_FinalStart.txt"
+++ "b/ner_6.0.0/miigunner/game_FinalStart.txt"
@@ -2,25 +2,25 @@ if(is_excute){
 	CHECK_VALID_FINAL_START_CAMERA(0, 7, 20, 0, 0, 0)
 	SLOW_OPPONENT(8, 70)
 	ArticleModule__generate_article(FIGHTER_MIIGUNNER_GENERATE_ARTICLE_LASER)
 	ArticleModule__generate_article(FIGHTER_MIIGUNNER_GENERATE_ARTICLE_FULLTHROTTLE)
 }
-if(0x12fc90(WorkModule__is_flag(FIGHTER_INSTANCE_WORK_ID_FLAG_DISABLE_FINAL_START_CAMERA), False, False)){
+if(0x12fcd0(WorkModule__is_flag(FIGHTER_INSTANCE_WORK_ID_FLAG_DISABLE_FINAL_START_CAMERA), False, False)){
 	frame(Frame=10)
 	if(is_excute){
 		FT_SET_FINAL_FEAR_FACE(60)
 	}
 }
 get_value_float(SO_VAR_FLOAT_LR)
-if(0x12fc90(1649568, 0)){
+if(0x12fcd0(1649632, 0)){
 	if(is_excute){
 		REQ_FINAL_START_CAMERA_arg3(0x1654ea70d3, True, False)
 	}
 	else{
 		if(is_excute){
 			PostureModule__scale(3, 0)
-			0x12fc90(-722815592, 2.2)
+			0x12fcd0(-98523339, 2.2)
 			CAM_ZOOM_IN_arg5(0, 0)
 		}
 		else{
 			if(is_excute){
 				REQ_FINAL_START_CAMERA_arg3(d04finalstart.nuanmb, True, False)
