diff --git "a/_10.1.0/richter/game_FinalAirStart.txt" "b/_11.0.0/richter/game_FinalAirStart.txt"
index 1f159aa..8260791 100644
--- "a/_10.1.0/richter/game_FinalAirStart.txt"
+++ "b/_11.0.0/richter/game_FinalAirStart.txt"
@@ -15,11 +15,11 @@ if(0x1155f0(32, 0)){
 		REQ_FINAL_START_CAMERA_arg3(hash40("d04finalstart02.nuanmb"), false, false)
 	}
 	else{
 		if(is_excute){
 			PostureModule::scale(5, 0)
-			0x1155f0(-1225610801, 2.1)
+			0x1155f0(2010766555, 2.1)
 			CAM_ZOOM_IN_arg5(0, 0)
 		}
 		else{
 			if(is_excute){
 				REQ_FINAL_START_CAMERA_arg3(hash40("d04finalstart.nuanmb"), false, false)
@@ -47,9 +47,10 @@ if(is_excute){
 wait(Frames=15)
 if(is_excute){
 	AttackModule::clear_all()
 }
 wait(Frames=1)
-operator___lib::L2CValueconst__const(WorkModule::is_flag(FIGHTER_SIMON_INSTANCE_WORK_ID_FLAG_FINAL_HIT), false, true)
-if(is_excute){
-	WorkModule::on_flag(Flag=FIGHTER_SIMON_STATUS_FINAL_WORK_ID_FLAG_START_COFFIN_ROT_Y)
+if(method.lib::L2CValue.operator___lib::L2CValueconst__const(WorkModule::is_flag(FIGHTER_SIMON_INSTANCE_WORK_ID_FLAG_FINAL_HIT), false, true)){
+	if(is_excute){
+		WorkModule::on_flag(Flag=FIGHTER_SIMON_STATUS_FINAL_WORK_ID_FLAG_START_COFFIN_ROT_Y)
+	}
 }
