diff --git "a/5.0.0/snake/game_FinalAirStart.txt" "b/6.0.0/snake/game_FinalAirStart.txt"
index d8d2663..77a33be 100644
--- "a/5.0.0/snake/game_FinalAirStart.txt"
+++ "b/6.0.0/snake/game_FinalAirStart.txt"
@@ -6,18 +6,18 @@ if(method.lib::L2CValue.operator___lib::L2CValueconst__const(WorkModule__is_flag
 	if(is_excute){
 		FT_SET_FINAL_FEAR_FACE(60)
 	}
 }
 get_value_float(SO_VAR_FLOAT_LR)
-if(0x113ce0(0, 0)){
+if(0x113ed0(0, 0)){
 	if(is_excute){
 		REQ_FINAL_START_CAMERA(0x1654ea70d3, False)
 	}
 	else{
 		if(is_excute){
 			PostureModule__scale(5, 0)
-			0x113ce0(424000171, 2)
+			0x113ed0(-701845496, 2)
 			CAM_ZOOM_IN_arg5(0, 0)
 		}
 		else{
 			if(is_excute){
 				REQ_FINAL_START_CAMERA(d04finalstart.nuanmb, False)
@@ -46,10 +46,10 @@ if(method.lib::L2CValue.operator___lib::L2CValueconst__const(WorkModule__is_flag
 frame(Frame=26)
 if(is_excute){
 	SlowModule__clear_whole()
 }
 else{
-	frame(Frame=424000171)
+	frame(Frame=-701845496)
 	if(is_excute){
 		CAM_ZOOM_OUT()
 	}
 }
