diff --git "a/7.0.0/chrom/game_FinalAirStart.txt" "b/8.0.0/chrom/game_FinalAirStart.txt"
index 337422e..61f140e 100644
--- "a/7.0.0/chrom/game_FinalAirStart.txt"
+++ "b/8.0.0/chrom/game_FinalAirStart.txt"
@@ -10,11 +10,11 @@ if(method.lib::L2CValue.operator___lib::L2CValueconst__const(WorkModule::is_flag
 		REQ_FINAL_START_CAMERA(hash40("d04finalstart.nuanmb"), false)
 	}
 	else{
 		if(is_excute){
 			PostureModule::scale(3, 0)
-			0x12b910(-981456494, 2)
+			0x12ba50(503437126, 2)
 			CAM_ZOOM_IN_arg5(0, 0)
 			FT_START_CUTIN()
 		}
 	}
 }
