diff --git "a/6.0.0/chrom/game_FinalAirStart.txt" "b/7.0.0/chrom/game_FinalAirStart.txt"
index 394d29c..fb3a4bf 100644
--- "a/6.0.0/chrom/game_FinalAirStart.txt"
+++ "b/7.0.0/chrom/game_FinalAirStart.txt"
@@ -10,11 +10,11 @@ if(method.lib::L2CValue.operator___lib::L2CValueconst__const(WorkModule__is_flag
 		REQ_FINAL_START_CAMERA(d04finalstart.nuanmb, False)
 	}
 	else{
 		if(is_excute){
 			PostureModule__scale(3, 0)
-			0x12b900(954636093, 2)
+			0x12b910(-981456494, 2)
 			CAM_ZOOM_IN_arg5(0, 0)
 			FT_START_CUTIN()
 		}
 	}
 }
