diff --git "a/ta_11.0.0/bayonetta/game_AttackS4Lw.txt" "b/ta_12.0.0/bayonetta/game_AttackS4Lw.txt"
index fba1fee..25f749d 100644
--- "a/ta_11.0.0/bayonetta/game_AttackS4Lw.txt"
+++ "b/ta_12.0.0/bayonetta/game_AttackS4Lw.txt"
@@ -4,11 +4,11 @@ frame(Frame=3)
 if(is_excute){
 	WorkModule::on_flag(Flag=FIGHTER_STATUS_ATTACK_FLAG_START_SMASH_HOLD)
 }
 execute(3)
 WorkModule::is_flag(FIGHTER_STATUS_ATTACK_FLAG_SMASH_SMASH_HOLD_TO_ATTACK)
-if(0x1cb230(false, true)){
+if(0x1cb400(false, true)){
 	if(is_excute){
 		ArticleModule::generate_article(FIGHTER_BAYONETTA_GENERATE_ARTICLE_WICKEDWEAVEARM)
 	}
 }
 if(is_excute){
