diff --git "a/tta_6.0.0/bayonetta/game_ShootingArmR_AtkOff_AttackHi3.txt" "b/tta_7.0.0/bayonetta/game_ShootingArmR_AtkOff_AttackHi3.txt"
index 9b08ec3..30708af 100644
--- "a/tta_6.0.0/bayonetta/game_ShootingArmR_AtkOff_AttackHi3.txt"
+++ "b/tta_7.0.0/bayonetta/game_ShootingArmR_AtkOff_AttackHi3.txt"
@@ -1,6 +1,6 @@
-if(0x190590(WorkModule__is_flag(FIGHTER_BAYONETTA_INSTANCE_WORK_ID_FLAG_SHOOTING_DISABLE_ROOT_ATTACK), False, False)){
+if(0x1905a0(WorkModule__is_flag(FIGHTER_BAYONETTA_INSTANCE_WORK_ID_FLAG_SHOOTING_DISABLE_ROOT_ATTACK), False, False)){
 	if(is_excute){
 		AttackModule__clear(ID=FIGHTER_BAYONETTA_SHOOTING_ATTACK_ID_00)
 	}
 }
 if(is_excute){
