diff --git "a/tta_6.0.0/bayonetta/game_AttackLw4.txt" "b/tta_7.0.0/bayonetta/game_AttackLw4.txt"
index 969e495..d665944 100644
--- "a/tta_6.0.0/bayonetta/game_AttackLw4.txt"
+++ "b/tta_7.0.0/bayonetta/game_AttackLw4.txt"
@@ -1,11 +1,11 @@
 frame(Frame=9)
 execute(9)
 if(is_excute){
 	WorkModule__on_flag(Flag=FIGHTER_STATUS_ATTACK_FLAG_START_SMASH_HOLD)
 }
-if(0x190590(WorkModule__is_flag(FIGHTER_STATUS_ATTACK_FLAG_SMASH_SMASH_HOLD_TO_ATTACK), False, True)){
+if(0x1905a0(WorkModule__is_flag(FIGHTER_STATUS_ATTACK_FLAG_SMASH_SMASH_HOLD_TO_ATTACK), False, True)){
 	if(is_excute){
 		ArticleModule__generate_article(FIGHTER_BAYONETTA_GENERATE_ARTICLE_WICKEDWEAVELEG)
 	}
 }
 frame(Frame=11)
