diff --git "a/_4.0.0/dedede/game_SpecialNSWallow.txt" "b/_5.0.0/dedede/game_SpecialNSWallow.txt"
index f12c1af..537ac80 100644
--- "a/_4.0.0/dedede/game_SpecialNSWallow.txt"
+++ "b/_5.0.0/dedede/game_SpecialNSWallow.txt"
@@ -7,10 +7,10 @@ if(is_excute){
 }
 frame(Frame=6)
 if(is_excute){
 	FighterAreaModuleImpl__enable_fix_jostle_area(14.7, 4)
 }
-if(0x148480(FIGHTER_INSTANCE_WORK_ID_INT_KIND, FIGHTER_KIND_KIRBY)){
+if(0x1486b0(FIGHTER_INSTANCE_WORK_ID_INT_KIND, FIGHTER_KIND_KIRBY)){
 	if(is_excute){
 		FighterAreaModuleImpl__enable_fix_jostle_area(4, 4)
 	}
 }
