diff --git "a/3.1.0/purin/game_FinalStartR.txt" "b/4.0.0/purin/game_FinalStartR.txt"
index e83b79f..4dc137f 100644
--- "a/3.1.0/purin/game_FinalStartR.txt"
+++ "b/4.0.0/purin/game_FinalStartR.txt"
@@ -1,19 +1,19 @@
 if(is_excute){
 	WHOLE_HIT(HIT_STATUS_XLU)
 	CHECK_VALID_FINAL_START_CAMERA(0, 7, 20, 0, 0, 0)
 	SLOW_OPPONENT(3, 120)
 }
-if(0xf3a40(WorkModule__is_flag(FIGHTER_INSTANCE_WORK_ID_FLAG_DISABLE_FINAL_START_CAMERA), False, False)){
+if(0xf3b80(WorkModule__is_flag(FIGHTER_INSTANCE_WORK_ID_FLAG_DISABLE_FINAL_START_CAMERA), False, False)){
 	if(is_excute){
 		FT_SET_FINAL_FEAR_FACE(60)
 		REQ_FINAL_START_CAMERA(0x158756dcc0, False)
 	}
 	else{
 		if(is_excute){
 			PostureModule__scale(5, 0)
-			0xf3a40(-1944671949, 2.1)
+			0xf3b80(-59272755, 2.1)
 			CAM_ZOOM_IN_arg5(0, 0)
 			FT_START_CUTIN()
 		}
 	}
 }
