diff --git "a/6.0.0/yoshi/game_GuardOn.txt" "b/7.0.0/yoshi/game_GuardOn.txt"
index 94f1878..cb78ca7 100644
--- "a/6.0.0/yoshi/game_GuardOn.txt"
+++ "b/7.0.0/yoshi/game_GuardOn.txt"
@@ -1,20 +1,25 @@
+frame(Frame=1)
 if(is_excute){
 	HIT_NODE(head, HIT_STATUS_XLU)
 	HIT_NODE(snout, HIT_STATUS_XLU)
 	HIT_NODE(armr, HIT_STATUS_XLU)
 	HIT_NODE(arml, HIT_STATUS_XLU)
+	HIT_NODE(toer, HIT_STATUS_XLU)
+	HIT_NODE(toel, HIT_STATUS_XLU)
 }
 frame(Frame=5)
 if(is_excute){
-	HIT_NODE(head, HIT_STATUS_NORMAL)
-	HIT_NODE(snout, HIT_STATUS_NORMAL)
 	HIT_NODE(armr, HIT_STATUS_NORMAL)
 	HIT_NODE(arml, HIT_STATUS_NORMAL)
 	HIT_NODE(hip, HIT_STATUS_XLU)
 	HIT_NODE(tail2, HIT_STATUS_XLU)
 }
 frame(Frame=7)
 if(is_excute){
+	HIT_NODE(head, HIT_STATUS_NORMAL)
+	HIT_NODE(snout, HIT_STATUS_NORMAL)
+	HIT_NODE(toer, HIT_STATUS_NORMAL)
+	HIT_NODE(toel, HIT_STATUS_NORMAL)
 	HIT_NODE(hip, HIT_STATUS_NORMAL)
 	HIT_NODE(tail2, HIT_STATUS_NORMAL)
 }
