diff --git "a/11.0.0/pickel_plate/game_Push.txt" "b/12.0.0/pickel_plate/game_Push.txt"
index 2a60370..aca65ea 100644
--- "a/11.0.0/pickel_plate/game_Push.txt"
+++ "b/12.0.0/pickel_plate/game_Push.txt"
@@ -1,6 +1,6 @@
 frame(Frame=2)
 if(is_excute){
-	WorkModule::on_flag(Flag=55700)
+	WorkModule::on_flag(Flag=55884)
 	methodlib::L2CValue::as_hash()const(0x113697d0ae, 3, false)
 	ControlModule::set_rumble()
 }
