diff --git "a/.0.0/popo/game_SpecialHiCheckLRSub.txt" "b/.0.0/popo/game_SpecialHiCheckLRSub.txt"
index fa41aab..22bdd6b 100644
--- "a/.0.0/popo/game_SpecialHiCheckLRSub.txt"
+++ "b/.0.0/popo/game_SpecialHiCheckLRSub.txt"
@@ -1,8 +1,8 @@
 get_value_float(FT_VAR_FLOAT_STICK_X_BACK)
 WorkModule::get_param_float(0, hash40("param_special_hi"), hash40("start_turn_cont_x"))
-if(0x161680(32)){
+if(0x161860(32)){
 	if(is_excute){
 		STICK_LR()
 		UPDATE_ROT()
 	}
 }
