diff --git "a/0.0/ike/game_FinalAirStart.txt" "b/0.1/ike/game_FinalAirStart.txt"
index 3746fc9..5dd1ed2 100644
--- "a/0.0/ike/game_FinalAirStart.txt"
+++ "b/0.1/ike/game_FinalAirStart.txt"
@@ -21,11 +21,11 @@ if(is_excute){
 	SlowModule::set_whole(20, 0)
 }
 else{
 	if(is_excute){
 		PostureModule::scale(5, 0)
-		0x140400(1354334974, 2)
+		0x149b00(1738818520, 2)
 		CAM_ZOOM_IN_arg5(0, 0)
 		FT_START_CUTIN()
 	}
 }
 frame(Frame=9)
