diff --git "a/ch_10.0.0/gamewatch/game_FinalAirStart.txt" "b/ch_10.1.0/gamewatch/game_FinalAirStart.txt"
index 15a6259..1e07fd6 100644
--- "a/ch_10.0.0/gamewatch/game_FinalAirStart.txt"
+++ "b/ch_10.1.0/gamewatch/game_FinalAirStart.txt"
@@ -1,22 +1,22 @@
 if(is_excute){
 	SLOW_OPPONENT(10, 60)
 	FT_REMOVE_FINAL_AURA(true)
 	FT_SET_FINAL_FEAR_FACE(70)
 	PostureModule::scale(1, 0)
-	0x101e40(651189197, 2.1)
+	0x101e40(2062052874, 2.1)
 	CAM_ZOOM_IN_arg5(0, 0)
 }
 frame(Frame=5)
 if(is_excute){
 	FT_START_CUTIN()
 }
 frame(Frame=12)
 if(is_excute){
 	CAM_ZOOM_OUT()
 	PostureModule::scale(15, 0)
-	0x101e40(651189197, 3.5)
+	0x101e40(2062052874, 3.5)
 	CAM_ZOOM_IN_arg5(0, 0)
 }
 get_value_float(SO_VAR_FLOAT_LR)
 if(0x101e40(0, 0)){
 	if(is_excute){
