diff --git "a/.0.0/jack_fire2/game_Hit.txt" "b/.1.0/jack_fire2/game_Hit.txt"
index 01000de..907e41a 100644
--- "a/.0.0/jack_fire2/game_Hit.txt"
+++ "b/.1.0/jack_fire2/game_Hit.txt"
@@ -1,9 +1,9 @@
 frame(Frame=1)
 if(is_excute){
 	QUAKE(CAMERA_QUAKE_KIND_S)
-	ATTACK(ID=0, Part=0, Bone=top, Damage=0.5, Angle=367, KBG=100, FKB=0, BKB=5, Size=4.0, X=0.0, Y=1.5, Z=0.0, X2=0.0, Y2=8.0, Z2=0.0, Hitlag=0.3, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=5, Reflectable=True, Absorbable=True, Flinchless=False, DisableHitlag=False, Direct/Indirect=False, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=0x1b2fb009bf, SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=COLLISION_SOUND_ATTR_FIRE, Type=ATTACK_REGION_MAGIC)
+	ATTACK(ID=0, Part=0, Bone=top, Damage=0.5, Angle=367, KBG=100, FKB=0, BKB=5, Size=4.0, X=0.0, Y=1.5, Z=0.0, X2=0.0, Y2=8.0, Z2=0.0, Hitlag=0.3, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_B, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=5, Reflectable=True, Absorbable=True, Flinchless=False, DisableHitlag=False, Direct/Indirect=False, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=0x1b2fb009bf, SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=COLLISION_SOUND_ATTR_FIRE, Type=ATTACK_REGION_MAGIC)
 	AttackModule__set_poison_param(ID=0, Frames=321, Rehit=40, Damage=1.5, Unk=False)
 }
 frame(Frame=5)
 if(is_excute){
 	as_hash__const(rbkind_explosion, 0, False)
