diff --git "a/gisou_4.0.0/pfushigisou/game_FinalAir.txt" "b/gisou_5.0.0/pfushigisou/game_FinalAir.txt"
index 068a804..fd0a564 100644
--- "a/gisou_4.0.0/pfushigisou/game_FinalAir.txt"
+++ "b/gisou_5.0.0/pfushigisou/game_FinalAir.txt"
@@ -11,11 +11,11 @@ if(0x1035f0(WorkModule__is_flag(FIGHTER_INSTANCE_WORK_ID_FLAG_DISABLE_FINAL_STAR
 		FT_START_CUTIN_arg1(True)
 	}
 	else{
 		if(is_excute){
 			PostureModule__scale(4, 0)
-			0x1035f0(-627980135, 2.1)
+			0x1035f0(-389617846, 2.1)
 			CAM_ZOOM_IN_arg5(0, 0)
 			FT_START_CUTIN_arg1(True)
 		}
 		frame(Frame=25)
 		if(is_excute){
