diff --git "a/3.1.0/kamui/game_Final.txt" "b/4.0.0/kamui/game_Final.txt"
index 6548ddb..5190b3a 100644
--- "a/3.1.0/kamui/game_Final.txt"
+++ "b/4.0.0/kamui/game_Final.txt"
@@ -19,11 +19,11 @@ if(is_excute){
 	SlowModule__clear_whole()
 }
 else{
 	if(is_excute){
 		PostureModule__scale(4, 0)
-		0x144210(-1311087587, 2.5)
+		0x1442f0(1189412136, 2.5)
 		CAM_ZOOM_IN_arg5(0, 0)
 		camera(MA_MSC_CMD_CAMERA_CAM_OFFSET, 0, 0)
 		FT_START_CUTIN()
 	}
 }
