diff --git "a/to_7.0.0/murabito/game_AttackHi3.txt" "b/to_8.0.0/murabito/game_AttackHi3.txt"
index 792aec1..e494c32 100644
--- "a/to_7.0.0/murabito/game_AttackHi3.txt"
+++ "b/to_8.0.0/murabito/game_AttackHi3.txt"
@@ -1,11 +1,16 @@
 frame(Frame=7)
+if(is_excute){
+	HIT_NODE(hash40("head"), HIT_STATUS_XLU)
+	HIT_NODE(hash40("arml"), HIT_STATUS_XLU)
+}
 FT_MOTION_RATE(FSM=0.85)
 if(is_excute){
 	ATTACK(ID=0, Part=0, Bone=hash40("havel"), Damage=5.0, Angle=108, KBG=100, FKB=75, BKB=0, Size=3.8, X=0.0, Y=-1.0, Z=0.0, Hitlag=0.8, SDI=1.0, Clang_Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=true, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=false, Absorbable=false, Flinchless=false, DisableHitlag=false, Direct_Hitbox=true, Ground_or_Air=COLLISION_SITUATION_MASK_G, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=false, Effect=hash40("collision_attr_normal"), SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=COLLISION_SOUND_ATTR_PUNCH, Type=ATTACK_REGION_OBJECT)
 	ATTACK(ID=1, Part=0, Bone=hash40("havel"), Damage=5.0, Angle=108, KBG=100, FKB=75, BKB=0, Size=4.0, X=0.0, Y=5.0, Z=0.0, Hitlag=0.8, SDI=1.0, Clang_Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=true, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=false, Absorbable=false, Flinchless=false, DisableHitlag=false, Direct_Hitbox=true, Ground_or_Air=COLLISION_SITUATION_MASK_G, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=false, Effect=hash40("collision_attr_normal"), SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=COLLISION_SOUND_ATTR_PUNCH, Type=ATTACK_REGION_OBJECT)
 	ATTACK(ID=2, Part=0, Bone=hash40("havel"), Damage=5.0, Angle=135, KBG=10, FKB=0, BKB=35, Size=4.0, X=0.0, Y=5.0, Z=0.0, Hitlag=0.8, SDI=1.0, Clang_Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=true, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=false, Absorbable=false, Flinchless=false, DisableHitlag=false, Direct_Hitbox=true, Ground_or_Air=COLLISION_SITUATION_MASK_A, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=false, Effect=hash40("collision_attr_normal"), SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=COLLISION_SOUND_ATTR_PUNCH, Type=ATTACK_REGION_OBJECT)
+	AttackModule::set_add_reaction_frame(ID=2, Frames=2, Unk=false)
 }
 wait(Frames=14)
 if(is_excute){
 	AttackModule::clear_all()
 }
@@ -16,6 +21,8 @@ if(is_excute){
 	ATTACK(ID=1, Part=0, Bone=hash40("havel"), Damage=6.0, Angle=80, KBG=140, FKB=0, BKB=50, Size=4.0, X=0.0, Y=1.0, Z=0.0, Hitlag=1.2, SDI=1.0, Clang_Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=false, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=false, Absorbable=false, Flinchless=false, DisableHitlag=false, Direct_Hitbox=true, Ground_or_Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=false, Effect=hash40("collision_attr_normal"), SFXLevel=ATTACK_SOUND_LEVEL_M, SFXType=COLLISION_SOUND_ATTR_PUNCH, Type=ATTACK_REGION_OBJECT)
 }
 wait(Frames=4)
 if(is_excute){
 	AttackModule::clear_all()
+	HIT_NODE(hash40("head"), HIT_STATUS_NORMAL)
+	HIT_NODE(hash40("arml"), HIT_STATUS_NORMAL)
 }
