diff --git "a/r_9.0.0/richter/game_FinalStart_Com.txt" "b/_10.0.0/richter/game_FinalStart_Com.txt"
index 92ebdac..411fc08 100644
--- "a/r_9.0.0/richter/game_FinalStart_Com.txt"
+++ "b/_10.0.0/richter/game_FinalStart_Com.txt"
@@ -15,11 +15,11 @@ if(0x1155f0(32, 0)){
 		REQ_FINAL_START_CAMERA_arg3(hash40("d04finalstart02.nuanmb"), false, false)
 	}
 	else{
 		if(is_excute){
 			PostureModule::scale(5, 0)
-			0x1155f0(-1391473823, 2.1)
+			0x1155f0(-130118445, 2.1)
 			CAM_ZOOM_IN_arg5(0, 0)
 		}
 		else{
 			if(is_excute){
 				REQ_FINAL_START_CAMERA_arg3(hash40("d04finalstart.nuanmb"), false, false)
