diff --git "a/tta_5.0.0/bayonetta/game_AttackHi4.txt" "b/tta_6.0.0/bayonetta/game_AttackHi4.txt"
index 5116ab2..6250131 100644
--- "a/tta_5.0.0/bayonetta/game_AttackHi4.txt"
+++ "b/tta_6.0.0/bayonetta/game_AttackHi4.txt"
@@ -1,11 +1,11 @@
 frame(Frame=7)
 execute(7)
 if(is_excute){
 	WorkModule__on_flag(Flag=FIGHTER_STATUS_ATTACK_FLAG_START_SMASH_HOLD)
 }
-if(0x1905b0(WorkModule__is_flag(FIGHTER_STATUS_ATTACK_FLAG_SMASH_SMASH_HOLD_TO_ATTACK), False, True)){
+if(0x190590(WorkModule__is_flag(FIGHTER_STATUS_ATTACK_FLAG_SMASH_SMASH_HOLD_TO_ATTACK), False, True)){
 	if(is_excute){
 		ArticleModule__generate_article(FIGHTER_BAYONETTA_GENERATE_ARTICLE_WICKEDWEAVEARM)
 	}
 }
 frame(Frame=9)
