diff --git "a/tta_5.0.0/bayonetta/game_AttackS4.txt" "b/tta_6.0.0/bayonetta/game_AttackS4.txt"
index 47ed4c1..eeb044a 100644
--- "a/tta_5.0.0/bayonetta/game_AttackS4.txt"
+++ "b/tta_6.0.0/bayonetta/game_AttackS4.txt"
@@ -3,11 +3,11 @@ FT_MOTION_RATE(FSM=0.75)
 frame(Frame=3)
 if(is_excute){
 	WorkModule__on_flag(Flag=FIGHTER_STATUS_ATTACK_FLAG_START_SMASH_HOLD)
 }
 execute(3)
-if(0x1905b0(WorkModule__is_flag(FIGHTER_STATUS_ATTACK_FLAG_SMASH_SMASH_HOLD_TO_ATTACK), False, True)){
+if(0x190590(WorkModule__is_flag(FIGHTER_STATUS_ATTACK_FLAG_SMASH_SMASH_HOLD_TO_ATTACK), False, True)){
 	if(is_excute){
 		ArticleModule__generate_article(FIGHTER_BAYONETTA_GENERATE_ARTICLE_WICKEDWEAVEARM)
 	}
 }
 if(is_excute){
