diff --git "a/tta_5.0.0/bayonetta/game_ShootingLegR_AtkOff_AttackS3.txt" "b/tta_6.0.0/bayonetta/game_ShootingLegR_AtkOff_AttackS3.txt"
index 69a7e4f..2f4d51a 100644
--- "a/tta_5.0.0/bayonetta/game_ShootingLegR_AtkOff_AttackS3.txt"
+++ "b/tta_6.0.0/bayonetta/game_ShootingLegR_AtkOff_AttackS3.txt"
@@ -1,6 +1,6 @@
-if(0x1905b0(WorkModule__is_flag(FIGHTER_BAYONETTA_INSTANCE_WORK_ID_FLAG_SHOOTING_DISABLE_ROOT_ATTACK), False, False)){
+if(0x190590(WorkModule__is_flag(FIGHTER_BAYONETTA_INSTANCE_WORK_ID_FLAG_SHOOTING_DISABLE_ROOT_ATTACK), False, False)){
 	if(is_excute){
 		AttackModule__clear(ID=FIGHTER_BAYONETTA_SHOOTING_ATTACK_ID_00)
 	}
 	else{
 	}
