diff --git "a/_4.0.0/ridley/game_SpecialLwStab.txt" "b/_5.0.0/ridley/game_SpecialLwStab.txt"
index 779a04b..961849c 100644
--- "a/_4.0.0/ridley/game_SpecialLwStab.txt"
+++ "b/_5.0.0/ridley/game_SpecialLwStab.txt"
@@ -1,9 +1,9 @@
 frame(Frame=30)
 if(method.lib::L2CValue.operator___lib::L2CValueconst__const(FIGHTER_RIDLEY_INSTANCE_WORK_ID_INT_DISABLE_SPECIAL_LW_FINISH_COUNT, 0)){
 	if(is_excute){
-		ATTACK(ID=0, Part=0, Bone=top, Damage=40.0, Angle=361, KBG=100, FKB=20, BKB=0, Size=2.1, X=0.0, Y=7.0, Z=24.5, X2=0.0, Y2=7.0, Z2=29.0, Hitlag=0.0, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_THRU, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=False, ShieldDamage=-37, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=49916, Hitbits=COLLISION_CATEGORY_MASK_FIGHTER, CollisionPart=COLLISION_PART_MASK_BODY_HEAD, FriendlyFire=False, Effect=collision_attr_sting, SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=COLLISION_SOUND_ATTR_NONE, Type=ATTACK_REGION_TAIL)
+		ATTACK(ID=0, Part=0, Bone=top, Damage=40.0, Angle=361, KBG=100, FKB=20, BKB=0, Size=2.1, X=0.0, Y=7.0, Z=24.5, X2=0.0, Y2=7.0, Z2=29.0, Hitlag=0.0, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_THRU, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=False, ShieldDamage=-37, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=50756, Hitbits=COLLISION_CATEGORY_MASK_FIGHTER, CollisionPart=COLLISION_PART_MASK_BODY_HEAD, FriendlyFire=False, Effect=collision_attr_sting, SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=COLLISION_SOUND_ATTR_NONE, Type=ATTACK_REGION_TAIL)
 		AttackModule__set_no_dead_all(True, False)
 	}
 }
 if(0x1090e0(WorkModule__is_flag(FIGHTER_RIDLEY_INSTANCE_WORK_ID_FLAG_DISABLE_CRITICAL_SPECIAL_LW), False, False)){
 	if(is_excute){
