diff --git "a/0.0.0/buddy/game_SpecialNEnd.txt" "b/0.1.0/buddy/game_SpecialNEnd.txt"
index 7d63578..e0cbc23 100644
--- "a/0.0.0/buddy/game_SpecialNEnd.txt"
+++ "b/0.1.0/buddy/game_SpecialNEnd.txt"
@@ -1,6 +1,6 @@
 frame(Frame=30)
-if(0x184a20(FIGHTER_INSTANCE_WORK_ID_INT_KIND, FIGHTER_KIND_KIRBY)){
+if(0x18e0b0(FIGHTER_INSTANCE_WORK_ID_INT_KIND, FIGHTER_KIND_KIRBY)){
 	if(is_excute){
 		ArticleModule::remove_exist(FIGHTER_BUDDY_GENERATE_ARTICLE_PARTNER)
 	}
 }
