diff --git "a/0.0/ken/game_AttackNearw.txt" "b/1.0/ken/game_AttackNearw.txt"
index 69723de..466946e 100644
--- "a/0.0/ken/game_AttackNearw.txt"
+++ "b/1.0/ken/game_AttackNearw.txt"
@@ -14,9 +14,9 @@ if(is_excute){
 }
 frame(Frame=8)
 if(is_excute){
 	WorkModule__on_flag(Flag=FIGHTER_RYU_STATUS_ATTACK_FLAG_HIT_CANCEL)
 }
-frame(Frame=15)
+frame(Frame=25)
 if(is_excute){
 	WorkModule__off_flag(Flag=FIGHTER_RYU_STATUS_ATTACK_FLAG_HIT_CANCEL)
 }
