diff --git "a/to_4.0.0/murabito/game_FinalmoneyR.txt" "b/to_5.0.0/murabito/game_FinalmoneyR.txt"
index d5022f3..6f1b89d 100644
--- "a/to_4.0.0/murabito/game_FinalmoneyR.txt"
+++ "b/to_5.0.0/murabito/game_FinalmoneyR.txt"
@@ -1,4 +1,4 @@
 frame(Frame=30)
 if(is_excute){
-	FILL_SCREEN_MODEL_COLOR(1, 10, 0.3, 0.3, 0.3, 0, 0, 0, 1, 1.3, 51192, EFFECT_SCREEN_PRIO_FINAL)
+	FILL_SCREEN_MODEL_COLOR(1, 10, 0.3, 0.3, 0.3, 0, 0, 0, 1, 1.3, 52032, EFFECT_SCREEN_PRIO_FINAL)
 }
