diff --git "a/2.0.0/chrom/game_FinalStart.txt" "b/3.0.0/chrom/game_FinalStart.txt"
index 1ebfbc7..374bf5f 100644
--- "a/2.0.0/chrom/game_FinalStart.txt"
+++ "b/3.0.0/chrom/game_FinalStart.txt"
@@ -11,11 +11,11 @@ if(methodlib::L2CValue::operator==(lib::L2CValueconst&)const(false, false)){
 		REQ_FINAL_START_CAMERA(hash40("d04finalstart.nuanmb"), false)
 	}
 	else{
 		if(is_excute){
 			PostureModule::scale(3, 0)
-			0x12bc30(294181909, 2)
+			0x12bc40(1807820073, 2)
 			CAM_ZOOM_IN_arg5(0, 0)
 			FT_START_CUTIN()
 		}
 	}
 }
