diff --git "a/.0.0/popo/game_Final2.txt" "b/.0.0/popo/game_Final2.txt"
index ccd9318..44e3a2b 100644
--- "a/.0.0/popo/game_Final2.txt"
+++ "b/.0.0/popo/game_Final2.txt"
@@ -2,27 +2,27 @@ if(is_excute){
 	WHOLE_HIT(HIT_STATUS_XLU)
 	FT_SET_FINAL_SMASH_LIGHT(True)
 	CHECK_VALID_FINAL_START_CAMERA(0, 7, 20, 0, 0, 0)
 	SLOW_OPPONENT(2, 60)
 }
-if(0x14bff0(WorkModule__is_flag(FIGHTER_INSTANCE_WORK_ID_FLAG_DISABLE_FINAL_START_CAMERA), False, False)){
+if(0x14c0b0(WorkModule__is_flag(FIGHTER_INSTANCE_WORK_ID_FLAG_DISABLE_FINAL_START_CAMERA), False, False)){
 	frame(Frame=10)
 	if(is_excute){
 		FT_SET_FINAL_FEAR_FACE(60)
 		REQ_FINAL_START_CAMERA(0x100bb16777, True)
 		FT_START_CUTIN()
 	}
 	else{
 		if(is_excute){
 			FT_REMOVE_FINAL_AURA(False)
 			PostureModule__scale(5, 0)
-			0x14bff0(-1995919863, 2.1)
+			0x14c0b0(1208653258, 2.1)
 			CAM_ZOOM_IN_arg5(0, 0)
 			FT_START_CUTIN()
 		}
 		get_value_float(SO_VAR_FLOAT_LR)
-		if(0x14bff0(0, 0)){
+		if(0x14c0b0(0, 0)){
 			if(is_excute){
 				camera(MA_MSC_CMD_CAMERA_CAM_OFFSET, 3, 0)
 			}
 			else{
 				if(is_excute){
