diff --git "a/4.0.0/falco/game_SpecialNEnd.txt" "b/5.0.0/falco/game_SpecialNEnd.txt"
index fb4542c..ef91e5f 100644
--- "a/4.0.0/falco/game_SpecialNEnd.txt"
+++ "b/5.0.0/falco/game_SpecialNEnd.txt"
@@ -1,9 +1,9 @@
 frame(Frame=1)
 FT_MOTION_RATE(FSM=0.75)
 IS_EXIST_ARTICLE(FIGHTER_FALCO_GENERATE_ARTICLE_BLASTER)
-if(0x101d80(False, True)){
+if(0x101ed0(False, True)){
 	if(is_excute){
 		as_hash__const(FIGHTER_FALCO_GENERATE_ARTICLE_BLASTER, 0x05130181c4)
 		ArticleModule__change_motion()
 	}
 }
