diff --git "a/4.0.0/simon/game_FinalAirStart.txt" "b/5.0.0/simon/game_FinalAirStart.txt"
index 486b804..8920f2a 100644
--- "a/4.0.0/simon/game_FinalAirStart.txt"
+++ "b/5.0.0/simon/game_FinalAirStart.txt"
@@ -15,11 +15,11 @@ if(0x119a60(32, 0)){
 		REQ_FINAL_START_CAMERA_arg3(0x1654ea70d3, False, False)
 	}
 	else{
 		if(is_excute){
 			PostureModule__scale(5, 0)
-			0x119a60(-327364042, 2.1)
+			0x119a60(-1801191224, 2.1)
 			CAM_ZOOM_IN_arg5(0, 0)
 		}
 		else{
 			if(is_excute){
 				REQ_FINAL_START_CAMERA_arg3(d04finalstart.nuanmb, False, False)
