diff --git "a/11.0.0/elight/game_SpecialSStart.txt" "b/12.0.0/elight/game_SpecialSStart.txt"
index 0c7a0a8..975525c 100644
--- "a/11.0.0/elight/game_SpecialSStart.txt"
+++ "b/12.0.0/elight/game_SpecialSStart.txt"
@@ -1,22 +1,22 @@
 frame(Frame=1)
 FT_MOTION_RATE(FSM=0.5)
 frame(Frame=3)
-IS_EXIST_ARTICLE(17456)
-if(0x353770(false, true)){
+IS_EXIST_ARTICLE(17632)
+if(0x354230(false, true)){
 	if(is_excute){
-		methodlib::L2CValue::as_hash()const(17456, 60344, 0x07439e926b, 5, 5, false, false, 0, false, true, false)
+		methodlib::L2CValue::as_hash()const(17632, 61372, 0x07439e926b, 5, 5, false, false, 0, false, true, false)
 		ArticleModule::add_motion_partial()
 	}
 }
 MotionModule::is_changing()
-if(0x353770(false, true)){
+if(0x354230(false, true)){
 	if(is_excute){
-		WorkModule::on_flag(Flag=60348)
+		WorkModule::on_flag(Flag=61376)
 	}
 }
 frame(0, 9)
 if(is_excute){
-	WorkModule::on_flag(Flag=56628)
+	WorkModule::on_flag(Flag=56816)
 }
 frame(Frame=21)
 FT_MOTION_RATE(FSM=1)
