diff --git "a/11.0.0/elight/game_AttackS4.txt" "b/12.0.0/elight/game_AttackS4.txt"
index 5ba2dc4..99100ad 100644
--- "a/11.0.0/elight/game_AttackS4.txt"
+++ "b/12.0.0/elight/game_AttackS4.txt"
@@ -1,24 +1,24 @@
 frame(Frame=1)
-IS_EXIST_ARTICLE(17456)
-if(0x353770(false, true)){
+IS_EXIST_ARTICLE(17632)
+if(0x354230(false, true)){
 	if(is_excute){
-		methodlib::L2CValue::as_hash()const(17456, 60344, 0x07439e926b, 5, 5, false, false, 0, false, true, false)
+		methodlib::L2CValue::as_hash()const(17632, 61372, 0x07439e926b, 5, 5, false, false, 0, false, true, false)
 		ArticleModule::add_motion_partial()
 	}
 }
 MotionModule::is_changing()
-if(0x353770(false, true)){
+if(0x354230(false, true)){
 	if(is_excute){
-		WorkModule::on_flag(Flag=60348)
+		WorkModule::on_flag(Flag=61376)
 	}
 }
 FT_MOTION_RATE(0, 0.5)
 frame(Frame=6)
 if(is_excute){
 	WorkModule::on_flag(Flag=FIGHTER_STATUS_ATTACK_FLAG_START_SMASH_HOLD)
-	WorkModule::set_int64(hash40("attack_s4_hold"), 60276)
+	WorkModule::set_int64(hash40("attack_s4_hold"), 61304)
 }
 frame(Frame=9)
 FT_MOTION_RATE(FSM=1)
 frame(Frame=18)
 if(is_excute){
@@ -48,20 +48,20 @@ frame(Frame=22)
 if(is_excute){
 	AttackModule::clear_all()
 }
 FT_MOTION_RATE(FSM=0.75)
 frame(Frame=37)
-IS_EXIST_ARTICLE(17456)
-if(0x353770(false, true)){
+IS_EXIST_ARTICLE(17632)
+if(0x354230(false, true)){
 	if(is_excute){
-		methodlib::L2CValue::as_hash()const(17456, 60344, 0x08183db0f4, 5, 5, false, false, 0, false, true, false)
+		methodlib::L2CValue::as_hash()const(17632, 61372, 0x08183db0f4, 5, 5, false, false, 0, false, true, false)
 		ArticleModule::add_motion_partial()
 	}
 }
 MotionModule::is_changing()
-if(0x353770(false, true)){
+if(0x354230(false, true)){
 	if(is_excute){
-		WorkModule::on_flag(Flag=60348)
+		WorkModule::on_flag(Flag=61376)
 	}
 }
 frame(0, 64)
 FT_MOTION_RATE(FSM=0.91)
