diff --git "a/11.0.0/elight/game_AttackDash.txt" "b/12.0.0/elight/game_AttackDash.txt"
index 115c043..420cdf4 100644
--- "a/11.0.0/elight/game_AttackDash.txt"
+++ "b/12.0.0/elight/game_AttackDash.txt"
@@ -1,21 +1,21 @@
 frame(Frame=1)
 FT_MOTION_RATE(FSM=0.5)
 frame(Frame=9)
 FT_MOTION_RATE(FSM=1)
 frame(Frame=10)
-IS_EXIST_ARTICLE(17456)
-if(0x353770(false, true)){
+IS_EXIST_ARTICLE(17632)
+if(0x354230(false, true)){
 	if(is_excute){
-		methodlib::L2CValue::as_hash()const(17456, 60344, 0x07439e926b, 5, 5, false, false, 0, false, true, false)
+		methodlib::L2CValue::as_hash()const(17632, 61372, 0x07439e926b, 5, 5, false, false, 0, false, true, false)
 		ArticleModule::add_motion_partial()
 	}
 }
 MotionModule::is_changing()
-if(0x353770(false, true)){
+if(0x354230(false, true)){
 	if(is_excute){
-		WorkModule::on_flag(Flag=60348)
+		WorkModule::on_flag(Flag=61376)
 	}
 }
 frame(0, 12)
 FT_MOTION_RATE(FSM=0.5)
 frame(Frame=14)
@@ -27,19 +27,19 @@ if(is_excute){
 frame(Frame=16)
 if(is_excute){
 	AttackModule::clear_all()
 }
 frame(Frame=29)
-IS_EXIST_ARTICLE(17456)
-if(0x353770(false, true)){
+IS_EXIST_ARTICLE(17632)
+if(0x354230(false, true)){
 	if(is_excute){
-		methodlib::L2CValue::as_hash()const(17456, 60344, 0x08183db0f4, 5, 5, false, false, 0, false, true, false)
+		methodlib::L2CValue::as_hash()const(17632, 61372, 0x08183db0f4, 5, 5, false, false, 0, false, true, false)
 		ArticleModule::add_motion_partial()
 	}
 }
 MotionModule::is_changing()
-if(0x353770(false, true)){
+if(0x354230(false, true)){
 	if(is_excute){
-		WorkModule::on_flag(Flag=60348)
+		WorkModule::on_flag(Flag=61376)
 	}
 }
 FT_MOTION_RATE(0, 0.75)
