diff --git "a/11.0.0/elight/game_SpecialAirHi1.txt" "b/12.0.0/elight/game_SpecialAirHi1.txt"
index 3bb6be5..f4e01a8 100644
--- "a/11.0.0/elight/game_SpecialAirHi1.txt"
+++ "b/12.0.0/elight/game_SpecialAirHi1.txt"
@@ -1,35 +1,35 @@
-IS_EXIST_ARTICLE(17456)
-if(0x353770(false, true)){
+IS_EXIST_ARTICLE(17632)
+if(0x354230(false, true)){
 	if(is_excute){
-		methodlib::L2CValue::as_hash()const(17456, 60344, 0x07439e926b, 5, 5, false, false, 0, false, true, false)
+		methodlib::L2CValue::as_hash()const(17632, 61372, 0x07439e926b, 5, 5, false, false, 0, false, true, false)
 		ArticleModule::add_motion_partial()
 	}
 }
 MotionModule::is_changing()
-if(0x353770(false, true)){
+if(0x354230(false, true)){
 	if(is_excute){
-		WorkModule::on_flag(Flag=60348)
+		WorkModule::on_flag(Flag=61376)
 	}
 }
 frame(0, 2)
 FT_MOTION_RATE(FSM=0.75)
 frame(Frame=10)
 FT_MOTION_RATE(FSM=1)
 frame(Frame=13)
 if(is_excute){
-	ArticleModule::generate_article(60328)
+	ArticleModule::generate_article(61356)
 }
 frame(Frame=14)
-IS_EXIST_ARTICLE(17456)
-if(0x353770(false, true)){
+IS_EXIST_ARTICLE(17632)
+if(0x354230(false, true)){
 	if(is_excute){
-		methodlib::L2CValue::as_hash()const(17456, 60344, 0x08183db0f4, 5, 5, false, false, 0, false, true, false)
+		methodlib::L2CValue::as_hash()const(17632, 61372, 0x08183db0f4, 5, 5, false, false, 0, false, true, false)
 		ArticleModule::add_motion_partial()
 	}
 }
 MotionModule::is_changing()
-if(0x353770(false, true)){
+if(0x354230(false, true)){
 	if(is_excute){
-		WorkModule::on_flag(Flag=60348)
+		WorkModule::on_flag(Flag=61376)
 	}
 }
