diff --git "a/a_11.0.0/gekkouga_gekkougas/game_FinalAttack.txt" "b/a_12.0.0/gekkouga_gekkougas/game_FinalAttack.txt"
index 531c48a..b63269a 100644
--- "a/a_11.0.0/gekkouga_gekkougas/game_FinalAttack.txt"
+++ "b/a_12.0.0/gekkouga_gekkougas/game_FinalAttack.txt"
@@ -1,23 +1,23 @@
 if(is_excute){
 	CAM_ZOOM_OUT()
 }
 frame(Frame=2)
 get_fov()
-if(0x141270(1, 25.0)){
+if(0x1412f0(1, 25.0)){
 	if(is_excute){
 		get_fov(5, 0)
-		0x141270(0, 60)
+		0x1412f0(0, 60)
 		CAM_ZOOM_IN_arg5(0.1, 0)
 	}
 }
 frame(Frame=3)
 get_fov()
-if(0x141270(1, 5.0)){
+if(0x1412f0(1, 5.0)){
 	if(is_excute){
 		get_fov(5, 0)
-		0x141270(0, 90)
+		0x1412f0(0, 90)
 		CAM_ZOOM_IN_arg5(0.1, 0)
 	}
 }
 frame(Frame=4)
 get_fov()
