diff --git "a/8.0.0/chrom/game_FinalStart.txt" "b/9.0.0/chrom/game_FinalStart.txt"
index 61f140e..846345f 100644
--- "a/8.0.0/chrom/game_FinalStart.txt"
+++ "b/9.0.0/chrom/game_FinalStart.txt"
@@ -10,11 +10,11 @@ if(method.lib::L2CValue.operator___lib::L2CValueconst__const(WorkModule::is_flag
 		REQ_FINAL_START_CAMERA(hash40("d04finalstart.nuanmb"), false)
 	}
 	else{
 		if(is_excute){
 			PostureModule::scale(3, 0)
-			0x12ba50(503437126, 2)
+			0x12ba60(1587853581, 2)
 			CAM_ZOOM_IN_arg5(0, 0)
 			FT_START_CUTIN()
 		}
 	}
 }
