diff --git "a/2.0.0/chrom/game_FinalAttack.txt" "b/3.0.0/chrom/game_FinalAttack.txt"
index 7ffd0e5..307e6c1 100644
--- "a/2.0.0/chrom/game_FinalAttack.txt"
+++ "b/3.0.0/chrom/game_FinalAttack.txt"
@@ -4,11 +4,11 @@ if(is_excute){
 	FT_REMOVE_FINAL_AURA(false)
 }
 WorkModule::is_flag(FIGHTER_ROY_STATUS_FINAL_FLAG_HIT_ALL_OPPONENT)
 if(methodlib::L2CValue::operator==(lib::L2CValueconst&)const(false, true)){
 	get_value_float(SO_VAR_FLOAT_LR)
-	if(0x12bc30(0, 0)){
+	if(0x12bc40(0, 0)){
 		if(is_excute){
 			CAM_ZOOM_IN_arg5(2, 0, 1.6, 0, 0)
 			camera(MA_MSC_CMD_CAMERA_CAM_OFFSET, -5, 5)
 		}
 		else{
