diff --git "a/er_13.0.0/miigunner/game_FinalAirStart.txt" "b/er_13.0.1/miigunner/game_FinalAirStart.txt"
index 837797c..1c5d4d0 100644
--- "a/er_13.0.0/miigunner/game_FinalAirStart.txt"
+++ "b/er_13.0.1/miigunner/game_FinalAirStart.txt"
@@ -17,11 +17,11 @@ if(0x1303f0(1657856, 0)){
 		REQ_FINAL_START_CAMERA_arg3(hash40("d04finalstart02.nuanmb"), true, false)
 	}
 	else{
 		if(is_excute){
 			PostureModule::scale(3, 0)
-			0x1303f0(43535466, 2.2)
+			0x1303f0(-1642871193, 2.2)
 			CAM_ZOOM_IN_arg5(0, 0)
 		}
 		else{
 			if(is_excute){
 				REQ_FINAL_START_CAMERA_arg3(hash40("d04finalstart.nuanmb"), true, false)
