diff --git "a/.1.0/link/game_SpecialAirHi.txt" "b/.0.0/link/game_SpecialAirHi.txt"
index 62b8562..89484ef 100644
--- "a/.1.0/link/game_SpecialAirHi.txt"
+++ "b/.0.0/link/game_SpecialAirHi.txt"
@@ -7,11 +7,11 @@ wait(Frames=2)
 if(is_excute){
 	AttackModule__clear_all()
 }
 frame(Frame=12)
 if(is_excute){
-	ATTACK(ID=1, Part=0, Bone=top, Damage=2.0, Angle=145, KBG=100, FKB=95, BKB=0, Size=5.0, X=0.0, Y=11.5, Z=-12.0, X2=0.0, Y2=11.0, Z2=-9.0, Hitlag=1.0, SDI=0.3, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_B, SetWeight=True, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_cutup, SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=COLLISION_SOUND_ATTR_CUTUP, Type=ATTACK_REGION_SWORD)
+	ATTACK(ID=1, Part=0, Bone=top, Damage=2.0, Angle=145, KBG=100, FKB=95, BKB=0, Size=5.0, X=0.0, Y=11.5, Z=-12.0, X2=0.0, Y2=11.0, Z2=-9.0, Hitlag=1.0, SDI=0.3, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_B, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_cutup, SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=COLLISION_SOUND_ATTR_CUTUP, Type=ATTACK_REGION_SWORD)
 	as_hash__const(1, top, 10, 20, 10)
 	AttackModule__set_vec_target_pos()
 	AttackModule__set_no_damage_fly_smoke_all(True, False)
 }
 wait(Frames=2)
@@ -28,11 +28,11 @@ wait(Frames=2)
 if(is_excute){
 	AttackModule__clear_all()
 }
 frame(Frame=19)
 if(is_excute){
-	ATTACK(ID=1, Part=0, Bone=top, Damage=2.0, Angle=145, KBG=100, FKB=95, BKB=0, Size=5.0, X=0.0, Y=11.5, Z=-12.5, X2=0.0, Y2=11.0, Z2=-9.0, Hitlag=1.0, SDI=0.3, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_B, SetWeight=True, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_cutup, SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=COLLISION_SOUND_ATTR_CUTUP, Type=ATTACK_REGION_SWORD)
+	ATTACK(ID=1, Part=0, Bone=top, Damage=2.0, Angle=145, KBG=100, FKB=95, BKB=0, Size=5.0, X=0.0, Y=11.5, Z=-12.5, X2=0.0, Y2=11.0, Z2=-9.0, Hitlag=1.0, SDI=0.3, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_B, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_cutup, SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=COLLISION_SOUND_ATTR_CUTUP, Type=ATTACK_REGION_SWORD)
 	as_hash__const(1, top, 10, 20, 10)
 	AttackModule__set_vec_target_pos()
 	AttackModule__set_no_damage_fly_smoke_all(True, False)
 }
 wait(Frames=2)
@@ -49,11 +49,11 @@ if(is_excute){
 	AttackModule__clear_all()
 }
 frame(Frame=26)
 if(is_excute){
 	JostleModule__set_status(False)
-	ATTACK(ID=1, Part=0, Bone=sword2, Damage=2.0, Angle=150, KBG=100, FKB=95, BKB=0, Size=5.0, X=2.0, Y=0.0, Z=0.0, X2=5.5, Y2=0.0, Z2=0.0, Hitlag=1.0, SDI=0.3, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_B, SetWeight=True, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_cutup, SFXLevel=ATTACK_SOUND_LEVEL_M, SFXType=COLLISION_SOUND_ATTR_CUTUP, Type=ATTACK_REGION_SWORD)
+	ATTACK(ID=1, Part=0, Bone=sword2, Damage=2.0, Angle=150, KBG=100, FKB=95, BKB=0, Size=5.0, X=2.0, Y=0.0, Z=0.0, X2=5.5, Y2=0.0, Z2=0.0, Hitlag=1.0, SDI=0.3, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_B, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_cutup, SFXLevel=ATTACK_SOUND_LEVEL_M, SFXType=COLLISION_SOUND_ATTR_CUTUP, Type=ATTACK_REGION_SWORD)
 	as_hash__const(1, top, 10, 20, 10)
 	AttackModule__set_vec_target_pos()
 	AttackModule__set_no_damage_fly_smoke_all(True, False)
 }
 wait(Frames=2)
@@ -70,11 +70,11 @@ if(is_excute){
 	AttackModule__clear_all()
 	notify_event_msc_cmd(0x2127e37c07, GROUND_CLIFF_CHECK_KIND_ALWAYS_BOTH_SIDES)
 }
 frame(Frame=38)
 if(is_excute){
-	ATTACK(ID=1, Part=0, Bone=sword2, Damage=2.0, Angle=160, KBG=100, FKB=95, BKB=0, Size=5.0, X=2.0, Y=0.0, Z=0.0, X2=5.5, Y2=0.0, Z2=0.0, Hitlag=1.0, SDI=0.3, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_B, SetWeight=True, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_cutup, SFXLevel=ATTACK_SOUND_LEVEL_M, SFXType=COLLISION_SOUND_ATTR_CUTUP, Type=ATTACK_REGION_SWORD)
+	ATTACK(ID=1, Part=0, Bone=sword2, Damage=2.0, Angle=160, KBG=100, FKB=95, BKB=0, Size=5.0, X=2.0, Y=0.0, Z=0.0, X2=5.5, Y2=0.0, Z2=0.0, Hitlag=1.0, SDI=0.3, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_B, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_cutup, SFXLevel=ATTACK_SOUND_LEVEL_M, SFXType=COLLISION_SOUND_ATTR_CUTUP, Type=ATTACK_REGION_SWORD)
 	as_hash__const(1, top, 10, 20, 10)
 	AttackModule__set_vec_target_pos()
 	AttackModule__set_no_damage_fly_smoke_all(True, False)
 }
 wait(Frames=2)
