diff --git "a/nk_8.0.0/toonlink/game_AttackAirLwOffGravity.txt" "b/nk_9.0.0/toonlink/game_AttackAirLwOffGravity.txt"
index 47de557..9a79c16 100644
--- "a/nk_8.0.0/toonlink/game_AttackAirLwOffGravity.txt"
+++ "b/nk_9.0.0/toonlink/game_AttackAirLwOffGravity.txt"
@@ -1,5 +1,5 @@
 get_value_int(SO_VAR_INT_PREV_STATUS)
-0x119130(0, FIGHTER_STATUS_KIND_DAMAGE_FLY)
+0x120860(0, FIGHTER_STATUS_KIND_DAMAGE_FLY)
 if(is_excute){
 	WorkModule::on_flag(Flag=FIGHTER_STATUS_WORK_ID_FLAG_RESERVE_GRAVITY_STABLE_UNABLE)
 }
