diff --git "a/ga_5.0.0/gekkouga/game_SpecialAirLwAttack.txt" "b/ga_6.0.0/gekkouga/game_SpecialAirLwAttack.txt"
index f1d5a2f..1df8727 100644
--- "a/ga_5.0.0/gekkouga/game_SpecialAirLwAttack.txt"
+++ "b/ga_6.0.0/gekkouga/game_SpecialAirLwAttack.txt"
@@ -1,39 +1,39 @@
 frame(Frame=5)
-if(0x1410d0(WorkModule__is_flag(FIGHTER_GEKKOUGA_STATUS_SPECIAL_LW_FLAG_ATTACK_UP), False, True)){
+if(0x1410f0(WorkModule__is_flag(FIGHTER_GEKKOUGA_STATUS_SPECIAL_LW_FLAG_ATTACK_UP), False, True)){
 	if(is_excute){
 		ATTACK(ID=0, Part=0, Bone=legl, Damage=13.0, Angle=90, KBG=95, FKB=0, BKB=50, Size=8.5, X=10.0, Y=0.0, Z=0.0, Hitlag=1.1, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=False, ShieldDamage=5, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_normal, SFXLevel=ATTACK_SOUND_LEVEL_L, SFXType=COLLISION_SOUND_ATTR_KICK, Type=ATTACK_REGION_KICK)
 		ATTACK(ID=1, Part=0, Bone=legl, Damage=13.0, Angle=90, KBG=95, FKB=0, BKB=50, Size=7.0, X=2.5, Y=0.0, Z=0.0, Hitlag=1.1, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=False, ShieldDamage=5, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_normal, SFXLevel=ATTACK_SOUND_LEVEL_L, SFXType=COLLISION_SOUND_ATTR_KICK, Type=ATTACK_REGION_KICK)
 		AttackModule__set_force_reaction(0, True, False)
 		AttackModule__set_force_reaction(1, True, False)
 	}
 	else{
-		if(0x1410d0(WorkModule__is_flag(FIGHTER_GEKKOUGA_STATUS_SPECIAL_LW_FLAG_ATTACK_UP_SIDE), False, True)){
+		if(0x1410f0(WorkModule__is_flag(FIGHTER_GEKKOUGA_STATUS_SPECIAL_LW_FLAG_ATTACK_UP_SIDE), False, True)){
 			if(is_excute){
 				ATTACK(ID=0, Part=0, Bone=legl, Damage=12.0, Angle=60, KBG=95, FKB=0, BKB=62, Size=7.5, X=10.0, Y=0.0, Z=0.0, Hitlag=1.1, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=False, ShieldDamage=5, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_normal, SFXLevel=ATTACK_SOUND_LEVEL_L, SFXType=COLLISION_SOUND_ATTR_KICK, Type=ATTACK_REGION_KICK)
 				ATTACK(ID=1, Part=0, Bone=legl, Damage=12.0, Angle=60, KBG=95, FKB=0, BKB=62, Size=6.0, X=2.5, Y=0.0, Z=0.0, Hitlag=1.1, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=False, ShieldDamage=5, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_normal, SFXLevel=ATTACK_SOUND_LEVEL_L, SFXType=COLLISION_SOUND_ATTR_KICK, Type=ATTACK_REGION_KICK)
 				AttackModule__set_force_reaction(0, True, False)
 				AttackModule__set_force_reaction(1, True, False)
 			}
 			else{
-				if(0x1410d0(WorkModule__is_flag(FIGHTER_GEKKOUGA_STATUS_SPECIAL_LW_FLAG_ATTACK_SIDE), False, True)){
+				if(0x1410f0(WorkModule__is_flag(FIGHTER_GEKKOUGA_STATUS_SPECIAL_LW_FLAG_ATTACK_SIDE), False, True)){
 					if(is_excute){
 						ATTACK(ID=0, Part=0, Bone=legl, Damage=11.0, Angle=361, KBG=85, FKB=0, BKB=62, Size=6.0, X=10.0, Y=0.0, Z=0.0, Hitlag=1.1, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=False, ShieldDamage=5, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_normal, SFXLevel=ATTACK_SOUND_LEVEL_L, SFXType=COLLISION_SOUND_ATTR_KICK, Type=ATTACK_REGION_KICK)
 						ATTACK(ID=1, Part=0, Bone=legl, Damage=11.0, Angle=361, KBG=85, FKB=0, BKB=62, Size=5.0, X=2.5, Y=0.0, Z=0.0, Hitlag=1.1, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=False, ShieldDamage=5, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_normal, SFXLevel=ATTACK_SOUND_LEVEL_L, SFXType=COLLISION_SOUND_ATTR_KICK, Type=ATTACK_REGION_KICK)
 						AttackModule__set_force_reaction(0, True, False)
 						AttackModule__set_force_reaction(1, True, False)
 					}
 					else{
-						if(0x1410d0(WorkModule__is_flag(FIGHTER_GEKKOUGA_STATUS_SPECIAL_LW_FLAG_ATTACK_DOWN_SIDE), False, True)){
+						if(0x1410f0(WorkModule__is_flag(FIGHTER_GEKKOUGA_STATUS_SPECIAL_LW_FLAG_ATTACK_DOWN_SIDE), False, True)){
 							if(is_excute){
 								ATTACK(ID=0, Part=0, Bone=legl, Damage=12.0, Angle=25, KBG=72, FKB=0, BKB=62, Size=7.5, X=10.0, Y=0.0, Z=0.0, Hitlag=1.1, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=False, ShieldDamage=5, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_normal, SFXLevel=ATTACK_SOUND_LEVEL_L, SFXType=COLLISION_SOUND_ATTR_KICK, Type=ATTACK_REGION_KICK)
 								ATTACK(ID=1, Part=0, Bone=legl, Damage=12.0, Angle=25, KBG=72, FKB=0, BKB=62, Size=6.0, X=2.5, Y=0.0, Z=0.0, Hitlag=1.1, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=False, ShieldDamage=5, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_normal, SFXLevel=ATTACK_SOUND_LEVEL_L, SFXType=COLLISION_SOUND_ATTR_KICK, Type=ATTACK_REGION_KICK)
 								AttackModule__set_force_reaction(0, True, False)
 								AttackModule__set_force_reaction(1, True, False)
 							}
 							else{
-								if(0x1410d0(WorkModule__is_flag(FIGHTER_GEKKOUGA_STATUS_SPECIAL_LW_FLAG_ATTACK_DOWN), False, True)){
+								if(0x1410f0(WorkModule__is_flag(FIGHTER_GEKKOUGA_STATUS_SPECIAL_LW_FLAG_ATTACK_DOWN), False, True)){
 									if(is_excute){
 										ATTACK(ID=0, Part=0, Bone=legl, Damage=13.0, Angle=270, KBG=65, FKB=0, BKB=25, Size=8.5, X=10.0, Y=0.0, Z=0.0, Hitlag=1.1, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=False, ShieldDamage=5, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_A, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_normal, SFXLevel=ATTACK_SOUND_LEVEL_L, SFXType=COLLISION_SOUND_ATTR_KICK, Type=ATTACK_REGION_KICK)
 										ATTACK(ID=1, Part=0, Bone=legl, Damage=13.0, Angle=45, KBG=95, FKB=0, BKB=50, Size=8.5, X=10.0, Y=0.0, Z=0.0, Hitlag=1.1, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=False, ShieldDamage=5, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_G, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_normal, SFXLevel=ATTACK_SOUND_LEVEL_L, SFXType=COLLISION_SOUND_ATTR_KICK, Type=ATTACK_REGION_KICK)
 										AttackModule__set_force_reaction(0, True, False)
 										AttackModule__set_force_reaction(1, True, False)
