diff --git "a/ner_9.0.0/miigunner/game_FinalStart.txt" "b/er_10.0.0/miigunner/game_FinalStart.txt"
index d732ad4..8f7f277 100644
--- "a/ner_9.0.0/miigunner/game_FinalStart.txt"
+++ "b/er_10.0.0/miigunner/game_FinalStart.txt"
@@ -16,11 +16,11 @@ if(0x12fe20(1649648, 0)){
 		REQ_FINAL_START_CAMERA_arg3(hash40("d04finalstart02.nuanmb"), true, false)
 	}
 	else{
 		if(is_excute){
 			PostureModule::scale(3, 0)
-			0x12fe20(913492079, 2.2)
+			0x12fe20(61294676, 2.2)
 			CAM_ZOOM_IN_arg5(0, 0)
 		}
 		else{
 			if(is_excute){
 				REQ_FINAL_START_CAMERA_arg3(hash40("d04finalstart.nuanmb"), true, false)
