diff --git "a/0.0/fox/game_LandingAirF.txt" "b/0.0/fox/game_LandingAirF.txt"
index ce248fe..48240a1 100644
--- "a/0.0/fox/game_LandingAirF.txt"
+++ "b/0.0/fox/game_LandingAirF.txt"
@@ -1,6 +1,6 @@
-if(0x10cc70(WorkModule__is_flag(FIGHTER_FOX_STATUS_ATTACK_AIR_FLAG_LANDING_DISABLE_ATTACK), False, True)){
+if(0x10ce40(WorkModule__is_flag(FIGHTER_FOX_STATUS_ATTACK_AIR_FLAG_LANDING_DISABLE_ATTACK), False, True)){
 	if(is_excute){
 		AttackModule__clear_all()
 	}
 }
 else{
