diff --git "a/g_8.0.0/inkling/game_RunBrakeL.txt" "b/g_9.0.0/inkling/game_RunBrakeL.txt"
index aec4b58..21e5d82 100644
--- "a/g_8.0.0/inkling/game_RunBrakeL.txt"
+++ "b/g_9.0.0/inkling/game_RunBrakeL.txt"
@@ -1,7 +1,7 @@
 get_value_float(SO_VAR_FLOAT_LR)
-if(0x13d0e0(0, 0)){
+if(0x13d330(0, 0)){
 	if(is_excute){
 		WorkModule::on_flag(Flag=FIGHTER_INKLING_INSTANCE_WORK_ID_FLAG_NO_FLIP_SQUID)
 	}
 }
 if(method.lib::L2CValue.operator___lib::L2CValueconst__const(WorkModule::is_flag(FIGHTER_INKLING_INSTANCE_WORK_ID_FLAG_EXIST_SQUID), false, false)){
@@ -9,12 +9,12 @@ if(method.lib::L2CValue.operator___lib::L2CValueconst__const(WorkModule::is_flag
 		ArticleModule::generate_article(FIGHTER_INKLING_GENERATE_ARTICLE_SQUID)
 	}
 }
 MotionModule::motion_kind()
 MotionModule::frame(0x00000002b0)
-MotionModule::rate(780876990)
-as_hash__const(780876990, FIGHTER_INKLING_GENERATE_ARTICLE_SQUID, false)
+MotionModule::rate(989672188)
+as_hash__const(989672188, FIGHTER_INKLING_GENERATE_ARTICLE_SQUID, false)
 ArticleModule::change_motion()
 if(is_excute){
 	ArticleModule::set_frame(FIGHTER_INKLING_GENERATE_ARTICLE_SQUID)
 	ArticleModule::set_rate(FIGHTER_INKLING_GENERATE_ARTICLE_SQUID)
 	ArticleModule::set_visibility_whole(FIGHTER_INKLING_GENERATE_ARTICLE_SQUID, false)
@@ -26,11 +26,11 @@ if(is_excute){
 if(is_excute){
 	VisibilityModule::set_whole(false)
 	ArticleModule::set_visibility_whole(FIGHTER_INKLING_GENERATE_ARTICLE_SQUID, true)
 }
 get_value_float(SO_VAR_FLOAT_LR)
-if(0x13d0e0(0, 0)){
+if(0x13d330(0, 0)){
 	if(is_excute){
 		WorkModule::on_flag(Flag=FIGHTER_INKLING_INSTANCE_WORK_ID_FLAG_NO_FLIP_SQUID)
 	}
 }
 if(method.lib::L2CValue.operator___lib::L2CValueconst__const(WorkModule::is_flag(FIGHTER_INKLING_INSTANCE_WORK_ID_FLAG_EXIST_SQUID), false, false)){
@@ -38,12 +38,12 @@ if(method.lib::L2CValue.operator___lib::L2CValueconst__const(WorkModule::is_flag
 		ArticleModule::generate_article(FIGHTER_INKLING_GENERATE_ARTICLE_SQUID)
 	}
 }
 MotionModule::motion_kind()
 MotionModule::frame(0x00000002b0)
-MotionModule::rate(780876990)
-as_hash__const(780876990, FIGHTER_INKLING_GENERATE_ARTICLE_SQUID, false)
+MotionModule::rate(989672188)
+as_hash__const(989672188, FIGHTER_INKLING_GENERATE_ARTICLE_SQUID, false)
 ArticleModule::change_motion()
 if(is_excute){
 	ArticleModule::set_frame(FIGHTER_INKLING_GENERATE_ARTICLE_SQUID)
 	ArticleModule::set_rate(FIGHTER_INKLING_GENERATE_ARTICLE_SQUID)
 	ArticleModule::set_visibility_whole(FIGHTER_INKLING_GENERATE_ARTICLE_SQUID, false)
@@ -51,11 +51,11 @@ if(is_excute){
 	StatusModule::status_kind()
 	operator___lib::L2CValueconst__const(0, FIGHTER_STATUS_KIND_REBIRTH)
 	WorkModule::on_flag(Flag=FIGHTER_INSTANCE_WORK_ID_FLAG_FORCE_LOUPE)
 }
 get_value_float(0, SO_VAR_FLOAT_LR)
-if(0x13d0e0(0, 0)){
+if(0x13d330(0, 0)){
 	if(is_excute){
 		as_hash__const(FIGHTER_INKLING_GENERATE_ARTICLE_SQUID, hash40("run_brake_l"))
 		ArticleModule::change_motion()
 	}
 	else{
