diff --git "a/11.0.0/eflame_esword/game_Rotate.txt" "b/12.0.0/eflame_esword/game_Rotate.txt"
index 283a24e..76dc5c2 100644
--- "a/11.0.0/eflame_esword/game_Rotate.txt"
+++ "b/12.0.0/eflame_esword/game_Rotate.txt"
@@ -55,11 +55,11 @@ if(is_excute){
 	methodlib::L2CValue::as_hash()const(hash40("rbkind_nohitm"), 0, false)
 	ControlModule::set_rumble()
 	ATTACK(ID=1, Part=0, Bone=hash40("sword1"), Damage=10.0, Angle=65, KBG=100, FKB=0, BKB=35, Size=3.5, X=5.0, Y=0.0, Z=0.0, X2=LUA_VOID, Y2=LUA_VOID, Z2=LUA_VOID, Hitlag=1.3, SDI=0.0, Clang_Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=false, ShieldDamage=0, Trip=-1.0, Rehit=0, Reflectable=false, Absorbable=false, Flinchless=false, DisableHitlag=false, Direct_Hitbox=false, Ground_or_Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=false, Effect=hash40("collision_attr_fire"), SFXLevel=ATTACK_SOUND_LEVEL_M, SFXType=COLLISION_SOUND_ATTR_FIRE, Type=ATTACK_REGION_SWORD)
 	ATTACK(ID=2, Part=0, Bone=hash40("sword1"), Damage=10.0, Angle=65, KBG=100, FKB=0, BKB=35, Size=3.5, X=9.0, Y=0.0, Z=0.0, X2=LUA_VOID, Y2=LUA_VOID, Z2=LUA_VOID, Hitlag=1.3, SDI=0.0, Clang_Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=false, ShieldDamage=0, Trip=-1.0, Rehit=0, Reflectable=false, Absorbable=false, Flinchless=false, DisableHitlag=false, Direct_Hitbox=false, Ground_or_Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=false, Effect=hash40("collision_attr_fire"), SFXLevel=ATTACK_SOUND_LEVEL_M, SFXType=COLLISION_SOUND_ATTR_FIRE, Type=ATTACK_REGION_SWORD)
 	ATTACK(ID=3, Part=0, Bone=hash40("sword1"), Damage=10.0, Angle=65, KBG=100, FKB=0, BKB=35, Size=3.5, X=11.0, Y=0.0, Z=0.0, X2=LUA_VOID, Y2=LUA_VOID, Z2=LUA_VOID, Hitlag=1.3, SDI=0.0, Clang_Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=false, ShieldDamage=0, Trip=-1.0, Rehit=0, Reflectable=false, Absorbable=false, Flinchless=false, DisableHitlag=false, Direct_Hitbox=false, Ground_or_Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=false, Effect=hash40("collision_attr_fire"), SFXLevel=ATTACK_SOUND_LEVEL_M, SFXType=COLLISION_SOUND_ATTR_FIRE, Type=ATTACK_REGION_SWORD)
-	WorkModule::on_flag(Flag=60304)
+	WorkModule::on_flag(Flag=61332)
 }
 frame(Frame=55)
 if(is_excute){
 	AttackModule::clear_all()
 }
