diff --git "a/_6.0.0/reflet/game_AttackAirB.txt" "b/_7.0.0/reflet/game_AttackAirB.txt"
index 8871cb4..936af00 100644
--- "a/_6.0.0/reflet/game_AttackAirB.txt"
+++ "b/_7.0.0/reflet/game_AttackAirB.txt"
@@ -18,11 +18,11 @@ if(is_excute){
 wait(Frames=6)
 if(is_excute){
 	AttackModule__clear_all()
 }
 frame(Frame=21)
-if(0x141ef0(FIGHTER_REFLET_INSTANCE_WORK_ID_INT_THUNDER_SWORD_CURRENT_POINT, 0)){
+if(0x141f00(FIGHTER_REFLET_INSTANCE_WORK_ID_INT_THUNDER_SWORD_CURRENT_POINT, 0)){
 	if(method.lib::L2CValue.operator___lib::L2CValueconst__const(WorkModule__is_flag(FIGHTER_REFLET_INSTANCE_WORK_ID_FLAG_THUNDER_SWORD_ON), False, True)){
 		if(is_excute){
 			VisibilityModule__set_int64(sword, sword_normal)
 			WorkModule__off_flag(Flag=FIGHTER_REFLET_INSTANCE_WORK_ID_FLAG_THUNDER_SWORD_ON)
 		}
@@ -47,9 +47,9 @@ if(is_excute){
 else{
 	L2CValue_lib::L2CValueconst()
 	as_pointer__const(-8, 14)
 	throwaway_sword_app::Fighter__phx::Vector2f__bool()
 }
-frame(1716128, 1716432, 32)
+frame(1716136, 1716440, 32)
 if(is_excute){
 	WorkModule__off_flag(Flag=FIGHTER_STATUS_ATTACK_AIR_FLAG_ENABLE_LANDING)
 }
