diff --git "a/4.0.0/kirby/game_Win3Wait.txt" "b/5.0.0/kirby/game_Win3Wait.txt"
index 4cd064d..5023441 100644
--- "a/4.0.0/kirby/game_Win3Wait.txt"
+++ "b/5.0.0/kirby/game_Win3Wait.txt"
@@ -1,5 +1,5 @@
-0x469f90(WorkModule__is_flag(FIGHTER_INSTANCE_WORK_ID_FLAG_RESULT_FIGHTER_SINGLE), False, True)
+0x48bff0(WorkModule__is_flag(FIGHTER_INSTANCE_WORK_ID_FLAG_RESULT_FIGHTER_SINGLE), False, True)
 if(is_excute){
 	as_hash__const(FIGHTER_KIRBY_GENERATE_ARTICLE_WINDUMMY, 0x0a73249b5f)
 	ArticleModule__change_motion()
 }
