diff --git "a/10.0.0/pikmin_pikmin/game_SpNTakenOutStart_W.txt" "b/10.1.0/pikmin_pikmin/game_SpNTakenOutStart_W.txt"
index 443486b..389b133 100644
--- "a/10.0.0/pikmin_pikmin/game_SpNTakenOutStart_W.txt"
+++ "b/10.1.0/pikmin_pikmin/game_SpNTakenOutStart_W.txt"
@@ -1,9 +1,9 @@
 frame(Frame=1)
 if(is_excute){
 	WorkModule::get_float(WEAPON_PIKMIN_PIKMIN_STATUS_PULL_OUT_START_WORK_FLOAT_MOT_RATE)
-	MotionModule::set_rate(2082482820)
+	MotionModule::set_rate(328571433)
 }
 frame(Frame=5)
 if(is_excute){
 	MotionModule::set_rate(1)
 }
