diff --git "a/isou_10.1.0/pfushigisou/game_FinalAir.txt" "b/isou_11.0.0/pfushigisou/game_FinalAir.txt"
index a014659..a362791 100644
--- "a/isou_10.1.0/pfushigisou/game_FinalAir.txt"
+++ "b/isou_11.0.0/pfushigisou/game_FinalAir.txt"
@@ -1,21 +1,21 @@
 if(is_excute){
 	CHECK_VALID_FINAL_START_CAMERA(0, 1, 20, 0, 0, 0)
 	SLOW_OPPONENT(5, 30)
 	VisibilityModule::set_int64(hash40("weapon"), hash40("weapon_normal"))
 }
-if(0x103780(WorkModule::is_flag(FIGHTER_INSTANCE_WORK_ID_FLAG_DISABLE_FINAL_START_CAMERA), false, false)){
+if(0x103850(WorkModule::is_flag(FIGHTER_INSTANCE_WORK_ID_FLAG_DISABLE_FINAL_START_CAMERA), false, false)){
 	frame(Frame=10)
 	if(is_excute){
 		FT_SET_FINAL_FEAR_FACE(60)
 		REQ_FINAL_START_CAMERA_arg3(hash40("d04final.nuanmb"), false, false)
 		FT_START_CUTIN_arg1(true)
 	}
 	else{
 		if(is_excute){
 			PostureModule::scale(4, 0)
-			0x103780(-1722768878, 2.1)
+			0x103850(-1025220042, 2.1)
 			CAM_ZOOM_IN_arg5(0, 0)
 			FT_START_CUTIN_arg1(true)
 		}
 		frame(Frame=25)
 		if(is_excute){
@@ -31,11 +31,11 @@ frame(Frame=60)
 if(is_excute){
 	AttackModule::clear_all()
 	camera(MA_MSC_CMD_CAMERA_CAM_RESET)
 }
 get_value_float(SO_VAR_FLOAT_LR)
-if(0x103780(464, 0)){
+if(0x103850(464, 0)){
 	if(is_excute){
 		camera(MA_MSC_CMD_CAMERA_CAM_OFFSET, -50, 0, 0)
 		camera(MA_MSC_CMD_CAMERA_CAM_RECT, 60, -56, 18, -8)
 	}
 	else{
