diff --git "a/uit_7.0.0/szerosuit/game_FinalAirStart.txt" "b/uit_8.0.0/szerosuit/game_FinalAirStart.txt"
index 50aedea..5eb1925 100644
--- "a/uit_7.0.0/szerosuit/game_FinalAirStart.txt"
+++ "b/uit_8.0.0/szerosuit/game_FinalAirStart.txt"
@@ -1,30 +1,30 @@
 if(is_excute){
 	CHECK_VALID_FINAL_START_CAMERA(0, 7, 20, 0, 0, 0)
 	SLOW_OPPONENT(4, 45)
 }
-if(0x1168e0(WorkModule::is_flag(FIGHTER_INSTANCE_WORK_ID_FLAG_DISABLE_FINAL_START_CAMERA), false, false)){
+if(0x116a20(WorkModule::is_flag(FIGHTER_INSTANCE_WORK_ID_FLAG_DISABLE_FINAL_START_CAMERA), false, false)){
 	if(is_excute){
 		FT_SET_FINAL_FEAR_FACE(60)
 		REQ_FINAL_START_CAMERA(hash40("d04finalstart.nuanmb"), false)
 		FT_START_CUTIN()
 		SlowModule::set_whole(2, 0)
 	}
 	else{
 		if(is_excute){
 			PostureModule::scale(3, 0)
-			0x1168e0(7406971, 2)
+			0x116a20(-93865576, 2)
 			CAM_ZOOM_IN_arg5(0, 0)
 			FT_START_CUTIN()
 		}
 	}
 }
 frame(Frame=20)
 if(is_excute){
 	SlowModule::clear_whole()
 }
-0x1168e0(WorkModule::is_flag(FIGHTER_INSTANCE_WORK_ID_FLAG_DISABLE_FINAL_START_CAMERA), false, false)
+0x116a20(WorkModule::is_flag(FIGHTER_INSTANCE_WORK_ID_FLAG_DISABLE_FINAL_START_CAMERA), false, false)
 frame(Frame=25)
 if(is_excute){
 	CAM_ZOOM_OUT()
 }
 frame(Frame=30)
