diff --git "a/_10.0.0/inkling/game_TurnRun.txt" "b/_10.1.0/inkling/game_TurnRun.txt"
index 971136b..a3ff73d 100644
--- "a/_10.0.0/inkling/game_TurnRun.txt"
+++ "b/_10.1.0/inkling/game_TurnRun.txt"
@@ -3,12 +3,12 @@ if(method.lib::L2CValue.operator___lib::L2CValueconst__const(WorkModule::is_flag
 		ArticleModule::generate_article(FIGHTER_INKLING_GENERATE_ARTICLE_SQUID)
 	}
 }
 MotionModule::motion_kind()
 MotionModule::frame(0x00000002b0)
-MotionModule::rate(1652099283)
-as_hash__const(1652099283, FIGHTER_INKLING_GENERATE_ARTICLE_SQUID, false)
+MotionModule::rate(-1594717753)
+as_hash__const(-1594717753, FIGHTER_INKLING_GENERATE_ARTICLE_SQUID, false)
 ArticleModule::change_motion()
 if(is_excute){
 	ArticleModule::set_frame(FIGHTER_INKLING_GENERATE_ARTICLE_SQUID)
 	ArticleModule::set_rate(FIGHTER_INKLING_GENERATE_ARTICLE_SQUID)
 	ArticleModule::set_visibility_whole(FIGHTER_INKLING_GENERATE_ARTICLE_SQUID, false)
