diff --git "a/_10.0.0/inkling/game_FinalAir.txt" "b/_10.1.0/inkling/game_FinalAir.txt"
index 69841cc..bc53e33 100644
--- "a/_10.0.0/inkling/game_FinalAir.txt"
+++ "b/_10.1.0/inkling/game_FinalAir.txt"
@@ -20,11 +20,11 @@ if(method.lib::L2CValue.operator___lib::L2CValueconst__const(WorkModule::is_flag
 	}
 	else{
 		if(is_excute){
 			camera(MA_MSC_CMD_CAMERA_CAM_RECT, 0, 10, 0, 0)
 			PostureModule::scale(5, 0)
-			0x13d360(1652099283, 2.5)
+			0x13d360(-1594717753, 2.5)
 			CAM_ZOOM_IN_arg5(0, 0)
 			FT_START_CUTIN()
 		}
 		frame(Frame=35)
 		if(is_excute){
