diff --git "a/mac_4.0.0/littlemac/game_SpecialSJump.txt" "b/mac_5.0.0/littlemac/game_SpecialSJump.txt"
index 1fe664e..53d8b82 100644
--- "a/mac_4.0.0/littlemac/game_SpecialSJump.txt"
+++ "b/mac_5.0.0/littlemac/game_SpecialSJump.txt"
@@ -6,11 +6,11 @@ if(is_excute){
 	HIT_NODE(kneel, HIT_STATUS_XLU)
 	HIT_NODE(kneer, HIT_STATUS_XLU)
 }
 frame(Frame=7)
 if(is_excute){
-	ATTACK(ID=0, Part=0, Bone=top, Damage=0.0, Angle=361, KBG=100, FKB=23, BKB=0, Size=3.0, X=0.0, Y=0.0, Z=10.0, X2=0.0, Y2=0.0, Z2=2.0, Hitlag=0.0, SDI=0.0, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=True, DisableHitlag=True, Direct/Indirect=False, Ground/Air=49912, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_normal, SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=COLLISION_SOUND_ATTR_NONE, Type=ATTACK_REGION_NONE)
+	ATTACK(ID=0, Part=0, Bone=top, Damage=0.0, Angle=361, KBG=100, FKB=23, BKB=0, Size=3.0, X=0.0, Y=0.0, Z=10.0, X2=0.0, Y2=0.0, Z2=2.0, Hitlag=0.0, SDI=0.0, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=True, DisableHitlag=True, Direct/Indirect=False, Ground/Air=50752, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_normal, SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=COLLISION_SOUND_ATTR_NONE, Type=ATTACK_REGION_NONE)
 }
 frame(Frame=18)
 if(is_excute){
 	HIT_NODE(hip, HIT_STATUS_NORMAL)
 	HIT_NODE(legl, HIT_STATUS_NORMAL)
