diff --git "a/2.0.0/yoshi/game_FinalStart.txt" "b/3.0.0/yoshi/game_FinalStart.txt"
index 17f6fc9..d498e6a 100644
--- "a/2.0.0/yoshi/game_FinalStart.txt"
+++ "b/3.0.0/yoshi/game_FinalStart.txt"
@@ -10,11 +10,11 @@ if(0x101bd0(false, false)){
 		REQ_FINAL_START_CAMERA(hash40("d04finalstart.nuanmb"), false)
 	}
 	else{
 		if(is_excute){
 			PostureModule::scale(3, 0)
-			0x101bd0(-1290233065, 2)
+			0x101bd0(482679033, 2)
 			CAM_ZOOM_IN_arg5(0, 0)
 			FT_START_CUTIN()
 		}
 	}
 }
