diff --git "a/_6.0.0/pacman/game_SpecialSDash.txt" "b/_7.0.0/pacman/game_SpecialSDash.txt"
index d528c13..1a4ede0 100644
--- "a/_6.0.0/pacman/game_SpecialSDash.txt"
+++ "b/_7.0.0/pacman/game_SpecialSDash.txt"
@@ -1,6 +1,6 @@
-if(0x119d30(WorkModule__is_flag(FIGHTER_PACMAN_STATUS_SPECIAL_S_WORK_FLAG_EAT_POWER_ESA), False, True)){
+if(0x119d40(WorkModule__is_flag(FIGHTER_PACMAN_STATUS_SPECIAL_S_WORK_FLAG_EAT_POWER_ESA), False, True)){
 	if(is_excute){
 		ATTACK(ID=0, Part=0, Bone=0x0bdd8b0afc, Damage=1.0, Angle=50, KBG=108, FKB=0, BKB=50, Size=6.0, X=0.0, Y=2.8, Z=0.0, Hitlag=1.0, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_SPEED, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_cutup, SFXLevel=ATTACK_SOUND_LEVEL_L, SFXType=COLLISION_SOUND_ATTR_KICK, Type=ATTACK_REGION_BODY)
 	}
 	else{
 		if(is_excute){
