diff --git "a/g_9.0.0/inkling/game_Final.txt" "b/_10.0.0/inkling/game_Final.txt"
index 1fcf792..ff2bcf8 100644
--- "a/g_9.0.0/inkling/game_Final.txt"
+++ "b/_10.0.0/inkling/game_Final.txt"
@@ -19,11 +19,11 @@ if(method.lib::L2CValue.operator___lib::L2CValueconst__const(WorkModule::is_flag
 		FT_START_CUTIN()
 	}
 	else{
 		if(is_excute){
 			PostureModule::scale(5, 0)
-			0x13d330(989672188, 2.5)
+			0x13d360(1652099283, 2.5)
 			CAM_ZOOM_IN_arg5(0, 0)
 			FT_START_CUTIN()
 		}
 		frame(Frame=35)
 		if(is_excute){
