diff --git "a/g_3.0.0/inkling/game_AttackHi4.txt" "b/g_3.1.0/inkling/game_AttackHi4.txt"
index 15f2ea9..df35989 100644
--- "a/g_3.0.0/inkling/game_AttackHi4.txt"
+++ "b/g_3.1.0/inkling/game_AttackHi4.txt"
@@ -9,11 +9,11 @@ if(is_excute){
 	WorkModule__on_flag(Flag=FIGHTER_STATUS_ATTACK_FLAG_START_SMASH_HOLD)
 }
 if(method.lib::L2CValue.operator___lib::L2CValueconst__const(WorkModule__is_flag(FIGHTER_STATUS_ATTACK_FLAG_SMASH_SMASH_HOLD_TO_ATTACK), False, True)){
 	if(is_excute){
 		WorkModule__get_float(FIGHTER_INKLING_GENERATE_ARTICLE_BLASTER, attack_hi4, True, FIGHTER_STATUS_ATTACK_WORK_FLOAT_SMASH_RESTART_FRAME)
-		as_hash__const(-1649488654)
+		as_hash__const(-1063256669)
 		ArticleModule__change_motion()
 	}
 }
 frame(Frame=9)
 if(is_excute){
@@ -22,11 +22,11 @@ if(is_excute){
 wait(Frames=2)
 if(is_excute){
 	AttackModule__clear_all()
 }
 frame(Frame=18)
-if(method.lib::L2CValue.operator___lib::L2CValueconst__const(WorkModule__is_flag(16320), False, True)){
+if(method.lib::L2CValue.operator___lib::L2CValueconst__const(WorkModule__is_flag(16324), False, True)){
 	if(is_excute){
 		ATTACK(ID=0, Part=0, Bone=top, Damage=15.0, Angle=88, KBG=76, FKB=0, BKB=55, Size=9.5, X=0.0, Y=20.0, Z=0.0, Hitlag=0.8, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=0x169c2f9aa4, SFXLevel=ATTACK_SOUND_LEVEL_L, SFXType=COLLISION_SOUND_ATTR_WATER, Type=ATTACK_REGION_WATER)
 		ATTACK(ID=1, Part=0, Bone=top, Damage=10.0, Angle=88, KBG=76, FKB=0, BKB=55, Size=14.0, X=0.0, Y=20.0, Z=0.0, Hitlag=0.5, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=0x169c2f9aa4, SFXLevel=ATTACK_SOUND_LEVEL_M, SFXType=COLLISION_SOUND_ATTR_WATER, Type=ATTACK_REGION_WATER)
 		AttackModule__set_ink_value(ID=0, Ink=120)
 		AttackModule__set_ink_value(ID=1, Ink=100)
