diff --git "a/11.0.0/eflame/game_AppealHiR.txt" "b/12.0.0/eflame/game_AppealHiR.txt"
index 7d596e3..1c5b558 100644
--- "a/11.0.0/eflame/game_AppealHiR.txt"
+++ "b/12.0.0/eflame/game_AppealHiR.txt"
@@ -1,8 +1,8 @@
-WorkModule::is_flag(56180)
-if(0x3cbcf0(false, true)){
+WorkModule::is_flag(56364)
+if(0x3cc3a0(false, true)){
 	if(is_excute){
-		ArticleModule::generate_article(56188)
-		methodlib::L2CValue::as_hash()const(56188, 0x0d323e89b4)
+		ArticleModule::generate_article(56372)
+		methodlib::L2CValue::as_hash()const(56372, 0x0d323e89b4)
 		ArticleModule::change_motion()
 	}
 }
