diff --git "a/11.0.0/eflame/game_CloseSword1F.txt" "b/12.0.0/eflame/game_CloseSword1F.txt"
index 6ae49b3..ab63bcf 100644
--- "a/11.0.0/eflame/game_CloseSword1F.txt"
+++ "b/12.0.0/eflame/game_CloseSword1F.txt"
@@ -1,13 +1,13 @@
-IS_EXIST_ARTICLE(17452)
-if(0x3cbcf0(false, true)){
+IS_EXIST_ARTICLE(17628)
+if(0x3cc3a0(false, true)){
 	if(is_excute){
-		methodlib::L2CValue::as_hash()const(17452, 60296, 0x08183db0f4, 10, 10, false, false, 0, false, true, false)
+		methodlib::L2CValue::as_hash()const(17628, 61324, 0x08183db0f4, 10, 10, false, false, 0, false, true, false)
 		ArticleModule::add_motion_partial()
 	}
 }
 MotionModule::is_changing()
-if(0x3cbcf0(false, true)){
+if(0x3cc3a0(false, true)){
 	if(is_excute){
-		WorkModule::on_flag(Flag=60300)
+		WorkModule::on_flag(Flag=61328)
 	}
 }
