diff --git "a/11.0.0/eflame/game_Win2Wait.txt" "b/12.0.0/eflame/game_Win2Wait.txt"
index 9d86595..73858c3 100644
--- "a/11.0.0/eflame/game_Win2Wait.txt"
+++ "b/12.0.0/eflame/game_Win2Wait.txt"
@@ -1,20 +1,20 @@
 WorkModule::is_flag(FIGHTER_INSTANCE_WORK_ID_FLAG_RESULT_FIGHTER_SINGLE)
-if(0x3cbcf0(false, false)){
+if(0x3cc3a0(false, false)){
 	if(is_excute){
-		methodlib::L2CValue::as_hash()const(56188, 0x0c0c0e06e9)
+		methodlib::L2CValue::as_hash()const(56372, 0x0c0c0e06e9)
 		ArticleModule::change_motion()
 	}
 }
-IS_EXIST_ARTICLE(17452)
-if(0x3cbcf0(false, true)){
+IS_EXIST_ARTICLE(17628)
+if(0x3cc3a0(false, true)){
 	if(is_excute){
-		methodlib::L2CValue::as_hash()const(17452, 60296, 0x07439e926b, 10, 10, false, false, 0, false, true, false)
+		methodlib::L2CValue::as_hash()const(17628, 61324, 0x07439e926b, 10, 10, false, false, 0, false, true, false)
 		ArticleModule::add_motion_partial()
 	}
 }
 MotionModule::is_changing()
-if(0x3cbcf0(false, true)){
+if(0x3cc3a0(false, true)){
 	if(is_excute){
-		WorkModule::on_flag(Flag=60300)
+		WorkModule::on_flag(Flag=61328)
 	}
 }
