diff --git "a/it_10.1.0/szerosuit/game_FinalAirStart.txt" "b/it_11.0.0/szerosuit/game_FinalAirStart.txt"
index 215a5ce..3095376 100644
--- "a/it_10.1.0/szerosuit/game_FinalAirStart.txt"
+++ "b/it_11.0.0/szerosuit/game_FinalAirStart.txt"
@@ -1,33 +1,34 @@
 if(is_excute){
 	CHECK_VALID_FINAL_START_CAMERA(0, 7, 20, 0, 0, 0)
 	SLOW_OPPONENT(4, 45)
 }
-if(0x116d20(WorkModule::is_flag(FIGHTER_INSTANCE_WORK_ID_FLAG_DISABLE_FINAL_START_CAMERA), false, false)){
+if(0x116fa0(WorkModule::is_flag(FIGHTER_INSTANCE_WORK_ID_FLAG_DISABLE_FINAL_START_CAMERA), false, false)){
 	if(is_excute){
 		FT_SET_FINAL_FEAR_FACE(60)
 		REQ_FINAL_START_CAMERA(hash40("d04finalstart.nuanmb"), false)
 		FT_START_CUTIN()
 		SlowModule::set_whole(2, 0)
 	}
 	else{
 		if(is_excute){
 			PostureModule::scale(3, 0)
-			0x116d20(507261018, 2)
+			0x116fa0(1483118158, 2)
 			CAM_ZOOM_IN_arg5(0, 0)
 			FT_START_CUTIN()
 		}
 	}
 }
 frame(Frame=20)
 if(is_excute){
 	SlowModule::clear_whole()
 }
-0x116d20(WorkModule::is_flag(FIGHTER_INSTANCE_WORK_ID_FLAG_DISABLE_FINAL_START_CAMERA), false, false)
-frame(Frame=25)
-if(is_excute){
-	CAM_ZOOM_OUT()
+if(0x116fa0(WorkModule::is_flag(FIGHTER_INSTANCE_WORK_ID_FLAG_DISABLE_FINAL_START_CAMERA), false, false)){
+	frame(Frame=25)
+	if(is_excute){
+		CAM_ZOOM_OUT()
+	}
 }
 frame(Frame=30)
 if(is_excute){
 	WorkModule::on_flag(Flag=FIGHTER_SZEROSUIT_STATUS_FINAL_FLAG_CAMERA_CHANGE_NEAR)
 }
