diff --git "a/9.0.0/buddy/game_SpecialHiStart.txt" "b/0.0.0/buddy/game_SpecialHiStart.txt"
index f059ca3..87ed174 100644
--- "a/9.0.0/buddy/game_SpecialHiStart.txt"
+++ "b/0.0.0/buddy/game_SpecialHiStart.txt"
@@ -1,7 +1,7 @@
 IS_EXIST_ARTICLE(FIGHTER_BUDDY_GENERATE_ARTICLE_PAD)
-if(0x184a20(false, true)){
+if(0x1845c0(false, true)){
 	if(is_excute){
 		ArticleModule::remove_exist(FIGHTER_BUDDY_GENERATE_ARTICLE_PAD)
 	}
 }
 if(is_excute){
