diff --git "a/nk_6.0.0/toonlink/game_AttackDash.txt" "b/nk_7.0.0/toonlink/game_AttackDash.txt"
index e79b6e7..c0fe358 100644
--- "a/nk_6.0.0/toonlink/game_AttackDash.txt"
+++ "b/nk_7.0.0/toonlink/game_AttackDash.txt"
@@ -5,9 +5,11 @@ if(is_excute){
 	ATTACK(ID=3, Part=0, Bone=sword2, Damage=8.0, Angle=65, KBG=70, FKB=0, BKB=70, Size=4.2, X=5.5, Y=0.0, Z=0.0, Hitlag=1.2, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=False, ShieldDamage=0, Trip=0.3, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_cutup, SFXLevel=ATTACK_SOUND_LEVEL_M, SFXType=COLLISION_SOUND_ATTR_TOONLINK_HIT, Type=ATTACK_REGION_SWORD)
 	ATTACK(ID=0, Part=0, Bone=sword2, Damage=8.0, Angle=65, KBG=70, FKB=0, BKB=70, Size=4.0, X=0.0, Y=0.0, Z=0.0, Hitlag=1.2, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=False, ShieldDamage=0, Trip=0.3, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_cutup, SFXLevel=ATTACK_SOUND_LEVEL_M, SFXType=COLLISION_SOUND_ATTR_TOONLINK_HIT, Type=ATTACK_REGION_SWORD)
 	ATTACK(ID=1, Part=0, Bone=arml, Damage=6.0, Angle=65, KBG=70, FKB=0, BKB=70, Size=3.5, X=0.0, Y=0.0, Z=0.0, Hitlag=1.0, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_THRU, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=False, ShieldDamage=0, Trip=0.3, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_cutup, SFXLevel=ATTACK_SOUND_LEVEL_M, SFXType=COLLISION_SOUND_ATTR_TOONLINK_HIT, Type=ATTACK_REGION_SWORD)
 }
 wait(Frames=2)
-FT_MOTION_RATE(FSM=1)
+FT_MOTION_RATE(FSM=0.6186)
 if(is_excute){
 	AttackModule__clear_all()
 }
+frame(Frame=43)
+FT_MOTION_RATE(FSM=1)
