diff --git "a/g_3.1.0/inkling/game_FinalAir.txt" "b/g_4.0.0/inkling/game_FinalAir.txt"
index d3fca76..899c01e 100644
--- "a/g_3.1.0/inkling/game_FinalAir.txt"
+++ "b/g_4.0.0/inkling/game_FinalAir.txt"
@@ -20,11 +20,11 @@ if(method.lib::L2CValue.operator___lib::L2CValueconst__const(WorkModule__is_flag
 	}
 	else{
 		if(is_excute){
 			camera(MA_MSC_CMD_CAMERA_CAM_RECT, 0, 10, 0, 0)
 			PostureModule__scale(5, 0)
-			0x13c860(-1063256669, 2.5)
+			0x13ce40(1617527991, 2.5)
 			CAM_ZOOM_IN_arg5(0, 0)
 			FT_START_CUTIN()
 		}
 		frame(Frame=35)
 		if(is_excute){
