diff --git "a/2.0.0/dolly/game_SpecialAirSFStart.txt" "b/3.0.0/dolly/game_SpecialAirSFStart.txt"
index d963248..16c8739 100644
--- "a/2.0.0/dolly/game_SpecialAirSFStart.txt"
+++ "b/3.0.0/dolly/game_SpecialAirSFStart.txt"
@@ -1,10 +1,10 @@
 frame(Frame=6)
 if(is_excute){
 	WorkModule::on_flag(Flag=FIGHTER_DOLLY_STATUS_SPECIAL_COMMON_WORK_FLAG_DECIDE_STRENGTH)
 }
-if(0x2508c0(FIGHTER_DOLLY_STATUS_SPECIAL_COMMON_WORK_INT_STRENGTH, FIGHTER_DOLLY_STRENGTH_W)){
+if(0x2508e0(FIGHTER_DOLLY_STATUS_SPECIAL_COMMON_WORK_INT_STRENGTH, FIGHTER_DOLLY_STRENGTH_W)){
 	if(is_excute){
 		MotionModule::set_rate(1.5)
 	}
 	else{
 		if(is_excute){
