diff --git "a/.0.0/link/game_FinalAir.txt" "b/.0.0/link/game_FinalAir.txt"
index 01332e5..de87215 100644
--- "a/.0.0/link/game_FinalAir.txt"
+++ "b/.0.0/link/game_FinalAir.txt"
@@ -12,11 +12,11 @@ if(0x1191f0(false, false)){
 		FT_START_CUTIN()
 	}
 	else{
 		if(is_excute){
 			PostureModule::scale(3, 0)
-			0x1191f0(-2097839447, 1.8)
+			0x1191f0(-1788204220, 1.8)
 			CAM_ZOOM_IN_arg5(0, 0)
 			FT_START_CUTIN()
 			camera(MA_MSC_CMD_CAMERA_CAM_RECT, 0, -5, -2, 0)
 		}
 	}
