diff --git "a/1.0.0/simon/game_AttackHi4.txt" "b/2.0.0/simon/game_AttackHi4.txt"
index ab1d129..83d2478 100644
--- "a/1.0.0/simon/game_AttackHi4.txt"
+++ "b/2.0.0/simon/game_AttackHi4.txt"
@@ -10,11 +10,11 @@ frame(Frame=13)
 FT_MOTION_RATE(FSM=0.5)
 frame(Frame=21)
 FT_MOTION_RATE(FSM=1)
 frame(Frame=22)
 if(is_excute){
-	SEARCH(0, 0, hash40("top"), 3.0, 0.0, 18.0, 2.0, 0.0, 52.0, 2.0, COLLISION_KIND_MASK_HIT, HIT_STATUS_MASK_NORMAL, 1, COLLISION_SITUATION_MASK_GA, 59664, COLLISION_PART_MASK_ALL, false)
+	SEARCH(0, 0, hash40("top"), 3.0, 0.0, 18.0, 2.0, 0.0, 52.0, 2.0, COLLISION_KIND_MASK_HIT, HIT_STATUS_MASK_NORMAL, 1, COLLISION_SITUATION_MASK_GA, 60692, COLLISION_PART_MASK_ALL, false)
 }
 frame(Frame=24)
 if(is_excute){
 	battle_object()
 	methodlib::L2CValue::L2CValue(void*)()
