diff --git "a/o_4.0.0/lucario/game_AttackAirLw.txt" "b/o_5.0.0/lucario/game_AttackAirLw.txt"
index 8ddccdf..79141df 100644
--- "a/o_4.0.0/lucario/game_AttackAirLw.txt"
+++ "b/o_5.0.0/lucario/game_AttackAirLw.txt"
@@ -2,11 +2,11 @@ if(is_excute){
 	FighterAreaModuleImpl__enable_fix_jostle_area_xy(3, 3, 8, 2)
 }
 frame(Frame=2)
 if(is_excute){
 	WorkModule__get_float(0, FIGHTER_LUCARIO_INSTANCE_WORK_ID_FLOAT_ATTACK_AIR_LW_SPEED)
-	SET_SPEED_EX(-54244549, KINETIC_ENERGY_RESERVE_ATTRIBUTE_MAIN)
+	SET_SPEED_EX(2100879093, KINETIC_ENERGY_RESERVE_ATTRIBUTE_MAIN)
 	WorkModule__on_flag(Flag=FIGHTER_LUCARIO_ATTACK_AIR_STATUS_WORK_ID_FLAG_DEC_SPEED)
 	KineticModule__suspend_energy(FIGHTER_KINETIC_ENERGY_ID_CONTROL)
 	WorkModule__on_flag(Flag=FIGHTER_STATUS_WORK_ID_FLAG_RESERVE_GRAVITY_STABLE_UNABLE)
 	WorkModule__on_flag(Flag=FIGHTER_STATUS_ATTACK_AIR_FLAG_ENABLE_LANDING)
 }
