diff --git "a/_5.0.0/reflet/game_FinalStart.txt" "b/_6.0.0/reflet/game_FinalStart.txt"
index 7d51a91..dcdd3d2 100644
--- "a/_5.0.0/reflet/game_FinalStart.txt"
+++ "b/_6.0.0/reflet/game_FinalStart.txt"
@@ -20,11 +20,11 @@ if(is_excute){
 	SlowModule__set_whole(6, 0)
 }
 else{
 	if(is_excute){
 		PostureModule__scale(3, 0)
-		0x141df0(-282680905, 2)
+		0x141ef0(-682412891, 2)
 		CAM_ZOOM_IN_arg5(0, 0)
 		camera(MA_MSC_CMD_CAMERA_CAM_OFFSET, 0, 6)
 		FT_START_CUTIN()
 	}
 }
