diff --git "a/3.0.0/zelda/game_FinalStart.txt" "b/3.0.1/zelda/game_FinalStart.txt"
index 1959213..320ba2e 100644
--- "a/3.0.0/zelda/game_FinalStart.txt"
+++ "b/3.0.1/zelda/game_FinalStart.txt"
@@ -10,11 +10,11 @@ if(0x10db70(false, false)){
 		REQ_FINAL_START_CAMERA_arg3(hash40("d04finalstart.nuanmb"), false, false)
 	}
 	else{
 		if(is_excute){
 			PostureModule::scale(3, 0)
-			0x10db70(-439948080, 2)
+			0x10db70(527766696, 2)
 			CAM_ZOOM_IN_arg5(0, 0)
 			FT_START_CUTIN()
 		}
 	}
 }
