diff --git "a/1.0.0/samus/game_FinalAir.txt" "b/2.0.0/samus/game_FinalAir.txt"
index aa9f64f..96a160d 100644
--- "a/1.0.0/samus/game_FinalAir.txt"
+++ "b/2.0.0/samus/game_FinalAir.txt"
@@ -1,7 +1,7 @@
 IS_EXIST_ARTICLE(FIGHTER_SAMUS_GENERATE_ARTICLE_LASER2)
-if(0x112490(false, false)){
+if(0x112510(false, false)){
 	if(is_excute){
 		ArticleModule::shoot(FIGHTER_SAMUS_GENERATE_ARTICLE_LASER, ARTICLE_OPE_TARGET_ALL, false)
 		WorkModule::set_float(14, FIGHTER_SAMUS_STATUS_FINAL_WORK_FLOAT_OY)
 		ArticleModule::generate_article(FIGHTER_SAMUS_GENERATE_ARTICLE_LASER2)
 		WorkModule::set_float(-14, FIGHTER_SAMUS_STATUS_FINAL_WORK_FLOAT_OY)
