diff --git "a/11.0.0/master/game_AttackLw4.txt" "b/12.0.0/master/game_AttackLw4.txt"
index 59c4380..550cbfe 100644
--- "a/11.0.0/master/game_AttackLw4.txt"
+++ "b/12.0.0/master/game_AttackLw4.txt"
@@ -10,11 +10,11 @@ if(is_excute){
 }
 WorkModule::is_flag(FIGHTER_STATUS_ATTACK_FLAG_SMASH_SMASH_HOLD_TO_ATTACK)
 if(methodlib::L2CValue::operator==(lib::L2CValueconst&)const(false, true)){
 	if(is_excute){
 		WorkModule::get_float(FIGHTER_MASTER_GENERATE_ARTICLE_AXE, hash40("attack_lw4"), true, FIGHTER_STATUS_ATTACK_WORK_FLOAT_SMASH_RESTART_FRAME)
-		methodlib::L2CValue::as_hash()const(-1522244318)
+		methodlib::L2CValue::as_hash()const(-1979100278)
 		ArticleModule::change_motion()
 	}
 }
 frame(Frame=5)
 FT_MOTION_RATE(FSM=1.3)
