diff --git "a/3.1.0/samus/game_Final.txt" "b/4.0.0/samus/game_Final.txt"
index 2dc53ff..814b6af 100644
--- "a/3.1.0/samus/game_Final.txt"
+++ "b/4.0.0/samus/game_Final.txt"
@@ -1,7 +1,7 @@
 IS_EXIST_ARTICLE(FIGHTER_SAMUS_GENERATE_ARTICLE_LASER2)
-if(0x111ea0(False, False)){
+if(0x111ec0(False, False)){
 	if(is_excute){
 		ArticleModule__shoot(FIGHTER_SAMUS_GENERATE_ARTICLE_LASER, ARTICLE_OPE_TARGET_ALL, False)
 		WorkModule__set_float(14, FIGHTER_SAMUS_STATUS_FINAL_WORK_FLOAT_OY)
 		ArticleModule__generate_article(FIGHTER_SAMUS_GENERATE_ARTICLE_LASER2)
 		WorkModule__set_float(-14, FIGHTER_SAMUS_STATUS_FINAL_WORK_FLOAT_OY)
