diff --git "a/2.0.0/diddy/game_FinalAirStart.txt" "b/3.0.0/diddy/game_FinalAirStart.txt"
index 00d2a4b..478d92b 100644
--- "a/2.0.0/diddy/game_FinalAirStart.txt"
+++ "b/3.0.0/diddy/game_FinalAirStart.txt"
@@ -10,11 +10,11 @@ if(methodlib::L2CValue::operator==(lib::L2CValueconst&)const(false, false)){
 		REQ_FINAL_START_CAMERA(hash40("d04finalstart.nuanmb"), false)
 	}
 	else{
 		if(is_excute){
 			PostureModule::scale(5, 0)
-			0x148470(-1205851967, 2.1)
+			0x148390(-793131399, 2.1)
 			CAM_ZOOM_IN_arg5(0, 0)
 			FT_START_CUTIN()
 		}
 	}
 }
