diff --git "a/to_9.0.0/murabito/game_SpecialLw1.txt" "b/o_10.0.0/murabito/game_SpecialLw1.txt"
index eb0803a..f690ccb 100644
--- "a/to_9.0.0/murabito/game_SpecialLw1.txt"
+++ "b/o_10.0.0/murabito/game_SpecialLw1.txt"
@@ -1,7 +1,7 @@
 frame(Frame=1)
-FT_MOTION_RATE(FSM=0.661)
+FT_MOTION_RATE(FSM=0.75)
 if(is_excute){
 	WorkModule::on_flag(Flag=FIGHTER_MURABITO_STATUS_SPECIAL_LW_PLANT_FLAG_CHECK_PLANT)
 }
 frame(Frame=2)
 if(is_excute){
