diff --git "a/hter_6.0.0/miifighter/game_FinalAirStart.txt" "b/hter_7.0.0/miifighter/game_FinalAirStart.txt"
index 5bda824..48c4784 100644
--- "a/hter_6.0.0/miifighter/game_FinalAirStart.txt"
+++ "b/hter_7.0.0/miifighter/game_FinalAirStart.txt"
@@ -27,11 +27,11 @@ if(is_excute){
 	SlowModule__set_whole(2, 0)
 }
 else{
 	if(is_excute){
 		PostureModule__scale(3, 0)
-		0x18ac90(-957306308, 2.4)
+		0x18b670(-688717853, 2.4)
 		CAM_ZOOM_IN_arg5(0, 0)
 		FT_START_CUTIN()
 	}
 }
 frame(Frame=20)
