diff --git "a/10.0.0/pickel/game_CatchPull.txt" "b/10.1.0/pickel/game_CatchPull.txt"
index 5c9dee3..6686702 100644
--- "a/10.0.0/pickel/game_CatchPull.txt"
+++ "b/10.1.0/pickel/game_CatchPull.txt"
@@ -1,8 +1,8 @@
 frame(Frame=9)
 if(is_excute){
-	ArticleModule::set_flag(54648, true, 55308)
+	ArticleModule::set_flag(54720, true, 55380)
 }
 frame(Frame=18)
 if(is_excute){
-	ArticleModule::remove_exist(54648)
+	ArticleModule::remove_exist(54720)
 }
