diff --git "a/10.0.0/pickel/game_TurnRun.txt" "b/10.1.0/pickel/game_TurnRun.txt"
index 29fdf78..fdace8a 100644
--- "a/10.0.0/pickel/game_TurnRun.txt"
+++ "b/10.1.0/pickel/game_TurnRun.txt"
@@ -1,8 +1,8 @@
 frame(Frame=9)
 if(is_excute){
-	WorkModule::on_flag(Flag=59180)
+	WorkModule::on_flag(Flag=59268)
 }
 frame(Frame=15)
 if(is_excute){
 	WorkModule::on_flag(Flag=FIGHTER_STATUS_RUN_BRAKE_FLAG_STOP)
 }
