diff --git "a/10.0.0/pickel/game_ThrowB.txt" "b/10.1.0/pickel/game_ThrowB.txt"
index 410974a..4e44de3 100644
--- "a/10.0.0/pickel/game_ThrowB.txt"
+++ "b/10.1.0/pickel/game_ThrowB.txt"
@@ -1,8 +1,8 @@
 if(is_excute){
-	ArticleModule::generate_article(54648)
-	as_hash__const(54648, hash40("throw_b"))
+	ArticleModule::generate_article(54720)
+	as_hash__const(54720, hash40("throw_b"))
 	ArticleModule::change_motion()
 	ATTACK_ABS(Kind=FIGHTER_ATTACK_ABSOLUTE_KIND_THROW, ID=2, Damage=10.0, Angle=55, KBG=66, FKB=0, BKB=70, Hitlag=0.0, Unk=1.0, FacingRestrict=ATTACK_LR_CHECK_F, Unk=0.0, Unk=true, Effect=hash40("collision_attr_normal"), SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=COLLISION_SOUND_ATTR_NONE, Type=ATTACK_REGION_THROW)
 	ATTACK_ABS(Kind=FIGHTER_ATTACK_ABSOLUTE_KIND_CATCH, ID=0, Damage=3.0, Angle=361, KBG=100, FKB=0, BKB=40, Hitlag=0.0, Unk=1.0, FacingRestrict=ATTACK_LR_CHECK_F, Unk=0.0, Unk=true, Effect=hash40("collision_attr_normal"), SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=COLLISION_SOUND_ATTR_NONE, Type=ATTACK_REGION_THROW)
 }
 frame(Frame=22)
