diff --git "a/n_7.0.0/gaogaen/game_SpecialLwStart.txt" "b/n_8.0.0/gaogaen/game_SpecialLwStart.txt"
index ff2da36..877a569 100644
--- "a/n_7.0.0/gaogaen/game_SpecialLwStart.txt"
+++ "b/n_8.0.0/gaogaen/game_SpecialLwStart.txt"
@@ -15,7 +15,7 @@ if(is_excute){
 FT_MOTION_RATE(FSM=0.5)
 frame(Frame=32)
 if(is_excute){
 	notify_event_msc_cmd(hash40("0x2127e37c07"), GROUND_CLIFF_CHECK_KIND_ALWAYS)
 }
-frame(Frame=38)
+frame(Frame=46)
 FT_MOTION_RATE(FSM=1)
