diff --git "a/5.0.0/cloud/game_SpecialS1.txt" "b/6.0.0/cloud/game_SpecialS1.txt"
index 1a04c0b..dfe225d 100644
--- "a/5.0.0/cloud/game_SpecialS1.txt"
+++ "b/6.0.0/cloud/game_SpecialS1.txt"
@@ -4,11 +4,11 @@ if(is_excute){
 frame(Frame=10)
 if(is_excute){
 	WorkModule__on_flag(Flag=FIGHTER_CLOUD_STATUS_SPECIAL_S_FLAG_DETACH_EFFECT)
 }
 get_value_float(SO_VAR_FLOAT_LR)
-if(0x14cbf0(0, 0)){
+if(0x14cb80(0, 0)){
 	if(is_excute){
 		ATTACK(ID=0, Part=0, Bone=top, Damage=4.0, Angle=75, KBG=35, FKB=0, BKB=30, Size=3.5, X=0.0, Y=9.0, Z=12.5, X2=0.0, Y2=6.5, Z2=13.0, Hitlag=0.6, SDI=0.0, Clang/Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=True, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_cutup, SFXLevel=ATTACK_SOUND_LEVEL_M, SFXType=COLLISION_SOUND_ATTR_CUTUP, Type=ATTACK_REGION_SWORD)
 		ATTACK(ID=1, Part=0, Bone=top, Damage=4.0, Angle=112, KBG=35, FKB=0, BKB=32, Size=3.5, X=0.0, Y=17.0, Z=18.0, X2=0.0, Y2=6.5, Z2=20.0, Hitlag=0.6, SDI=0.0, Clang/Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=True, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_G, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_cutup, SFXLevel=ATTACK_SOUND_LEVEL_M, SFXType=COLLISION_SOUND_ATTR_CUTUP, Type=ATTACK_REGION_SWORD)
 		ATTACK(ID=2, Part=0, Bone=top, Damage=4.0, Angle=366, KBG=40, FKB=0, BKB=30, Size=7.0, X=0.0, Y=19.0, Z=14.0, X2=0.0, Y2=9.0, Z2=16.0, Hitlag=0.5, SDI=0.0, Clang/Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=True, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_cutup, SFXLevel=ATTACK_SOUND_LEVEL_M, SFXType=COLLISION_SOUND_ATTR_CUTUP, Type=ATTACK_REGION_SWORD)
 	}
