diff --git "a/11.0.0/pickel/game_SleepStart.txt" "b/12.0.0/pickel/game_SleepStart.txt"
index 4e6cba0..ed01012 100644
--- "a/11.0.0/pickel/game_SleepStart.txt"
+++ "b/12.0.0/pickel/game_SleepStart.txt"
@@ -1,7 +1,7 @@
 if(is_excute){
-	WorkModule::on_flag(Flag=60192)
+	WorkModule::on_flag(Flag=61220)
 }
 frame(Frame=12)
 if(is_excute){
-	ArticleModule::generate_article(60196)
+	ArticleModule::generate_article(61224)
 }
