diff --git "a/8.0.0/kirby/game_Win1.txt" "b/9.0.0/kirby/game_Win1.txt"
index 59df993..ce03b1e 100644
--- "a/8.0.0/kirby/game_Win1.txt"
+++ "b/9.0.0/kirby/game_Win1.txt"
@@ -1,6 +1,6 @@
-0x4a64b0(WorkModule::is_flag(FIGHTER_INSTANCE_WORK_ID_FLAG_RESULT_FIGHTER_SINGLE), false, true)
+0x4e2d70(WorkModule::is_flag(FIGHTER_INSTANCE_WORK_ID_FLAG_RESULT_FIGHTER_SINGLE), false, true)
 if(is_excute){
 	ArticleModule::generate_article(FIGHTER_KIRBY_GENERATE_ARTICLE_WINDUMMY)
 	ArticleModule::generate_article(FIGHTER_KIRBY_GENERATE_ARTICLE_WINDUMMY)
 	as_hash__const(FIGHTER_KIRBY_GENERATE_ARTICLE_WINDUMMY, hash40("win_1"))
 	ArticleModule::change_motion()
