diff --git "a/0.1.0/simon/game_AttackS4.txt" "b/1.0.0/simon/game_AttackS4.txt"
index 89bea01..754e3d3 100644
--- "a/0.1.0/simon/game_AttackS4.txt"
+++ "b/1.0.0/simon/game_AttackS4.txt"
@@ -2,11 +2,11 @@ frame(Frame=6)
 if(is_excute){
 	WorkModule::on_flag(Flag=FIGHTER_STATUS_ATTACK_FLAG_START_SMASH_HOLD)
 }
 frame(Frame=22)
 if(is_excute){
-	SEARCH(0, 0, hash40("top"), 3.0, 0.0, 7.0, 17.0, 0.0, 7.0, 45.5, COLLISION_KIND_MASK_HIT, HIT_STATUS_MASK_NORMAL, 1, COLLISION_SITUATION_MASK_GA, 58740, COLLISION_PART_MASK_ALL, false)
+	SEARCH(0, 0, hash40("top"), 3.0, 0.0, 7.0, 17.0, 0.0, 7.0, 45.5, COLLISION_KIND_MASK_HIT, HIT_STATUS_MASK_NORMAL, 1, COLLISION_SITUATION_MASK_GA, 59664, COLLISION_PART_MASK_ALL, false)
 }
 frame(Frame=24)
 if(is_excute){
 	battle_object()
 	L2CValue()
