diff --git "a/ac_10.1.0/littlemac/game_FinalStart.txt" "b/ac_11.0.0/littlemac/game_FinalStart.txt"
index bfaa239..7299ea8 100644
--- "a/ac_10.1.0/littlemac/game_FinalStart.txt"
+++ "b/ac_11.0.0/littlemac/game_FinalStart.txt"
@@ -16,11 +16,11 @@ if(is_excute){
 	SlowModule::set_whole(3, 0)
 }
 else{
 	if(is_excute){
 		PostureModule::scale(5, 0)
-		0x144420(-1153644838, 2.0)
+		0x1444f0(577296125, 2.0)
 		CAM_ZOOM_IN_arg5(-0.1, 0)
 		FT_START_CUTIN()
 		camera(MA_MSC_CMD_CAMERA_CAM_RECT, 0, 4, 6, 0)
 		camera(MA_MSC_CMD_CAMERA_CAM_OFFSET, 0, 6)
 	}
