diff --git "a/10.1.0/donkey/game_FinalStart.txt" "b/11.0.0/donkey/game_FinalStart.txt"
index e3ba1d2..ade3f9d 100644
--- "a/10.1.0/donkey/game_FinalStart.txt"
+++ "b/11.0.0/donkey/game_FinalStart.txt"
@@ -19,11 +19,11 @@ if(is_excute){
 	SlowModule::clear_whole()
 }
 else{
 	if(is_excute){
 		PostureModule::scale(5, 0)
-		0x11e000(1202330612, 2)
+		0x11e000(-1638354939, 2)
 		CAM_ZOOM_IN_arg5(0, 0)
 		FT_START_CUTIN()
 	}
 }
 frame(Frame=24)
