diff --git "a/7.0.0/buddy/game_SpecialAirNEnd.txt" "b/8.0.0/buddy/game_SpecialAirNEnd.txt"
index b08476b..e6af02a 100644
--- "a/7.0.0/buddy/game_SpecialAirNEnd.txt"
+++ "b/8.0.0/buddy/game_SpecialAirNEnd.txt"
@@ -1,5 +1,5 @@
 frame(Frame=30)
-0x1845c0(FIGHTER_INSTANCE_WORK_ID_INT_KIND, FIGHTER_KIND_KIRBY)
+0x184720(FIGHTER_INSTANCE_WORK_ID_INT_KIND, FIGHTER_KIND_KIRBY)
 if(is_excute){
 	ArticleModule::remove_exist(FIGHTER_BUDDY_GENERATE_ARTICLE_PARTNER)
 }
