diff --git "a/mac_3.1.0/littlemac/game_FinalAirStart.txt" "b/mac_4.0.0/littlemac/game_FinalAirStart.txt"
index 783cb0f..d1866b6 100644
--- "a/mac_3.1.0/littlemac/game_FinalAirStart.txt"
+++ "b/mac_4.0.0/littlemac/game_FinalAirStart.txt"
@@ -16,11 +16,11 @@ if(is_excute){
 	SlowModule__set_whole(3, 0)
 }
 else{
 	if(is_excute){
 		PostureModule__scale(5, 0)
-		0x1441f0(-1510825960, 2.0)
+		0x1442a0(642632499, 2.0)
 		CAM_ZOOM_IN_arg5(-0.1, 0)
 		FT_START_CUTIN()
 		camera(MA_MSC_CMD_CAMERA_CAM_RECT, 0, 4, 6, 0)
 		camera(MA_MSC_CMD_CAMERA_CAM_OFFSET, 0, 6)
 	}
