diff --git "a/11.0.0/tantan/game_AttackLongStartLB1.txt" "b/12.0.0/tantan/game_AttackLongStartLB1.txt"
index 0dc8730..3566c59 100644
--- "a/11.0.0/tantan/game_AttackLongStartLB1.txt"
+++ "b/12.0.0/tantan/game_AttackLongStartLB1.txt"
@@ -1,11 +1,11 @@
 if(is_excute){
 	FighterAreaModuleImpl::enable_fix_jostle_area(3.5, 3.5)
 }
 frame(Frame=9)
 if(is_excute){
-	WorkModule::on_flag(Flag=53632)
+	WorkModule::on_flag(Flag=53816)
 }
 frame(Frame=15)
 if(is_excute){
 	HIT_NO(3, HIT_STATUS_OFF)
 	HIT_NO(9, HIT_STATUS_OFF)
