diff --git "a/13.0.0/tantan_punch1/game_AttackBeamLoop.txt" "b/13.0.1/tantan_punch1/game_AttackBeamLoop.txt"
index f6b648a..15583ae 100644
--- "a/13.0.0/tantan_punch1/game_AttackBeamLoop.txt"
+++ "b/13.0.1/tantan_punch1/game_AttackBeamLoop.txt"
@@ -1,7 +1,7 @@
 IS_EXIST_ARTICLE(WEAPON_TANTAN_PUNCH1_GENERATE_ARTICLE_BEAM)
-if(0x39fa00(false, true)){
+if(0x3a02e0(false, true)){
 	if(is_excute){
 		ArticleModule::remove_exist(WEAPON_TANTAN_PUNCH1_GENERATE_ARTICLE_BEAM)
 	}
 }
 if(is_excute){
