diff --git "a/3.0.0/purin/game_FinalAirStartR.txt" "b/3.0.1/purin/game_FinalAirStartR.txt"
index a054602..726e169 100644
--- "a/3.0.0/purin/game_FinalAirStartR.txt"
+++ "b/3.0.1/purin/game_FinalAirStartR.txt"
@@ -10,11 +10,11 @@ if(0xf85d0(false, false)){
 		REQ_FINAL_START_CAMERA(hash40("d04finalstartr.nuanmb"), false)
 	}
 	else{
 		if(is_excute){
 			PostureModule::scale(5, 0)
-			0xf85d0(1834796743, 2.1)
+			0xf85d0(-467097480, 2.1)
 			CAM_ZOOM_IN_arg5(0, 0)
 			FT_START_CUTIN()
 		}
 	}
 }
