diff --git "a/ta_11.0.0/bayonetta/game_ShootingArmR_AtkOff_AttackAirF3.txt" "b/ta_12.0.0/bayonetta/game_ShootingArmR_AtkOff_AttackAirF3.txt"
index 4f94161..992d20b 100644
--- "a/ta_11.0.0/bayonetta/game_ShootingArmR_AtkOff_AttackAirF3.txt"
+++ "b/ta_12.0.0/bayonetta/game_ShootingArmR_AtkOff_AttackAirF3.txt"
@@ -1,7 +1,7 @@
 WorkModule::is_flag(FIGHTER_BAYONETTA_INSTANCE_WORK_ID_FLAG_SHOOTING_DISABLE_ROOT_ATTACK)
-if(0x1cb230(false, false)){
+if(0x1cb400(false, false)){
 	if(is_excute){
 		AttackModule::clear(ID=FIGHTER_BAYONETTA_SHOOTING_ATTACK_ID_00)
 	}
 }
 if(is_excute){
