diff --git "a/13.0.0/mewtwo/game_FinalAir.txt" "b/13.0.1/mewtwo/game_FinalAir.txt"
index d346910..75a2448 100644
--- "a/13.0.0/mewtwo/game_FinalAir.txt"
+++ "b/13.0.1/mewtwo/game_FinalAir.txt"
@@ -5,11 +5,11 @@ if(is_excute){
 }
 WorkModule::is_flag(FIGHTER_INSTANCE_WORK_ID_FLAG_DISABLE_FINAL_START_CAMERA)
 methodlib::L2CValue::operator==(lib::L2CValueconst&)const(false, false)
 if(is_excute){
 	PostureModule::scale(5, 0)
-	0xe77b0(944930413, 2)
+	0xe77b0(-1411323488, 2)
 	CAM_ZOOM_IN_arg5(0, 0)
 	camera(MA_MSC_CMD_CAMERA_CAM_OFFSET, 0, 2)
 	FT_START_CUTIN()
 }
 frame(Frame=5)
