diff --git "a/_13.0.0/inkling/game_TurnDash.txt" "b/_13.0.1/inkling/game_TurnDash.txt"
index 9bfff4d..a0ff141 100644
--- "a/_13.0.0/inkling/game_TurnDash.txt"
+++ "b/_13.0.1/inkling/game_TurnDash.txt"
@@ -1,7 +1,7 @@
 get_value_float(SO_VAR_FLOAT_LR)
-if(0x13d490(0, 0)){
+if(0x13d500(0, 0)){
 	if(is_excute){
 		WorkModule::on_flag(Flag=FIGHTER_INKLING_INSTANCE_WORK_ID_FLAG_NO_FLIP_SQUID)
 	}
 }
 WorkModule::is_flag(FIGHTER_INKLING_INSTANCE_WORK_ID_FLAG_EXIST_SQUID)
@@ -10,12 +10,12 @@ if(methodlib::L2CValue::operator==(lib::L2CValueconst&)const(false, false)){
 		ArticleModule::generate_article(FIGHTER_INKLING_GENERATE_ARTICLE_SQUID)
 	}
 }
 MotionModule::motion_kind()
 MotionModule::frame(0x00000002b0)
-MotionModule::rate(-718230741)
-methodlib::L2CValue::as_hash()const(-718230741, FIGHTER_INKLING_GENERATE_ARTICLE_SQUID, false)
+MotionModule::rate(-1407754940)
+methodlib::L2CValue::as_hash()const(-1407754940, FIGHTER_INKLING_GENERATE_ARTICLE_SQUID, false)
 ArticleModule::change_motion()
 if(is_excute){
 	ArticleModule::set_frame(FIGHTER_INKLING_GENERATE_ARTICLE_SQUID)
 	ArticleModule::set_rate(FIGHTER_INKLING_GENERATE_ARTICLE_SQUID)
 	ArticleModule::set_visibility_whole(FIGHTER_INKLING_GENERATE_ARTICLE_SQUID, false)
@@ -23,11 +23,11 @@ if(is_excute){
 	StatusModule::status_kind()
 	methodlib::L2CValue::operator==(lib::L2CValueconst&)const(0, FIGHTER_STATUS_KIND_REBIRTH)
 	WorkModule::on_flag(Flag=FIGHTER_INSTANCE_WORK_ID_FLAG_FORCE_LOUPE)
 }
 get_value_float(0, SO_VAR_FLOAT_LR)
-if(0x13d490(0, 0)){
+if(0x13d500(0, 0)){
 	if(is_excute){
 		methodlib::L2CValue::as_hash()const(FIGHTER_INKLING_GENERATE_ARTICLE_SQUID, hash40("turn_dash_l"))
 		ArticleModule::change_motion()
 	}
 	else{
