diff --git "a/tch_9.0.0/gamewatch/game_LandingAirN.txt" "b/ch_10.0.0/gamewatch/game_LandingAirN.txt"
index 94a5e4f..306e144 100644
--- "a/tch_9.0.0/gamewatch/game_LandingAirN.txt"
+++ "b/ch_10.0.0/gamewatch/game_LandingAirN.txt"
@@ -1,9 +1,6 @@
-IS_EXIST_ARTICLE(FIGHTER_GAMEWATCH_GENERATE_ARTICLE_NORMAL_WEAPON)
-if(method.lib::L2CValue.operator___lib::L2CValueconst__const(false, true)){
-	if(is_excute){
-		as_hash__const(FIGHTER_GAMEWATCH_GENERATE_ARTICLE_NORMAL_WEAPON, hash40("landing_air_n"))
-		ArticleModule::change_motion()
-		get_value_float(FIGHTER_GAMEWATCH_GENERATE_ARTICLE_NORMAL_WEAPON, SO_VAR_FLOAT_MOTION_RATE_TOTAL)
-		SET_RATE_ARTICLE_V(256)
-	}
+if(is_excute){
+	as_hash__const(FIGHTER_GAMEWATCH_GENERATE_ARTICLE_NORMAL_WEAPON, hash40("landing_air_n"))
+	ArticleModule::change_motion()
+	get_value_float(FIGHTER_GAMEWATCH_GENERATE_ARTICLE_NORMAL_WEAPON, SO_VAR_FLOAT_MOTION_RATE_TOTAL)
+	SET_RATE_ARTICLE_V(144)
 }
