diff --git "a/5.0.0/krool_ironball/game_Shoot.txt" "b/6.0.0/krool_ironball/game_Shoot.txt"
index 48ac707..913f9de 100644
--- "a/5.0.0/krool_ironball/game_Shoot.txt"
+++ "b/6.0.0/krool_ironball/game_Shoot.txt"
@@ -1,4 +1,4 @@
 if(is_excute){
-	ATTACK(ID=0, Part=0, Bone=top, Damage=13.0, Angle=60, KBG=70, FKB=0, BKB=30, Size=4.5, X=0.0, Y=0.0, Z=0.0, Hitlag=1.0, SDI=0.0, Clang/Rebound=ATTACK_SETOFF_KIND_THRU, FacingRestrict=ATTACK_LR_CHECK_SPEED, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=True, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=False, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_normal, SFXLevel=ATTACK_SOUND_LEVEL_L, SFXType=COLLISION_SOUND_ATTR_PUNCH, Type=ATTACK_REGION_OBJECT)
+	ATTACK(ID=0, Part=0, Bone=top, Damage=13.0, Angle=60, KBG=92, FKB=0, BKB=18, Size=4.5, X=0.0, Y=0.0, Z=0.0, Hitlag=1.0, SDI=0.0, Clang/Rebound=ATTACK_SETOFF_KIND_THRU, FacingRestrict=ATTACK_LR_CHECK_SPEED, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=True, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=False, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_normal, SFXLevel=ATTACK_SOUND_LEVEL_L, SFXType=COLLISION_SOUND_ATTR_PUNCH, Type=ATTACK_REGION_OBJECT)
 	AttackModule__enable_safe_pos()
 }
