diff --git "a/hter_3.0.0/miifighter/game_FinalStart.txt" "b/hter_3.1.0/miifighter/game_FinalStart.txt"
index f20b99f..7a6116b 100644
--- "a/hter_3.0.0/miifighter/game_FinalStart.txt"
+++ "b/hter_3.1.0/miifighter/game_FinalStart.txt"
@@ -27,11 +27,11 @@ if(is_excute){
 	SlowModule__set_whole(2, 0)
 }
 else{
 	if(is_excute){
 		PostureModule__scale(3, 0)
-		0x188e90(-1344469785, 2.4)
+		0x18abf0(1516162342, 2.4)
 		CAM_ZOOM_IN_arg5(0, 0)
 		FT_START_CUTIN()
 	}
 }
 frame(Frame=20)
