diff --git "a/1.0.0/diddy/game_FinalCharge.txt" "b/2.0.0/diddy/game_FinalCharge.txt"
index 488b99c..9eec6bc 100644
--- "a/1.0.0/diddy/game_FinalCharge.txt"
+++ "b/2.0.0/diddy/game_FinalCharge.txt"
@@ -1,8 +1,8 @@
 if(is_excute){
 	PostureModule::scale(10, 0)
-	0x1483f0(103150178, 2.1)
+	0x148470(-1205851967, 2.1)
 	CAM_ZOOM_IN_arg5(0, 0)
 	SLOW_OPPONENT(30, 60)
 }
 frame(Frame=40)
 if(is_excute){
