diff --git "a/1.0.0/diddy/game_FinalStart.txt" "b/2.0.0/diddy/game_FinalStart.txt"
index f138ace..00d2a4b 100644
--- "a/1.0.0/diddy/game_FinalStart.txt"
+++ "b/2.0.0/diddy/game_FinalStart.txt"
@@ -10,11 +10,11 @@ if(methodlib::L2CValue::operator==(lib::L2CValueconst&)const(false, false)){
 		REQ_FINAL_START_CAMERA(hash40("d04finalstart.nuanmb"), false)
 	}
 	else{
 		if(is_excute){
 			PostureModule::scale(5, 0)
-			0x1483f0(103150178, 2.1)
+			0x148470(-1205851967, 2.1)
 			CAM_ZOOM_IN_arg5(0, 0)
 			FT_START_CUTIN()
 		}
 	}
 }
