diff --git "a/3.0.0/pichu/game_AttackAirLw.txt" "b/3.1.0/pichu/game_AttackAirLw.txt"
index 66b6136..61465cc 100644
--- "a/3.0.0/pichu/game_AttackAirLw.txt"
+++ "b/3.1.0/pichu/game_AttackAirLw.txt"
@@ -1,11 +1,11 @@
 if(is_excute){
 	WorkModule__on_flag(Flag=FIGHTER_STATUS_ATTACK_AIR_FLAG_ENABLE_LANDING)
 }
 frame(Frame=14)
 if(is_excute){
-	FT_ADD_DAMAGE(SelfDamage=1)
+	FT_ADD_DAMAGE(SelfDamage=1.5)
 	HIT_NODE(mimir1, HIT_STATUS_XLU)
 	HIT_NODE(mimil1, HIT_STATUS_XLU)
 	ATTACK(ID=0, Part=0, Bone=neck, Damage=13.0, Angle=270, KBG=86, FKB=0, BKB=16, Size=5.5, X=4.5, Y=-1.3, Z=0.0, Hitlag=1.0, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_elec, SFXLevel=ATTACK_SOUND_LEVEL_L, SFXType=COLLISION_SOUND_ATTR_ELEC, Type=ATTACK_REGION_HEAD)
 }
 frame(Frame=18)
