diff --git "a/_9.0.0/pickel/game_JustShieldOff.txt" "b/10.0.0/pickel/game_JustShieldOff.txt"
index 823b73a..02c2bb3 100644
--- "a/_9.0.0/pickel/game_JustShieldOff.txt"
+++ "b/10.0.0/pickel/game_JustShieldOff.txt"
@@ -1,7 +1,7 @@
 if(is_excute){
-	ArticleModule::generate_article(58512)
+	ArticleModule::generate_article(59184)
 }
 frame(Frame=8)
 if(is_excute){
-	ArticleModule::remove_exist(58512)
+	ArticleModule::remove_exist(59184)
 }
