diff --git "a/_7.0.0/pikmin_pikmin/game_SpNTakenOutStart_y.txt" "b/_8.0.0/pikmin_pikmin/game_SpNTakenOutStart_y.txt"
index 91137d4..72a4ade 100644
--- "a/_7.0.0/pikmin_pikmin/game_SpNTakenOutStart_y.txt"
+++ "b/_8.0.0/pikmin_pikmin/game_SpNTakenOutStart_y.txt"
@@ -1,9 +1,9 @@
 frame(Frame=1)
 if(is_excute){
 	WorkModule::get_float(WEAPON_PIKMIN_PIKMIN_STATUS_PULL_OUT_START_WORK_FLOAT_MOT_RATE)
-	MotionModule::set_rate(-1157984911)
+	MotionModule::set_rate(-502677940)
 }
 frame(Frame=5)
 if(is_excute){
 	MotionModule::set_rate(1)
 }
