diff --git "a/me_13.0.0/pzenigame/game_Final.txt" "b/me_13.0.1/pzenigame/game_Final.txt"
index 509b6cf..95ab047 100644
--- "a/me_13.0.0/pzenigame/game_Final.txt"
+++ "b/me_13.0.1/pzenigame/game_Final.txt"
@@ -11,11 +11,11 @@ if(methodlib::L2CValue::operator==(lib::L2CValueconst&)const(false, false)){
 		FT_START_CUTIN_arg1(true)
 	}
 	else{
 		if(is_excute){
 			PostureModule::scale(4, 0)
-			0xe2bd0(1166712247, 2.1)
+			0xe2bd0(1570364760, 2.1)
 			CAM_ZOOM_IN_arg5(0, 0)
 			FT_START_CUTIN_arg1(true)
 		}
 		frame(Frame=25)
 		if(is_excute){
