diff --git "a/tta_7.0.0/bayonetta/game_AttackLw4.txt" "b/tta_8.0.0/bayonetta/game_AttackLw4.txt"
index 5c20bf1..0a27bfc 100644
--- "a/tta_7.0.0/bayonetta/game_AttackLw4.txt"
+++ "b/tta_8.0.0/bayonetta/game_AttackLw4.txt"
@@ -1,11 +1,11 @@
 frame(Frame=9)
 execute(9)
 if(is_excute){
 	WorkModule::on_flag(Flag=FIGHTER_STATUS_ATTACK_FLAG_START_SMASH_HOLD)
 }
-if(0x1905a0(WorkModule::is_flag(FIGHTER_STATUS_ATTACK_FLAG_SMASH_SMASH_HOLD_TO_ATTACK), false, true)){
+if(0x1cbe00(WorkModule::is_flag(FIGHTER_STATUS_ATTACK_FLAG_SMASH_SMASH_HOLD_TO_ATTACK), false, true)){
 	if(is_excute){
 		ArticleModule::generate_article(FIGHTER_BAYONETTA_GENERATE_ARTICLE_WICKEDWEAVELEG)
 	}
 }
 frame(Frame=11)
