diff --git "a/tta_7.0.0/bayonetta/game_SpecialS.txt" "b/tta_8.0.0/bayonetta/game_SpecialS.txt"
index 448df75..0384c64 100644
--- "a/tta_7.0.0/bayonetta/game_SpecialS.txt"
+++ "b/tta_8.0.0/bayonetta/game_SpecialS.txt"
@@ -28,5 +28,7 @@ if(is_excute){
 frame(Frame=40)
 if(is_excute){
 	AttackModule::clear_all()
 	WorkModule::off_flag(Flag=FIGHTER_BAYONETTA_STATUS_WORK_ID_SPECIAL_S_FLAG_WALL_CHECK)
 }
+frame(Frame=40)
+FT_MOTION_RATE(FSM=0.843)
