diff --git "a/_3.0.0/samusd/game_AttackHi4.txt" "b/_3.1.0/samusd/game_AttackHi4.txt"
index 866b32f..1472b22 100644
--- "a/_3.0.0/samusd/game_AttackHi4.txt"
+++ "b/_3.1.0/samusd/game_AttackHi4.txt"
@@ -5,47 +5,62 @@ if(is_excute){
 frame(Frame=11)
 if(is_excute){
 	ATTACK(ID=0, Part=0, Bone=armr, Damage=3.0, Angle=100, KBG=30, FKB=0, BKB=60, Size=4.5, X=-3.5, Y=0.0, Z=0.0, Hitlag=0.65, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_G, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_elec, SFXLevel=ATTACK_SOUND_LEVEL_M, SFXType=COLLISION_SOUND_ATTR_ELEC, Type=ATTACK_REGION_BOMB)
 	ATTACK(ID=1, Part=0, Bone=armr, Damage=3.0, Angle=368, KBG=30, FKB=0, BKB=0, Size=5.0, X=6.5, Y=0.0, Z=0.0, Hitlag=0.65, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_G, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_elec, SFXLevel=ATTACK_SOUND_LEVEL_M, SFXType=COLLISION_SOUND_ATTR_ELEC, Type=ATTACK_REGION_BOMB)
 	ATTACK(ID=2, Part=0, Bone=armr, Damage=3.0, Angle=368, KBG=30, FKB=0, BKB=0, Size=5.0, X=6.5, Y=0.0, Z=0.0, Hitlag=0.65, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_A, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_elec, SFXLevel=ATTACK_SOUND_LEVEL_M, SFXType=COLLISION_SOUND_ATTR_ELEC, Type=ATTACK_REGION_BOMB)
-	as_hash__const(1, top, 5, 25, 8)
+	as_hash__const(1, top, 4, 22, 4)
 	AttackModule__set_vec_target_pos()
-	as_hash__const(2, top, 5, 25, 8)
+	as_hash__const(2, top, 4, 22, 4)
 	AttackModule__set_vec_target_pos()
+	AttackModule__set_no_finish_camera(0, True, False)
+	AttackModule__set_no_finish_camera(1, True, False)
+	AttackModule__set_no_finish_camera(2, True, False)
 }
 wait(Frames=2)
 if(is_excute){
 	AttackModule__clear_all()
 }
 wait(Frames=2)
 if(is_excute){
 	ATTACK(ID=0, Part=0, Bone=armr, Damage=3.0, Angle=130, KBG=30, FKB=0, BKB=30, Size=4.5, X=-3.5, Y=0.0, Z=0.0, Hitlag=0.65, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=True, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_elec, SFXLevel=ATTACK_SOUND_LEVEL_M, SFXType=COLLISION_SOUND_ATTR_ELEC, Type=ATTACK_REGION_BOMB)
-	ATTACK(ID=1, Part=0, Bone=armr, Damage=3.0, Angle=200, KBG=30, FKB=0, BKB=30, Size=5.0, X=6.5, Y=0.0, Z=0.0, Hitlag=0.65, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=True, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_A, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_elec, SFXLevel=ATTACK_SOUND_LEVEL_M, SFXType=COLLISION_SOUND_ATTR_ELEC, Type=ATTACK_REGION_BOMB)
-	ATTACK(ID=2, Part=0, Bone=armr, Damage=3.0, Angle=366, KBG=30, FKB=0, BKB=20, Size=5.0, X=6.5, Y=0.0, Z=0.0, Hitlag=0.65, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=True, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_A, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_elec, SFXLevel=ATTACK_SOUND_LEVEL_M, SFXType=COLLISION_SOUND_ATTR_ELEC, Type=ATTACK_REGION_BOMB)
-	ATTACK(ID=3, Part=0, Bone=armr, Damage=3.0, Angle=160, KBG=30, FKB=0, BKB=30, Size=5.0, X=6.5, Y=0.0, Z=0.0, Hitlag=0.65, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=True, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_G, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_elec, SFXLevel=ATTACK_SOUND_LEVEL_M, SFXType=COLLISION_SOUND_ATTR_ELEC, Type=ATTACK_REGION_BOMB)
+	ATTACK(ID=1, Part=0, Bone=armr, Damage=3.0, Angle=368, KBG=30, FKB=0, BKB=0, Size=5.0, X=6.5, Y=0.0, Z=0.0, Hitlag=0.65, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=True, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_A, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_elec, SFXLevel=ATTACK_SOUND_LEVEL_M, SFXType=COLLISION_SOUND_ATTR_ELEC, Type=ATTACK_REGION_BOMB)
+	ATTACK(ID=2, Part=0, Bone=armr, Damage=3.0, Angle=160, KBG=30, FKB=0, BKB=30, Size=5.0, X=6.5, Y=0.0, Z=0.0, Hitlag=0.65, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=True, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_G, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_elec, SFXLevel=ATTACK_SOUND_LEVEL_M, SFXType=COLLISION_SOUND_ATTR_ELEC, Type=ATTACK_REGION_BOMB)
+	as_hash__const(1, top, -1, 23, 4)
+	AttackModule__set_vec_target_pos()
+	AttackModule__set_no_finish_camera(0, True, False)
+	AttackModule__set_no_finish_camera(1, True, False)
+	AttackModule__set_no_finish_camera(2, True, False)
 }
 wait(Frames=2)
 if(is_excute){
 	AttackModule__clear_all()
 }
 wait(Frames=2)
 if(is_excute){
 	ATTACK(ID=0, Part=0, Bone=armr, Damage=3.0, Angle=130, KBG=30, FKB=0, BKB=40, Size=4.5, X=-3.5, Y=0.0, Z=0.0, Hitlag=0.65, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=True, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_elec, SFXLevel=ATTACK_SOUND_LEVEL_M, SFXType=COLLISION_SOUND_ATTR_ELEC, Type=ATTACK_REGION_BOMB)
-	ATTACK(ID=1, Part=0, Bone=armr, Damage=3.0, Angle=190, KBG=30, FKB=0, BKB=30, Size=5.0, X=6.5, Y=0.0, Z=0.0, Hitlag=0.65, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=True, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_A, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_elec, SFXLevel=ATTACK_SOUND_LEVEL_M, SFXType=COLLISION_SOUND_ATTR_ELEC, Type=ATTACK_REGION_BOMB)
-	ATTACK(ID=2, Part=0, Bone=armr, Damage=3.0, Angle=366, KBG=30, FKB=0, BKB=20, Size=5.0, X=6.5, Y=0.0, Z=0.0, Hitlag=0.65, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=True, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_A, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_elec, SFXLevel=ATTACK_SOUND_LEVEL_M, SFXType=COLLISION_SOUND_ATTR_ELEC, Type=ATTACK_REGION_BOMB)
-	ATTACK(ID=3, Part=0, Bone=armr, Damage=3.0, Angle=160, KBG=30, FKB=0, BKB=30, Size=5.0, X=6.5, Y=0.0, Z=0.0, Hitlag=0.65, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=True, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_G, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_elec, SFXLevel=ATTACK_SOUND_LEVEL_M, SFXType=COLLISION_SOUND_ATTR_ELEC, Type=ATTACK_REGION_BOMB)
+	ATTACK(ID=1, Part=0, Bone=armr, Damage=3.0, Angle=368, KBG=30, FKB=0, BKB=0, Size=5.0, X=6.5, Y=0.0, Z=0.0, Hitlag=0.65, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=True, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_A, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_elec, SFXLevel=ATTACK_SOUND_LEVEL_M, SFXType=COLLISION_SOUND_ATTR_ELEC, Type=ATTACK_REGION_BOMB)
+	ATTACK(ID=2, Part=0, Bone=armr, Damage=3.0, Angle=160, KBG=30, FKB=0, BKB=30, Size=5.0, X=6.5, Y=0.0, Z=0.0, Hitlag=0.65, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=True, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_G, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_elec, SFXLevel=ATTACK_SOUND_LEVEL_M, SFXType=COLLISION_SOUND_ATTR_ELEC, Type=ATTACK_REGION_BOMB)
+	as_hash__const(1, top, -6, 22, 4)
+	AttackModule__set_vec_target_pos()
+	AttackModule__set_no_finish_camera(0, True, False)
+	AttackModule__set_no_finish_camera(1, True, False)
+	AttackModule__set_no_finish_camera(2, True, False)
 }
 wait(Frames=2)
 if(is_excute){
 	AttackModule__clear_all()
 }
 wait(Frames=2)
 if(is_excute){
 	ATTACK(ID=0, Part=0, Bone=armr, Damage=3.0, Angle=120, KBG=30, FKB=0, BKB=30, Size=4.5, X=-3.5, Y=0.0, Z=0.0, Hitlag=0.65, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=True, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_elec, SFXLevel=ATTACK_SOUND_LEVEL_M, SFXType=COLLISION_SOUND_ATTR_ELEC, Type=ATTACK_REGION_BOMB)
-	ATTACK(ID=1, Part=0, Bone=armr, Damage=3.0, Angle=190, KBG=30, FKB=0, BKB=20, Size=5.0, X=6.5, Y=0.0, Z=0.0, Hitlag=0.65, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=True, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_A, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_elec, SFXLevel=ATTACK_SOUND_LEVEL_M, SFXType=COLLISION_SOUND_ATTR_ELEC, Type=ATTACK_REGION_BOMB)
-	ATTACK(ID=2, Part=0, Bone=armr, Damage=3.0, Angle=366, KBG=30, FKB=0, BKB=20, Size=5.0, X=6.5, Y=0.0, Z=0.0, Hitlag=0.65, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=True, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_A, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_elec, SFXLevel=ATTACK_SOUND_LEVEL_M, SFXType=COLLISION_SOUND_ATTR_ELEC, Type=ATTACK_REGION_BOMB)
-	ATTACK(ID=3, Part=0, Bone=armr, Damage=3.0, Angle=170, KBG=30, FKB=0, BKB=30, Size=5.0, X=6.5, Y=0.0, Z=0.0, Hitlag=0.65, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=True, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_G, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_elec, SFXLevel=ATTACK_SOUND_LEVEL_M, SFXType=COLLISION_SOUND_ATTR_ELEC, Type=ATTACK_REGION_BOMB)
+	ATTACK(ID=1, Part=0, Bone=armr, Damage=3.0, Angle=368, KBG=30, FKB=0, BKB=0, Size=5.0, X=6.5, Y=0.0, Z=0.0, Hitlag=0.65, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=True, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_A, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_elec, SFXLevel=ATTACK_SOUND_LEVEL_M, SFXType=COLLISION_SOUND_ATTR_ELEC, Type=ATTACK_REGION_BOMB)
+	ATTACK(ID=2, Part=0, Bone=armr, Damage=3.0, Angle=170, KBG=30, FKB=0, BKB=30, Size=5.0, X=6.5, Y=0.0, Z=0.0, Hitlag=0.65, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=True, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_G, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_elec, SFXLevel=ATTACK_SOUND_LEVEL_M, SFXType=COLLISION_SOUND_ATTR_ELEC, Type=ATTACK_REGION_BOMB)
+	as_hash__const(1, top, -10, 20, 4)
+	AttackModule__set_vec_target_pos()
+	AttackModule__set_no_finish_camera(0, True, False)
+	AttackModule__set_no_finish_camera(1, True, False)
+	AttackModule__set_no_finish_camera(2, True, False)
 }
 wait(Frames=2)
 if(is_excute){
 	AttackModule__clear_all()
 }
