diff --git "a/_9.0.0/reflet/game_FinalStart.txt" "b/10.0.0/reflet/game_FinalStart.txt"
index 2ac6426..dc754ce 100644
--- "a/_9.0.0/reflet/game_FinalStart.txt"
+++ "b/10.0.0/reflet/game_FinalStart.txt"
@@ -20,11 +20,11 @@ if(is_excute){
 	SlowModule::set_whole(6, 0)
 }
 else{
 	if(is_excute){
 		PostureModule::scale(3, 0)
-		0x142050(-1489493555, 2)
+		0x142050(1035765528, 2)
 		CAM_ZOOM_IN_arg5(0, 0)
 		camera(MA_MSC_CMD_CAMERA_CAM_OFFSET, 0, 6)
 		FT_START_CUTIN()
 	}
 }
