diff --git "a/0.0.0/wario/game_FinalAirStart.txt" "b/0.1.0/wario/game_FinalAirStart.txt"
index 145d36f..f0132e5 100644
--- "a/0.0.0/wario/game_FinalAirStart.txt"
+++ "b/0.1.0/wario/game_FinalAirStart.txt"
@@ -11,11 +11,11 @@ if(0x103130(WorkModule::is_flag(FIGHTER_INSTANCE_WORK_ID_FLAG_DISABLE_FINAL_STAR
 		REQ_FINAL_START_CAMERA(hash40("d04finalstart.nuanmb"), false)
 	}
 	else{
 		if(is_excute){
 			PostureModule::scale(4, 0)
-			0x103130(861692302, 2.1)
+			0x103130(-288151298, 2.1)
 			CAM_ZOOM_IN_arg5(0, 0)
 			FT_START_CUTIN()
 		}
 	}
 }
