diff --git "a/_6.0.0/pikmin/game_SpecialAirS.txt" "b/_7.0.0/pikmin/game_SpecialAirS.txt"
index 12926c8..17cc8b3 100644
--- "a/_6.0.0/pikmin/game_SpecialAirS.txt"
+++ "b/_7.0.0/pikmin/game_SpecialAirS.txt"
@@ -2,11 +2,11 @@ if(method.lib::L2CValue.operator___lib::L2CValueconst__const(WorkModule__is_flag
 	frame(Frame=9)
 	if(is_excute){
 		WorkModule__on_flag(Flag=FIGHTER_PIKMIN_STATUS_SPECIAL_S_FLAG_THROW)
 	}
 	else{
-		frame(Frame=-1298307000)
+		frame(Frame=-1157984911)
 		if(is_excute){
 			MotionModule__set_rate(0.6)
 		}
 	}
 }
