diff --git "a/2.0.0/simon/game_FinalAirStart.txt" "b/3.0.0/simon/game_FinalAirStart.txt"
index 8f6489d..4a7a8df 100644
--- "a/2.0.0/simon/game_FinalAirStart.txt"
+++ "b/3.0.0/simon/game_FinalAirStart.txt"
@@ -16,11 +16,11 @@ if(0x11a2c0(32, 0)){
 		REQ_FINAL_START_CAMERA_arg3(hash40("d04finalstart02.nuanmb"), false, false)
 	}
 	else{
 		if(is_excute){
 			PostureModule::scale(5, 0)
-			0x11a2c0(2005439778, 2.1)
+			0x11a2c0(-608094172, 2.1)
 			CAM_ZOOM_IN_arg5(0, 0)
 		}
 		else{
 			if(is_excute){
 				REQ_FINAL_START_CAMERA_arg3(hash40("d04finalstart.nuanmb"), false, false)
