diff --git "a/6.0.0/sheik/game_Catch.txt" "b/7.0.0/sheik/game_Catch.txt"
index 73502c8..2e982c6 100644
--- "a/6.0.0/sheik/game_Catch.txt"
+++ "b/7.0.0/sheik/game_Catch.txt"
@@ -4,12 +4,12 @@ frame(Frame=6)
 if(is_excute){
 	GrabModule__set_rebound(CanCatchRebound=True)
 }
 frame(Frame=7)
 if(is_excute){
-	CATCH(ID=0, Bone=top, Size=3.1, X=0.0, Y=7.0, Z=4.0, X2=0.0, Y2=7.0, Z2=7.9, Status=FIGHTER_STATUS_KIND_CAPTURE_PULLED, Ground/Air=COLLISION_SITUATION_MASK_G)
-	CATCH(ID=1, Bone=top, Size=1.55, X=0.0, Y=7.0, Z=2.45, X2=0.0, Y2=7.0, Z2=9.45, Status=FIGHTER_STATUS_KIND_CAPTURE_PULLED, Ground/Air=COLLISION_SITUATION_MASK_A)
+	CATCH(ID=0, Bone=top, Size=3.1, X=0.0, Y=7.0, Z=4.0, X2=0.0, Y2=7.0, Z2=9.4, Status=FIGHTER_STATUS_KIND_CAPTURE_PULLED, Ground/Air=COLLISION_SITUATION_MASK_G)
+	CATCH(ID=1, Bone=top, Size=1.55, X=0.0, Y=7.0, Z=2.45, X2=0.0, Y2=7.0, Z2=10.95, Status=FIGHTER_STATUS_KIND_CAPTURE_PULLED, Ground/Air=COLLISION_SITUATION_MASK_A)
 }
 game_CaptureCutCommon()
 FT_MOTION_RATE(FSM=1)
 wait(Frames=2)
 if(is_excute){
