diff --git "a/_8.0.0/tantan/game_AttackLongStartLB1.txt" "b/_9.0.0/tantan/game_AttackLongStartLB1.txt"
index 504bb34..02be5c3 100644
--- "a/_8.0.0/tantan/game_AttackLongStartLB1.txt"
+++ "b/_9.0.0/tantan/game_AttackLongStartLB1.txt"
@@ -1,11 +1,11 @@
 if(is_excute){
 	FighterAreaModuleImpl::enable_fix_jostle_area(3.5, 3.5)
 }
 frame(Frame=9)
 if(is_excute){
-	WorkModule::on_flag(Flag=52692)
+	WorkModule::on_flag(Flag=53200)
 }
 frame(Frame=15)
 if(is_excute){
 	HIT_NO(3, HIT_STATUS_OFF)
 	HIT_NO(9, HIT_STATUS_OFF)
