diff --git "a/_6.0.0/ridley/game_GuardOn.txt" "b/_7.0.0/ridley/game_GuardOn.txt"
index 76f476b..6fee090 100644
--- "a/_6.0.0/ridley/game_GuardOn.txt"
+++ "b/_7.0.0/ridley/game_GuardOn.txt"
@@ -2,8 +2,12 @@ frame(Frame=1)
 if(is_excute){
 	HIT_NODE(wingr2, HIT_STATUS_XLU)
 	HIT_NODE(wingl2, HIT_STATUS_XLU)
 }
 frame(Frame=4)
+if(is_excute){
+	HIT_NODE(wingr2, HIT_STATUS_NORMAL)
+}
+frame(Frame=5)
 if(is_excute){
 	HitModule__set_status_all(HIT_STATUS_NORMAL)
 }
