diff --git "a/a_13.0.0/palutena/game_FinalAir.txt" "b/a_13.0.1/palutena/game_FinalAir.txt"
index f93010e..09906ef 100644
--- "a/a_13.0.0/palutena/game_FinalAir.txt"
+++ "b/a_13.0.1/palutena/game_FinalAir.txt"
@@ -12,11 +12,11 @@ if(0xee630(false, false)){
 		REQ_FINAL_START_CAMERA_arg3(hash40("d04final.nuanmb"), false, false)
 	}
 	else{
 		if(is_excute){
 			PostureModule::scale(10, 0)
-			0xee630(-270076305, 2)
+			0xee630(-748688718, 2)
 			CAM_ZOOM_IN_arg5(0, 0)
 			camera(MA_MSC_CMD_CAMERA_CAM_OFFSET, 0, 6)
 			FT_START_CUTIN()
 		}
 	}
