diff --git "a/r_3.0.0/koopajr/0x162aecbdfd.txt" "b/r_3.1.0/koopajr/0x162aecbdfd.txt"
index 70d97d4..6cbf47f 100644
--- "a/r_3.0.0/koopajr/0x162aecbdfd.txt"
+++ "b/r_3.1.0/koopajr/0x162aecbdfd.txt"
@@ -1,25 +1,33 @@
 if(is_excute){
 	ArticleModule__generate_article_enable(FIGHTER_KOOPAJR_GENERATE_ARTICLE_HAMMER)
 }
 frame(Frame=8)
+FT_MOTION_RATE(FSM=2)
 if(is_excute){
-	ATTACK(ID=0, Part=0, Bone=top, Damage=15.0, Angle=361, KBG=90, FKB=0, BKB=50, Size=6.5, X=0.0, Y=8.0, Z=-12.0, Hitlag=1.2, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_B, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_normal, SFXLevel=ATTACK_SOUND_LEVEL_L, SFXType=COLLISION_SOUND_ATTR_KICK, Type=ATTACK_REGION_OBJECT)
-	ATTACK(ID=1, Part=0, Bone=top, Damage=10.0, Angle=361, KBG=90, FKB=0, BKB=50, Size=3.0, X=0.0, Y=8.0, Z=-10.0, X2=0.0, Y2=8.0, Z2=-3.0, Hitlag=1.2, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_B, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_normal, SFXLevel=ATTACK_SOUND_LEVEL_M, SFXType=COLLISION_SOUND_ATTR_KICK, Type=ATTACK_REGION_OBJECT)
+	ATTACK(ID=0, Part=0, Bone=top, Damage=15.0, Angle=361, KBG=90, FKB=0, BKB=50, Size=7.0, X=0.0, Y=8.0, Z=-12.0, Hitlag=1.2, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_B, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_normal, SFXLevel=ATTACK_SOUND_LEVEL_L, SFXType=COLLISION_SOUND_ATTR_KICK, Type=ATTACK_REGION_OBJECT)
+	ATTACK(ID=1, Part=0, Bone=top, Damage=10.0, Angle=361, KBG=90, FKB=0, BKB=50, Size=3.5, X=0.0, Y=8.0, Z=-10.0, X2=0.0, Y2=8.0, Z2=-3.0, Hitlag=1.2, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_B, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_normal, SFXLevel=ATTACK_SOUND_LEVEL_M, SFXType=COLLISION_SOUND_ATTR_KICK, Type=ATTACK_REGION_OBJECT)
 }
 frame(Frame=9)
+FT_MOTION_RATE(FSM=0.5)
 if(is_excute){
 	AttackModule__clear_all()
 }
+frame(Frame=11)
+FT_MOTION_RATE(FSM=1)
 frame(Frame=13)
+FT_MOTION_RATE(FSM=2)
 if(is_excute){
-	ATTACK(ID=0, Part=0, Bone=top, Damage=15.0, Angle=361, KBG=90, FKB=0, BKB=50, Size=6.5, X=0.0, Y=8.0, Z=12.5, Hitlag=1.2, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_normal, SFXLevel=ATTACK_SOUND_LEVEL_L, SFXType=COLLISION_SOUND_ATTR_KICK, Type=ATTACK_REGION_OBJECT)
-	ATTACK(ID=1, Part=0, Bone=top, Damage=10.0, Angle=361, KBG=90, FKB=0, BKB=50, Size=3.0, X=0.0, Y=8.0, Z=10.0, X2=0.0, Y2=8.0, Z2=3.0, Hitlag=1.2, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_normal, SFXLevel=ATTACK_SOUND_LEVEL_M, SFXType=COLLISION_SOUND_ATTR_KICK, Type=ATTACK_REGION_OBJECT)
+	ATTACK(ID=0, Part=0, Bone=top, Damage=15.0, Angle=361, KBG=90, FKB=0, BKB=50, Size=7.0, X=0.0, Y=8.0, Z=12.5, Hitlag=1.2, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_normal, SFXLevel=ATTACK_SOUND_LEVEL_L, SFXType=COLLISION_SOUND_ATTR_KICK, Type=ATTACK_REGION_OBJECT)
+	ATTACK(ID=1, Part=0, Bone=top, Damage=10.0, Angle=361, KBG=90, FKB=0, BKB=50, Size=3.5, X=0.0, Y=8.0, Z=10.0, X2=0.0, Y2=8.0, Z2=3.0, Hitlag=1.2, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_normal, SFXLevel=ATTACK_SOUND_LEVEL_M, SFXType=COLLISION_SOUND_ATTR_KICK, Type=ATTACK_REGION_OBJECT)
 }
 frame(Frame=14)
+FT_MOTION_RATE(FSM=0.5)
 if(is_excute){
 	AttackModule__clear_all()
 }
+frame(Frame=15)
+FT_MOTION_RATE(FSM=1)
 frame(Frame=35)
 if(is_excute){
 	ArticleModule__remove(FIGHTER_KOOPAJR_GENERATE_ARTICLE_HAMMER)
 }
