diff --git "a/3.0.0/kirby/game_Win3.txt" "b/3.1.0/kirby/game_Win3.txt"
index 9d3b35b..c2ddc13 100644
--- "a/3.0.0/kirby/game_Win3.txt"
+++ "b/3.1.0/kirby/game_Win3.txt"
@@ -1,6 +1,6 @@
-0x451cd0(WorkModule__is_flag(FIGHTER_INSTANCE_WORK_ID_FLAG_RESULT_FIGHTER_SINGLE), False, True)
+0x452170(WorkModule__is_flag(FIGHTER_INSTANCE_WORK_ID_FLAG_RESULT_FIGHTER_SINGLE), False, True)
 if(is_excute){
 	ArticleModule__generate_article(FIGHTER_KIRBY_GENERATE_ARTICLE_WINDUMMY)
 	ArticleModule__generate_article(FIGHTER_KIRBY_GENERATE_ARTICLE_WINDUMMY)
 	as_hash__const(FIGHTER_KIRBY_GENERATE_ARTICLE_WINDUMMY, 0x050b44b5ec)
 	ArticleModule__change_motion()
