diff --git "a/_8.0.0/tantan_punch1/game_AttackBeamLoop.txt" "b/_9.0.0/tantan_punch1/game_AttackBeamLoop.txt"
index 206a7e6..08379f1 100644
--- "a/_8.0.0/tantan_punch1/game_AttackBeamLoop.txt"
+++ "b/_9.0.0/tantan_punch1/game_AttackBeamLoop.txt"
@@ -1,13 +1,13 @@
-IS_EXIST_ARTICLE(53460)
-if(0x3a1870(false, true)){
+IS_EXIST_ARTICLE(53960)
+if(0x3a06b0(false, true)){
 	if(is_excute){
-		ArticleModule::remove_exist(53460)
+		ArticleModule::remove_exist(53960)
 	}
 }
 if(is_excute){
-	ArticleModule::generate_article(53460)
-	ArticleModule::shoot_exist(53460, ARTICLE_OPE_TARGET_ALL, false)
+	ArticleModule::generate_article(53960)
+	ArticleModule::shoot_exist(53960, ARTICLE_OPE_TARGET_ALL, false)
 	as_hash__const(0x14cc97af5b, 30, false)
 	ControlModule::set_rumble()
 	QUAKE(CAMERA_QUAKE_KIND_S)
 }
