diff --git "a/dsman_11.0.0/miiswordsman/game_FinalStart.txt" "b/dsman_12.0.0/miiswordsman/game_FinalStart.txt"
index 4e049a5..bc75354 100644
--- "a/dsman_11.0.0/miiswordsman/game_FinalStart.txt"
+++ "b/dsman_12.0.0/miiswordsman/game_FinalStart.txt"
@@ -8,18 +8,18 @@ if(methodlib::L2CValue::operator==(lib::L2CValueconst&)const(false, false)){
 	if(is_excute){
 		FT_SET_FINAL_FEAR_FACE(60)
 	}
 }
 get_value_float(SO_VAR_FLOAT_LR)
-if(0x190e30(0, 0)){
+if(0x191390(0, 0)){
 	if(is_excute){
 		REQ_FINAL_START_CAMERA(hash40("d04finalstart02.nuanmb"), false)
 	}
 	else{
 		if(is_excute){
 			PostureModule::scale(5, 0)
-			0x190e30(502963252, 2.2)
+			0x191390(-27793954, 2.2)
 			CAM_ZOOM_IN_arg5(0, 0)
 		}
 		else{
 			if(is_excute){
 				REQ_FINAL_START_CAMERA(hash40("d04finalstart.nuanmb"), false)
