diff --git "a/g_3.1.0/inkling/game_Attack100.txt" "b/g_4.0.0/inkling/game_Attack100.txt"
index dbce5af..16e3695 100644
--- "a/g_3.1.0/inkling/game_Attack100.txt"
+++ "b/g_4.0.0/inkling/game_Attack100.txt"
@@ -1,6 +1,6 @@
-if(method.lib::L2CValue.operator___lib::L2CValueconst__const(WorkModule__is_flag(16324), False, True)){
+if(method.lib::L2CValue.operator___lib::L2CValueconst__const(WorkModule__is_flag(16560), False, True)){
 	if(is_excute){
 		ATTACK(ID=0, Part=0, Bone=top, Damage=0.4, Angle=361, KBG=10, FKB=0, BKB=12, Size=3.5, X=0.0, Y=8.0, Z=11.0, X2=0.0, Y2=8.0, Z2=11.0, Hitlag=0.5, SDI=0.4, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=0x169c2f9aa4, SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=COLLISION_SOUND_ATTR_INKLING_HIT, Type=ATTACK_REGION_WATER)
 		ATTACK(ID=1, Part=0, Bone=top, Damage=0.4, Angle=361, KBG=10, FKB=0, BKB=7, Size=4.5, X=0.0, Y=8.0, Z=19.5, X2=0.0, Y2=8.0, Z2=11.0, Hitlag=0.5, SDI=0.4, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=0x169c2f9aa4, SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=COLLISION_SOUND_ATTR_INKLING_HIT, Type=ATTACK_REGION_WATER)
 		AttackModule__set_add_reaction_frame(ID=0, Frames=2, Unk=False)
 		AttackModule__set_add_reaction_frame(ID=1, Frames=2, Unk=False)
@@ -14,11 +14,11 @@ if(is_excute){
 	AttackModule__clear_all()
 	WorkModule__on_flag(Flag=FIGHTER_STATUS_ATTACK_FLAG_100_CONTINUE_CHECK)
 }
 wait_loop_clear(0)
 frame(Frame=2)
-if(method.lib::L2CValue.operator___lib::L2CValueconst__const(WorkModule__is_flag(16324), False, True)){
+if(method.lib::L2CValue.operator___lib::L2CValueconst__const(WorkModule__is_flag(16560), False, True)){
 	if(is_excute){
 		ATTACK(ID=0, Part=0, Bone=top, Damage=0.4, Angle=361, KBG=10, FKB=0, BKB=12, Size=3.5, X=0.0, Y=8.0, Z=11.0, X2=0.0, Y2=8.0, Z2=11.0, Hitlag=0.5, SDI=0.4, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=0x169c2f9aa4, SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=COLLISION_SOUND_ATTR_INKLING_HIT, Type=ATTACK_REGION_WATER)
 		ATTACK(ID=1, Part=0, Bone=top, Damage=0.4, Angle=361, KBG=10, FKB=0, BKB=7, Size=4.5, X=0.0, Y=8.0, Z=19.5, X2=0.0, Y2=8.0, Z2=11.0, Hitlag=0.5, SDI=0.4, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=0x169c2f9aa4, SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=COLLISION_SOUND_ATTR_INKLING_HIT, Type=ATTACK_REGION_WATER)
 		AttackModule__set_add_reaction_frame(ID=0, Frames=2, Unk=False)
 		AttackModule__set_add_reaction_frame(ID=1, Frames=2, Unk=False)
@@ -31,11 +31,11 @@ wait(Frames=1)
 if(is_excute){
 	AttackModule__clear_all()
 	WorkModule__on_flag(Flag=FIGHTER_STATUS_ATTACK_FLAG_100_CONTINUE_CHECK)
 }
 frame(0, 4)
-if(method.lib::L2CValue.operator___lib::L2CValueconst__const(WorkModule__is_flag(16324), False, True)){
+if(method.lib::L2CValue.operator___lib::L2CValueconst__const(WorkModule__is_flag(16560), False, True)){
 	if(is_excute){
 		ATTACK(ID=0, Part=0, Bone=top, Damage=0.4, Angle=361, KBG=10, FKB=0, BKB=12, Size=3.5, X=0.0, Y=8.0, Z=11.0, X2=0.0, Y2=8.0, Z2=11.0, Hitlag=0.5, SDI=0.4, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=0x169c2f9aa4, SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=COLLISION_SOUND_ATTR_INKLING_HIT, Type=ATTACK_REGION_WATER)
 		ATTACK(ID=1, Part=0, Bone=top, Damage=0.4, Angle=361, KBG=10, FKB=0, BKB=7, Size=4.5, X=0.0, Y=8.0, Z=19.5, X2=0.0, Y2=8.0, Z2=11.0, Hitlag=0.5, SDI=0.4, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=0x169c2f9aa4, SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=COLLISION_SOUND_ATTR_INKLING_HIT, Type=ATTACK_REGION_WATER)
 		AttackModule__set_add_reaction_frame(ID=0, Frames=2, Unk=False)
 		AttackModule__set_add_reaction_frame(ID=1, Frames=2, Unk=False)
@@ -48,11 +48,11 @@ wait(Frames=1)
 if(is_excute){
 	AttackModule__clear_all()
 	WorkModule__on_flag(Flag=FIGHTER_STATUS_ATTACK_FLAG_100_CONTINUE_CHECK)
 }
 frame(0, 6)
-if(method.lib::L2CValue.operator___lib::L2CValueconst__const(WorkModule__is_flag(16324), False, True)){
+if(method.lib::L2CValue.operator___lib::L2CValueconst__const(WorkModule__is_flag(16560), False, True)){
 	if(is_excute){
 		ATTACK(ID=0, Part=0, Bone=top, Damage=0.4, Angle=361, KBG=10, FKB=0, BKB=12, Size=3.5, X=0.0, Y=8.0, Z=11.0, X2=0.0, Y2=8.0, Z2=11.0, Hitlag=0.5, SDI=0.4, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=0x169c2f9aa4, SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=COLLISION_SOUND_ATTR_INKLING_HIT, Type=ATTACK_REGION_WATER)
 		ATTACK(ID=1, Part=0, Bone=top, Damage=0.4, Angle=361, KBG=10, FKB=0, BKB=7, Size=4.5, X=0.0, Y=8.0, Z=19.5, X2=0.0, Y2=8.0, Z2=11.0, Hitlag=0.5, SDI=0.4, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=0x169c2f9aa4, SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=COLLISION_SOUND_ATTR_INKLING_HIT, Type=ATTACK_REGION_WATER)
 		AttackModule__set_add_reaction_frame(ID=0, Frames=2, Unk=False)
 		AttackModule__set_add_reaction_frame(ID=1, Frames=2, Unk=False)
@@ -65,11 +65,11 @@ wait(Frames=1)
 if(is_excute){
 	AttackModule__clear_all()
 	WorkModule__on_flag(Flag=FIGHTER_STATUS_ATTACK_FLAG_100_CONTINUE_CHECK)
 }
 frame(0, 8)
-if(method.lib::L2CValue.operator___lib::L2CValueconst__const(WorkModule__is_flag(16324), False, True)){
+if(method.lib::L2CValue.operator___lib::L2CValueconst__const(WorkModule__is_flag(16560), False, True)){
 	if(is_excute){
 		ATTACK(ID=0, Part=0, Bone=top, Damage=0.4, Angle=361, KBG=10, FKB=0, BKB=12, Size=3.5, X=0.0, Y=8.0, Z=11.0, X2=0.0, Y2=8.0, Z2=11.0, Hitlag=0.5, SDI=0.4, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=0x169c2f9aa4, SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=COLLISION_SOUND_ATTR_INKLING_HIT, Type=ATTACK_REGION_WATER)
 		ATTACK(ID=1, Part=0, Bone=top, Damage=0.4, Angle=361, KBG=10, FKB=0, BKB=7, Size=4.5, X=0.0, Y=8.0, Z=19.5, X2=0.0, Y2=8.0, Z2=11.0, Hitlag=0.5, SDI=0.4, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=0x169c2f9aa4, SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=COLLISION_SOUND_ATTR_INKLING_HIT, Type=ATTACK_REGION_WATER)
 		AttackModule__set_add_reaction_frame(ID=0, Frames=2, Unk=False)
 		AttackModule__set_add_reaction_frame(ID=1, Frames=2, Unk=False)
@@ -82,11 +82,11 @@ wait(Frames=1)
 if(is_excute){
 	AttackModule__clear_all()
 	WorkModule__on_flag(Flag=FIGHTER_STATUS_ATTACK_FLAG_100_CONTINUE_CHECK)
 }
 frame(0, 10)
-if(method.lib::L2CValue.operator___lib::L2CValueconst__const(WorkModule__is_flag(16324), False, True)){
+if(method.lib::L2CValue.operator___lib::L2CValueconst__const(WorkModule__is_flag(16560), False, True)){
 	if(is_excute){
 		ATTACK(ID=0, Part=0, Bone=top, Damage=0.4, Angle=361, KBG=10, FKB=0, BKB=12, Size=3.5, X=0.0, Y=8.0, Z=11.0, X2=0.0, Y2=8.0, Z2=11.0, Hitlag=0.5, SDI=0.4, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=0x169c2f9aa4, SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=COLLISION_SOUND_ATTR_INKLING_HIT, Type=ATTACK_REGION_WATER)
 		ATTACK(ID=1, Part=0, Bone=top, Damage=0.4, Angle=361, KBG=10, FKB=0, BKB=7, Size=4.5, X=0.0, Y=8.0, Z=19.5, X2=0.0, Y2=8.0, Z2=11.0, Hitlag=0.5, SDI=0.4, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=0x169c2f9aa4, SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=COLLISION_SOUND_ATTR_INKLING_HIT, Type=ATTACK_REGION_WATER)
 		AttackModule__set_add_reaction_frame(ID=0, Frames=2, Unk=False)
 		AttackModule__set_add_reaction_frame(ID=1, Frames=2, Unk=False)
@@ -99,11 +99,11 @@ wait(Frames=1)
 if(is_excute){
 	AttackModule__clear_all()
 	WorkModule__on_flag(Flag=FIGHTER_STATUS_ATTACK_FLAG_100_CONTINUE_CHECK)
 }
 frame(0, 12)
-if(method.lib::L2CValue.operator___lib::L2CValueconst__const(WorkModule__is_flag(16324), False, True)){
+if(method.lib::L2CValue.operator___lib::L2CValueconst__const(WorkModule__is_flag(16560), False, True)){
 	if(is_excute){
 		ATTACK(ID=0, Part=0, Bone=top, Damage=0.4, Angle=361, KBG=10, FKB=0, BKB=12, Size=3.5, X=0.0, Y=8.0, Z=11.0, X2=0.0, Y2=8.0, Z2=11.0, Hitlag=0.5, SDI=0.4, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=0x169c2f9aa4, SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=COLLISION_SOUND_ATTR_INKLING_HIT, Type=ATTACK_REGION_WATER)
 		ATTACK(ID=1, Part=0, Bone=top, Damage=0.4, Angle=361, KBG=10, FKB=0, BKB=7, Size=4.5, X=0.0, Y=8.0, Z=19.5, X2=0.0, Y2=8.0, Z2=11.0, Hitlag=0.5, SDI=0.4, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=0x169c2f9aa4, SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=COLLISION_SOUND_ATTR_INKLING_HIT, Type=ATTACK_REGION_WATER)
 		AttackModule__set_add_reaction_frame(ID=0, Frames=2, Unk=False)
 		AttackModule__set_add_reaction_frame(ID=1, Frames=2, Unk=False)
@@ -116,11 +116,11 @@ wait(Frames=1)
 if(is_excute){
 	AttackModule__clear_all()
 	WorkModule__on_flag(Flag=FIGHTER_STATUS_ATTACK_FLAG_100_CONTINUE_CHECK)
 }
 frame(0, 14)
-if(method.lib::L2CValue.operator___lib::L2CValueconst__const(WorkModule__is_flag(16324), False, True)){
+if(method.lib::L2CValue.operator___lib::L2CValueconst__const(WorkModule__is_flag(16560), False, True)){
 	if(is_excute){
 		ATTACK(ID=0, Part=0, Bone=top, Damage=0.4, Angle=361, KBG=10, FKB=0, BKB=12, Size=3.5, X=0.0, Y=8.0, Z=11.0, X2=0.0, Y2=8.0, Z2=11.0, Hitlag=0.5, SDI=0.4, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=0x169c2f9aa4, SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=COLLISION_SOUND_ATTR_INKLING_HIT, Type=ATTACK_REGION_WATER)
 		ATTACK(ID=1, Part=0, Bone=top, Damage=0.4, Angle=361, KBG=10, FKB=0, BKB=7, Size=4.5, X=0.0, Y=8.0, Z=19.5, X2=0.0, Y2=8.0, Z2=11.0, Hitlag=0.5, SDI=0.4, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=0x169c2f9aa4, SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=COLLISION_SOUND_ATTR_INKLING_HIT, Type=ATTACK_REGION_WATER)
 		AttackModule__set_add_reaction_frame(ID=0, Frames=2, Unk=False)
 		AttackModule__set_add_reaction_frame(ID=1, Frames=2, Unk=False)
