diff --git "a/tta_6.0.0/bayonetta/game_SpecialAirHi.txt" "b/tta_7.0.0/bayonetta/game_SpecialAirHi.txt"
index 35e982a..65231a2 100644
--- "a/tta_6.0.0/bayonetta/game_SpecialAirHi.txt"
+++ "b/tta_7.0.0/bayonetta/game_SpecialAirHi.txt"
@@ -5,11 +5,11 @@ if(is_excute){
 	notify_event_msc_cmd(0x2b7cb92b79, FIGHTER_BAYONETTA_SHOOTING_SLOT_R_LEG, False, False, True, 20)
 	notify_event_msc_cmd(0x2b7cb92b79, FIGHTER_BAYONETTA_SHOOTING_SLOT_L_LEG, False, False, True, 20)
 }
 FT_MOTION_RATE(FSM=0.7)
 frame(Frame=7)
-if(0x190590(WorkModule__is_flag(FIGHTER_BAYONETTA_STATUS_WORK_ID_SPECIAL_HI_FLAG_GROUND_START), False, True)){
+if(0x1905a0(WorkModule__is_flag(FIGHTER_BAYONETTA_STATUS_WORK_ID_SPECIAL_HI_FLAG_GROUND_START), False, True)){
 	if(is_excute){
 		ATTACK(ID=0, Part=0, Bone=top, Damage=3.0, Angle=96, KBG=100, FKB=120, BKB=0, Size=2.5, X=0.0, Y=4.5, Z=0.5, X2=0.0, Y2=9.5, Z2=0.5, Hitlag=0.8, SDI=0.0, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=True, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_normal, SFXLevel=ATTACK_SOUND_LEVEL_L, SFXType=COLLISION_SOUND_ATTR_KICK, Type=ATTACK_REGION_PUNCH)
 		ATTACK(ID=1, Part=0, Bone=top, Damage=3.0, Angle=103, KBG=100, FKB=130, BKB=0, Size=4.0, X=0.0, Y=6.0, Z=5.0, X2=0.0, Y2=8.0, Z2=5.0, Hitlag=0.8, SDI=0.0, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=True, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_normal, SFXLevel=ATTACK_SOUND_LEVEL_L, SFXType=COLLISION_SOUND_ATTR_KICK, Type=ATTACK_REGION_PUNCH)
 	}
 	else{
@@ -55,11 +55,11 @@ if(is_excute){
 	notify_event_msc_cmd(0x2127e37c07, GROUND_CLIFF_CHECK_KIND_ALWAYS)
 	AttackModule__clear_all()
 	WorkModule__on_flag(Flag=FIGHTER_BAYONETTA_INSTANCE_WORK_ID_FLAG_SHOOTING_CHECK_END)
 }
 frame(Frame=31)
-if(0x190590(WorkModule__is_flag(FIGHTER_BAYONETTA_STATUS_WORK_ID_SPECIAL_HI_FLAG_REUSE), False, False)){
+if(0x1905a0(WorkModule__is_flag(FIGHTER_BAYONETTA_STATUS_WORK_ID_SPECIAL_HI_FLAG_REUSE), False, False)){
 	if(is_excute){
 		ATTACK(ID=0, Part=0, Bone=top, Damage=3.0, Angle=32, KBG=80, FKB=0, BKB=45, Size=6.5, X=0.0, Y=26.5, Z=0.0, X2=0.0, Y2=19.0, Z2=0.0, Hitlag=0.8, SDI=2.0, Clang/Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=True, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_normal, SFXLevel=ATTACK_SOUND_LEVEL_L, SFXType=COLLISION_SOUND_ATTR_KICK, Type=ATTACK_REGION_PUNCH)
 		AttackModule__set_add_reaction_frame(ID=0, Frames=1, Unk=False)
 	}
 	else{
