diff --git "a/11.0.0/eflame/game_AttackHi3.txt" "b/12.0.0/eflame/game_AttackHi3.txt"
index 33b90d9..a407656 100644
--- "a/11.0.0/eflame/game_AttackHi3.txt"
+++ "b/12.0.0/eflame/game_AttackHi3.txt"
@@ -1,21 +1,21 @@
 frame(Frame=1)
 FT_MOTION_RATE(FSM=2)
 frame(Frame=3)
 FT_MOTION_RATE(FSM=1)
 frame(Frame=7)
-IS_EXIST_ARTICLE(17452)
-if(0x3cbcf0(false, true)){
+IS_EXIST_ARTICLE(17628)
+if(0x3cc3a0(false, true)){
 	if(is_excute){
-		methodlib::L2CValue::as_hash()const(17452, 60296, 0x07439e926b, 5, 5, false, false, 0, false, true, false)
+		methodlib::L2CValue::as_hash()const(17628, 61324, 0x07439e926b, 5, 5, false, false, 0, false, true, false)
 		ArticleModule::add_motion_partial()
 	}
 }
 MotionModule::is_changing()
-if(0x3cbcf0(false, true)){
+if(0x3cc3a0(false, true)){
 	if(is_excute){
-		WorkModule::on_flag(Flag=60300)
+		WorkModule::on_flag(Flag=61328)
 	}
 }
 frame(0, 9)
 if(is_excute){
 	ATTACK(ID=0, Part=0, Bone=hash40("top"), Damage=9.5, Angle=96, KBG=98, FKB=0, BKB=66, Size=2.6, X=0.0, Y=17.0, Z=-7.5, X2=LUA_VOID, Y2=LUA_VOID, Z2=LUA_VOID, Hitlag=1.25, SDI=1.0, Clang_Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=false, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=false, Absorbable=false, Flinchless=false, DisableHitlag=false, Direct_Hitbox=true, Ground_or_Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=false, Effect=hash40("collision_attr_cutup"), SFXLevel=ATTACK_SOUND_LEVEL_L, SFXType=COLLISION_SOUND_ATTR_CUTUP, Type=ATTACK_REGION_SWORD)
@@ -67,18 +67,18 @@ if(is_excute){
 frame(Frame=15)
 if(is_excute){
 	AttackModule::clear_all()
 }
 frame(Frame=37)
-IS_EXIST_ARTICLE(17452)
-if(0x3cbcf0(false, true)){
+IS_EXIST_ARTICLE(17628)
+if(0x3cc3a0(false, true)){
 	if(is_excute){
-		methodlib::L2CValue::as_hash()const(17452, 60296, 0x08183db0f4, 5, 5, false, false, 0, false, true, false)
+		methodlib::L2CValue::as_hash()const(17628, 61324, 0x08183db0f4, 5, 5, false, false, 0, false, true, false)
 		ArticleModule::add_motion_partial()
 	}
 }
 MotionModule::is_changing()
-if(0x3cbcf0(false, true)){
+if(0x3cc3a0(false, true)){
 	if(is_excute){
-		WorkModule::on_flag(Flag=60300)
+		WorkModule::on_flag(Flag=61328)
 	}
 }
