diff --git "a/0.1.0/ganon/game_FinalStart.txt" "b/1.0.0/ganon/game_FinalStart.txt"
index c9d243f..9eedbbf 100644
--- "a/0.1.0/ganon/game_FinalStart.txt"
+++ "b/1.0.0/ganon/game_FinalStart.txt"
@@ -13,11 +13,11 @@ if(method.lib::L2CValue.operator___lib::L2CValueconst__const(WorkModule::is_flag
 		REQ_FINAL_START_CAMERA(hash40("d04finalstart.nuanmb"), false)
 	}
 	else{
 		if(is_excute){
 			PostureModule::scale(3, 0)
-			0xfde20(1070524262, 1.7)
+			0xffdb0(514459429, 1.7)
 			CAM_ZOOM_IN_arg5(0, 0)
 			FT_START_CUTIN()
 		}
 	}
 }
