diff --git "a/10.1.0/master/game_SpecialHi.txt" "b/11.0.0/master/game_SpecialHi.txt"
index 75c70c1..5c19951 100644
--- "a/10.1.0/master/game_SpecialHi.txt"
+++ "b/11.0.0/master/game_SpecialHi.txt"
@@ -1,7 +1,7 @@
 frame(Frame=1)
-FT_MOTION_RATE(FSM=0.5)
+FT_MOTION_RATE(FSM=0.3)
 if(is_excute){
 	FighterAreaModuleImpl::enable_fix_jostle_area(2, 6)
 	ArticleModule::generate_article(FIGHTER_MASTER_GENERATE_ARTICLE_SWORD)
 	as_hash__const(FIGHTER_MASTER_GENERATE_ARTICLE_SWORD, hash40("special_hi"))
 	ArticleModule::change_motion()
