diff --git "a/o_3.0.0/lucario/game_AttackAirLw.txt" "b/o_3.1.0/lucario/game_AttackAirLw.txt"
index f36b6d7..187034a 100644
--- "a/o_3.0.0/lucario/game_AttackAirLw.txt"
+++ "b/o_3.1.0/lucario/game_AttackAirLw.txt"
@@ -1,28 +1,32 @@
+if(is_excute){
+	FighterAreaModuleImpl__enable_fix_jostle_area_xy(3, 3, 8, 2)
+}
 frame(Frame=2)
 if(is_excute){
 	WorkModule__get_float(0, FIGHTER_LUCARIO_INSTANCE_WORK_ID_FLOAT_ATTACK_AIR_LW_SPEED)
-	SET_SPEED_EX(1860873677, KINETIC_ENERGY_RESERVE_ATTRIBUTE_MAIN)
+	SET_SPEED_EX(-23050645, KINETIC_ENERGY_RESERVE_ATTRIBUTE_MAIN)
 	WorkModule__on_flag(Flag=FIGHTER_LUCARIO_ATTACK_AIR_STATUS_WORK_ID_FLAG_DEC_SPEED)
 	KineticModule__suspend_energy(FIGHTER_KINETIC_ENERGY_ID_CONTROL)
 	WorkModule__on_flag(Flag=FIGHTER_STATUS_WORK_ID_FLAG_RESERVE_GRAVITY_STABLE_UNABLE)
 	WorkModule__on_flag(Flag=FIGHTER_STATUS_ATTACK_AIR_FLAG_ENABLE_LANDING)
 }
 frame(Frame=4)
 if(is_excute){
 	WorkModule__off_flag(Flag=FIGHTER_STATUS_WORK_ID_FLAG_RESERVE_GRAVITY_STABLE_UNABLE)
-	ATTACK(ID=0, Part=0, Bone=top, Damage=5.0, Angle=80, KBG=30, FKB=0, BKB=20, Size=3.8, X=0.0, Y=-2.2, Z=0.0, Hitlag=1.0, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=0x13313725f6, SFXLevel=ATTACK_SOUND_LEVEL_M, SFXType=COLLISION_SOUND_ATTR_LUCARIO, Type=ATTACK_REGION_KICK)
-	ATTACK(ID=1, Part=0, Bone=top, Damage=5.0, Angle=80, KBG=30, FKB=0, BKB=20, Size=4.2, X=0.0, Y=2.0, Z=0.3, Hitlag=1.0, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=0x13313725f6, SFXLevel=ATTACK_SOUND_LEVEL_M, SFXType=COLLISION_SOUND_ATTR_LUCARIO, Type=ATTACK_REGION_KICK)
+	ATTACK(ID=1, Part=0, Bone=top, Damage=5.0, Angle=100, KBG=30, FKB=0, BKB=20, Size=3.8, X=0.0, Y=-2.2, Z=0.0, Hitlag=1.0, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=True, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=0x13313725f6, SFXLevel=ATTACK_SOUND_LEVEL_M, SFXType=COLLISION_SOUND_ATTR_LUCARIO, Type=ATTACK_REGION_KICK)
+	ATTACK(ID=0, Part=0, Bone=top, Damage=5.0, Angle=367, KBG=100, FKB=50, BKB=0, Size=4.6, X=0.0, Y=2.2, Z=0.3, Hitlag=1.0, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=True, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_A, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=0x13313725f6, SFXLevel=ATTACK_SOUND_LEVEL_M, SFXType=COLLISION_SOUND_ATTR_LUCARIO, Type=ATTACK_REGION_KICK)
+	ATTACK(ID=2, Part=0, Bone=top, Damage=5.0, Angle=150, KBG=100, FKB=25, BKB=0, Size=4.6, X=0.0, Y=2.2, Z=0.3, Hitlag=1.0, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=True, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_G, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=0x13313725f6, SFXLevel=ATTACK_SOUND_LEVEL_M, SFXType=COLLISION_SOUND_ATTR_LUCARIO, Type=ATTACK_REGION_KICK)
 }
 wait(Frames=2)
 if(is_excute){
 	AttackModule__clear_all()
 }
 frame(Frame=11)
 if(is_excute){
 	ATTACK(ID=0, Part=0, Bone=top, Damage=6.0, Angle=361, KBG=100, FKB=0, BKB=60, Size=5.8, X=0.0, Y=-2.5, Z=0.0, Hitlag=1.0, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=0x13313725f6, SFXLevel=ATTACK_SOUND_LEVEL_M, SFXType=COLLISION_SOUND_ATTR_LUCARIO, Type=ATTACK_REGION_KICK)
-	ATTACK(ID=1, Part=0, Bone=top, Damage=6.0, Angle=361, KBG=100, FKB=0, BKB=60, Size=4.2, X=0.0, Y=2.0, Z=0.3, Hitlag=1.0, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=0x13313725f6, SFXLevel=ATTACK_SOUND_LEVEL_M, SFXType=COLLISION_SOUND_ATTR_LUCARIO, Type=ATTACK_REGION_KICK)
+	ATTACK(ID=1, Part=0, Bone=top, Damage=6.0, Angle=361, KBG=100, FKB=0, BKB=60, Size=4.6, X=0.0, Y=2.2, Z=0.3, Hitlag=1.0, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=0x13313725f6, SFXLevel=ATTACK_SOUND_LEVEL_M, SFXType=COLLISION_SOUND_ATTR_LUCARIO, Type=ATTACK_REGION_KICK)
 }
 wait(Frames=2)
 if(is_excute){
 	AttackModule__clear_all()
 }
