diff --git "a/o_3.0.0/lucario/game_Attack12.txt" "b/o_3.1.0/lucario/game_Attack12.txt"
index 14a3b94..d321a80 100644
--- "a/o_3.0.0/lucario/game_Attack12.txt"
+++ "b/o_3.1.0/lucario/game_Attack12.txt"
@@ -1,12 +1,17 @@
+frame(Frame=1)
+FT_MOTION_RATE(FSM=1)
 frame(Frame=5)
 if(is_excute){
 	ATTACK(ID=0, Part=0, Bone=top, Damage=2.0, Angle=361, KBG=20, FKB=0, BKB=20, Size=4.0, X=0.0, Y=8.8, Z=8.0, Hitlag=1.2, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=0x13313725f6, SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=COLLISION_SOUND_ATTR_LUCARIO, Type=ATTACK_REGION_PUNCH)
 	ATTACK(ID=1, Part=0, Bone=top, Damage=2.0, Angle=361, KBG=20, FKB=0, BKB=35, Size=3.8, X=0.0, Y=8.8, Z=4.0, Hitlag=1.2, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=0x13313725f6, SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=COLLISION_SOUND_ATTR_LUCARIO, Type=ATTACK_REGION_PUNCH)
 	AttackModule__set_add_reaction_frame(ID=0, Frames=6, Unk=False)
 	AttackModule__set_add_reaction_frame(ID=1, Frames=6, Unk=False)
 }
 wait(Frames=2)
+FT_MOTION_RATE(FSM=0.75)
 if(is_excute){
 	AttackModule__clear_all()
 	WorkModule__on_flag(Flag=FIGHTER_STATUS_ATTACK_FLAG_ENABLE_COMBO)
 }
+frame(Frame=35)
+FT_MOTION_RATE(FSM=1)
