diff --git "a/ta_13.0.0/bayonetta/game_Final.txt" "b/ta_13.0.1/bayonetta/game_Final.txt"
index a39ecd6..ca6f6c6 100644
--- "a/ta_13.0.0/bayonetta/game_Final.txt"
+++ "b/ta_13.0.1/bayonetta/game_Final.txt"
@@ -20,11 +20,11 @@ if(is_excute){
 }
 else{
 	frame(Frame=1)
 	if(is_excute){
 		PostureModule::scale(3, 0)
-		0x1cb400(478658803, 2.0)
+		0x1cb400(-460198162, 2.0)
 		CAM_ZOOM_IN_arg5(0, 0)
 		FT_START_CUTIN()
 	}
 }
 frame(Frame=25)
