diff --git "a/_3.1.0/reflet/game_AttackAirLw.txt" "b/_4.0.0/reflet/game_AttackAirLw.txt"
index 6ff749a..2acaa43 100644
--- "a/_3.1.0/reflet/game_AttackAirLw.txt"
+++ "b/_4.0.0/reflet/game_AttackAirLw.txt"
@@ -21,11 +21,11 @@ if(is_excute){
 frame(Frame=25)
 if(is_excute){
 	AttackModule__clear_all()
 }
 frame(Frame=27)
-if(0x141d30(FIGHTER_REFLET_INSTANCE_WORK_ID_INT_THUNDER_SWORD_CURRENT_POINT, 0)){
+if(0x141de0(FIGHTER_REFLET_INSTANCE_WORK_ID_INT_THUNDER_SWORD_CURRENT_POINT, 0)){
 	if(method.lib::L2CValue.operator___lib::L2CValueconst__const(WorkModule__is_flag(FIGHTER_REFLET_INSTANCE_WORK_ID_FLAG_THUNDER_SWORD_ON), False, True)){
 		if(is_excute){
 			VisibilityModule__set_int64(sword, sword_normal)
 			WorkModule__off_flag(Flag=FIGHTER_REFLET_INSTANCE_WORK_ID_FLAG_THUNDER_SWORD_ON)
 		}
