diff --git "a/_8.0.0/donkey/game_FinalAirStart.txt" "b/_9.0.0/donkey/game_FinalAirStart.txt"
index eeffd55..7d0352c 100644
--- "a/_8.0.0/donkey/game_FinalAirStart.txt"
+++ "b/_9.0.0/donkey/game_FinalAirStart.txt"
@@ -19,11 +19,11 @@ if(is_excute){
 	SlowModule::clear_whole()
 }
 else{
 	if(is_excute){
 		PostureModule::scale(5, 0)
-		0x11d9a0(105213974, 2)
+		0x11e000(2082549632, 2)
 		CAM_ZOOM_IN_arg5(0, 0)
 		FT_START_CUTIN()
 	}
 }
 frame(Frame=24)
