diff --git "a/_8.0.0/donkey/game_FinalAttack.txt" "b/_9.0.0/donkey/game_FinalAttack.txt"
index 11d2b19..6f4db6b 100644
--- "a/_8.0.0/donkey/game_FinalAttack.txt"
+++ "b/_9.0.0/donkey/game_FinalAttack.txt"
@@ -1,11 +1,11 @@
 if(is_excute){
 	CAM_ZOOM_OUT()
 }
 if(method.lib::L2CValue.operator___lib::L2CValueconst__const(WorkModule::is_flag(FIGHTER_DONKEY_STATUS_FINAL_FLAG_HIT_ALL_OPPONENT), false, true)){
 	get_value_float(SO_VAR_FLOAT_LR)
-	0x11d9a0(0, 0)
+	0x11e000(0, 0)
 }
 if(is_excute){
 	WHOLE_HIT(HIT_STATUS_XLU)
 }
 frame(Frame=6)
