diff --git "a/13.0.0/samusd/game_FinalAir.txt" "b/13.0.1/samusd/game_FinalAir.txt"
index 64ad762..5e9ada4 100644
--- "a/13.0.0/samusd/game_FinalAir.txt"
+++ "b/13.0.1/samusd/game_FinalAir.txt"
@@ -1,7 +1,7 @@
 IS_EXIST_ARTICLE(FIGHTER_SAMUSD_GENERATE_ARTICLE_LASER2)
-if(0x1123f0(false, false)){
+if(0x112510(false, false)){
 	if(is_excute){
 		ArticleModule::shoot(FIGHTER_SAMUSD_GENERATE_ARTICLE_LASER, ARTICLE_OPE_TARGET_ALL, false)
 		WorkModule::set_float(14, FIGHTER_SAMUS_STATUS_FINAL_WORK_FLOAT_OY)
 		ArticleModule::generate_article(FIGHTER_SAMUSD_GENERATE_ARTICLE_LASER2)
 		WorkModule::set_float(-14, FIGHTER_SAMUS_STATUS_FINAL_WORK_FLOAT_OY)
