diff --git "a/8.0.0/brave/game_SpecialAirS3.txt" "b/9.0.0/brave/game_SpecialAirS3.txt"
index 11f89c9..dd6a6c9 100644
--- "a/8.0.0/brave/game_SpecialAirS3.txt"
+++ "b/9.0.0/brave/game_SpecialAirS3.txt"
@@ -1,6 +1,6 @@
 frame(Frame=10)
-if(0x21c050(WorkModule::is_flag(FIGHTER_BRAVE_STATUS_SPECIAL_S_FLAG_SUCCESS_SP), false, true)){
+if(0x239e40(WorkModule::is_flag(FIGHTER_BRAVE_STATUS_SPECIAL_S_FLAG_SUCCESS_SP), false, true)){
 	if(is_excute){
 		WorkModule::on_flag(Flag=FIGHTER_BRAVE_STATUS_SPECIAL_S_FLAG_ACTIVATE_LIGHTNING)
 	}
 }
