diff --git "a/0.0/fox/game_SpecialNEnd.txt" "b/0.0/fox/game_SpecialNEnd.txt"
index 4469878..6da60b5 100644
--- "a/0.0/fox/game_SpecialNEnd.txt"
+++ "b/0.0/fox/game_SpecialNEnd.txt"
@@ -1,7 +1,7 @@
 IS_EXIST_ARTICLE(FIGHTER_FOX_GENERATE_ARTICLE_BLASTER)
-if(0x10ce40(False, True)){
+if(0x10ce60(False, True)){
 	if(is_excute){
 		as_hash__const(FIGHTER_FOX_GENERATE_ARTICLE_BLASTER, 0x05130181c4)
 		ArticleModule__change_motion()
 	}
 }
