diff --git "a/6.0.0/dolly/0x11a8026a20.txt" "b/7.0.0/dolly/0x11a8026a20.txt"
index 9a8bd4e..daa0764 100644
--- "a/6.0.0/dolly/0x11a8026a20.txt"
+++ "b/7.0.0/dolly/0x11a8026a20.txt"
@@ -1,6 +1,6 @@
-if(0x24c750(FIGHTER_DOLLY_STATUS_SPECIAL_COMMON_WORK_INT_STRENGTH, FIGHTER_DOLLY_STRENGTH_W)){
+if(0x24c760(FIGHTER_DOLLY_STATUS_SPECIAL_COMMON_WORK_INT_STRENGTH, FIGHTER_DOLLY_STRENGTH_W)){
 	if(is_excute){
 		MotionModule__set_rate(1.2)
 	}
 }
 frame(Frame=25)
