diff --git "a/6.0.0/dolly/0x16ae247550.txt" "b/7.0.0/dolly/0x16ae247550.txt"
index 7311dc6..fdf0ed2 100644
--- "a/6.0.0/dolly/0x16ae247550.txt"
+++ "b/7.0.0/dolly/0x16ae247550.txt"
@@ -1,10 +1,10 @@
 frame(Frame=6)
 if(is_excute){
 	WorkModule__on_flag(Flag=FIGHTER_DOLLY_STATUS_SPECIAL_COMMON_WORK_FLAG_DECIDE_STRENGTH)
 }
-if(0x24c750(FIGHTER_DOLLY_STATUS_SPECIAL_COMMON_WORK_INT_STRENGTH, FIGHTER_DOLLY_STRENGTH_W)){
+if(0x24c760(FIGHTER_DOLLY_STATUS_SPECIAL_COMMON_WORK_INT_STRENGTH, FIGHTER_DOLLY_STRENGTH_W)){
 	if(is_excute){
 		MotionModule__set_rate(1.5)
 	}
 	else{
 		if(is_excute){
