diff --git "a/_4.0.0/lucina/game_FinalAirStart.txt" "b/_5.0.0/lucina/game_FinalAirStart.txt"
index decccbe..c11d2c9 100644
--- "a/_4.0.0/lucina/game_FinalAirStart.txt"
+++ "b/_5.0.0/lucina/game_FinalAirStart.txt"
@@ -16,11 +16,11 @@ if(0x1324d0(0, 0)){
 	}
 	else{
 		if(is_excute){
 			camera(MA_MSC_CMD_CAMERA_CAM_RECT, 0, 10, 0, 0)
 			PostureModule__scale(5, 0)
-			0x1324d0(-726396593, 2)
+			0x1324d0(-457966279, 2)
 			CAM_ZOOM_IN_arg5(0, 0)
 			FT_START_CUTIN()
 		}
 		frame(Frame=25)
 		if(is_excute){
