diff --git "a/a_3.0.0/rosetta_tico/game_AttackS4.txt" "b/a_3.1.0/rosetta_tico/game_AttackS4.txt"
index 42bffe6..a7e3aad 100644
--- "a/a_3.0.0/rosetta_tico/game_AttackS4.txt"
+++ "b/a_3.1.0/rosetta_tico/game_AttackS4.txt"
@@ -3,11 +3,11 @@ if(is_excute){
 	WorkModule__on_flag(Flag=WEAPON_ROSETTA_TICO_STATUS_ATTACK_4_WORK_FLAG_START_SMASH_HOLD)
 }
 frame(Frame=16)
 if(is_excute){
 	ATTACK(ID=0, Part=0, Bone=armr, Damage=7.0, Angle=361, KBG=183, FKB=0, BKB=43, Size=5.0, X=6.0, Y=0.0, Z=0.0, Hitlag=1.7, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_normal, SFXLevel=ATTACK_SOUND_LEVEL_L, SFXType=COLLISION_SOUND_ATTR_KICK, Type=ATTACK_REGION_PUNCH)
-	ATTACK(ID=1, Part=0, Bone=armr, Damage=5.0, Angle=45, KBG=174, FKB=0, BKB=43, Size=2.0, X=0.0, Y=0.0, Z=0.0, X2=-4.0, Y2=0.0, Z2=0.4, Hitlag=1.2, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_normal, SFXLevel=ATTACK_SOUND_LEVEL_L, SFXType=COLLISION_SOUND_ATTR_KICK, Type=ATTACK_REGION_PUNCH)
+	ATTACK(ID=1, Part=0, Bone=armr, Damage=5.0, Angle=361, KBG=219, FKB=0, BKB=23, Size=2.0, X=0.0, Y=0.0, Z=0.0, X2=-4.0, Y2=0.0, Z2=0.4, Hitlag=1.2, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_normal, SFXLevel=ATTACK_SOUND_LEVEL_L, SFXType=COLLISION_SOUND_ATTR_KICK, Type=ATTACK_REGION_PUNCH)
 }
 wait(Frames=2)
 if(is_excute){
 	AttackModule__clear_all()
 }
