diff --git "a/7.0.0/kamui/game_FinalAir.txt" "b/8.0.0/kamui/game_FinalAir.txt"
index 56bfa61..e40f44b 100644
--- "a/7.0.0/kamui/game_FinalAir.txt"
+++ "b/8.0.0/kamui/game_FinalAir.txt"
@@ -19,11 +19,11 @@ if(is_excute){
 	SlowModule::clear_whole()
 }
 else{
 	if(is_excute){
 		PostureModule::scale(4, 0)
-		0x144320(-2027306936, 2.5)
+		0x14e020(-1303854836, 2.5)
 		CAM_ZOOM_IN_arg5(0, 0)
 		camera(MA_MSC_CMD_CAMERA_CAM_OFFSET, 0, 0)
 		FT_START_CUTIN()
 	}
 }
