diff --git "a/3.0.0/ganon/game_FinalStart.txt" "b/3.1.0/ganon/game_FinalStart.txt"
index 171bace..1270a63 100644
--- "a/3.0.0/ganon/game_FinalStart.txt"
+++ "b/3.1.0/ganon/game_FinalStart.txt"
@@ -13,11 +13,11 @@ if(method.lib::L2CValue.operator___lib::L2CValueconst__const(WorkModule__is_flag
 		REQ_FINAL_START_CAMERA(d04finalstart.nuanmb, False)
 	}
 	else{
 		if(is_excute){
 			PostureModule__scale(3, 0)
-			0xfda30(1066641932, 1.7)
+			0xfda30(-1005776680, 1.7)
 			CAM_ZOOM_IN_arg5(0, 0)
 			FT_START_CUTIN()
 		}
 	}
 }
