diff --git "a/ac_10.0.0/littlemac/game_FinalAirAttack.txt" "b/ac_10.1.0/littlemac/game_FinalAirAttack.txt"
index 1e461cc..3a0d7eb 100644
--- "a/ac_10.0.0/littlemac/game_FinalAirAttack.txt"
+++ "b/ac_10.1.0/littlemac/game_FinalAirAttack.txt"
@@ -8,11 +8,11 @@ if(is_excute){
 if(method.lib::L2CValue.operator___lib::L2CValueconst__const(WorkModule::is_flag(FIGHTER_LITTLEMAC_STATUS_FINAL_FLAG_HIT_ALL_OPPONENT), false, true)){
 	get_value_float(SO_VAR_FLOAT_LR)
 	if(0x144420(0, 0)){
 		if(is_excute){
 			PostureModule::scale(300, 0)
-			0x144420(-6214052, 2.5)
+			0x144420(-1153644838, 2.5)
 			CAM_ZOOM_IN_FINAL_arg13(-30, 10, -20, -5, 70, true, FIGHTER_LITTLEMAC_STATUS_FINAL_WORK_INT_ATTACK_HIT_OBJECT_ID, 0, 20, 5)
 		}
 		else{
 			if(is_excute){
 				CAM_ZOOM_IN_FINAL_arg13(300, 0, 2.5, 30, 10, 20, -5, 70, true, FIGHTER_LITTLEMAC_STATUS_FINAL_WORK_INT_ATTACK_HIT_OBJECT_ID, 0, -15, 5)
@@ -59,16 +59,16 @@ if(is_excute){
 	AttackModule::set_final_finish_cut_in(0, true, false, -1, false)
 }
 if(method.lib::L2CValue.operator___lib::L2CValueconst__const(WorkModule::is_flag(FIGHTER_LITTLEMAC_STATUS_FINAL_FLAG_HIT_ALL_OPPONENT), false, true)){
 	if(is_excute){
 		PostureModule::scale(2, 0)
-		0x144420(-6214052, 2.5)
+		0x144420(-1153644838, 2.5)
 		CAM_ZOOM_IN_arg5(0, 0)
 	}
 }
 get_value_float(SO_VAR_FLOAT_LR)
-if(0x144420(1417224, 0)){
+if(0x144420(1417232, 0)){
 	if(is_excute){
 		camera(MA_MSC_CMD_CAMERA_CAM_OFFSET, -10, 10)
 	}
 	else{
 		if(is_excute){
