diff --git "a/5.0.0/diddy/game_FinalStart.txt" "b/6.0.0/diddy/game_FinalStart.txt"
index 45a59b9..7dba9da 100644
--- "a/5.0.0/diddy/game_FinalStart.txt"
+++ "b/6.0.0/diddy/game_FinalStart.txt"
@@ -9,11 +9,11 @@ if(method.lib::L2CValue.operator___lib::L2CValueconst__const(WorkModule__is_flag
 		REQ_FINAL_START_CAMERA(d04finalstart.nuanmb, False)
 	}
 	else{
 		if(is_excute){
 			PostureModule__scale(5, 0)
-			0x13c530(242694103, 2.1)
+			0x13c540(-903504264, 2.1)
 			CAM_ZOOM_IN_arg5(0, 0)
 			FT_START_CUTIN()
 		}
 	}
 }
