diff --git "a/_9.0.0/pikmin/game_SpecialS.txt" "b/10.0.0/pikmin/game_SpecialS.txt"
index 6b4f781..5a16630 100644
--- "a/_9.0.0/pikmin/game_SpecialS.txt"
+++ "b/10.0.0/pikmin/game_SpecialS.txt"
@@ -2,11 +2,11 @@ if(method.lib::L2CValue.operator___lib::L2CValueconst__const(WorkModule::is_flag
 	frame(Frame=9)
 	if(is_excute){
 		WorkModule::on_flag(Flag=FIGHTER_PIKMIN_STATUS_SPECIAL_S_FLAG_THROW)
 	}
 	else{
-		frame(Frame=617544972)
+		frame(Frame=2082482820)
 		if(is_excute){
 			MotionModule::set_rate(0.6)
 		}
 	}
 }
