diff --git "a/_8.0.0/pacman/game_SpecialHiStart.txt" "b/_9.0.0/pacman/game_SpecialHiStart.txt"
index 23c34b9..8ad28b4 100644
--- "a/_8.0.0/pacman/game_SpecialHiStart.txt"
+++ "b/_9.0.0/pacman/game_SpecialHiStart.txt"
@@ -1,4 +1,4 @@
-0x119ec0(WorkModule::is_flag(FIGHTER_PACMAN_INSTANCE_WORK_ID_FLAG_SPECIAL_HI_TRAMPOLINE_JUMP), false, true)
+0x119ed0(WorkModule::is_flag(FIGHTER_PACMAN_INSTANCE_WORK_ID_FLAG_SPECIAL_HI_TRAMPOLINE_JUMP), false, true)
 if(is_excute){
 	ArticleModule::generate_article(FIGHTER_PACMAN_GENERATE_ARTICLE_TRAMPOLINE)
 }
