diff --git "a/g_4.0.0/inkling/0x152edc65bb.txt" "b/g_5.0.0/inkling/0x152edc65bb.txt"
index cbc9f96..490fbb4 100644
--- "a/g_4.0.0/inkling/0x152edc65bb.txt"
+++ "b/g_5.0.0/inkling/0x152edc65bb.txt"
@@ -6,12 +6,12 @@ if(method.lib::L2CValue.operator___lib::L2CValueconst__const(WorkModule__is_flag
 		ArticleModule__generate_article(FIGHTER_INKLING_GENERATE_ARTICLE_SQUID)
 	}
 }
 MotionModule__motion_kind()
 MotionModule__frame(0x00000002b0)
-MotionModule__rate(1617527991)
-as_hash__const(1617527991, FIGHTER_INKLING_GENERATE_ARTICLE_SQUID, False)
+MotionModule__rate(1535628542)
+as_hash__const(1535628542, FIGHTER_INKLING_GENERATE_ARTICLE_SQUID, False)
 ArticleModule__change_motion()
 if(is_excute){
 	ArticleModule__set_frame(FIGHTER_INKLING_GENERATE_ARTICLE_SQUID)
 	ArticleModule__set_rate(FIGHTER_INKLING_GENERATE_ARTICLE_SQUID)
 	ArticleModule__set_visibility_whole(FIGHTER_INKLING_GENERATE_ARTICLE_SQUID, False)
@@ -19,11 +19,11 @@ if(is_excute){
 	StatusModule__status_kind()
 	operator___lib::L2CValueconst__const(0, FIGHTER_STATUS_KIND_REBIRTH)
 	WorkModule__on_flag(Flag=FIGHTER_INSTANCE_WORK_ID_FLAG_FORCE_LOUPE)
 }
 get_value_float(0, SO_VAR_FLOAT_LR)
-if(0x13ce40(0, 0)){
+if(0x13cf10(0, 0)){
 	if(is_excute){
 		as_hash__const(FIGHTER_INKLING_GENERATE_ARTICLE_SQUID, 0x14a7eb92f2)
 		ArticleModule__change_motion()
 		get_value_float(FIGHTER_INKLING_GENERATE_ARTICLE_SQUID, SO_VAR_FLOAT_MOTION_RATE)
 		SET_RATE_ARTICLE(144)
