diff --git "a/g_4.0.0/inkling/game_Final.txt" "b/g_5.0.0/inkling/game_Final.txt"
index 9e20a33..696999d 100644
--- "a/g_4.0.0/inkling/game_Final.txt"
+++ "b/g_5.0.0/inkling/game_Final.txt"
@@ -19,11 +19,11 @@ if(method.lib::L2CValue.operator___lib::L2CValueconst__const(WorkModule__is_flag
 		FT_START_CUTIN()
 	}
 	else{
 		if(is_excute){
 			PostureModule__scale(5, 0)
-			0x13ce40(1617527991, 2.5)
+			0x13cf10(1535628542, 2.5)
 			CAM_ZOOM_IN_arg5(0, 0)
 			FT_START_CUTIN()
 		}
 		frame(Frame=35)
 		if(is_excute){
