diff --git "a/tta_9.0.0/bayonetta/game_ShootingArmR_AtkOn_AttackS4.txt" "b/ta_10.0.0/bayonetta/game_ShootingArmR_AtkOn_AttackS4.txt"
index 422ebb9..648e414 100644
--- "a/tta_9.0.0/bayonetta/game_ShootingArmR_AtkOn_AttackS4.txt"
+++ "b/ta_10.0.0/bayonetta/game_ShootingArmR_AtkOn_AttackS4.txt"
@@ -1,6 +1,6 @@
-if(0x1cb230(WorkModule::is_flag(FIGHTER_BAYONETTA_INSTANCE_WORK_ID_FLAG_SHOOTING_DISABLE_ROOT_ATTACK), false, false)){
+if(0x1905a0(WorkModule::is_flag(FIGHTER_BAYONETTA_INSTANCE_WORK_ID_FLAG_SHOOTING_DISABLE_ROOT_ATTACK), false, false)){
 	if(is_excute){
 		ATTACK(ID=FIGHTER_BAYONETTA_SHOOTING_ATTACK_ID_00, Part=0, Bone=hash40("top"), Damage=0.5, Angle=361, KBG=100, FKB=0, BKB=10, Size=2.5, X=0.0, Y=9.5, Z=10.0, X2=0.0, Y2=9.5, Z2=14.0, Hitlag=0.0, SDI=0.0, Clang_Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=false, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=false, Absorbable=false, Flinchless=false, DisableHitlag=false, Direct_Hitbox=false, Ground_or_Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=false, Effect=hash40("collision_attr_normal_bullet"), SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=COLLISION_SOUND_ATTR_BAYONETTA_HIT_01, Type=ATTACK_REGION_NONE)
 	}
 }
 if(is_excute){
