diff --git "a/0.0/fox/game_FinalAirLockOn.txt" "b/0.1/fox/game_FinalAirLockOn.txt"
index 9bbbfd1..f7c29da 100644
--- "a/0.0/fox/game_FinalAirLockOn.txt"
+++ "b/0.1/fox/game_FinalAirLockOn.txt"
@@ -27,11 +27,11 @@ if(0x10d030(false, false)){
 		REQ_FINAL_START_CAMERA_arg3(hash40("d04finallockon.nuanmb"), false, false)
 	}
 	else{
 		if(is_excute){
 			PostureModule::scale(4, 0)
-			0x10d030(504346932, 1.2)
+			0x10d030(999741611, 1.2)
 			CAM_ZOOM_IN_arg5(0, 0)
 			FT_START_CUTIN()
 		}
 	}
 }
