diff --git "a/6.0.0/buddy/game_SpecialHiStart.txt" "b/7.0.0/buddy/game_SpecialHiStart.txt"
index 0578275..24f78a5 100644
--- "a/6.0.0/buddy/game_SpecialHiStart.txt"
+++ "b/7.0.0/buddy/game_SpecialHiStart.txt"
@@ -1,7 +1,7 @@
 IS_EXIST_ARTICLE(FIGHTER_BUDDY_GENERATE_ARTICLE_PAD)
-if(0x184410(False, True)){
+if(0x1845c0(False, True)){
 	if(is_excute){
 		ArticleModule__remove_exist(FIGHTER_BUDDY_GENERATE_ARTICLE_PAD)
 	}
 }
 if(is_excute){
