diff --git "a/hter_4.0.0/miifighter/game_SpecialAirS3Dash.txt" "b/hter_5.0.0/miifighter/game_SpecialAirS3Dash.txt"
index 524f005..35565ac 100644
--- "a/hter_4.0.0/miifighter/game_SpecialAirS3Dash.txt"
+++ "b/hter_5.0.0/miifighter/game_SpecialAirS3Dash.txt"
@@ -2,15 +2,15 @@ frame(Frame=6)
 if(is_excute){
 	FighterAreaModuleImpl__enable_fix_jostle_area_xy(3, 3, 6, 3)
 }
 frame(Frame=9)
 if(is_excute){
-	ATTACK(ID=0, Part=0, Bone=top, Damage=0.0, Angle=350, KBG=100, FKB=40, BKB=0, Size=3.0, X=0.0, Y=4.0, Z=3.0, X2=0.0, Y2=9.0, Z2=3.0, Hitlag=0.0, SDI=0.0, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=2, Reflectable=False, Absorbable=False, Flinchless=True, DisableHitlag=True, Direct/Indirect=False, Ground/Air=49916, Hitbits=COLLISION_CATEGORY_MASK_FIGHTER, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_normal, SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=COLLISION_SOUND_ATTR_NONE, Type=ATTACK_REGION_THROW)
+	ATTACK(ID=0, Part=0, Bone=top, Damage=0.0, Angle=350, KBG=100, FKB=40, BKB=0, Size=3.0, X=0.0, Y=4.0, Z=3.0, X2=0.0, Y2=9.0, Z2=3.0, Hitlag=0.0, SDI=0.0, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=2, Reflectable=False, Absorbable=False, Flinchless=True, DisableHitlag=True, Direct/Indirect=False, Ground/Air=50756, Hitbits=COLLISION_CATEGORY_MASK_FIGHTER, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_normal, SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=COLLISION_SOUND_ATTR_NONE, Type=ATTACK_REGION_THROW)
 }
 frame(Frame=10)
 if(is_excute){
-	ATTACK(ID=1, Part=0, Bone=top, Damage=0.0, Angle=350, KBG=100, FKB=15, BKB=0, Size=3.0, X=0.0, Y=7.0, Z=3.0, X2=0.0, Y2=8.0, Z2=3.0, Hitlag=0.0, SDI=0.0, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=True, DisableHitlag=True, Direct/Indirect=False, Ground/Air=49916, Hitbits=COLLISION_CATEGORY_MASK_NO_STAGE, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_normal, SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=COLLISION_SOUND_ATTR_NONE, Type=ATTACK_REGION_THROW)
+	ATTACK(ID=1, Part=0, Bone=top, Damage=0.0, Angle=350, KBG=100, FKB=15, BKB=0, Size=3.0, X=0.0, Y=7.0, Z=3.0, X2=0.0, Y2=8.0, Z2=3.0, Hitlag=0.0, SDI=0.0, Clang/Rebound=ATTACK_SETOFF_KIND_OFF, FacingRestrict=ATTACK_LR_CHECK_F, SetWeight=False, ShieldDamage=0, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=True, DisableHitlag=True, Direct/Indirect=False, Ground/Air=50756, Hitbits=COLLISION_CATEGORY_MASK_NO_STAGE, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_normal, SFXLevel=ATTACK_SOUND_LEVEL_S, SFXType=COLLISION_SOUND_ATTR_NONE, Type=ATTACK_REGION_THROW)
 }
 frame(Frame=12)
 if(is_excute){
 	GrabModule__set_rebound(CanCatchRebound=True)
 }
