diff --git "a/4.0.0/wario/game_FinalAirStart.txt" "b/5.0.0/wario/game_FinalAirStart.txt"
index ca21fad..15b1389 100644
--- "a/4.0.0/wario/game_FinalAirStart.txt"
+++ "b/5.0.0/wario/game_FinalAirStart.txt"
@@ -11,11 +11,11 @@ if(0x102ee0(WorkModule__is_flag(FIGHTER_INSTANCE_WORK_ID_FLAG_DISABLE_FINAL_STAR
 		REQ_FINAL_START_CAMERA(d04finalstart.nuanmb, False)
 	}
 	else{
 		if(is_excute){
 			PostureModule__scale(4, 0)
-			0x102ee0(-1909525514, 2.1)
+			0x102ee0(-1689006174, 2.1)
 			CAM_ZOOM_IN_arg5(0, 0)
 			FT_START_CUTIN()
 		}
 	}
 }
