diff --git "a/1.0.0/simon/game_AttackS3.txt" "b/2.0.0/simon/game_AttackS3.txt"
index a5098a4..ca41f2a 100644
--- "a/1.0.0/simon/game_AttackS3.txt"
+++ "b/2.0.0/simon/game_AttackS3.txt"
@@ -2,11 +2,11 @@ frame(Frame=1)
 FT_MOTION_RATE(FSM=0.5)
 frame(Frame=4)
 FT_MOTION_RATE(FSM=1)
 frame(Frame=12)
 if(is_excute){
-	SEARCH(0, 0, hash40("top"), 2.5, 0.0, 10.0, 5.0, 0.0, 10.0, 34.0, COLLISION_KIND_MASK_HIT, HIT_STATUS_MASK_NORMAL, 1, COLLISION_SITUATION_MASK_GA, 59664, COLLISION_PART_MASK_ALL, false)
+	SEARCH(0, 0, hash40("top"), 2.5, 0.0, 10.0, 5.0, 0.0, 10.0, 34.0, COLLISION_KIND_MASK_HIT, HIT_STATUS_MASK_NORMAL, 1, COLLISION_SITUATION_MASK_GA, 60692, COLLISION_PART_MASK_ALL, false)
 }
 frame(Frame=14)
 if(is_excute){
 	battle_object()
 	methodlib::L2CValue::L2CValue(void*)()
