diff --git "a/6.0.0/dolly/0x150210c694.txt" "b/7.0.0/dolly/0x150210c694.txt"
index cb5f547..ed19447 100644
--- "a/6.0.0/dolly/0x150210c694.txt"
+++ "b/7.0.0/dolly/0x150210c694.txt"
@@ -2,11 +2,11 @@ if(is_excute){
 	WorkModule__on_flag(Flag=FIGHTER_DOLLY_INSTANCE_WORK_ID_FLAG_FINAL_HIT_CANCEL)
 	WorkModule__on_flag(Flag=FIGHTER_INSTANCE_WORK_ID_FLAG_NO_SPEED_OPERATION_CHK)
 	KineticModule__clear_speed_all()
 	WorkModule__off_flag(Flag=FIGHTER_INSTANCE_WORK_ID_FLAG_NO_SPEED_OPERATION_CHK)
 }
-if(0x24c750(WorkModule__is_flag(FIGHTER_DOLLY_STATUS_SPECIAL_COMMON_WORK_FLAG_COMMAND), False, True)){
+if(0x24c760(WorkModule__is_flag(FIGHTER_DOLLY_STATUS_SPECIAL_COMMON_WORK_FLAG_COMMAND), False, True)){
 	if(is_excute){
 		WHOLE_HIT(HIT_STATUS_XLU)
 	}
 }
 if(is_excute){
