diff --git "a/5.0.0/brave/game_AttackLw4.txt" "b/6.0.0/brave/game_AttackLw4.txt"
index c66f459..cdf83c8 100644
--- "a/5.0.0/brave/game_AttackLw4.txt"
+++ "b/6.0.0/brave/game_AttackLw4.txt"
@@ -11,11 +11,11 @@ if(is_excute){
 else{
 	L2CValue_lib::L2CValueconst()
 }
 as_pointer__const()
 lot_critical_app::Fighter()
-if(0x21bab0(WorkModule__is_flag(FIGHTER_BRAVE_INSTANCE_WORK_ID_FLAG_CRITICAL_HIT), False, True)){
+if(0x21b9e0(WorkModule__is_flag(FIGHTER_BRAVE_INSTANCE_WORK_ID_FLAG_CRITICAL_HIT), False, True)){
 	if(is_excute){
 		ATTACK(ID=0, Part=0, Bone=top, Damage=22.0, Angle=56, KBG=113, FKB=0, BKB=10, Size=4.0, X=0.0, Y=5.0, Z=4.5, Hitlag=2.0, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=False, ShieldDamage=-11, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_cutup, SFXLevel=ATTACK_SOUND_LEVEL_L, SFXType=COLLISION_SOUND_ATTR_BRAVE_CRITICALHIT, Type=ATTACK_REGION_SWORD)
 		ATTACK(ID=1, Part=0, Bone=top, Damage=26.0, Angle=56, KBG=100, FKB=0, BKB=10, Size=3.6, X=0.0, Y=4.0, Z=11.5, Hitlag=2.0, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=False, ShieldDamage=-13, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_cutup, SFXLevel=ATTACK_SOUND_LEVEL_L, SFXType=COLLISION_SOUND_ATTR_BRAVE_CRITICALHIT, Type=ATTACK_REGION_SWORD)
 		ATTACK(ID=2, Part=0, Bone=sword1, Damage=26.0, Angle=56, KBG=100, FKB=0, BKB=10, Size=3.6, X=8.6, Y=0.0, Z=0.0, Hitlag=2.0, SDI=1.0, Clang/Rebound=ATTACK_SETOFF_KIND_ON, FacingRestrict=ATTACK_LR_CHECK_POS, SetWeight=False, ShieldDamage=-13, Trip=0.0, Rehit=0, Reflectable=False, Absorbable=False, Flinchless=False, DisableHitlag=False, Direct/Indirect=True, Ground/Air=COLLISION_SITUATION_MASK_GA, Hitbits=COLLISION_CATEGORY_MASK_ALL, CollisionPart=COLLISION_PART_MASK_ALL, FriendlyFire=False, Effect=collision_attr_cutup, SFXLevel=ATTACK_SOUND_LEVEL_L, SFXType=COLLISION_SOUND_ATTR_BRAVE_CRITICALHIT, Type=ATTACK_REGION_SWORD)
 	}
