diff --git "a/_7.0.0/pikmin/game_SpecialNStart.txt" "b/_8.0.0/pikmin/game_SpecialNStart.txt"
index 58c85d5..23e95b9 100644
--- "a/_7.0.0/pikmin/game_SpecialNStart.txt"
+++ "b/_8.0.0/pikmin/game_SpecialNStart.txt"
@@ -2,11 +2,11 @@ if(is_excute){
 	ArticleModule::generate_article(FIGHTER_PIKMIN_GENERATE_ARTICLE_PIKMIN)
 }
 frame(Frame=3)
 if(is_excute){
 	WorkModule::get_float(FIGHTER_PIKMIN_STATUS_PULL_OUT_WORK_FLOAT_MOT_RATE)
-	MotionModule::set_rate(-1157984911)
+	MotionModule::set_rate(-502677940)
 }
 frame(Frame=6)
 if(is_excute){
 	MotionModule::set_rate(1)
 }
